# Faecalibacterium prausnitzii enhances intestinal IgA response by host-microbe derived inecalcitol in colitis.

<!DOCTYPE article PUBLIC "-//NLM//DTD JATS (Z39.96) Journal Archiving and Interchange DTD with MathML3 v1.3 20210610//EN" "JATS-archivearticle1-3-mathml3.dtd"> 
<article xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article" dtd-version="1.3"><?properties open_access?><?DTDIdentifier.IdentifierValue -//Springer-Verlag//DTD A++ V2.4//EN?><?DTDIdentifier.IdentifierType public?><?SourceDTD.DTDName A++V2.4.dtd?><?SourceDTD.Version 2.4?><?ConverterInfo.XSLTName springer2nlmx2.xsl?><?ConverterInfo.Version 1?><processing-meta base-tagset="archiving" mathml-version="3.0" table-model="xhtml" tagset-family="jats"><restricted-by>pmc</restricted-by></processing-meta><front><journal-meta><journal-id journal-id-type="nlm-ta">BMC Med</journal-id><journal-id journal-id-type="iso-abbrev">BMC Med</journal-id><journal-title-group><journal-title>BMC Medicine</journal-title></journal-title-group><issn pub-type="epub">1741-7015</issn><publisher><publisher-name>BioMed Central</publisher-name><publisher-loc>London</publisher-loc></publisher></journal-meta>
<article-meta><article-id pub-id-type="pmcid">12261651</article-id><article-id pub-id-type="pmid">40660288</article-id>
<article-id pub-id-type="publisher-id">4260</article-id><article-id pub-id-type="doi">10.1186/s12916-025-04260-2</article-id><article-categories><subj-group subj-group-type="heading"><subject>Research</subject></subj-group></article-categories><title-group><article-title><italic>Faecalibacterium prausnitzii</italic> enhances intestinal IgA response by host-microbe derived inecalcitol in colitis</article-title></title-group><contrib-group><contrib contrib-type="author" equal-contrib="yes"><name><surname>Qin</surname><given-names>Wenfei</given-names></name><xref ref-type="aff" rid="Aff1">1</xref><xref ref-type="aff" rid="Aff2">2</xref><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author" equal-contrib="yes"><name><surname>Yin</surname><given-names>Nuoming</given-names></name><xref ref-type="aff" rid="Aff2">2</xref><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author" equal-contrib="yes"><name><surname>Xu</surname><given-names>Binqiang</given-names></name><xref ref-type="aff" rid="Aff2">2</xref><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author"><name><surname>Mei</surname><given-names>Qixiang</given-names></name><xref ref-type="aff" rid="Aff2">2</xref><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author"><name><surname>Fu</surname><given-names>Yang</given-names></name><xref ref-type="aff" rid="Aff2">2</xref><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author"><name><surname>Fan</surname><given-names>Junjie</given-names></name><xref ref-type="aff" rid="Aff2">2</xref><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author"><name><surname>Lu</surname><given-names>Yingying</given-names></name><xref ref-type="aff" rid="Aff2">2</xref><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author"><name><surname>Wang</surname><given-names>Guangqiang</given-names></name><xref ref-type="aff" rid="Aff1">1</xref></contrib><contrib contrib-type="author"><name><surname>Ai</surname><given-names>Lianzhong</given-names></name><xref ref-type="aff" rid="Aff1">1</xref></contrib><contrib contrib-type="author"><name><surname>Lu</surname><given-names>Zhanjun</given-names></name><xref ref-type="aff" rid="Aff2">2</xref></contrib><contrib contrib-type="author" corresp="yes"><name><surname>Zeng</surname><given-names>Yue</given-names></name><address><email>carrie_1004@sjtu.edu.cn</email></address><xref ref-type="aff" rid="Aff2">2</xref><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author" corresp="yes"><name><surname>Huang</surname><given-names>Chunlan</given-names></name><address><email>chunlan.huang@shgh.cn</email></address><xref ref-type="aff" rid="Aff2">2</xref><xref ref-type="aff" rid="Aff3">3</xref></contrib><aff id="Aff1"><label>1</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/00ay9v204</institution-id><institution-id institution-id-type="GRID">grid.267139.8</institution-id><institution-id institution-id-type="ISNI">0000 0000 9188 055X</institution-id><institution>Shanghai Engineering Research Center of Food Microbiology, </institution><institution>School of Health Science and Engineering, University of Shanghai for Science and Technology, </institution></institution-wrap>Shanghai, 200093 China </aff><aff id="Aff2"><label>2</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/0220qvk04</institution-id><institution-id institution-id-type="GRID">grid.16821.3c</institution-id><institution-id institution-id-type="ISNI">0000 0004 0368 8293</institution-id><institution>Department of Gastroenterology, </institution><institution>Shanghai General Hospital, Shanghai Jiao Tong University School of Medicine, </institution></institution-wrap>Shanghai, 201620 China </aff><aff id="Aff3"><label>3</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/0220qvk04</institution-id><institution-id institution-id-type="GRID">grid.16821.3c</institution-id><institution-id institution-id-type="ISNI">0000 0004 0368 8293</institution-id><institution>Shanghai Key Laboratory of Pancreatic Diseases, </institution><institution>Shanghai Jiao Tong University School of Medicine, </institution></institution-wrap>Shanghai, 201620 China </aff></contrib-group><pub-date pub-type="epub"><day>15</day><month>7</month><year>2025</year></pub-date><pub-date pub-type="pmc-release"><day>15</day><month>7</month><year>2025</year></pub-date><pub-date pub-type="collection"><year>2025</year></pub-date><volume>23</volume><elocation-id>425</elocation-id><history><date date-type="received"><day>2</day><month>1</month><year>2025</year></date><date date-type="accepted"><day>4</day><month>7</month><year>2025</year></date></history><permissions><copyright-statement>&#x000a9; The Author(s) 2025</copyright-statement><copyright-year>2025</copyright-year><license><ali:license_ref specific-use="textmining" content-type="ccbyncndlicense">https://creativecommons.org/licenses/by-nc-nd/4.0/</ali:license_ref><license-p><bold>Open Access</bold> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article&#x02019;s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article&#x02019;s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <ext-link ext-link-type="uri" xlink:href="https://creativecommons.org/licenses/by-nc-nd/4.0/">http://creativecommons.org/licenses/by-nc-nd/4.0/</ext-link>.</license-p></license></permissions><abstract id="Abs1"><sec><title>Background</title><p id="Par1"><italic>Faecalibacterium prausnitzii</italic> plays a crucial role in ulcerative colitis (UC) remission, but its action mechanism is unknown. Here, we aimed to explore the potential mechanisms focusing on the interaction of <italic>F. prausnitzii</italic> with host immune response and its potential modulation on gut microbiome.</p></sec><sec><title>Methods</title><p id="Par2">RNA-seq analysis together with 16S rRNA sequencing and metabolomics were performed in a dextran sodium sulfate (DSS)-induced colitis mouse model followed by <italic>F. prausnitzii</italic> gavage. To present evidence of sIgA involved in the anti-inflammatory effects of <italic>F. prausnitzii</italic>, we further applied immunoglobulin A (IgA) knockout mice and secretory IgA (sIgA) depletion mouse models using polymeric immunoglobulin receptor (pIgR) neutralizing antibody. Colonic immune cells were characterized by flow cytometry. The fecal relative abundance of <italic>F. prausnitzii</italic>, inecalcitol, and colonic IgA expression were assessed in UC patients.</p></sec><sec><title>Results</title><p id="Par3"><italic>F. prausnitzii</italic> markedly ameliorated colitis by alleviating intestinal inflammation and barrier dysfunction, with significantly decreased abundance of pro-inflammatory taxa (<italic>Enterococcus</italic>, <italic>Desulfovibrio</italic>, <italic>Escherichia-Shigella</italic>, and <italic>Enterorhabdus</italic>) and increased abundance of <italic>Lachnospiraceae NK4A136_group</italic>. Functions related to intestinal immune network for IgA production pathway were up-regulated shown by transcriptomics and KEGG pathway analysis. Increased expression of IgA production associated genes including MHCII-related genes, Aicda, and Tnfrsfl3c were verified, accompanied by up-regulated colonic IgA and pIgR. The IgA knockout mice and sIgA depletion model weakened the anti-inflammation and microbiota-modulation effects of <italic>F. prausnitzii</italic>, which was further proved by fecal microbiota transplantation (FMT)<italic>.</italic> The shift profile of fecal metabolites after <italic>F. prausnitzii</italic> supplement was characterized by increased production of inecalcitol, which may account for the enhanced IgA response. In a cohort of UC patients, the relative abundance of <italic>F. prausnitzii</italic> was decreased and positively correlated with colonic IgA expression and negatively correlated with disease severity.</p></sec><sec><title>Conclusions</title><p id="Par4"><italic>F. prausnitzii</italic> effectively alleviated colonic inflammation and modulated dysbiosis via enhancing colonic IgA response, thus showing promise as a UC treatment.</p></sec><sec><title>Graphical Abstract</title><p id="Par5">
<graphic position="anchor" xlink:href="12916_2025_4260_Figa_HTML" id="MO1"/></p></sec><sec><title>Supplementary Information</title><p>The online version contains supplementary material available at 10.1186/s12916-025-04260-2.</p></sec></abstract><kwd-group xml:lang="en"><title>Keywords</title><kwd><italic>Faecalibacterium prausnitzii</italic></kwd><kwd>Experimental colitis</kwd><kwd>Immunoglobulin a</kwd><kwd>Inecalcitol</kwd></kwd-group><funding-group><award-group><funding-source><institution-wrap><institution-id institution-id-type="FundRef">https://doi.org/10.13039/501100001809</institution-id><institution>National Natural Science Foundation of China</institution></institution-wrap></funding-source><award-id>No. 81970555</award-id></award-group></funding-group><funding-group><award-group><funding-source><institution-wrap><institution-id institution-id-type="FundRef">https://doi.org/10.13039/501100013103</institution-id><institution>Shanghai General Hospital</institution></institution-wrap></funding-source><award-id>No.CCTR-2022B02</award-id></award-group></funding-group><funding-group><award-group><funding-source><institution>Shanghai Municipal Health Commission</institution></funding-source><award-id>202440170</award-id></award-group></funding-group><funding-group><award-group><funding-source><institution>Science and Technology Commission of Songjiang District</institution></funding-source><award-id>No.22SJKGGG28</award-id></award-group></funding-group><funding-group><award-group><funding-source><institution>Shanghai Jiao Tong University School of Medicine, Digestive Institute</institution></funding-source><award-id>No.KY-2023-03-02</award-id></award-group></funding-group><custom-meta-group><custom-meta><meta-name>issue-copyright-statement</meta-name><meta-value>&#x000a9; BioMed Central Ltd., part of Springer Nature 2025</meta-value></custom-meta></custom-meta-group></article-meta></front><body><sec id="Sec1"><title>Background
</title><p id="Par31">Ulcerative colitis (UC), one of the sub-types of inflammatory bowel disease (IBD), is characterized by chronic relapse and remission stages of intestinal inflammation. Its incidence is increasing worldwide [<xref ref-type="bibr" rid="CR1">1</xref>, <xref ref-type="bibr" rid="CR2">2</xref>] with substantial health care and societal costs [<xref ref-type="bibr" rid="CR3">3</xref>]. Patients have been reported to experience poor disease control and an adverse impact on their quality of life with a mentally exhausting condition [<xref ref-type="bibr" rid="CR4">4</xref>].
</p><p id="Par32">UC pathophysiology has been reported to be associated with dysregulated mucosal immune response to commensal gut microbiota [<xref ref-type="bibr" rid="CR5">5</xref>, <xref ref-type="bibr" rid="CR6">6</xref>]. Identifying effector microorganisms and deciphering the underlying mechanisms enables the development of microbiota-based therapeutics [<xref ref-type="bibr" rid="CR7">7</xref>]. Our previous study reported that the <italic>Faecalibacterium</italic> colonization plays a crucial role in UC remission by alleviating intestinal inflammation [<xref ref-type="bibr" rid="CR8">8</xref>]. However, the underlying mechanisms need to be further revealed.</p><p id="Par33"><italic>Faecalibacterium prausnitzii</italic> (<italic>F. prausnitzii</italic>) is one of the most abundant bacterial species in the human gut microbiome and represents more than 5% of the total bacterial population [<xref ref-type="bibr" rid="CR9">9</xref>]. Lower abundance of this anti-inflammatory bacterium has been reported in various diseases, including IBD [<xref ref-type="bibr" rid="CR10">10</xref>]. A study reported that <italic>F. prausnitzii</italic> produced butyrate to decrease Th17 differentiation and attenuate colitis through inhibiting HDAC3 and c-Myc-related metabolism in T cells [<xref ref-type="bibr" rid="CR11">11</xref>], whereas another study presented that the anti-inflammatory effects of <italic>F. prausnitzii</italic> in colitis were partly associated with secreted metabolites capable of blocking NF-&#x003ba;B activation and IL-8 production, which were not related to butyrate presence [<xref ref-type="bibr" rid="CR12">12</xref>]. Some studies also focused on its production of the Microbial Anti-inflammatory Molecule (MAM), which is able to block the NF-&#x003ba;B pathway and consequently reduce the production of IL-8 [<xref ref-type="bibr" rid="CR13">13</xref>] or Th1, Th2, and Th17 immune responses [<xref ref-type="bibr" rid="CR14">14</xref>] in colitis. The commercial bacteria can also inhibit the efficacy of enteropathogens through a variety of molecular mechanisms involving microbe-microbe and/or microbe-host interactions [<xref ref-type="bibr" rid="CR15">15</xref>]. They competitively consume the nutrients of pathogenic bacteria and compete for their adhesion sites. Moreover, they can induce host immune responses to restrict pathogen growth and colonization [<xref ref-type="bibr" rid="CR16">16</xref>]. Few studies focused on the interaction of <italic>F. prausnitzii</italic> with host immune response and its potential modulation of the gut microbiome and microbial metabolites to ameliorate colitis.</p><p id="Par34">To explore the host signaling pathways of <italic>F. prausnitzii</italic>, which contributed potentially to its microbiota modulation and anti-inflammatory effects in active UC, we performed transcriptomics and metabolomics approaches in a dextran sodium sulfate (DSS)-induced colitis mouse model with <italic>F. prausnitzii</italic> supplement. The 16S rRNA analysis was used to validate the interaction of <italic>F. prausnitzii</italic> with gut microbiota in colitis. These analyses led us to focus on colonic mucosal secretory immunoglobulin A (sIgA), which plays a key role in the host-microbiota mutualism by excluding pathogens and preventing pathogen attachment to epithelial surfaces [<xref ref-type="bibr" rid="CR17">17</xref>]. We further applied IgA knockout mice and sIgA depletion mouse models using polymeric immunoglobulin receptor (pIgR) neutralizing antibody to present evidence of sIgA involved in the anti-inflammatory effects of the beneficial commensal bacterium <italic>F. prausnitzii</italic>. The involvement of the gut metabolite inecalcitol in colitis was also studied.</p></sec><sec id="Sec2"><title>Methods</title><sec id="Sec3"><title>Sex as a biological variable</title><p id="Par35">Sex has no impact on the IBD phenotype in both wild-type (WT) and gene-edited mice [<xref ref-type="bibr" rid="CR18">18</xref>], and epidemiological evidence also indicates no association between sex and the severity of IBD [<xref ref-type="bibr" rid="CR19">19</xref>]. Therefore, we examined male mice only in our study.</p></sec><sec id="Sec4"><title>Animals, DSS model induction and intervention</title><p id="Par36">All the animal experiments were approved by the Institutional Animal Care and Use Committee (IACUC) of Shanghai General Hospital and conducted according to the instructions of the IACUC. Male C57BL/6 mice (6&#x02013;8&#x000a0;weeks, 20&#x02013;22&#x000a0;g) were purchased from Shanghai SLAC Laboratory Animal Co. Ltd (Shanghai, China). The mice were allowed free access to rodent and housed in a constant temperature (23&#x02009;&#x000b1;&#x02009;2 &#x02103;) and humidity (50&#x02009;&#x000b1;&#x02009;5%) environment with a 12-h dark/light cycle.</p><p id="Par37">To study the effect of <italic>F. prausnitzii</italic> treatment on colitis, the mice were randomly divided into four groups: WT mice with phosphate-buffered saline (PBS) (CON group) or <italic>F. prausnitzii</italic> supplement (<italic>Fp</italic> group) and DSS-induced colitis mice with PBS (DSS group) or <italic>F. prausnitzii</italic> supplement (DSS&#x02009;+&#x02009;<italic>Fp</italic> group) (<italic>n</italic>&#x02009;=&#x02009;6 per group). Experimental colitis was induced with a 3.2% (wt/vol) DSS solution (MP Biomedicals, St Ana, USA), which was added to their drinking water for 7&#x000a0;days [<xref ref-type="bibr" rid="CR20">20</xref>]. 200 &#x003bc;L of <italic>F. prausnitzii</italic> resuspension fluid or PBS was administrated by gavage for the next 7&#x000a0;days (Fig.&#x000a0;<xref rid="Fig1" ref-type="fig">1</xref>A).<fig id="Fig1"><label>Fig. 1</label><caption><p><italic>F. prausnitzii</italic> ameliorates intestinal inflammation and barrier dysfunction in DSS colitis. <bold>A</bold> Experiment design of DSS colitis induction and <italic>F. prausnitzii</italic> administration. <bold>B</bold> The body weight changes. <bold>C</bold> The DAI score of mice during the intervention. <bold>D</bold> Photographs of colon length 14&#x000a0;days after interventions and the corresponding quantified lengths of colons. <bold>E</bold> The MPO activity of colon. <bold>F</bold> Representative H&#x00026;E staining images of colon Sects.&#x000a0;(200&#x02009;&#x000d7;&#x02009;magnification) and the histological score. <bold>G</bold> mRNA expression levels of TNF-&#x003b1;, TGF-&#x003b2;, IL-1&#x003b2;, IL-6, IL-10, and IL-17 in the colon. <bold>H</bold>&#x02013;<bold>J</bold> Representative immunofluorescence staining images of colon sections to indicate the expression of TJPs, including Claudin1, Occludin, and ZO-1 (200&#x02009;&#x000d7;&#x02009;magnification). <bold>K</bold> The corresponding relative fluorescence intensity of Claudin1, Occludin, and ZO-1. <bold>L</bold> mRNA expression levels of Claudin1, Occludin, and ZO-1 in the colon. <bold>M</bold> Serum D-lactate concentration and DAO activity. The data are presented as the means&#x02009;&#x000b1;&#x02009;SEM (<italic>n</italic>&#x02009;=&#x02009;6 mice per group). ***, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.001</p></caption><graphic xlink:href="12916_2025_4260_Fig1_HTML" id="MO2"/></fig></p><p id="Par38">The pIgR neutralizing antibody was applied for sIgA depletion in mice [<xref ref-type="bibr" rid="CR21">21</xref>]. A pilot study was performed to assess its effect and optimal regimen. The pIgR neutralizing antibody (5&#x000a0;&#x003bc;g or 10&#x000a0;&#x003bc;g per mouse) was administrated by intraperitoneal injection over a 6-day period (day 1, 3 and 6) and euthanized at day 7 (Additional file 1: Fig. S1A). Both the pIgR and IgA expression were significantly downregulated in 5&#x000a0;&#x003bc;g and 10&#x000a0;&#x003bc;g groups compared to the control group, respectively. Notably, the pIgR and IgA transcription was downregulated by more than 50% in the 10&#x000a0;&#x003bc;g group (Additional file 1: Fig. S1B&#x02013;C). The colonic expression of pIgR and IgA was also assessed by immunofluorescence (Additional file 1: Fig. S1D&#x02013;E), showing the dramatic reduction in the 10&#x000a0;&#x003bc;g group. Therefore, the protocol of pIgR neutralizing antibody intervention by intraperitoneal injection of 10&#x000a0;&#x003bc;g per time was determined in our study. The protocol of sIgA depletion in the DSS-induced colitis mice with or without <italic>F. prausnitzii</italic> supplement was shown in Additional file 1: Fig. S2A.</p><p id="Par39">The inecalcitol intervention experiment was performed as described previously [<xref ref-type="bibr" rid="CR22">22</xref>, <xref ref-type="bibr" rid="CR23">23</xref>] with some modifications (Additional file 1: Fig. S3A). Mice were randomly divided into CON, inecalcitol, DSS, and DSS&#x02009;+&#x02009;inecalcitol groups (<italic>n</italic>&#x02009;=&#x02009;6 per group). In DSS and DSS&#x02009;+&#x02009;inecalcitol groups, mice were given 3.2% DSS solution for 7&#x000a0;days. Afterwards, mice in inecalcitol and DSS&#x02009;+&#x02009;inecalcitol groups were gavaged with inecalcitol (3&#x000a0;&#x003bc;g/kg/day) for 7&#x000a0;days, while mice in CON and DSS groups received an equal volume of sterile corn oil instead.</p><p id="Par40">Mice were monitored daily for weight loss, stool consistency, and fecal occult blood (JianCheng, Nanjing, China). Disease activity index (DAI) was calculated to assess the mucosal inflammation [<xref ref-type="bibr" rid="CR24">24</xref>]. After the mice were euthanized, the distance from cecum to anus was measured. The blood samples were collected, centrifuged at 6&#x000a0;&#x000b0;C, and stored at&#x02009;&#x02212;&#x02009;80&#x000a0;&#x000b0;C for further detection of intestinal permeability. Fecal contents were harvested for 16S rRNA gene sequencing analysis and metabolomics approach. Segments of the colon were harvested in RNase-free cryovial tubes, snap-frozen in liquid nitrogen, and stored at&#x02009;&#x02212;&#x02009;80 &#x02103; for further RNA-seq analysis, myeloperoxidase (MPO) activity, and cytokine detection. Parts of the colon were fixed in 4% paraformaldehyde or Carnoy&#x02019;s solution for later histology and immunofluorescence examinations.</p></sec><sec id="Sec5"><title>Bacterial strains and culture</title><p id="Par41"><italic>F. prausnitzii</italic> ATCC27766 was purchased from ATCC (Manassas, VA, USA). LYHBHI medium, containing brain&#x02013;heart infusion (37&#x000a0;g/L), yeast extract (5&#x000a0;g/L) and hemin (5&#x000a0;mg/L), supplemented with cellobiose (1&#x000a0;g/L), maltose (1&#x000a0;g/L) and cysteine (0.5&#x000a0;g/L), was used. For solid medium preparation, 1.5% agar was added to LYHBHI medium at 115 &#x02103; for 20&#x000a0;min. <italic>F. prausnitzii</italic> was grown in LYHBHI solid medium at 37 &#x02103; in an anaerobic incubator containing 80% N<sub>2</sub>, 10% CO<sub>2</sub>, and 10% H<sub>2</sub>. After two generations, a single colony was picked and inoculated into the liquid medium and cultured for 24 to 48&#x000a0;h. The bacteria were collected by centrifuging at 4 &#x02103; for 10&#x000a0;min at 4500&#x02009;&#x000d7;&#x02009;g, washed twice, and resuspended to the concentration of 5&#x02009;&#x000d7;&#x02009;10<sup>8</sup> colony-forming units (CFUs) per mL with PBS.</p><p id="Par42">To verify the metabolites produced by <italic>F. prausnitzii </italic>in vitro, in addition to LYHBHI medium, the M2GSC medium consisted of yeast extract (5&#x000a0;g/L), NaHCO<sub>3</sub> (4&#x000a0;g/L), glucose (5&#x000a0;g/L), soluble starch (2&#x000a0;g/L), cellobiose (2&#x000a0;g/L), casitone (10&#x000a0;g/L), cysteine (1&#x000a0;g/L), K<sub>2</sub>HPO<sub>4</sub> (0.45&#x000a0;g/L), KH<sub>2</sub>PO<sub>4</sub> (0.45&#x000a0;g/L), NaCl (0.9&#x000a0;g/L), (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> (0.9&#x000a0;g/L), MgSO<sub>4</sub> (0.09&#x000a0;g/L), CaCl<sub>2</sub> (0.09&#x000a0;g/L), and clarified rumen fluid (10&#x000a0;mL/L) was also used. <italic>F. prausnitzii</italic> was cultured in 20&#x000a0;mL LYHBHI or M2GSC medium for 48&#x000a0;h, respectively. The supernatant was collected after centrifugation, packaged for freeze-drying, and stored at&#x02009;&#x02212;&#x02009;80 &#x02103; for later analysis. LYHBHI or M2GSC medium supernatant was used as the control group.</p></sec><sec id="Sec6"><title>Assessment of intestinal inflammation and barrier dysfunction</title><p id="Par43">Colonic histopathological changes, MPO activity, and inflammatory cytokines (TNF-&#x003b1;, TGF-&#x003b2;, IL-1&#x003b2;, IL-6, IL-10, and IL-17) were measured to assess intestinal inflammation, while intestinal permeability (diamine oxidase (DAO) and D-lactate) and tight junction proteins (TJPs) (Claudin1, Occludin, and ZO-1) were measured to assess intestinal barrier dysfunction.</p><p id="Par44">The colon tissues were fixed in 4% paraformaldehyde at 4 &#x02103; overnight and embedded in paraffin. The slides were then cut into 4&#x000a0;&#x000b5;m slices and stained with hematoxylin and eosin (H&#x00026;E). The colonic histopathological changes were evaluated based on the previous study [<xref ref-type="bibr" rid="CR25">25</xref>].</p><p id="Par45">The MPO activity of colon tissues was detected by Myeloperoxidase Assay Kit (Jiancheng, Nanjing, China) according to the manufacturer&#x02019;s instructions.</p><p id="Par46">The levels of TNF-&#x003b1;, TGF-&#x003b2;, IL-1&#x003b2;, IL-6, IL-10, and IL-17 in the colon were determined by using quantitative real-time PCR. RNA was extracted from fresh frozen colon tissues using FastPure Cell/Tissue Total RNA Isolation Kit V2 (Vazyme, Nanjin, China) according to the manufacturer&#x02019;s instructions. Complementary DNA was then synthesized from 2&#x000a0;&#x003bc;g aliquots of RNA using HyperScriptTM III RT SuperMix (EnzyArtisan, China). Quantitative real-time PCR was performed using 2&#x02009;&#x000d7;&#x02009;S6 Universal SYBR qPCR Mix (EnzyArtisan, China) in the QuantStudio 7 Flex real-time PCR system (Applied Biosystems). The primer sequences used for this analysis were listed in Additional file 2: Table S1, and &#x003b2;-actin served as the internal reference gene.</p><p id="Par47">The TJPs were measured by quantitative real-time PCR and immunofluorescence. The primer sequences used for this analysis were listed in Additional file 2: Table S2. To assess TJPs by immunofluorescence, the slides of colon tissue were deparaffinized, blocked with immunostaining blocking buffer (Sangon Biotech, China) at room temperature for 1&#x000a0;h, and incubated with primary antibodies against Claudin1 (Abcam, USA), Occludin (Abclonal, China) and ZO-1 (Cell Signaling Technology, USA) at 4 &#x02103; overnight. Slides were washed with PBS and incubated with Alexa Fluor 488 AffiniPure donkey antirabbit IgG (Yeason, China) for 1&#x000a0;h at room temperature. The slides were then washed with PBS again and stained with dihydrochloride (Yeason, China) for 10&#x000a0;min. Images were captured with a fluorescence microscope (Leica, USA).</p><p id="Par48">The DAO activity and D-lactate concentration in the serum were measured by Diamine oxidase ELISA Kit (WELLBIO, China) and D-LA ELISA kit (Krishgen Biosystems, USA) according to the manufacturers&#x02019; protocols, respectively.</p></sec><sec id="Sec7"><title>16S rRNA gene sequencing</title><p id="Par49">16S rRNA gene sequencing was performed to verify the changes of gut microbiota in experimental colitis with or without <italic>F. prausnitzii</italic> supplement. The microbiota changes between the DSS&#x02009;+&#x02009;<italic>Fp</italic> mice with or without sIgA depletion were also tested. The cecal contents were collected as previously described. Total microbial genomic DNA was extracted using the E.Z.N.A.&#x000ae; soil DNA Kit (Omega Bio-tek, Norcross, GA, USA). The concentration of DNA was determined by 1.0% agarose gel electrophoresis.</p><p id="Par50">The hypervariable region V3&#x02013;V4 of the bacterial 16S rRNA gene was amplified using an ABI GeneAmp&#x000ae; 9700 PCR thermocycler (ABI, CA, USA) with primer pairs 338&#x02009;F (5&#x02032;-ACTCCTACGGGAGGCAGCAG-3&#x02032;) and 806R (5&#x02032;-GGACTACHVGGGTWTCTAAT-3&#x02032;). The PCR reaction mixture consisted of 4 &#x003bc;L 5&#x02009;&#x000d7;&#x02009;Fast Pfu buffer, 2 &#x003bc;L 2.5&#x000a0;mM dNTPs, 0.8 &#x003bc;L each primer (5&#x000a0;&#x003bc;M), 0.4 &#x003bc;L Fast Pfu polymerase, and 10&#x000a0;ng of template DNA with ddH2O to a final volume of 20 &#x000b5;L. After PCR amplification, the product was extracted with 2% agarose gel, then purified using the AxyPrep DNA Gel Extraction Kit (Axygen Biosciences, Union City, CA, USA) and quantified using Quantus&#x02122; Fluorometer (Promega, USA).</p><p id="Par51">The amplicons were pooled in equal amounts and sequenced on MiSeq PE300 platform/NovaSeq PE250 platform (Illumina, San Diego, USA). Raw sequence data files were converted into fastq files and de-multiplexed using an in-house Perl script. The clean data were then quality-filtered by fastp and merged by FLASH. Sequences were clustered into operational taxonomic units (OTUs) using UPARSE 7.1 with a threshold of 97% similarity. The taxonomy of each OTU representative sequence was analyzed by RDP Classifier (version 2.13) against the Silva database (version 138) using a confidence threshold of 0.7.</p></sec><sec id="Sec8"><title>RNA extraction, cDNA library preparation and sequencing</title><p id="Par52">Total RNA was extracted from 50 to 100&#x000a0;mg colon tissue per sample using TRIzol&#x000ae; Reagent (Invitrogen, Burlington, ON, Canada) and genomic DNA was removed using DNase I (TaKara, Otsu, Japan). RNA degradation and contamination was monitored by electrophoresis on 1% agarose gels. The concentration and integrity of RNA was assessed by using NanoDrop 2000 (Thermo Scientific, USA) and Agilent 2100 Bioanalyzer (Agilent, USA), respectively.</p><p id="Par53">The sequencing library was prepared based on 1&#x000a0;&#x003bc;g of total RNA using TruSeq TM RNA sample preparation Kit from Illumina (San Diego, CA). Double-stranded cDNA was synthesized using a SuperScript double-stranded cDNA synthesis kit (Invitrogen, CA) with random hexamer primers (Illumina). cDNA target fragments of 300&#x000a0;bp were selected on 2% low range ultra agarose and then amplified using Phusion DNA polymerase (NEB) for 15 PCR cycles. Paired-end RNA-seq sequencing library was quantified by TBS380 before sequenced with the Illumina NovaSeq 6000 sequencer. The depth of sequencing coverage and the length of sequence reads were 2&#x02009;&#x000d7;&#x02009;150&#x000a0;bp.</p><p id="Par54">The raw paired-end reads were trimmed and quality controlled by fastp to obtain clean reads, which were then separately aligned to the reference genome using HISAT2 software. The mapped reads of each sample were assembled by StringTie software.</p><p id="Par55">Differential genes expression analysis was performed using the DESeq2 with <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.05 and fold change&#x02009;&#x0003e;&#x02009;2 as the threshold for differential expressed genes (DEGs). KEGG pathway analysis was performed by KOBAS.</p><p id="Par56">The most up- and downregulated genes involved in the intestinal IgA production pathway were validated by RT-qPCR as previously described. The genes analyzed and the related primers were listed in Additional file 2: Table S3.</p></sec><sec id="Sec9"><title>Assessment of pIgR and IgA in colon</title><p id="Par57">Colon tissue was weighed, minced to small pieces, and homogenized (50&#x000a0;Hz, 15&#x000a0;s) in ice-cold PBS (pH&#x02009;=&#x02009;7.4, 9&#x000a0;mL PBS would be appropriate to 1&#x000a0;g tissue pieces). The homogenates were then centrifuged for 15&#x000a0;min at 5000&#x02009;&#x000d7;&#x02009;g to get the supernatant. The pIgR (JINGMEI BIOTECHNOLOGY, China) and IgA (JINGMEI BIOTECHNOLOGY, China) levels were measured according to the reagent experimental protocol.</p><p id="Par58">To further assess the colonic pIgR and IgA expression by immunofluorescence, the Carnoy&#x02019;s solution-fixed colon tissues were employed to preserve the surface mucous layer over the epithelial cells. The slides were treated as described previously, and the primary antibodies against pIgR (Abclonal, China) and IgA (A-9) (SantaCruz, USA) were used.</p></sec><sec id="Sec10"><title>IgA knockout mouse models</title><p id="Par59">IgA knockout mice (20&#x02013;25&#x000a0;g) were kindly gifted by the University of Science and Technology of China. IgA knockout mice were constructed by deleting the <italic>Igha</italic> gene encoding IgA in mice. In brief, the <italic>Igha</italic> gene is located on chromosome 12 in mice, and gene editing techniques were used to delete exons 1 to 4 of this gene to obtain IgA knockout mice. WT mice (20&#x02013;25&#x000a0;g) were randomly divided into DSS group (<italic>n</italic>&#x02009;=&#x02009;4) and DSS&#x02009;+&#x02009;<italic>Fp</italic> group (<italic>n</italic>&#x02009;=&#x02009;3). IgA knockout mice were randomly divided into KO&#x02009;+&#x02009;DSS group (<italic>n</italic>&#x02009;=&#x02009;3) and KO&#x02009;+&#x02009;DSS&#x02009;+&#x02009;<italic>Fp</italic> group (<italic>n</italic>&#x02009;=&#x02009;4). All mice were given 1.0% DSS solution for 7&#x000a0;days. Since the knockout mice experienced excessive weight loss and a high mortality when exposed to 3.2% DSS in a pilot study, the DSS concentration was adjusted to 1.0% [<xref ref-type="bibr" rid="CR26">26</xref>&#x02013;<xref ref-type="bibr" rid="CR28">28</xref>] in order to ensure animal welfare and effective disease induction. Afterwards, mice in DSS&#x02009;+&#x02009;<italic>Fp</italic> and KO&#x02009;+&#x02009;DSS&#x02009;+&#x02009;<italic>Fp</italic> groups were gavaged with <italic>F. prausnitzii</italic> (5&#x02009;&#x000d7;&#x02009;10<sup>8</sup> CFUs/mL, 200 &#x003bc;L per day) for 7&#x000a0;days, while mice in DSS and KO&#x02009;+&#x02009;DSS groups received an equal volume of PBS.</p><p id="Par60">The inecalcitol intervention experiment was also performed in IgA knockout mice. After DSS models induction, mice in WT&#x02009;+&#x02009;DSS&#x02009;+&#x02009;inecalcitol and IgA KO&#x02009;+&#x02009;DSS&#x02009;+&#x02009;inecalcitol groups (<italic>n</italic>&#x02009;=&#x02009;6 per group) were gavaged with inecalcitol (3&#x000a0;&#x003bc;g/kg/day) for 7&#x000a0;days.</p></sec><sec id="Sec11"><title>Fecal microbiota transplantation (FMT)</title><p id="Par61">Feces were collected from DSS&#x02009;+&#x02009;<italic>Fp</italic> mice and DSS&#x02009;+&#x02009;sIgA depleting&#x02009;+&#x02009;<italic>Fp</italic> mice on day 14, 15, and 16. Processing of the FMT suspension was done within 2&#x000a0;h. One hundred milligrams of feces was resuspended in 1&#x000a0;mL of saline and centrifuged at 100&#x02009;&#x000d7;&#x02009;g for 5&#x000a0;min. The supernatant was stored at&#x02009;&#x02212;&#x02009;80 &#x02103;. Before freezing, glycerol was added up to a final concentration of 10%. The supernatant was used as the fecal suspension after thawed in a water bath at 37&#x000a0;&#x000b0;C on the day of FMT.</p><p id="Par62">All the recipient mice were fed 3.2% DSS in the drinking water for 7&#x000a0;days and divided into three groups. Afterwards, the DSS group received no treatment. The other two groups were gavaged with 200 &#x003bc;L of an FMT suspension from DSS&#x02009;+&#x02009;<italic>Fp</italic> mice and DSS&#x02009;+&#x02009;sIgA depleting&#x02009;+&#x02009;<italic>Fp</italic> mice for 1&#x000a0;week, respectively (Fig.&#x000a0;<xref rid="Fig5" ref-type="fig">5</xref>A).</p></sec><sec id="Sec12"><title>Metabolomics</title><p id="Par63">The cecal contents of mice and fecal samples from 15 patients with UC were obtained. 50&#x000a0;mg of each sample was mixed with 400 &#x003bc;L of extraction solution (methanol:water&#x02009;=&#x02009;4: 1 (v:v)) containing 0.02&#x000a0;mg/mL of internal standard (L-2-chlorophenylalanine) and a 6&#x000a0;mm diameter grinding bead. Samples were ground by the Wonbio-96c (Shanghai wanbo biotechnology co., LTD) frozen tissue grinder at&#x02009;&#x02212;&#x02009;10&#x000a0;&#x000b0;C and 50&#x000a0;Hz for 6&#x000a0;min, followed by ultrasonic extraction at 5&#x000a0;&#x000b0;C and 40&#x000a0;kHz for 30&#x000a0;min. The samples were kept at&#x02009;&#x02212;&#x02009;20&#x000a0;&#x000b0;C for 30&#x000a0;min and centrifuged at 13,000&#x02009;&#x000d7;&#x02009;g and 4&#x000a0;&#x000b0;C for 15&#x000a0;min, and the supernatant was transferred for LC&#x02013;MS/MS analysis.</p><p id="Par64">Two milliliters <italic>F. prausnitzii</italic> culture supernatant and LYHBHI and M2GSC media was collected and freeze-dried, respectively. The sample was extracted using a 400 &#x000b5;L extraction solution (methanol:water&#x02009;=&#x02009;4: 1 (v:v)) containing 0.02&#x000a0;mg/mL of internal standard (L-2-chlorophenylalanine). The mixture was sonicated at 40&#x000a0;kHz for 30&#x000a0;min at 5&#x000a0;&#x000b0;C, and placed at&#x02009;&#x02212;&#x02009;20&#x000a0;&#x000b0;C for 30&#x000a0;min to precipitate proteins. After centrifugation at 13,000&#x02009;&#x000d7;&#x02009;g 4&#x000a0;&#x000b0;C for 15&#x000a0;min, the supernatant was transferred to new microtubes and evaporated to dryness under a gentle stream of nitrogen. The samples were reconstituted in 100 &#x000b5;L loading solution of acetonitrile:water (1:1, v/v) by brief sonication in a 5&#x000a0;&#x000b0;C water bath. After being centrifuged at 13,000&#x02009;&#x000d7;&#x02009;g 4&#x000a0;&#x000b0;C for 15&#x000a0;min, the supernatant was ready for LC&#x02013;MS/MS analysis.</p><p id="Par65">The LC&#x02013;MS/MS analysis of the sample was conducted on a Thermo UHPLC-Q Exactive system equipped with an ACQUITY HSS T3 column (100&#x000a0;mm&#x02009;&#x000d7;&#x02009;2.1&#x000a0;mm i.d., 1.8&#x000a0;&#x003bc;m; Waters, USA). The LC/MS raw data was processed by Progenesis QI (Waters Corporation, Milford, USA) software for baseline denoising, peak picking, and peak signal alignment. The metabolites were identified by searching databases including HMDB (<ext-link ext-link-type="uri" xlink:href="http://www.hmdb.ca/">http://www.hmdb.ca/</ext-link>) and Metlin (<ext-link ext-link-type="uri" xlink:href="https://metlin.scripps.edu/">https://metlin.scripps.edu/</ext-link>).</p></sec><sec id="Sec13"><title>Lamina propria mononuclear cells (LPMC) isolation</title><p id="Par66">LPMCs were isolated from freshly obtained colon tissue as previously described [<xref ref-type="bibr" rid="CR29">29</xref>]. In brief, following euthanasia of the mice, the colon was isolated and longitudinally incised, subsequently sectioned into 1&#x000a0;cm fragments. The intestinal pieces were subjected to two rounds of pre-digestion in 5&#x000a0;mL solution (1&#x02009;&#x000d7;&#x02009;Hank&#x02019;s balanced salt solution (HBSS, Servicebio, Wuhan, China) containing 5&#x000a0;mM ethylene diamine tetra acetic acid (EDTA, Sangon Biotech, Shanghai, China) and 1&#x000a0;mM dithiothreitol (DTT, Yeason, Shanghai, China)), and incubated by shaking at 40&#x000a0;g for 20&#x000a0;min at 37&#x000a0;&#x000b0;C. The remaining tissue was filtered through a 100-&#x000b5;m cell strainer and subsequently cut into 1&#x000a0;mm fragments, then collected into a 50-mL centrifuge tube containing 5&#x000a0;mL of digestion solution and was incubated by shaking at 40&#x000a0;g for 30&#x000a0;min at 37&#x000a0;&#x000b0;C to facilitate digestion. The digestion solution was prepared by dissolving 0.05&#x000a0;g of collagenase D (Roche, Basel, Switzerland), 0.05&#x000a0;g of DNase I (Sigma, St. Louis, Missouri, USA), and 0.3&#x000a0;g of dispase II (Roche, Basel, Switzerland) in 100&#x000a0;mL of PBS. After incubation, the cell suspension was strongly vortexed and filtered through a 40-&#x003bc;m cell strainer. The filtrate was collected and centrifuged at 20&#x000a0;&#x000b0;C for 10&#x000a0;min at 500&#x02009;&#x000d7;&#x02009;g. Following the removal of the supernatant, the precipitate was resuspended in 1&#x000a0;mL of cold fluorescence-activated cell sorting (FACS) buffer (1&#x02009;&#x000d7;&#x02009;PBS containing 3% (v/v) fetal bovine serum) and transferred to a 1.5-mL centrifuge tube.</p></sec><sec id="Sec14"><title>Flow cytometry</title><p id="Par67">Flow cytometric analysis was conducted using the LSR Fortessa (BD Biosciences, San Jose, CA, USA) instrument, and data processing was performed utilizing FlowJo software. Briefly, dead cells were excluded by staining with Fixable Viability Stain 510 (BD Pharmingen, San Diego, CA, USA). LPMCs were first subjected to surface marker staining with the following markers: CD45-APC-CY7, CD3-FITC, B220-R718, CD138-BV421, IgA-PE, CD4-BB700, CD25-BV421, CD11b-FITC, F4/80-BV605, CD11c-PE-CY7, and I-A/I-E-BB700 (BD Pharmingen, San Diego, CA, USA). Then, cells were fixed and permeabilized by Transcription Factor Buffer Set (BD Pharmingen, San Diego, CA, USA). Finally, cells were stained for intracellular markers: vitamin D receptor (VDR)-AF647, Foxp3-PE, and IL-17A-PE (BD Pharmingen, San Diego, CA, USA).</p><p id="Par68">Colonic IgA expression and relative abundance of <italic>F. prausnitzii</italic> assessment in patients with ulcerative colitis.</p><p id="Par69">Colonic biopsy specimens and fecal samples were obtained from 15 patients with UC in the Department of Gastroenterology of Shanghai General Hospital. All the written informed consent was obtained. The exclusion criteria were intestinal infection, intestinal cancers, or use of antibiotics or probiotics within two weeks. The severity of UC was assessed by the Mayo score and endoscopic Mayo score. The biopsy specimens were fixed in 4% paraformaldehyde, and IgA expression was assessed by immunofluorescence. The relative abundance of <italic>F. prausnitzii</italic> was assessed in human feces. The E.Z.N.A.&#x000ae; Stool DNA Kit (D4015, Omega, Inc., USA) was used to extract microbial DNA from the fecal samples according to the manufacturer&#x02019;s protocols. DNA was diluted to 10&#x000a0;ng/&#x003bc;L and quantitative real-time PCR was performed using 16S rRNA primers (Forward: 5&#x02032;-GTGSTGCAYGGYTGTCGTCA-3&#x02032;; Reverse: 5&#x02032;-ACGTCRTCCMCACCTTCCTC-3&#x02032;) and <italic>F. prausnitzii</italic> primers (Forward: 5&#x02032;-GGAGGAAGAAGGTCTTCGG-3&#x02032;; Reverse: 5&#x02032;-AATTCCGCCTACCTCTGCACT-3&#x02032;), respectively [<xref ref-type="bibr" rid="CR30">30</xref>, <xref ref-type="bibr" rid="CR31">31</xref>]. The study involving human samples was approved by the Ethics Committee of Shanghai General Hospital (2024KS446).</p></sec><sec id="Sec15"><title>Statistical analysis</title><p id="Par70">Data were presented as the mean&#x02009;&#x000b1;&#x02009;SEM. Spearman&#x02019;s rank correlation coefficient was used to analyze the correlation. Statistical analyses were performed using Statistical Product and Service Solutions V17 software (SPSS, USA). Graphs were plotted using GraphPad Prism 8.0 software (San Diego, USA). Comparisons between two groups were performed by the Student&#x02019;s <italic>t</italic>-test, while comparisons among multiple groups were performed by one-way analysis of variance (ANOVA). Kruskal&#x02013;Wallis test was applied for data that did not meet the normal distribution. Permutational multivariate analysis of variance (PERMANOVA) was used to evaluate differences in beta diversity. A <italic>p</italic>-value less than 0.05 was considered a statistically significant difference.</p></sec></sec><sec id="Sec16"><title>Results</title><sec id="Sec17"><title><italic>F. prausnitzii</italic> maintains intestinal homeostasis in DSS-induced colitis</title><p id="Par71">To explore the potential effect of <italic>F. prausnitzii</italic> in colitis, DSS-induced colitis mice were treated with <italic>F. prausnitzii</italic> (Fig.&#x000a0;<xref rid="Fig1" ref-type="fig">1</xref>A). The increase of body weight and reduction of DAI were more obvious in the DSS&#x02009;+&#x02009;<italic>Fp</italic> group than those of the DSS group, although without significant difference (Fig.&#x000a0;<xref rid="Fig1" ref-type="fig">1</xref>B&#x02013;C). <italic>F. prausnitzii</italic> supplement significantly ameliorated the shortening of the colon (Fig.&#x000a0;<xref rid="Fig1" ref-type="fig">1</xref>D) and MPO activity (Fig.&#x000a0;<xref rid="Fig1" ref-type="fig">1</xref>E). Histopathological analysis confirmed the significant alleviation of colitis in the DSS&#x02009;+&#x02009;<italic>Fp</italic> group (Fig.&#x000a0;<xref rid="Fig1" ref-type="fig">1</xref>F). Furthermore, the DSS&#x02009;+&#x02009;<italic>Fp</italic> group showed significantly lower mRNA levels of TNF-&#x003b1;, IL-1&#x003b2;, IL-6, and IL-17, and higher levels of TGF-&#x003b2; and IL-10 compared with those of the DSS group (Fig.&#x000a0;<xref rid="Fig1" ref-type="fig">1</xref>G), indicating reduced colonic inflammation.</p><p id="Par72">DSS-induced colitis resulted in the mucosal barrier disruption, while the supplement of <italic>F. prausnitzii</italic> significantly improved the TJPs (Claudin1, Occludin, and ZO-1) expression (Fig.&#x000a0;<xref rid="Fig1" ref-type="fig">1</xref>H&#x02013;L) and intestinal permeability including serum enzyme DAO activity and D-lactate concentration (Fig.&#x000a0;<xref rid="Fig1" ref-type="fig">1</xref>M). These results suggested the protective effect of <italic>F. prausnitzii</italic> on the intestinal barrier dysfunction in colitis.</p><p id="Par73">Given the key role of gut microbiota in maintaining intestinal homeostasis, the microbiota composition changes after <italic>F. prausnitzii</italic> supplement was characterized by 16S rRNA sequencing. The &#x003b1;-diversity by Chao1 index and Shannon index showed significantly higher microbial diversity and richness in DSS&#x02009;+&#x02009;<italic>Fp</italic> group than that in DSS group (Fig.&#x000a0;<xref rid="Fig2" ref-type="fig">2</xref>A&#x02013;B). The principal coordinates analysis (PCoA) revealed significant differences in bacterial communities between DSS and DSS&#x02009;+&#x02009;<italic>Fp</italic> groups (PERMANOVA, <italic>p</italic>&#x02009;=&#x02009;0.001) (Fig.&#x000a0;<xref rid="Fig2" ref-type="fig">2</xref>C).<fig id="Fig2"><label>Fig. 2</label><caption><p><italic>F. prausnitzii</italic> supplement modulates intestinal dysbiosis in colitis. <bold>A</bold> Alpha diversity shown by the Chao1 index. <bold>B</bold> Alpha diversity shown by the Shannon index. <bold>C</bold> Principal coordinates analysis (PCoA) of the microbial communities. <bold>D</bold> Relative abundance of phyla in gut microbiota. <bold>E</bold> Relative abundances of the Firmicutes, Bacteroidota, Actinobacteria, and Proteobacteria phyla between groups. <bold>F</bold> Relative abundance of genera in gut microbiota. <bold>G</bold> Representative genera alteration after <italic>F. prausnitzii</italic> supplement. <bold>H</bold> Heatmap showing a correlation among MPO activity, inflammation cytokines, tight junction proteins (TJPs) and the gut microbiota at the genus level. The data are presented as the means&#x02009;&#x000b1;&#x02009;SEM (<italic>n</italic>&#x02009;=&#x02009;6 mice per group). ns, no significant difference; *, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.05; **, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.01; ***, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.001</p></caption><graphic xlink:href="12916_2025_4260_Fig2_HTML" id="MO3"/></fig></p><p id="Par74">At the phylum level, Firmicutes dominated the microbial communities of CON group (Fig.&#x000a0;<xref rid="Fig2" ref-type="fig">2</xref>D), which decreased while Bacteroidota increased significantly in DSS group (Fig.&#x000a0;<xref rid="Fig2" ref-type="fig">2</xref>E). <italic>F. prausnitzii</italic> intervention decreased the Actinobacteria and Proteobacteria abundance in DSS-induced colitis, although without significant difference (Fig.&#x000a0;<xref rid="Fig2" ref-type="fig">2</xref>E).</p><p id="Par75"><italic>Turicibacter</italic>, <italic>Staphylococcus</italic>, and <italic>Lachnospiraceae NK4A136_group</italic> were the top three dominant taxa in the CON group at the genus level. The dominant genera shifted to <italic>Enterococcus</italic>, <italic>Lactobacillus</italic>, and <italic>Turicibacter</italic> in the DSS group, which did not change after <italic>F. prausnitzii</italic> supplement in colitis mice (Fig.&#x000a0;<xref rid="Fig2" ref-type="fig">2</xref>F). The top ten genera with substantial changes among the four groups were shown in Additional file 1: Fig. S4A. Significantly decreased abundance of <italic>Lachnospiraceae NK4A136_group</italic> in the DSS group was elevated after <italic>F. prausnitzii</italic> supplement (Fig.&#x000a0;<xref rid="Fig2" ref-type="fig">2</xref>G). Of the remaining enriched genera, <italic>Enterococcus</italic>, <italic>Desulfovibrio</italic>, <italic>Escherichia-Shigella</italic>, and <italic>Enterorhabdus</italic> increased significantly in the DSS group, while decreasing after <italic>F. prausnitzii</italic> treatment in DSS&#x02009;+&#x02009;<italic>Fp</italic> mice (Fig.&#x000a0;<xref rid="Fig2" ref-type="fig">2</xref>G). <italic>F. prausnitzii</italic> treatment did not reverse the elevated abundance of <italic>Lactobacillus</italic> and <italic>Alistipes</italic> and the decreased abundance of <italic>Staphylococcus</italic> in the DSS group (Additional file 1: Fig. S4B). No significant difference in abundance of <italic>norank</italic><sub><italic>_</italic></sub><italic>f_Muribaculaceae</italic> or <italic>Bacteroides</italic> was found between the CON and DSS groups or the DSS and DSS&#x02009;+&#x02009;<italic>Fp</italic> groups (Additional file 1: Fig. S4B). To further explore how altered gut microbiota correlated with intestinal inflammation and barrier dysfunction, the heatmap by Spearman&#x02019;s correlation analysis was performed (Fig.&#x000a0;<xref rid="Fig2" ref-type="fig">2</xref>H). The abundance of <italic>Lactobacillus</italic>, <italic>Enterococcus</italic>, <italic>Desulfovibrio</italic>, <italic>norank</italic><sub><italic>_</italic></sub><italic>f_Muribaculaceae</italic>, <italic>Escherichia-Shigella</italic>, <italic>Enterorhabdus</italic>, and <italic>Alistipes</italic> showed positive correlation with MPO activity, IL-1&#x003b2;, IL-6, and IL-17 and negative correlation with IL-10 and TJPs. In contrast, <italic>Staphylococcus</italic> and <italic>Lachnospiraceae NK4A136_group</italic> presented negative correlation with MPO activity, IL-1&#x003b2;, IL-6, and IL-17 and positive correlation with IL-10 and TJPs. Besides, <italic>Bacteroides</italic> only showed negative correlation with IL-10 and positive correlation with IL-17. No correlation between the bacteria and levels of TNF-&#x003b1; and IL-1&#x003b2;, the serum DAO activity, or D-lactate concentration was found.</p><p id="Par76">All these results indicate that alteration of gut microbiota composition is crucial to the intestinal inflammation and barrier dysfunction in colitis. Meanwhile, <italic>F. prausnitzii</italic> supplement could maintain the intestinal homeostasis to alleviate colitis by partial modulation of dysbiosis.</p><sec id="Sec18"><title>Supplementary of <italic>F. prausnitzii</italic> enhances the intestinal IgA response in colitis</title><p id="Par77">To elucidate the potential mechanisms through which <italic>F. prausnitzii</italic> interact with the host to alleviate colitis, the gene profiles of colons were analyzed by RNA sequencing analysis. After <italic>F. prausnitzii</italic> supplement, 299 genes were up-regulated and 145 genes down-regulated in DSS&#x02009;+&#x02009;<italic>Fp</italic> group compared with DSS group (<italic>p</italic>&#x02009;&#x0003c;&#x02009;0.05 and fold change&#x02009;&#x0003e;&#x02009;2) (Fig.&#x000a0;<xref rid="Fig3" ref-type="fig">3</xref>A). KEGG pathway analysis showed that the differential gene expression was mainly enriched in the intestinal immune network for IgA production, B cell receptor signaling pathway, cell adhesion molecules, cytokine-cytokine receptor interaction, NF-kappa B signaling pathway, etc. (Fig.&#x000a0;<xref rid="Fig3" ref-type="fig">3</xref>B). We found the intestinal immune network for IgA production was the most significantly enhanced pathway among these top ten up-regulated KEGG pathways with the highest rich factor (<italic>p</italic>&#x02009;=&#x02009;1.701&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;9</sup>, rich factor&#x02009;=&#x02009;0.209) (Fig.&#x000a0;<xref rid="Fig3" ref-type="fig">3</xref>B). In addition, high enrichment of genes (fold change&#x02009;&#x0003e;&#x02009;2) was focused on MHCII-related genes (H2-DMb2, H2-Eb2, H2-Ob, and H2-Oa), Aicda and Tnfrsfl3c (Fig.&#x000a0;<xref rid="Fig3" ref-type="fig">3</xref>C). The most up- and down-regulated genes involved in the intestinal immune network for IgA production were validated by RT-qPCR (Fig.&#x000a0;<xref rid="Fig3" ref-type="fig">3</xref>D), which was consistent with the regulation trends observed in the RNA-seq analysis.<fig id="Fig3"><label>Fig. 3</label><caption><p><italic>F. prausnitzii</italic> supplement enhances the colonic IgA response in colitis. <bold>A</bold> The volcano plots illustrated the differentially expressed genes (<italic>p</italic>&#x02009;&#x0003c;&#x02009;0.05 and fold change&#x02009;&#x0003e;&#x02009;2) in colon between DSS&#x02009;+&#x02009;<italic>Fp</italic> and DSS groups. <bold>B</bold> KEGG pathway analysis indicated up-regulation of the intestinal immune network for IgA production pathway in colitis mice after <italic>F. prausnitzii</italic> intervention. <bold>C</bold> Genes encoding components of the intestinal IgA production pathway (<italic>p</italic>&#x02009;&#x0003c;&#x02009;0.05 and fold change&#x02009;&#x0003e;&#x02009;2). <bold>D</bold> The mRNA expression of genes encoding IgA production, including MHCII-related genes, Aicda and Tnfrsfl3c. <bold>E</bold> The expression of IgA in colon assessed by ELISA. <bold>F</bold> Representative immunofluorescence staining images of colon sections to indicate the expression of IgA (200&#x02009;&#x000d7;&#x02009;magnification), and corresponding relative fluorescence intensity. <bold>G</bold> Relative abundance of <italic>F. prausnitzii</italic> between different groups. <bold>H</bold> The positive correlation between <italic>F. prausnitzii</italic> abundance and IgA expression (<italic>r</italic>&#x02009;=&#x02009;0.9102, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.0001). <bold>I</bold> Heatmap showing a correlation between IgA and the gut microbiota at genus level. The data are presented as the means&#x02009;&#x000b1;&#x02009;SEM (<italic>n</italic>&#x02009;=&#x02009;6 mice per group). ns, no significant difference; *, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.05; **, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.01; ***, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.001</p></caption><graphic xlink:href="12916_2025_4260_Fig3_HTML" id="MO4"/></fig></p><p id="Par78">To further verify the relationship between <italic>F. prausnitzii</italic> and IgA in colitis, the expression of colonic IgA was evaluated by ELISA, showing significantly decreased IgA in colitis models was dramatically increased after <italic>F. prausnitzii</italic> supplement (Fig.&#x000a0;<xref rid="Fig3" ref-type="fig">3</xref>E). Consistently, immunofluorescence staining also showed significantly increased colonic IgA expression in DSS&#x02009;+&#x02009;<italic>Fp</italic> group compared with that in DSS group (Fig.&#x000a0;<xref rid="Fig3" ref-type="fig">3</xref>F). Comparison of the relative abundance of <italic>F. prausnitzii</italic> between different groups (Fig.&#x000a0;<xref rid="Fig3" ref-type="fig">3</xref>G) and the positive correlation between <italic>F. prausnitzii</italic> abundance and IgA expression (<italic>r</italic>&#x02009;=&#x02009;0.9102, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.0001) (Fig.&#x000a0;<xref rid="Fig3" ref-type="fig">3</xref>H) further demonstrated the potential enhancement of IgA response after <italic>F. prausnitzii</italic> supplement in colitis.</p><p id="Par79">The composition of the gut microbiota is determined by the local habitat, which is shaped by immune response including mucosal IgA. To highlight the contribution of upregulated IgA in the resident microbial community, the correlation between IgA expression and the altered microbiota composition was performed. We found that IgA showed a significantly positive correlation with <italic>Staphylococcus</italic> and <italic>Lachnospiraceae NK4A136_group</italic>, and a significantly negative correlation with <italic>Lactobacillus</italic>, <italic>Enterococcus</italic>, <italic>Desulfovibrio</italic>, <italic>norank_f_Muribaculaceae</italic>, <italic>Escherichia-Shigella</italic>, <italic>Enterorhabdus</italic>, <italic>Bacteroides</italic>, and <italic>Alistipes</italic>, respectively (Fig.&#x000a0;<xref rid="Fig3" ref-type="fig">3</xref>I). The data suggest that the enhanced colonic IgA response after <italic>F. prausnitzii</italic> supplement might be involved in the gut microbiota modulation.</p></sec></sec><sec id="Sec19"><title>Intestinal IgA expression is involved in the protective effect of <italic>F. prausnitzii</italic></title><p id="Par80">We established IgA knockout mice to determine the role of intestinal IgA in colitis treated with <italic>F. prausnitzii</italic>. Colonic IgA expression in IgA knockout mice showed a remarkable reduction (Fig.&#x000a0;<xref rid="Fig4" ref-type="fig">4</xref>A). After DSS colitis induction, IgA knockout mice with <italic>F. prausnitzii</italic> supplement (IgA KO&#x02009;+&#x02009;DSS&#x02009;+&#x02009;<italic>Fp</italic> group) displayed shortened colon length and increased colonic injury and inflammatory cytokines (TNF-&#x003b1; and IL-6) compared with WT&#x02009;+&#x02009;DSS&#x02009;+&#x02009;<italic>Fp</italic> group (Fig.&#x000a0;<xref rid="Fig4" ref-type="fig">4</xref>B&#x02013;E). The improvement of TJPs expression in IgA KO&#x02009;+&#x02009;DSS&#x02009;+&#x02009;<italic>Fp</italic> group was also weakened (Fig.&#x000a0;<xref rid="Fig4" ref-type="fig">4</xref>F).<fig id="Fig4"><label>Fig. 4</label><caption><p>Anti-inflammatory effects of <italic>F. prausnitzii</italic> are weakened in IgA knockout mice with colitis. <bold>A</bold> Representative immunofluorescence staining images of colon sections to indicate the expression of IgA (200&#x02009;&#x000d7;&#x02009;magnification) and corresponding relative fluorescence intensity. <bold>B</bold> Experiment design and time axis. <bold>C</bold> Colon length after different interventions. <bold>D</bold> Representative H&#x00026;E staining images of colon Sects.&#x000a0;(200&#x02009;&#x000d7;&#x02009;magnification) and the histological scores. <bold>E</bold> mRNA expression levels of TNF-&#x003b1;, IL-1&#x003b2;, IL-6, and IL-10 in the colon. <bold>F</bold> Representative immunofluorescence staining images of colon sections to indicate the expression of tight junction proteins (200&#x02009;&#x000d7;&#x02009;magnification) and corresponding relative fluorescence intensity, including Claudin1 and Occludin. The data are presented as the means&#x02009;&#x000b1;&#x02009;SEM (<italic>n</italic>&#x02009;=&#x02009;3 or 4 mice per group). ns, no significant difference; *, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.05; **, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.01; ***, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.001</p></caption><graphic xlink:href="12916_2025_4260_Fig4_HTML" id="MO5"/></fig></p><p id="Par81">We then further explore the function of IgA in <italic>F. prausnitzii</italic> treatment. Since dimeric IgA is transcytosed by pIgR from the lamina propria through epithelial cells into external secretion to perform its function, and the change trends of pIgR after <italic>F. prausnitzii</italic> supplement were similar to that of IgA (Additional file 1: Fig. S5A&#x02013;C), the neutralizing antibody against pIgR was applied to block the pIgR and deplete sIgA in the pilot study (Additional file 1: Fig. S1).</p><p id="Par82">The pIgR neutralizing antibody was applied in DSS-induced colitis mice with or without <italic>F. prausnitzii</italic> supplement (Additional file 1: Fig. S2A). The successful depletion of colonic sIgA and pIgR was demonstrated by immunofluorescence staining (Fig. <xref rid="Fig5" ref-type="fig">5</xref>A, Additional file 1: Fig. S2B). The body weight loss was significantly lower in DSS&#x02009;+&#x02009;sIgA depletion&#x02009;+&#x02009;<italic>Fp</italic> group than that in DSS&#x02009;+&#x02009;<italic>Fp</italic> group (Additional file 1: Fig. S2C). The DAI score showed no significant difference among the different groups (Additional file 1: Fig. S2D). All the improvements of colon length shortening, colonic injury, and TJPs were weakened in DSS&#x02009;+&#x02009;sIgA depletion&#x02009;+&#x02009;<italic>Fp</italic> group (Additional file 1: Fig. S2E, Fig.&#x000a0;<xref rid="Fig5" ref-type="fig">5</xref>B&#x02013;C). The sIgA depletion also impaired the anti-inflammatory effect of <italic>F. prausnitzii</italic> shown by the comparison of inflammatory factors between DSS&#x02009;+&#x02009;<italic>Fp</italic> and DSS&#x02009;+&#x02009;sIgA depletion&#x02009;+&#x02009;<italic>Fp</italic> groups (Fig.&#x000a0;<xref rid="Fig5" ref-type="fig">5</xref>D).<fig id="Fig5"><label>Fig. 5</label><caption><p>sIgA depletion blocks the anti-inflammatory effects of F. prausnitzii in colitis. A Representative immunofluorescence staining images of colon sections to indicate the expression of IgA (200&#x000d7; magnification) and corresponding relative fluorescence intensity. B Representative H&#x00026;E staining images of colon sections (200&#x000d7; magnification) and the histological scores. C Representative immunofluorescence staining images of colon sections to indicate the expression of tight junction proteins (200&#x000d7; magnification) and corresponding relative fluorescence intensity, including Claudin1, Occludin, and ZO-1. D mRNA expression levels of TNF-&#x003b1;, TGF-&#x003b2;, IL-1&#x003b2;, IL-6, IL-10, and IL-17 in the colon. The data are presented as the means &#x000b1; SEM (n = 4 mice per group). ns, no significant difference; *, <italic>p</italic> &#x0003c; 0.05; **, <italic>p</italic> &#x0003c; 0.01; ***, <italic>p</italic> &#x0003c; 0.001
</p></caption><graphic xlink:href="12916_2025_4260_Fig5_HTML" id="MO6"/></fig></p><p id="Par83">FMT experiment was performed to evaluate the contribution of the sIgA-modulated microbiota in colitis. DSS&#x02009;+&#x02009;<italic>Fp</italic> and DSS&#x02009;+&#x02009;sIgA depletion&#x02009;+&#x02009;<italic>Fp</italic> groups were served as the donor mice (Additional file 1: Fig. S6A). By day 7 post-FMT, compared with the recipient colitis mice that received feces from the DSS&#x02009;+&#x02009;sIgA depletion&#x02009;+&#x02009;<italic>Fp</italic> group, the recipient mice that received feces from the DSS&#x02009;+&#x02009;<italic>Fp</italic> group showed dramatically improved shortened colon length (Additional file 1: Fig. S6B), alleviated intestinal injury (Additional file 1: Fig. S6C) and reduced TNF-&#x003b1; and elevated IL-10 expression in the colon (Additional file 1: Fig. S6D).</p><p id="Par84">The gut microbiota changes after sIgA depletion were further clarified. Significantly reduced richness and diversity of the microbial community were found in the DSS&#x02009;+&#x02009;sIgA depletion&#x02009;+&#x02009;<italic>Fp</italic> group compared to the DSS&#x02009;+&#x02009;<italic>Fp</italic> group (Additional file 1: Fig. S7A&#x02013;B). The PCoA presented clear grouping of the two groups, although no significant difference was found (PERMANOVA, <italic>p</italic>&#x02009;=&#x02009;0.1) (Additional file 1: Fig. S7C). At the phylum level, the dominant microbial communities shifted from Firmicutes and Actinobacteria in the DSS&#x02009;+&#x02009;<italic>Fp</italic> group to Firmicutes and Proteobacteria in the DSS&#x02009;+&#x02009;sIgA depletion&#x02009;+&#x02009;<italic>Fp</italic> group (Additional file 1: Fig. S7D). The composition of the gut microbiota between the two groups also differed at the genus level (Additional file 1: Fig. S7E). It was intriguing to find that the microbiota changes after sIgA depletion were mainly consistent with the four bacteria changes followed by <italic>F. prausnitzii</italic> treatment observed in Fig.&#x000a0;<xref rid="Fig2" ref-type="fig">2</xref>G. Compared to the DSS&#x02009;+&#x02009;<italic>Fp</italic> group, the relative abundance of <italic>Enterococcus</italic>, <italic>Desulfovibrio</italic>, and <italic>Escherichia-Shigella</italic> increased, and <italic>Enterorhabdus</italic> decreased in the DSS&#x02009;+&#x02009;sIgA depletion&#x02009;+&#x02009;<italic>Fp</italic> group (Additional file 1: Fig. S7F).</p><p id="Par85">These data partially establish the protective effect of <italic>F. prausnitzii</italic> to maintain intestinal homeostasis in colitis might be conducted via colonic sIgA.</p></sec><sec id="Sec20"><title><italic>F. prausnitzii</italic> upregulates production of inecalcitol to enhance intestinal IgA response in inflammatory colon</title><p id="Par86">The influence of gut microbiome on the host immune response may depend on their metabolites. The key distinguishing changes in the fecal metabolic profile were identified by metabolomics analysis. DSS group showed 816 up-regulated metabolites and 1111 down-regulated metabolites compared to CON group by volcano plot (Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>A). It also revealed distinct changes of metabolites in DSS&#x02009;+&#x02009;<italic>Fp</italic> group, including 118 upregulated metabolites and 215 downregulated metabolites, compared to DSS group (Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>B). We focused on the top fifteen important metabolites according to the relative abundance with importance of variables (VIP) value&#x02009;&#x0003e;&#x02009;2 between DSS&#x02009;+&#x02009;<italic>Fp</italic> and DSS groups by the orthogonal projection to latent structure-discriminant analysis (OPLS-DA) models (Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>C). The VIP values of these fifteen metabolites between DSS and CON groups were ranked, among which ten metabolites with distinct changes (VIP&#x02009;&#x0003e;&#x02009;1) after DSS colitis induction were chosen (Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>D). The metabolites produced by <italic>F. prausnitzii </italic>in vitro were also verified. In addition to the LYHBHI medium (Additional file 1: Fig. S8A&#x02013;C), the M2GSC medium, characterized by its well-defined composition to minimize background interference in metabolomic analyses, was further used for <italic>F. prausnitzii</italic> inoculation in vitro (Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>E&#x02013;F, Additional file 1: Fig. S8D). The volcano plot showed the up-regulated 239 metabolites in <italic>F. prausnitzii</italic> culture supernatant compared to M2GSC medium (Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>E). Among the top 10 discriminating metabolites between DSS and CON groups (Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>D), inecalcitol (19-nor-14-epi-23-yne-1,25-(OH)<sub>2</sub>D<sub>3</sub>) was the only metabolite dramatically produced by <italic>F. prausnitzii </italic>in vitro (<italic>p</italic>&#x02009;&#x0003c;&#x02009;0.01) (Fig. <xref rid="Fig6" ref-type="fig">6</xref>F). Inecalcitol was significantly down-regulated after DSS colitis induction (DSS <italic>vs.</italic> CON groups, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.01, VIP&#x02009;&#x0003e;&#x02009;1) (Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>D) and significantly up-regulated after <italic>F. prausnitzii</italic> supplement (DSS&#x02009;+&#x02009;<italic>Fp vs.</italic> DSS groups, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.01, VIP&#x02009;&#x0003e;&#x02009;2) (Fig. <xref rid="Fig6" ref-type="fig">6</xref>&#x02009;C), which was also confirmed by the comparison of relative abundance in different groups (<italic>p</italic>&#x02009;&#x0003c;&#x02009;0.001) (Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>G). Therefore, inecalcitol might be the candidate of key metabolites in <italic>F. prausnitzii-</italic>associated ameliorated colitis.<fig id="Fig6"><label>Fig. 6</label><caption><p>Colitis treated with <italic>F. prausnitzii</italic> shows different metabolite profiles. <bold>A</bold> Volcano plot illustrated key differential metabolites between DSS and CON groups (<italic>p</italic>&#x02009;&#x0003c;&#x02009;0.05 and fold change&#x02009;&#x0003e;&#x02009;1.0). <bold>B</bold> Volcano plot between DSS&#x02009;+&#x02009;<italic>Fp</italic> and DSS groups (<italic>p</italic>&#x02009;&#x0003c;&#x02009;0.05 and fold change&#x02009;&#x0003e;&#x02009;1.0). <bold>C</bold> Heatmap of top 15 discriminating metabolites between DSS&#x02009;+&#x02009;<italic>Fp</italic> and DSS groups with VIP plots&#x02009;&#x0003e;&#x02009;2. <bold>D</bold> Heatmap of top 10 discriminating metabolites between DSS and CON groups. <bold>E</bold> Volcano plot illustrated key differential metabolites between <italic>F. prausnitzii</italic> supernatant and M2GSC medium. <bold>F</bold> Comparison of relative abundance of inecalcitol in vitro. <bold>G</bold> Comparison of relative abundance of inecalcitol in vivo. The data are presented as the means&#x02009;&#x000b1;&#x02009;SEM (<italic>n</italic>&#x02009;=&#x02009;6 mice per group). ns, no significant difference; *, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.05; **, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.01; ***, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.001</p></caption><graphic xlink:href="12916_2025_4260_Fig6_HTML" id="MO7"/></fig></p><p id="Par87">Inecalcitol is one of the vitamin D analogs which have been considered to relieve colitis [<xref ref-type="bibr" rid="CR23">23</xref>]. We found that inecalcitol supplementation exerted an anti-colitis effect and improved the intestinal barrier dysfunction (Fig.&#x000a0;<xref rid="Fig7" ref-type="fig">7</xref>A&#x02013;D, Additional file 1: Fig. S3B&#x02013;D). Similar to <italic>F. prausnitzii</italic>, inecalcitol increased the expression of colonic IgA (Fig.&#x000a0;<xref rid="Fig7" ref-type="fig">7</xref>E) and pIgR (Additional file 1: Fig. S3E) in colitis. IgA knockout blunted the effect of inecalcitol supplementation in colitis (Additional file 1: Fig. S9A&#x02013;E).<fig id="Fig7"><label>Fig. 7</label><caption><p>Inecalcitol supplement alleviates colitis and promotes IgA production. <bold>A</bold> Photographs of colon length and the corresponding quantified lengths of colons. <bold>B</bold> Representative H&#x00026;E staining images of colon Sects.&#x000a0;(200&#x02009;&#x000d7;&#x02009;magnification) and the histological score. <bold>C</bold> The MPO activity of colon. <bold>D</bold> mRNA expression levels of TGF-&#x003b2;, IL-6, and IL-10 in the colon. <bold>E</bold> Representative immunofluorescence staining images of colon sections to indicate the expression of IgA (200&#x02009;&#x000d7;&#x02009;magnification) and corresponding relative fluorescence intensity. <bold>F</bold> Representative flow cytometry plots representing the proportion of CD3<sup>&#x02212;</sup>B220<sup>+</sup> B cells. <bold>G</bold> B220<sup>&#x02212;</sup>CD138<sup>+</sup> plasma cells. <bold>H</bold> IgA<sup>+</sup>CD138.<sup>+</sup> plasma cells. The data are presented as the means&#x02009;&#x000b1;&#x02009;SEM (<italic>n</italic>&#x02009;=&#x02009;4&#x02013;6 mice per group). ns, no significant difference; *, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.05; **, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.01; ***, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.001</p></caption><graphic xlink:href="12916_2025_4260_Fig7_HTML" id="MO8"/></fig></p><p id="Par88">We further investigated the immunomodulatory effects of inecalcitol in colitis by flow cytometric analysis. Significantly higher levels of CD3<sup>&#x02212;</sup>B220<sup>+</sup> B lymphocytes were found in the inecalcitol&#x02009;+&#x02009;DSS group, compared to those in the DSS group (Fig. <xref rid="Fig7" ref-type="fig">7</xref>F). Although no significant difference was found in B220<sup>&#x02212;</sup>CD138<sup>+</sup> plasma cells (Fig.&#x000a0;<xref rid="Fig7" ref-type="fig">7</xref>G), IgA<sup>+</sup>CD138<sup>+</sup> plasma cells increased obviously (Fig.&#x000a0;<xref rid="Fig7" ref-type="fig">7</xref>H). Furthermore, dendritic cells, macrophages, Treg cells, and Th17 cells were not changed significantly by inecalcitol (Additional file 1: Fig. S10A&#x02013;D). Vitamin D or its analog exerts its biological activity by binding to the vitamin D3 receptor (VDR), which functions as a transcription factor [<xref ref-type="bibr" rid="CR32">32</xref>]. Colitis mice treated with inecalcitol showed significantly increased VDR<sup>+</sup>B cells in colon, although the levels of VDR<sup>+</sup>CD138<sup>+</sup> plasma cells were very low in the CON, DSS, and DSS&#x02009;+&#x02009;inecalcitol groups (Additional file 1: Fig. S11A&#x02013;B). Taken together, these results suggested that inecalcitol might be involved in <italic>F. prausnitzii</italic>-induced change in colonic IgA expression.</p></sec><sec id="Sec21"><title><italic>F. prausnitzii</italic> has potential therapeutic effects on ulcerative colitis</title><p id="Par89">We collected colon specimens by endoscopy from active UC patients (moderate and severe severity, <italic>n</italic>&#x02009;=&#x02009;5, respectively) and UC patients with remission (<italic>n</italic>&#x02009;=&#x02009;5). Compared with UC patients with remission, active UC patients showed significantly decreased IgA expression. UC patients with severe severity had lower levels of IgA expression than patients with moderate severity (Fig. <xref rid="Fig8" ref-type="fig">8</xref>A), which was also confirmed by the negative correlation between IgA expression and endoscopic Mayo score (<italic>r</italic>&#x02009;= &#x02212;&#x02009;0.8382, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.001) and Mayo score (<italic>r</italic>&#x02009;= &#x02212;&#x02009;0.9638, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.001) (Fig.&#x000a0;<xref rid="Fig8" ref-type="fig">8</xref>B).<fig id="Fig8"><label>Fig. 8</label><caption><p><italic>F. prausnitzii</italic> enhanced the colonic IgA response in UC patients and was correlated with the severity of colitis. <bold>A</bold> IgA expression in colon samples was compared between active UC patients and UC patients with remission by immunofluorescence staining (200&#x02009;&#x000d7;&#x02009;magnification). <bold>B</bold> The negative correlation between colonic IgA and endoscopic Mayo score (<italic>r</italic>&#x02009;= &#x02212;&#x02009;0.8382, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.001) and Mayo score (<italic>r</italic>&#x02009;= &#x02212;&#x02009;0.9638, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.001). <bold>C</bold> Relative abundance of <italic>F. prausnitzii</italic> between active UC patients and UC patients with remission. <bold>D</bold> The negative correlation between <italic>F. prausnitzii</italic> abundance and endoscopic Mayo score (<italic>r</italic>&#x02009;= &#x02212;&#x02009;0.7115, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.01) and Mayo score (<italic>r</italic>&#x02009;= &#x02212;&#x02009;0.8310, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.001). <bold>E</bold> The positive correlation between <italic>F. prausnitzii</italic> abundance and colonic IgA expression (<italic>r</italic>&#x02009;=&#x02009;0.8786, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.001). <bold>F</bold> The relative abundance of inecalcitol between active UC patients and UC patients with remission. <bold>G</bold> The negative correlation between inecalcitol abundance and endoscopic Mayo score (<italic>r</italic>&#x02009;= &#x02212;&#x02009;0.6920, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.001) and Mayo score (<italic>r</italic>&#x02009;= &#x02212;&#x02009;0.8456, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.0001). <bold>H</bold> The positive correlation between inecalcitol abundance and colonic IgA (r&#x02009;=&#x02009;0.8536, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.0001). The data are presented as the means&#x02009;&#x000b1;&#x02009;SEM (<italic>n</italic>&#x02009;=&#x02009;5 per group). *, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.05; **, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.01</p></caption><graphic xlink:href="12916_2025_4260_Fig8_HTML" id="MO9"/></fig></p><p id="Par90">In human feces, we found that the relative abundance of <italic>F. prausnitzii</italic> in UC patients with remission was significantly higher than those with moderate or severe severity (Fig. <xref rid="Fig8" ref-type="fig">8</xref>C). The <italic>F. prausnitzii</italic> abundance was negatively correlated with endoscopic Mayo score and Mayo score (Fig.&#x000a0;<xref rid="Fig8" ref-type="fig">8</xref>D). The correlation between <italic>F. prausnitzii</italic> and colonic IgA response was next analyzed. The <italic>F. prausnitzii</italic> abundance was positively correlated with IgA expression (Fig.&#x000a0;<xref rid="Fig8" ref-type="fig">8</xref>E). Furthermore, the relative abundance of inecalcitol in the fecal samples of UC patients was measured by metabolomics analysis. The relative abundance of inecalcitol was significantly decreased in active UC patients compared to UC patients with remission (Fig. <xref rid="Fig8" ref-type="fig">8</xref>F), which was negatively correlated with endoscopic Mayo score (<italic>r</italic>&#x02009;= &#x02212;&#x02009;0.6920, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.001) and Mayo score (<italic>r</italic>&#x02009;= &#x02212;&#x02009;0.8456, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.0001) (Fig.&#x000a0;<xref rid="Fig8" ref-type="fig">8</xref>G). The positive correlation between inecalcitol abundance and colonic IgA (<italic>r</italic>&#x02009;=&#x02009;0.8536, <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.0001) was also found (Fig.&#x000a0;<xref rid="Fig8" ref-type="fig">8</xref>H). Collectively, these results imply that increased abundance of <italic>F. prausnitzii</italic> and inecalcitol is associated with enhanced colonic IgA response and decreased UC severity.</p></sec></sec><sec id="Sec22"><title>Discussion</title><p id="Par91">Varies human studies demonstrated significantly reduced abundance of <italic>F. prausnitzii</italic> in UC, especially in active UC [<xref ref-type="bibr" rid="CR33">33</xref>]. The supplement of <italic>F. prausnitzii</italic> or culture supernatant attenuated the severity of intestinal inflammation and enhanced gut barrier dysfunction in the experimental colitis model [<xref ref-type="bibr" rid="CR34">34</xref>]. Our study showed similar results, representing decreased intestinal histopathological scores and pro-inflammatory factors and ameliorated TJPs expression and serum DAO activity and D-lactate concentration. Despite its character as a probiotic candidate to treat IBD, the underlying mechanism to alleviate colitis needs further investigation.</p><p id="Par92">Gut dysbiosis plays a critical role in the pathogenesis of UC [<xref ref-type="bibr" rid="CR35">35</xref>]. Both the decline in the abundance of Firmicutes and Bacteroidetes and the increase of Proteobacteria have been reported in UC patients [<xref ref-type="bibr" rid="CR6">6</xref>]. In our study, although <italic>F. prausnitzii</italic> treatment in the DSS mice could not restore the altered abundance of Firmicutes and Bacteroidota, the elevated abundance of Actinobacteria and Proteobacteria was decreased. At the genus level, the elevated abundance of <italic>Enterococcus</italic>, <italic>Desulfovibrio</italic>, <italic>Escherichia-Shigella</italic>, and <italic>Enterorhabdus</italic> in the DSS group was reversed after <italic>F. prausnitzii</italic> treatment. These potentially pathogenic bacteria were reported to be related to intestinal inflammation and gut dysfunction [<xref ref-type="bibr" rid="CR36">36</xref>&#x02013;<xref ref-type="bibr" rid="CR39">39</xref>], although a negative association between <italic>Enterorhabdus</italic> and IBD was also reported [<xref ref-type="bibr" rid="CR40">40</xref>]. We systematically reviewed previous studies regarding changes in <italic>Enterorhabdus</italic> in colitis and found inconsistent conclusions. Increased abundance of <italic>Enterorhabdus</italic> was observed in diet-induced colitis models [<xref ref-type="bibr" rid="CR41">41</xref>]. Researchers even isolated a member of the <italic>Enterorhabdus</italic> genus, <italic>Enterorhabdus caecimuris</italic>, in the cecum of the C3H/HeJBir spontaneous colitis mouse model, which showed a significantly increased abundance in the inflammatory environment [<xref ref-type="bibr" rid="CR42">42</xref>]. However, Han et al. [<xref ref-type="bibr" rid="CR43">43</xref>] and Zhong et al. [<xref ref-type="bibr" rid="CR44">44</xref>] reported that the abundance of <italic>Enterorhabdus</italic> is significantly decreased in DSS-induced colitis mice compared to control mice, which is contrary to our results. What&#x02019;s more intriguing, Liu et al. [<xref ref-type="bibr" rid="CR45">45</xref>] and Wang et al. [<xref ref-type="bibr" rid="CR46">46</xref>] observed no significant changes in <italic>Enterorhabdus</italic> abundance in DSS-induced colitis mice. These results suggested heterogeneity in the abundance of <italic>Enterorhabdus</italic> across different research, highlighting that the pro-inflammatory or anti-inflammatory role of <italic>Enterorhabdus</italic> has not been fully defined. In our study, the abundance of <italic>Enterorhabdus</italic> significantly increased in DSS models and significantly decreased after <italic>F. prausnitzii</italic> treatment. Although <italic>Enterorhabdus</italic> showed a positive correlation with several inflammatory markers and a negative correlation with anti-inflammatory markers in our study, partially suggesting its potential pro-inflammatory role, further validation experiments are needed.</p><p id="Par93">Probiotics are part of the intestinal microbial barrier and help to regulate the structure of the microbiota. They can competitively consume the nutrients of pathogenic bacteria and compete for the adhesion sites. Moreover, they can induce the host immune response to indirectly restrict pathogen growth and colonization [<xref ref-type="bibr" rid="CR47">47</xref>]. Transcripts linked with <italic>F. prausnitzii</italic> treatment to DSS-induced colitis were revealed by using RNA-Seq, focusing on the intestinal immune network for IgA production. The high enrichment of genes was focused on MHCII-related genes, Aicda and Tnfrsfl3c in the colitis mice after <italic>F. prausnitzii</italic> treatment. The upregulation of MHCII genes may represent the augmentation of follicular helper T cells (T<sub>FH</sub>), while CD40 and CD40L interactions between T<sub>FH</sub> and B cells are critical for germinal center (GC) formation [<xref ref-type="bibr" rid="CR48">48</xref>]. Plasma cell-produced IgA is essential to ensure appropriate mucosal immunity between the host and microbiota. Compared to the IgA in circulation, IgA from the lamina propria can be transcytosed by pIgR into the intestinal lumen as sIgA and anchor itself on the outer mucosal surface to interact with the gut bacteria for proper immune-microbiota stability [<xref ref-type="bibr" rid="CR49">49</xref>].</p><p id="Par94">It has been reported that sIgA aids in preventing IBD pathogenesis by helping to facilitate microbiota stability [<xref ref-type="bibr" rid="CR50">50</xref>], whereas other research found the high level of IgA coat inflammatory commensals that preferentially drive intestinal disease [<xref ref-type="bibr" rid="CR51">51</xref>]. The generation of high-affinity class switched IgA results from class switch recombination (CSR), which is initiated by the enzyme activation-induced cytidine deaminase (AID), encoded by Aicda [<xref ref-type="bibr" rid="CR52">52</xref>]. Therefore, enrichment of Aicda may indicate high affinity IgA production. IgA knockout mice allow us to validate the role of IgA in <italic>F. prausnitzii</italic>-mediated protection against colitis; however, they might introduce potential confounders such as altered immune response and microbiota composition [<xref ref-type="bibr" rid="CR53">53</xref>, <xref ref-type="bibr" rid="CR54">54</xref>]. The pIgR neutralization blocks the secretion of IgA into the gut lumen without affecting systemic IgA levels, which is temporally controlled, although pIgR neutralization may impact other pIgR-dependent processes such as the secretion of IgM [<xref ref-type="bibr" rid="CR55">55</xref>&#x02013;<xref ref-type="bibr" rid="CR57">57</xref>]. Taken together, IgA knockout mice highlight the necessity of IgA, while sIgA depletion models clarify the specific contribution of sIgA in <italic>F. prausnitzii</italic>-mediated anti-inflammatory effects. In our study, we noticed that the protective effect of <italic>F. prausnitzii</italic> on intestinal inflammation and gut barrier function in DSS-induced colitis was abolished in sIgA-depletion mice, suggesting that sIgA is the key target of <italic>F. prausnitzii</italic> in controlling gut injury. We hypothesized that the <italic>F. prausnitzii</italic> intervention might increase the level of sIgA in the colon, which might be related to the reduction of the potentially pathogenic organisms in the colitis. However, the specific microbiota changes due to sIgA induction after the <italic>F. prausnitzii</italic> intervention have not been explored thoroughly in our study, and further investigations might explain certain issues arising in the results, such as the contradictory changes in the abundance of the <italic>Enterorhabdus</italic> genus in sIgA depletion mice after <italic>F. prausnitzii</italic> supplementation. To define the IgA-coated microbiota, the sorting and sequencing of IgA-coated microbiota (called IgA-seq) [<xref ref-type="bibr" rid="CR58">58</xref>] might be used in the future.</p><p id="Par95">Since <italic>F. prausnitzii</italic> is an obligate anaerobe, the screening for certain metabolites is essential, which could represent a potential therapeutic tool. Butyrate, specifically produced by <italic>F. prausnitzii</italic>, is important for intestinal physiology. However, other conflicting studies showed metabolic molecules such as shikimic and salicylic acids, rather than butyrate, were associated with its protective effects [<xref ref-type="bibr" rid="CR59">59</xref>]. We found that <italic>F. prausnitzii</italic> treatment elevated fecal inecalcitol levels in colitis mice, as well as in two different culture media, including LYHBHI and M2GSC, inoculated with <italic>F. prausnitzii.</italic> Inecalcitol is an analog of the active form of vitamin D, characterized by a high anti-proliferative effect and a low calcemic potential [<xref ref-type="bibr" rid="CR60">60</xref>]. Another vitamin D analog, TX527, has been improved to ameliorate disease symptoms in a DSS-induced colitis with decreased levels of inflammatory cytokines such as IL-1, IL-6, IFN-&#x003b3; and TNF-&#x003b1; and gastrointestinal glutathione peroxidase 2 [<xref ref-type="bibr" rid="CR23">23</xref>]. Vitamin D can impact the function of immune cells in the gut by binding to intracellular VDR and subsequently transcribing relevant genes to maintain intestinal homeostasis. Inflammation-related signaling, including E-cadherin, RXR&#x003b1;, and wnt/&#x003b2;-catenin, seems to work via VDR [<xref ref-type="bibr" rid="CR61">61</xref>]. Therefore, it implies that <italic>F. prausnitzii</italic> might modulate the IgA expression via inecalcitol, which needs further study in the future.</p></sec><sec id="Sec23"><title>Conclusions</title><p id="Par96">In the present study, the anti-inflammatory and dysbiosis modulation effects of <italic>F. prausnitzii</italic> in the DSS-induced colitis mice were strongly correlated with the intestinal immune network of the IgA production pathway based on multi-omics analysis. The beneficial effects of <italic>F. prausnitzii</italic> were weakened in IgA knockout mice and sIgA depleted mouse model. The shift profile of fecal metabolites after <italic>F. prausnitzii</italic> supplement was characterized by increased production of inecalcitol, which may account for the enhanced IgA response. In a cohort of UC patients, the relative abundance of <italic>F. prausnitzii</italic> and inecalcitol was decreased and positively correlated with colonic IgA expression and negatively correlated with disease severity. These results suggest that <italic>F. prausnitzii</italic> effectively alleviated colonic inflammation and modulated dysbiosis via enhancing colonic IgA response, thus showing promise as a UC treatment.</p></sec><sec id="Sec24" sec-type="supplementary-material"><title>Supplementary Information</title><p>
<supplementary-material content-type="local-data" id="MOESM1"><media xlink:href="12916_2025_4260_MOESM1_ESM.docx"><caption><p>Additional file 1: Fig S1. The pilot study assessed the effect and optimal regimen of pIgR neutralizing antibody for sIgA depletion. Fig S2. sIgA depletion blocked the anti-inflammatory effects of F. prausnitzii in colitis. Fig S3. Inecalcitol supplement in DSS-induced colitis. Fig S4. The modulation of intestinal microbiota dysbiosis in colitis after F. prausnitzii supplement. Fig S5. The assessment of colonic pIgR expression after F. prausnitzii supplement in colitis. Fig S6. Fecal microbiota transplantationexperiment to evaluate the contribution of the sIgA-modulated microbiota in colitis mice. Fig S7. Intestinal microbiota changes in colitis mice with F. prausnitzii supplement after sIgA depletion. Fig S8. Metabolite profiling of F. prausnitzii in LYHBHI and M2GSC media. Fig S9. IgA knockout weakens the effect of inecalcitol supplementation in colitis. Fig S10. The immunomodulatory effects of inecalcitol in colitis by flow cytometric analysis. Fig S11. VDR is involved in IgA production through inecalcitol supplement</p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="MOESM2"><media xlink:href="12916_2025_4260_MOESM2_ESM.docx"><caption><p>Additional file 2. Table S1. Primers sequences of cytokines. Table S2. Primers sequences of tight junction proteins. Table S3. Primers sequences of validated different expressed genes</p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="MOESM3"><media xlink:href="12916_2025_4260_MOESM3_ESM.pdf"><caption><p>Additional file 3.</p></caption></media></supplementary-material></p></sec></body><back><glossary id="AGS1"><title>Abbreviations</title><def-list><def-item><term>AID</term><def><p id="Par6">Activation-induced cytidine deaminase</p></def></def-item><def-item><term>CSR</term><def><p id="Par7">Class switch recombination</p></def></def-item><def-item><term>CFU</term><def><p id="Par8">Colony forming units</p></def></def-item><def-item><term>DAI</term><def><p id="Par9">Disease activity index</p></def></def-item><def-item><term>DAO</term><def><p id="Par10">Diamine oxidase</p></def></def-item><def-item><term>DEGs</term><def><p id="Par11">Differential expressed genes</p></def></def-item><def-item><term>DSS</term><def><p id="Par12">Dextran sodium sulfate</p></def></def-item><def-item><term>FC</term><def><p id="Par13">Fold change</p></def></def-item><def-item><term>GC</term><def><p id="Par14">Germinal center</p></def></def-item><def-item><term>H&#x00026;E</term><def><p id="Par15">Hematoxylin and eosin</p></def></def-item><def-item><term>IBD</term><def><p id="Par16">Inflammatory bowel disease</p></def></def-item><def-item><term>IgA</term><def><p id="Par17">Immunoglobulin A</p></def></def-item><def-item><term>MAM</term><def><p id="Par18">Microbial anti-inflammatory molecule</p></def></def-item><def-item><term>MPO</term><def><p id="Par19">Myeloperoxidase</p></def></def-item><def-item><term>OPLS-DA</term><def><p id="Par20">Orthogonal projection to latent structure-discriminant analysis</p></def></def-item><def-item><term>OUT</term><def><p id="Par21">Pperational taxonomic unit</p></def></def-item><def-item><term>PBS</term><def><p id="Par22">Phosphate-buffered saline</p></def></def-item><def-item><term>PCoA</term><def><p id="Par23">Principal coordinates analysis</p></def></def-item><def-item><term>PERMANOVA</term><def><p id="Par24">Permutational multivariate analysis of variance</p></def></def-item><def-item><term>pIgR</term><def><p id="Par25">Polymeric immunoglobulin receptor</p></def></def-item><def-item><term>sIgA</term><def><p id="Par26">Secretory immunoglobulin A</p></def></def-item><def-item><term>TJP</term><def><p id="Par27">Tight junction protein</p></def></def-item><def-item><term>T<sub>FH</sub></term><def><p id="Par28">Follicular helper T cell</p></def></def-item><def-item><term>UC</term><def><p id="Par29">Ulcerative colitis</p></def></def-item><def-item><term>VDR</term><def><p id="Par30">Vitamin D receptor</p></def></def-item></def-list></glossary><fn-group><fn><p><bold>Publisher&#x02019;s Note</bold></p><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></fn><fn><p>Wenfei Qin, Nuoming Yin and Binqiang Xu contributed equally to this work.</p></fn></fn-group><ack><title>Acknowledgements</title><p>Not applicable.</p></ack><notes notes-type="author-contribution"><title>Authors&#x02019; contributions</title><p>CH, YZ, WQ and ZL designed the experiments. WQ, NY, BX, YF, JF and YL performed the experiments. CH, QM, WQ, NY, GW and LA analyzed and discussed the data. CH, and YZ wrote and edited the manuscript. All authors contributed to the critical revision of the manuscript and approved the final draft. All authors have read and agreed to the published version of the manuscript.</p></notes><notes notes-type="funding-information"><title>Funding</title><p>This work was supported by the National Natural Science Foundation of China (No. 81970555, 82270671), Science and Technology Commission of Songjiang District (22SJKGGG28), Shanghai Jiao Tong University School of Medicine, Digestive Institute (KY-2023&#x02013;03-02), Shanghai General Hospital (CCTR-2022B02) and Shanghai Municipal Health Commission (202440170).</p><p>National Natural Science Foundation of China,No. 81970555,Shanghai General Hospital,No.CCTR-2022B02,Shanghai Municipal Health Commission,202440170,Science and Technology Commission of Songjiang District,No.22SJKGGG28,Shanghai Jiao Tong University School of Medicine,Digestive Institute,No.KY-2023-03-02
</p></notes><notes notes-type="data-availability"><title>Data availability
</title><p>The raw data were deposited into the NCBI Sequence Read Archive (SRA) database (Accession Number of 16S rRNA sequencing: PRJNA1093442, accession Number of RNA-seq: PRJNA1044084). The raw data can be accessed via links (https://dataview.ncbi.nlm.nih.gov/object/PRJNA1093442?reviewer&#x02009;=&#x02009;ij2nujlkoj47cn2vlnaej3svup, https://dataview.ncbi.nlm.nih.gov/object/PRJNA1044084?reviewer&#x02009;=&#x02009;61n2ufbe0pbecjnu1uqjc6qdf5) and are available publicly as of the data of publication.</p></notes><notes><title>Declarations</title><notes id="FPar1"><title>Ethics approval and consent to participate</title><p id="Par97">The study involving human samples was approved by the Ethics Committee of Shanghai General Hospital (2024KS446). All the written informed consents were obtained.</p></notes><notes id="FPar2"><title>Consent for publication</title><p id="Par98">Not applicable.</p></notes><notes id="FPar3" notes-type="COI-statement"><title>Competing interests
</title><p id="Par99">The authors declare no competing interests.</p></notes></notes><ref-list id="Bib1"><title>References</title><ref id="CR1"><label>1.</label><citation-alternatives><element-citation id="ec-CR1" publication-type="journal"><person-group person-group-type="author"><name><surname>Kaplan</surname><given-names>GG</given-names></name></person-group><article-title>The global burden of IBD: from 2015 to 2025</article-title><source>Nat Rev Gastroenterol Hepatol</source><year>2015</year><volume>12</volume><fpage>720</fpage><lpage>727</lpage><?supplied-pmid 26323879?><pub-id pub-id-type="pmid">26323879</pub-id>
</element-citation><mixed-citation id="mc-CR1" publication-type="journal">Kaplan GG. The global burden of IBD: from 2015 to 2025. Nat Rev Gastroenterol Hepatol. 2015;12:720&#x02013;7.<pub-id pub-id-type="pmid">26323879</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR2"><label>2.</label><citation-alternatives><element-citation id="ec-CR2" publication-type="journal"><person-group person-group-type="author"><name><surname>Ng</surname><given-names>SC</given-names></name><name><surname>Kaplan</surname><given-names>GG</given-names></name><name><surname>Tang</surname><given-names>W</given-names></name><name><surname>Banerjee</surname><given-names>R</given-names></name><name><surname>Adigopula</surname><given-names>B</given-names></name><name><surname>Underwood</surname><given-names>FE</given-names></name><etal/></person-group><article-title>Population density and risk of inflammatory bowel disease: a prospective population-based study in 13 countries or regions in Asia-pacific</article-title><source>Am J Gastroenterol</source><year>2019</year><volume>114</volume><fpage>107</fpage><lpage>115</lpage><?supplied-pmid 30177785?><pub-id pub-id-type="pmid">30177785</pub-id>
</element-citation><mixed-citation id="mc-CR2" publication-type="journal">Ng SC, Kaplan GG, Tang W, Banerjee R, Adigopula B, Underwood FE, et al. Population density and risk of inflammatory bowel disease: a prospective population-based study in 13 countries or regions in Asia-pacific. Am J Gastroenterol. 2019;114:107&#x02013;15.<pub-id pub-id-type="pmid">30177785</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR3"><label>3.</label><citation-alternatives><element-citation id="ec-CR3" publication-type="journal"><person-group person-group-type="author"><name><surname>Du</surname><given-names>L</given-names></name><name><surname>Ha</surname><given-names>C</given-names></name></person-group><article-title>Epidemiology and pathogenesis of ulcerative colitis</article-title><source>Gastroenterol Clin North Am</source><year>2020</year><volume>49</volume><fpage>643</fpage><lpage>654</lpage><?supplied-pmid 33121686?><pub-id pub-id-type="pmid">33121686</pub-id>
</element-citation><mixed-citation id="mc-CR3" publication-type="journal">Du L, Ha C. Epidemiology and pathogenesis of ulcerative colitis. Gastroenterol Clin North Am. 2020;49:643&#x02013;54.<pub-id pub-id-type="pmid">33121686</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR4"><label>4.</label><citation-alternatives><element-citation id="ec-CR4" publication-type="journal"><person-group person-group-type="author"><name><surname>Dubinsky</surname><given-names>MC</given-names></name><name><surname>Watanabe</surname><given-names>K</given-names></name><name><surname>Molander</surname><given-names>P</given-names></name><name><surname>Peyrin-Biroulet</surname><given-names>L</given-names></name><name><surname>Rubin</surname><given-names>M</given-names></name><name><surname>Melmed</surname><given-names>GY</given-names></name><etal/></person-group><article-title>Ulcerative colitis narrative global survey findings: the impact of living with ulcerative colitis-patients' and physicians' view</article-title><source>Inflamm Bowel Dis</source><year>2021</year><volume>27</volume><fpage>1747</fpage><lpage>55</lpage><?supplied-pmid 33529314?><pub-id pub-id-type="pmid">33529314</pub-id>
</element-citation><mixed-citation id="mc-CR4" publication-type="journal">Dubinsky MC, Watanabe K, Molander P, Peyrin-Biroulet L, Rubin M, Melmed GY, et al. Ulcerative colitis narrative global survey findings: the impact of living with ulcerative colitis-patients&#x02019; and physicians&#x02019; view. Inflamm Bowel Dis. 2021;27:1747&#x02013;55.<pub-id pub-id-type="pmid">33529314</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR5"><label>5.</label><citation-alternatives><element-citation id="ec-CR5" publication-type="journal"><person-group person-group-type="author"><name><surname>Ord&#x000e1;s</surname><given-names>I</given-names></name><name><surname>Eckmann</surname><given-names>L</given-names></name><name><surname>Talamini</surname><given-names>M</given-names></name><name><surname>Baumgart</surname><given-names>DC</given-names></name><name><surname>Sandborn</surname><given-names>WJ</given-names></name></person-group><article-title>Ulcerative colitis</article-title><source>Lancet</source><year>2012</year><volume>380</volume><fpage>1606</fpage><lpage>1619</lpage><?supplied-pmid 22914296?><pub-id pub-id-type="pmid">22914296</pub-id>
</element-citation><mixed-citation id="mc-CR5" publication-type="journal">Ord&#x000e1;s I, Eckmann L, Talamini M, Baumgart DC, Sandborn WJ. Ulcerative colitis. Lancet. 2012;380:1606&#x02013;19.<pub-id pub-id-type="pmid">22914296</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR6"><label>6.</label><citation-alternatives><element-citation id="ec-CR6" publication-type="journal"><person-group person-group-type="author"><name><surname>Zou</surname><given-names>J</given-names></name><name><surname>Liu</surname><given-names>C</given-names></name><name><surname>Jiang</surname><given-names>S</given-names></name><name><surname>Qian</surname><given-names>D</given-names></name><name><surname>Duan</surname><given-names>J</given-names></name></person-group><article-title>Cross talk between gut microbiota and intestinal mucosal immunity in the development of ulcerative colitis</article-title><source>Infect Immun</source><year>2021</year><volume>89</volume><fpage>e0001421</fpage><?supplied-pmid 33526559?><pub-id pub-id-type="pmid">33526559</pub-id>
</element-citation><mixed-citation id="mc-CR6" publication-type="journal">Zou J, Liu C, Jiang S, Qian D, Duan J. Cross talk between gut microbiota and intestinal mucosal immunity in the development of ulcerative colitis. Infect Immun. 2021;89:e0001421.<pub-id pub-id-type="pmid">33526559</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR7"><label>7.</label><citation-alternatives><element-citation id="ec-CR7" publication-type="journal"><person-group person-group-type="author"><name><surname>Skelly</surname><given-names>AN</given-names></name><name><surname>Sato</surname><given-names>Y</given-names></name><name><surname>Kearney</surname><given-names>S</given-names></name><name><surname>Honda</surname><given-names>K</given-names></name></person-group><article-title>Mining the microbiota for microbial and metabolite-based immunotherapies</article-title><source>Nat Rev Immunol</source><year>2019</year><volume>19</volume><fpage>305</fpage><lpage>323</lpage><?supplied-pmid 30858494?><pub-id pub-id-type="pmid">30858494</pub-id>
</element-citation><mixed-citation id="mc-CR7" publication-type="journal">Skelly AN, Sato Y, Kearney S, Honda K. Mining the microbiota for microbial and metabolite-based immunotherapies. Nat Rev Immunol. 2019;19:305&#x02013;23.<pub-id pub-id-type="pmid">30858494</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR8"><label>8.</label><citation-alternatives><element-citation id="ec-CR8" publication-type="journal"><person-group person-group-type="author"><name><surname>Huang</surname><given-names>CL</given-names></name><name><surname>Mei</surname><given-names>QX</given-names></name><name><surname>Lou</surname><given-names>LH</given-names></name><name><surname>Huang</surname><given-names>ZH</given-names></name><name><surname>Yang</surname><given-names>Fu</given-names></name><name><surname>Fan</surname><given-names>JJ</given-names></name><etal/></person-group><article-title>Ulcerative colitis in response to fecal microbiota transplantation via modulation of gut microbiota and Th17/Treg cell balance</article-title><source>Cells</source><year>2022</year><volume>11</volume><fpage>1851</fpage><?supplied-pmid 35681546?><pub-id pub-id-type="pmid">35681546</pub-id>
</element-citation><mixed-citation id="mc-CR8" publication-type="journal">Huang CL, Mei QX, Lou LH, Huang ZH, Yang Fu, Fan JJ, et al. Ulcerative colitis in response to fecal microbiota transplantation via modulation of gut microbiota and Th17/Treg cell balance. Cells. 2022;11:1851.<pub-id pub-id-type="pmid">35681546</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR9"><label>9.</label><citation-alternatives><element-citation id="ec-CR9" publication-type="journal"><person-group person-group-type="author"><name><surname>Leylabadlo</surname><given-names>HE</given-names></name><name><surname>Ghotaslou</surname><given-names>R</given-names></name><name><surname>Feizabadi</surname><given-names>MM</given-names></name><name><surname>Farajnia</surname><given-names>S</given-names></name><name><surname>Moaddab</surname><given-names>SY</given-names></name><name><surname>Ganbarov</surname><given-names>K</given-names></name><etal/></person-group><article-title>The critical role of <italic>Faecalibacterium prausnitzii</italic> in human health: an overview</article-title><source>Microb Pathog</source><year>2020</year><volume>149</volume><fpage>104344</fpage><?supplied-pmid 32534182?><pub-id pub-id-type="pmid">32534182</pub-id>
</element-citation><mixed-citation id="mc-CR9" publication-type="journal">Leylabadlo HE, Ghotaslou R, Feizabadi MM, Farajnia S, Moaddab SY, Ganbarov K, et al. The critical role of <italic>Faecalibacterium prausnitzii</italic> in human health: an overview. Microb Pathog. 2020;149: 104344.<pub-id pub-id-type="pmid">32534182</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR10"><label>10.</label><citation-alternatives><element-citation id="ec-CR10" publication-type="journal"><person-group person-group-type="author"><name><surname>Sokol</surname><given-names>H</given-names></name><name><surname>Pigneur</surname><given-names>B</given-names></name><name><surname>Watterlot</surname><given-names>L</given-names></name><name><surname>Lakhdari</surname><given-names>O</given-names></name><name><surname>Berm&#x000fa;dez-Humar&#x000e1;n</surname><given-names>LG</given-names></name><name><surname>Gratadoux</surname><given-names>JJ</given-names></name><etal/></person-group><article-title><italic>Faecalibacterium prausnitzii</italic> is an anti-inflammatory commensal bacterium identified by gut microbiota analysis of Crohn disease patients</article-title><source>Proc Natl Acad Sci USA</source><year>2008</year><volume>105</volume><fpage>16731</fpage><lpage>16736</lpage><?supplied-pmid 18936492?><pub-id pub-id-type="pmid">18936492</pub-id>
</element-citation><mixed-citation id="mc-CR10" publication-type="journal">Sokol H, Pigneur B, Watterlot L, Lakhdari O, Berm&#x000fa;dez-Humar&#x000e1;n LG, Gratadoux JJ, et al. <italic>Faecalibacterium prausnitzii</italic> is an anti-inflammatory commensal bacterium identified by gut microbiota analysis of Crohn disease patients. Proc Natl Acad Sci USA. 2008;105:16731&#x02013;6.
<pub-id pub-id-type="pmid">18936492</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR11"><label>11.</label><citation-alternatives><element-citation id="ec-CR11" publication-type="journal"><person-group person-group-type="author"><name><surname>Zhang</surname><given-names>M</given-names></name><name><surname>Zhou</surname><given-names>L</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Dorfman</surname><given-names>RG</given-names></name><name><surname>Tang</surname><given-names>D</given-names></name><name><surname>Xu</surname><given-names>L</given-names></name><etal/></person-group><article-title><italic>Faecalibacterium prausnitzii</italic> produces butyrate to decrease c-Myc-related metabolism and Th17 differentiation by inhibiting histone deacetylase 3</article-title><source>Int Immunol</source><year>2019</year><volume>31</volume><fpage>499</fpage><lpage>514</lpage><?supplied-pmid 30809639?><pub-id pub-id-type="pmid">30809639</pub-id>
</element-citation><mixed-citation id="mc-CR11" publication-type="journal">Zhang M, Zhou L, Wang Y, Dorfman RG, Tang D, Xu L, et al. <italic>Faecalibacterium prausnitzii</italic> produces butyrate to decrease c-Myc-related metabolism and Th17 differentiation by inhibiting histone deacetylase 3. Int Immunol. 2019;31:499&#x02013;514.<pub-id pub-id-type="pmid">30809639</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR12"><label>12.</label><citation-alternatives><element-citation id="ec-CR12" publication-type="journal"><person-group person-group-type="author"><name><surname>Sokol</surname><given-names>H</given-names></name><name><surname>Seksik</surname><given-names>P</given-names></name><name><surname>Furet</surname><given-names>JP</given-names></name><name><surname>Firmesse</surname><given-names>O</given-names></name><name><surname>Larmurier</surname><given-names>IN</given-names></name><name><surname>Beaugerie</surname><given-names>L</given-names></name><etal/></person-group><article-title>Low counts of <italic>Faecalibacterium prausnitzii</italic> in colitis microbiota</article-title><source>Inflamm Bowel Dis</source><year>2009</year><volume>15</volume><fpage>1183</fpage><lpage>1189</lpage><?supplied-pmid 19235886?><pub-id pub-id-type="pmid">19235886</pub-id>
</element-citation><mixed-citation id="mc-CR12" publication-type="journal">Sokol H, Seksik P, Furet JP, Firmesse O, Larmurier IN, Beaugerie L, et al. Low counts of <italic>Faecalibacterium prausnitzii</italic> in colitis microbiota. Inflamm Bowel Dis. 2009;15:1183&#x02013;9.<pub-id pub-id-type="pmid">19235886</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR13"><label>13.</label><citation-alternatives><element-citation id="ec-CR13" publication-type="journal"><person-group person-group-type="author"><name><surname>Qu&#x000e9;vrain</surname><given-names>E</given-names></name><name><surname>Maubert</surname><given-names>MA</given-names></name><name><surname>Michon</surname><given-names>C</given-names></name><name><surname>Chain</surname><given-names>F</given-names></name><name><surname>Marquant</surname><given-names>R</given-names></name><name><surname>Tailhades</surname><given-names>J</given-names></name><etal/></person-group><article-title>Identification of an anti-inflammatory protein from <italic>Faecalibacterium prausnitzii</italic>, a commensal bacterium deficient in Crohn's disease</article-title><source>Gut</source><year>2016</year><volume>65</volume><fpage>415</fpage><lpage>425</lpage><?supplied-pmid 26045134?><pub-id pub-id-type="pmid">26045134</pub-id>
</element-citation><mixed-citation id="mc-CR13" publication-type="journal">Qu&#x000e9;vrain E, Maubert MA, Michon C, Chain F, Marquant R, Tailhades J, et al. Identification of an anti-inflammatory protein from <italic>Faecalibacterium prausnitzii</italic>, a commensal bacterium deficient in Crohn&#x02019;s disease. Gut. 2016;65:415&#x02013;25.<pub-id pub-id-type="pmid">26045134</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR14"><label>14.</label><citation-alternatives><element-citation id="ec-CR14" publication-type="journal"><person-group person-group-type="author"><name><surname>Breyner</surname><given-names>NM</given-names></name><name><surname>Michon</surname><given-names>C</given-names></name><name><surname>de Sousa</surname><given-names>CS</given-names></name><name><surname>Vilas Boas</surname><given-names>PB</given-names></name><name><surname>Chain</surname><given-names>F</given-names></name><name><surname>Azevedo</surname><given-names>VA</given-names></name><etal/></person-group><article-title>Microbial anti-Inflammatory molecule (MAM) from <italic>Faecalibacterium prausnitzii</italic> shows a protective effect on DNBS and DSS-induced colitis model in mice through inhibition of NF-&#x003ba;B pathway</article-title><source>Front Microbiol</source><year>2017</year><volume>8</volume><fpage>114</fpage><?supplied-pmid 28203226?><pub-id pub-id-type="pmid">28203226</pub-id>
</element-citation><mixed-citation id="mc-CR14" publication-type="journal">Breyner NM, Michon C, de Sousa CS, Vilas Boas PB, Chain F, Azevedo VA, et al. Microbial anti-Inflammatory molecule (MAM) from <italic>Faecalibacterium prausnitzii</italic> shows a protective effect on DNBS and DSS-induced colitis model in mice through inhibition of NF-&#x003ba;B pathway. Front Microbiol. 2017;8:114.<pub-id pub-id-type="pmid">28203226</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR15"><label>15.</label><citation-alternatives><element-citation id="ec-CR15" publication-type="journal"><person-group person-group-type="author"><name><surname>Bron</surname><given-names>PA</given-names></name><name><surname>Kleerebezem</surname><given-names>M</given-names></name><name><surname>Brummer</surname><given-names>RJ</given-names></name><name><surname>Cani</surname><given-names>PD</given-names></name><name><surname>Mercenier</surname><given-names>A</given-names></name><name><surname>MacDonald</surname><given-names>TT</given-names></name><etal/></person-group><article-title>Can probiotics modulate human disease by impacting intestinal barrier function?</article-title><source>Br J Nutr</source><year>2017</year><volume>117</volume><fpage>93</fpage><lpage>107</lpage><?supplied-pmid 28102115?><pub-id pub-id-type="pmid">28102115</pub-id>
</element-citation><mixed-citation id="mc-CR15" publication-type="journal">Bron PA, Kleerebezem M, Brummer RJ, Cani PD, Mercenier A, MacDonald TT, et al. Can probiotics modulate human disease by impacting intestinal barrier function? Br J Nutr. 2017;117:93&#x02013;107.<pub-id pub-id-type="pmid">28102115</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR16"><label>16.</label><citation-alternatives><element-citation id="ec-CR16" publication-type="journal"><person-group person-group-type="author"><name><surname>Gou</surname><given-names>HZ</given-names></name><name><surname>Zhang</surname><given-names>YL</given-names></name><name><surname>Ren</surname><given-names>LF</given-names></name><name><surname>Li</surname><given-names>ZJ</given-names></name><name><surname>Zhang</surname><given-names>L</given-names></name></person-group><article-title>How do intestinal probiotics restore the intestinal barrier?</article-title><source>Front Microbiol</source><year>2022</year><volume>13</volume><fpage>929346</fpage><?supplied-pmid 35910620?><pub-id pub-id-type="pmid">35910620</pub-id>
</element-citation><mixed-citation id="mc-CR16" publication-type="journal">Gou HZ, Zhang YL, Ren LF, Li ZJ, Zhang L. How do intestinal probiotics restore the intestinal barrier? Front Microbiol. 2022;13: 929346.<pub-id pub-id-type="pmid">35910620</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR17"><label>17.</label><citation-alternatives><element-citation id="ec-CR17" publication-type="journal"><person-group person-group-type="author"><name><surname>Mantis</surname><given-names>NJ</given-names></name><name><surname>Rol</surname><given-names>N</given-names></name><name><surname>Corth&#x000e9;sy</surname><given-names>B</given-names></name></person-group><article-title>Secretory IgA's complex roles in immunity and mucosal homeostasis in the gut</article-title><source>Mucosal Immunol</source><year>2011</year><volume>4</volume><fpage>603</fpage><lpage>611</lpage><?supplied-pmid 21975936?><pub-id pub-id-type="pmid">21975936</pub-id>
</element-citation><mixed-citation id="mc-CR17" publication-type="journal">Mantis NJ, Rol N, Corth&#x000e9;sy B. Secretory IgA&#x02019;s complex roles in immunity and mucosal homeostasis in the gut. Mucosal Immunol. 2011;4:603&#x02013;11.<pub-id pub-id-type="pmid">21975936</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR18"><label>18.</label><citation-alternatives><element-citation id="ec-CR18" publication-type="journal"><person-group person-group-type="author"><name><surname>Hu</surname><given-names>X</given-names></name><name><surname>Deng</surname><given-names>J</given-names></name><name><surname>Yu</surname><given-names>T</given-names></name><name><surname>Chen</surname><given-names>SH</given-names></name><name><surname>Ge</surname><given-names>YD</given-names></name><name><surname>Zhou</surname><given-names>ZH</given-names></name><etal/></person-group><article-title>ATF4 deficiency promotes intestinal inflammation in mice by reducing uptake of glutamine and expression of antimicrobial peptides</article-title><source>Gastroenterology</source><year>2019</year><volume>156</volume><fpage>1098</fpage><lpage>1111</lpage><?supplied-pmid 30452920?><pub-id pub-id-type="pmid">30452920</pub-id>
</element-citation><mixed-citation id="mc-CR18" publication-type="journal">Hu X, Deng J, Yu T, Chen SH, Ge YD, Zhou ZH, et al. ATF4 deficiency promotes intestinal inflammation in mice by reducing uptake of glutamine and expression of antimicrobial peptides. Gastroenterology. 2019;156:1098&#x02013;111.<pub-id pub-id-type="pmid">30452920</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR19"><label>19.</label><citation-alternatives><element-citation id="ec-CR19" publication-type="journal"><person-group person-group-type="author"><name><surname>Ananthakrishnan</surname><given-names>AN</given-names></name></person-group><article-title>Epidemiology and risk factors for IBD</article-title><source>Nat Rev Gastroenterol Hepatol</source><year>2015</year><volume>12</volume><fpage>205</fpage><lpage>217</lpage><?supplied-pmid 25732745?><pub-id pub-id-type="pmid">25732745</pub-id>
</element-citation><mixed-citation id="mc-CR19" publication-type="journal">Ananthakrishnan AN. Epidemiology and risk factors for IBD. Nat Rev Gastroenterol Hepatol. 2015;12:205&#x02013;17.
<pub-id pub-id-type="pmid">25732745</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR20"><label>20.</label><citation-alternatives><element-citation id="ec-CR20" publication-type="journal"><person-group person-group-type="author"><name><surname>Okayasu</surname><given-names>I</given-names></name><name><surname>Hatakeyama</surname><given-names>S</given-names></name><name><surname>Yamada</surname><given-names>M</given-names></name><name><surname>Ohkusa</surname><given-names>T</given-names></name><name><surname>Inagaki</surname><given-names>Y</given-names></name><name><surname>Nakaya</surname><given-names>R</given-names></name></person-group><article-title>A novel method in the induction of reliable experimental acute and chronic ulcerative colitis in mice</article-title><source>Gastroenterology</source><year>1990</year><volume>98</volume><fpage>694</fpage><lpage>702</lpage><?supplied-pmid 1688816?><pub-id pub-id-type="pmid">1688816</pub-id>
</element-citation><mixed-citation id="mc-CR20" publication-type="journal">Okayasu I, Hatakeyama S, Yamada M, Ohkusa T, Inagaki Y, Nakaya R. A novel method in the induction of reliable experimental acute and chronic ulcerative colitis in mice. Gastroenterology. 1990;98:694&#x02013;702.<pub-id pub-id-type="pmid">1688816</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR21"><label>21.</label><citation-alternatives><element-citation id="ec-CR21" publication-type="journal"><person-group person-group-type="author"><name><surname>Tey</surname><given-names>SK</given-names></name><name><surname>Wong</surname><given-names>SWK</given-names></name><name><surname>Chan</surname><given-names>JYT</given-names></name><name><surname>Mao</surname><given-names>XW</given-names></name><name><surname>Ng</surname><given-names>TH</given-names></name><name><surname>Yeung</surname><given-names>CLS</given-names></name><etal/></person-group><article-title>Patient pIgR-enriched extracellular vesicles drive cancer stemness, tumorigenesis and metastasis in hepatocellular carcinoma</article-title><source>J Hepatol</source><year>2022</year><volume>76</volume><fpage>883</fpage><lpage>895</lpage><?supplied-pmid 34922977?><pub-id pub-id-type="pmid">34922977</pub-id>
</element-citation><mixed-citation id="mc-CR21" publication-type="journal">Tey SK, Wong SWK, Chan JYT, Mao XW, Ng TH, Yeung CLS, et al. Patient pIgR-enriched extracellular vesicles drive cancer stemness, tumorigenesis and metastasis in hepatocellular carcinoma. J Hepatol. 2022;76:883&#x02013;95.<pub-id pub-id-type="pmid">34922977</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR22"><label>22.</label><citation-alternatives><element-citation id="ec-CR22" publication-type="journal"><person-group person-group-type="author"><name><surname>Wang</surname><given-names>HQ</given-names></name><name><surname>Zhao</surname><given-names>MX</given-names></name><name><surname>Hong</surname><given-names>SC</given-names></name><name><surname>He</surname><given-names>X</given-names></name><name><surname>Tao</surname><given-names>L</given-names></name><name><surname>Tong</surname><given-names>CC</given-names></name><etal/></person-group><article-title>1,25(OH)2D3 alleviates oxidative stress and inflammation through up-regulating HMGCS2 in DSS-induced colitis</article-title><source>Int Immunopharmacol</source><year>2023</year><volume>125</volume><fpage>111131</fpage><?supplied-pmid 38149572?><pub-id pub-id-type="pmid">38149572</pub-id>
</element-citation><mixed-citation id="mc-CR22" publication-type="journal">Wang HQ, Zhao MX, Hong SC, He X, Tao L, Tong CC, et al. 1,25(OH)2D3 alleviates oxidative stress and inflammation through up-regulating HMGCS2 in DSS-induced colitis. Int Immunopharmacol. 2023;125:111131.<pub-id pub-id-type="pmid">38149572</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR23"><label>23.</label><citation-alternatives><element-citation id="ec-CR23" publication-type="journal"><person-group person-group-type="author"><name><surname>Verlinden</surname><given-names>L</given-names></name><name><surname>Leyssens</surname><given-names>C</given-names></name><name><surname>Beullens</surname><given-names>I</given-names></name><name><surname>Marcelis</surname><given-names>S</given-names></name><name><surname>Mathieu</surname><given-names>C</given-names></name><name><surname>De Clercq</surname><given-names>P</given-names></name><etal/></person-group><article-title>The vitamin D analog TX527 ameliorates disease symptoms in a chemically induced model of inflammatory bowel disease</article-title><source>J Steroid Biochem Mol Biol</source><year>2013</year><volume>136</volume><fpage>107</fpage><lpage>111</lpage><?supplied-pmid 23000190?><pub-id pub-id-type="pmid">23000190</pub-id>
</element-citation><mixed-citation id="mc-CR23" publication-type="journal">Verlinden L, Leyssens C, Beullens I, Marcelis S, Mathieu C, De Clercq P, et al. The vitamin D analog TX527 ameliorates disease symptoms in a chemically induced model of inflammatory bowel disease. J Steroid Biochem Mol Biol. 2013;136:107&#x02013;11.<pub-id pub-id-type="pmid">23000190</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR24"><label>24.</label><citation-alternatives><element-citation id="ec-CR24" publication-type="journal"><person-group person-group-type="author"><name><surname>Murthy</surname><given-names>SN</given-names></name><name><surname>Cooper</surname><given-names>HS</given-names></name><name><surname>Shim</surname><given-names>H</given-names></name><name><surname>Shah</surname><given-names>RS</given-names></name><name><surname>Ibrahim</surname><given-names>SA</given-names></name><name><surname>Sedergran</surname><given-names>DJ</given-names></name></person-group><article-title>Treatment of dextran sulfate sodium-induced murine colitis by intracolonic cyclosporin</article-title><source>Dig Dis Sci</source><year>1993</year><volume>38</volume><fpage>1722</fpage><lpage>1734</lpage><?supplied-pmid 8359087?><pub-id pub-id-type="pmid">8359087</pub-id>
</element-citation><mixed-citation id="mc-CR24" publication-type="journal">Murthy SN, Cooper HS, Shim H, Shah RS, Ibrahim SA, Sedergran DJ. Treatment of dextran sulfate sodium-induced murine colitis by intracolonic cyclosporin. Dig Dis Sci. 1993;38:1722&#x02013;34.<pub-id pub-id-type="pmid">8359087</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR25"><label>25.</label><citation-alternatives><element-citation id="ec-CR25" publication-type="journal"><person-group person-group-type="author"><name><surname>Wang</surname><given-names>X</given-names></name><name><surname>Cai</surname><given-names>J</given-names></name><name><surname>Lin</surname><given-names>B</given-names></name><name><surname>Ma</surname><given-names>M</given-names></name><name><surname>Tao</surname><given-names>Y</given-names></name><name><surname>Zhou</surname><given-names>YB</given-names></name><etal/></person-group><article-title>GPR34-mediated sensing of lysophosphatidylserine released by apoptotic neutrophils activates type 3 innate lymphoid cells to mediate tissue repair</article-title><source>Immunity</source><year>2021</year><volume>54</volume><fpage>1123</fpage><lpage>1136</lpage><?supplied-pmid 34107271?><pub-id pub-id-type="pmid">34107271</pub-id>
</element-citation><mixed-citation id="mc-CR25" publication-type="journal">Wang X, Cai J, Lin B, Ma M, Tao Y, Zhou YB, et al. GPR34-mediated sensing of lysophosphatidylserine released by apoptotic neutrophils activates type 3 innate lymphoid cells to mediate tissue repair. Immunity. 2021;54:1123&#x02013;36.<pub-id pub-id-type="pmid">34107271</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR26"><label>26.</label><citation-alternatives><element-citation id="ec-CR26" publication-type="journal"><person-group person-group-type="author"><name><surname>Chang</surname><given-names>YC</given-names></name><name><surname>Ching</surname><given-names>YH</given-names></name><name><surname>Chiu</surname><given-names>CC</given-names></name><name><surname>Liu</surname><given-names>JY</given-names></name><name><surname>Hung</surname><given-names>SW</given-names></name><name><surname>Huang</surname><given-names>WC</given-names></name><etal/></person-group><article-title>TLR2 and interleukin-10 are involved in <italic>Bacteroides fragilis</italic>-mediated prevention of DSS-induced colitis in gnotobiotic mice</article-title><source>PLoS ONE</source><year>2017</year><volume>12</volume><issue>7</issue><fpage>e0180025</fpage><?supplied-pmid 28683149?><pub-id pub-id-type="pmid">28683149</pub-id>
</element-citation><mixed-citation id="mc-CR26" publication-type="journal">Chang YC, Ching YH, Chiu CC, Liu JY, Hung SW, Huang WC, et al. TLR2 and interleukin-10 are involved in <italic>Bacteroides fragilis</italic>-mediated prevention of DSS-induced colitis in gnotobiotic mice. PLoS ONE. 2017;12(7): e0180025.<pub-id pub-id-type="pmid">28683149</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR27"><label>27.</label><citation-alternatives><element-citation id="ec-CR27" publication-type="journal"><person-group person-group-type="author"><name><surname>Malik</surname><given-names>A</given-names></name><name><surname>Sharma</surname><given-names>D</given-names></name><name><surname>Zhu</surname><given-names>Q</given-names></name><name><surname>Karki</surname><given-names>R</given-names></name><name><surname>Guy</surname><given-names>CS</given-names></name><name><surname>Vogel</surname><given-names>P</given-names></name><etal/></person-group><article-title>IL-33 regulates the IgA-microbiota axis to restrain IL-1&#x003b1;-dependent colitis and tumorigenesis</article-title><source>J Clin Invest</source><year>2016</year><volume>126</volume><issue>12</issue><fpage>4469</fpage><lpage>4481</lpage><?supplied-pmid 27775548?><pub-id pub-id-type="pmid">27775548</pub-id>
</element-citation><mixed-citation id="mc-CR27" publication-type="journal">Malik A, Sharma D, Zhu Q, Karki R, Guy CS, Vogel P, et al. IL-33 regulates the IgA-microbiota axis to restrain IL-1&#x003b1;-dependent colitis and tumorigenesis. J Clin Invest. 2016;126(12):4469&#x02013;81.<pub-id pub-id-type="pmid">27775548</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR28"><label>28.</label><citation-alternatives><element-citation id="ec-CR28" publication-type="journal"><person-group person-group-type="author"><name><surname>Nakanishi</surname><given-names>M</given-names></name><name><surname>Perret</surname><given-names>C</given-names></name><name><surname>Meuillet</surname><given-names>EJ</given-names></name><name><surname>Rosenberg</surname><given-names>DW</given-names></name></person-group><article-title>Non-cell autonomous effects of targeting inducible PGE2 synthesis during inflammation-associated colon carcinogenesis</article-title><source>Carcinogenesis</source><year>2015</year><volume>36</volume><issue>4</issue><fpage>478</fpage><lpage>486</lpage><?supplied-pmid 25634334?><pub-id pub-id-type="pmid">25634334</pub-id>
</element-citation><mixed-citation id="mc-CR28" publication-type="journal">Nakanishi M, Perret C, Meuillet EJ, Rosenberg DW. Non-cell autonomous effects of targeting inducible PGE2 synthesis during inflammation-associated colon carcinogenesis. Carcinogenesis. 2015;36(4):478&#x02013;86.<pub-id pub-id-type="pmid">25634334</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR29"><label>29.</label><citation-alternatives><element-citation id="ec-CR29" publication-type="journal"><person-group person-group-type="author"><name><surname>Weigmann</surname><given-names>B</given-names></name><name><surname>Tubbe</surname><given-names>I</given-names></name><name><surname>Seidel</surname><given-names>D</given-names></name><name><surname>Nicolaev</surname><given-names>A</given-names></name><name><surname>Becker</surname><given-names>C</given-names></name><name><surname>Neurath</surname><given-names>MF</given-names></name></person-group><article-title>Isolation and subsequent analysis of murine lamina propria mononuclear cells from colonic tissue</article-title><source>Nat Protoc</source><year>2007</year><volume>2</volume><fpage>2307</fpage><lpage>2311</lpage><?supplied-pmid 17947970?><pub-id pub-id-type="pmid">17947970</pub-id>
</element-citation><mixed-citation id="mc-CR29" publication-type="journal">Weigmann B, Tubbe I, Seidel D, Nicolaev A, Becker C, Neurath MF. Isolation and subsequent analysis of murine lamina propria mononuclear cells from colonic tissue. Nat Protoc. 2007;2:2307&#x02013;11.<pub-id pub-id-type="pmid">17947970</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR30"><label>30.</label><citation-alternatives><element-citation id="ec-CR30" publication-type="journal"><person-group person-group-type="author"><name><surname>Duan</surname><given-names>Y</given-names></name><name><surname>Llorente</surname><given-names>C</given-names></name><name><surname>Lang</surname><given-names>S</given-names></name><name><surname>Brandl</surname><given-names>K</given-names></name><name><surname>Chu</surname><given-names>HK</given-names></name><name><surname>Jiang</surname><given-names>L</given-names></name><etal/></person-group><article-title>Bacteriophage targeting of gut bacterium attenuates alcoholic liver disease</article-title><source>Nature</source><year>2019</year><volume>575</volume><fpage>505</fpage><lpage>511</lpage><?supplied-pmid 31723265?><pub-id pub-id-type="pmid">31723265</pub-id>
</element-citation><mixed-citation id="mc-CR30" publication-type="journal">Duan Y, Llorente C, Lang S, Brandl K, Chu HK, Jiang L, et al. Bacteriophage targeting of gut bacterium attenuates alcoholic liver disease. Nature. 2019;575:505&#x02013;11.<pub-id pub-id-type="pmid">31723265</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR31"><label>31.</label><citation-alternatives><element-citation id="ec-CR31" publication-type="journal"><person-group person-group-type="author"><name><surname>Ramirez-Farias</surname><given-names>C</given-names></name><name><surname>Slezak</surname><given-names>K</given-names></name><name><surname>Fuller</surname><given-names>Z</given-names></name><name><surname>Duncan</surname><given-names>A</given-names></name><name><surname>Holtrop</surname><given-names>G</given-names></name><name><surname>Louis</surname><given-names>P</given-names></name></person-group><article-title>Effect of inulin on the human gut microbiota: stimulation of <italic>Bifidobacterium adolescentis</italic> and <italic>Faecalibacterium prausnitzii</italic></article-title><source>Br J Nutr</source><year>2009</year><volume>101</volume><fpage>541</fpage><lpage>550</lpage><?supplied-pmid 18590586?><pub-id pub-id-type="pmid">18590586</pub-id>
</element-citation><mixed-citation id="mc-CR31" publication-type="journal">Ramirez-Farias C, Slezak K, Fuller Z, Duncan A, Holtrop G, Louis P. Effect of inulin on the human gut microbiota: stimulation of <italic>Bifidobacterium adolescentis</italic> and <italic>Faecalibacterium prausnitzii</italic>. Br J Nutr. 2009;101:541&#x02013;50.<pub-id pub-id-type="pmid">18590586</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR32"><label>32.</label><citation-alternatives><element-citation id="ec-CR32" publication-type="journal"><person-group person-group-type="author"><name><surname>Carlberg</surname><given-names>C</given-names></name></person-group><article-title>Genomic signaling of vitamin D</article-title><source>Steroids</source><year>2023</year><volume>198</volume><fpage>109271</fpage><?supplied-pmid 37442517?><pub-id pub-id-type="pmid">37442517</pub-id>
</element-citation><mixed-citation id="mc-CR32" publication-type="journal">Carlberg C. Genomic signaling of vitamin D. Steroids. 2023;198: 109271.<pub-id pub-id-type="pmid">37442517</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR33"><label>33.</label><citation-alternatives><element-citation id="ec-CR33" publication-type="journal"><person-group person-group-type="author"><name><surname>Zhao</surname><given-names>H</given-names></name><name><surname>Xu</surname><given-names>H</given-names></name><name><surname>Chen</surname><given-names>S</given-names></name><name><surname>He</surname><given-names>J</given-names></name><name><surname>Zhou</surname><given-names>Y</given-names></name><name><surname>Nie</surname><given-names>Y</given-names></name></person-group><article-title>Systematic review and meta-analysis of the role of <italic>Faecalibacterium prausnitzii</italic> alteration in inflammatory bowel disease</article-title><source>J Gastroenterol Hepatol</source><year>2021</year><volume>36</volume><issue>2</issue><fpage>320</fpage><lpage>328</lpage><?supplied-pmid 32815163?><pub-id pub-id-type="pmid">32815163</pub-id>
</element-citation><mixed-citation id="mc-CR33" publication-type="journal">Zhao H, Xu H, Chen S, He J, Zhou Y, Nie Y. Systematic review and meta-analysis of the role of <italic>Faecalibacterium prausnitzii</italic> alteration in inflammatory bowel disease. J Gastroenterol Hepatol. 2021;36(2):320&#x02013;8.<pub-id pub-id-type="pmid">32815163</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR34"><label>34.</label><citation-alternatives><element-citation id="ec-CR34" publication-type="journal"><person-group person-group-type="author"><name><surname>Lopez-Siles</surname><given-names>M</given-names></name><name><surname>Duncan</surname><given-names>SH</given-names></name><name><surname>Garcia-Gil</surname><given-names>LJ</given-names></name><name><surname>Martinez-Medina</surname><given-names>M</given-names></name></person-group><article-title><italic>Faecalibacterium prausnitzii</italic>: from microbiology to diagnostics and prognostics</article-title><source>ISME J</source><year>2017</year><volume>11</volume><issue>4</issue><fpage>841</fpage><lpage>852</lpage><?supplied-pmid 28045459?><pub-id pub-id-type="pmid">28045459</pub-id>
</element-citation><mixed-citation id="mc-CR34" publication-type="journal">Lopez-Siles M, Duncan SH, Garcia-Gil LJ, Martinez-Medina M. <italic>Faecalibacterium prausnitzii</italic>: from microbiology to diagnostics and prognostics. ISME J. 2017;11(4):841&#x02013;52.<pub-id pub-id-type="pmid">28045459</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR35"><label>35.</label><citation-alternatives><element-citation id="ec-CR35" publication-type="journal"><person-group person-group-type="author"><name><surname>Kostic</surname><given-names>AD</given-names></name><name><surname>Xavier</surname><given-names>RJ</given-names></name><name><surname>Gevers</surname><given-names>D</given-names></name></person-group><article-title>The microbiome in inflammatory bowel disease: current status and the future ahead</article-title><source>Gastroenterology</source><year>2014</year><volume>146</volume><issue>6</issue><fpage>1489</fpage><lpage>1499</lpage><?supplied-pmid 24560869?><pub-id pub-id-type="pmid">24560869</pub-id>
</element-citation><mixed-citation id="mc-CR35" publication-type="journal">Kostic AD, Xavier RJ, Gevers D. The microbiome in inflammatory bowel disease: current status and the future ahead. Gastroenterology. 2014;146(6):1489&#x02013;99.<pub-id pub-id-type="pmid">24560869</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR36"><label>36.</label><citation-alternatives><element-citation id="ec-CR36" publication-type="journal"><person-group person-group-type="author"><name><surname>Lengfelder</surname><given-names>I</given-names></name><name><surname>Sava</surname><given-names>IG</given-names></name><name><surname>Hansen</surname><given-names>JJ</given-names></name><name><surname>Kleigrewe</surname><given-names>K</given-names></name><name><surname>Herzog</surname><given-names>J</given-names></name><name><surname>Neuhaus</surname><given-names>K</given-names></name><etal/></person-group><article-title>Complex bacterial consortia reprogram the colitogenic activity of enterococcus faecalis in a gnotobiotic mouse model of chronic, immune-mediated colitis</article-title><source>Front Immunol</source><year>2019</year><volume>10</volume><fpage>1420</fpage><?supplied-pmid 31281321?><pub-id pub-id-type="pmid">31281321</pub-id>
</element-citation><mixed-citation id="mc-CR36" publication-type="journal">Lengfelder I, Sava IG, Hansen JJ, Kleigrewe K, Herzog J, Neuhaus K, et al. Complex bacterial consortia reprogram the colitogenic activity of enterococcus faecalis in a gnotobiotic mouse model of chronic, immune-mediated colitis. Front Immunol. 2019;10: 1420.<pub-id pub-id-type="pmid">31281321</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR37"><label>37.</label><citation-alternatives><element-citation id="ec-CR37" publication-type="journal"><person-group person-group-type="author"><name><surname>Rowan</surname><given-names>F</given-names></name><name><surname>Docherty</surname><given-names>NG</given-names></name><name><surname>Murphy</surname><given-names>M</given-names></name><name><surname>Murphy</surname><given-names>B</given-names></name><name><surname>Calvin Coffey</surname><given-names>J</given-names></name><name><surname>O'Connell</surname><given-names>PR</given-names></name></person-group><article-title><italic>Desulfovibrio</italic> bacterial species are increased in ulcerative colitis</article-title><source>Dis Colon Rectum</source><year>2010</year><volume>53</volume><fpage>1530</fpage><lpage>1536</lpage><?supplied-pmid 20940602?><pub-id pub-id-type="pmid">20940602</pub-id>
</element-citation><mixed-citation id="mc-CR37" publication-type="journal">Rowan F, Docherty NG, Murphy M, Murphy B, Calvin Coffey J, O&#x02019;Connell PR. <italic>Desulfovibrio</italic> bacterial species are increased in ulcerative colitis. Dis Colon Rectum. 2010;53:1530&#x02013;6.<pub-id pub-id-type="pmid">20940602</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR38"><label>38.</label><citation-alternatives><element-citation id="ec-CR38" publication-type="journal"><person-group person-group-type="author"><name><surname>Abdelbary</surname><given-names>MMH</given-names></name><name><surname>Hatting</surname><given-names>M</given-names></name><name><surname>Bott</surname><given-names>A</given-names></name><name><surname>Dahlhausen</surname><given-names>A</given-names></name><name><surname>Keller</surname><given-names>D</given-names></name><name><surname>Trautwein</surname><given-names>C</given-names></name><etal/></person-group><article-title>The oral-gut axis: salivary and fecal microbiome dysbiosis in patients with inflammatory bowel disease</article-title><source>Front Cell Infect Microbiol</source><year>2022</year><volume>12</volume><fpage>1010853</fpage><?supplied-pmid 36275026?><pub-id pub-id-type="pmid">36275026</pub-id>
</element-citation><mixed-citation id="mc-CR38" publication-type="journal">Abdelbary MMH, Hatting M, Bott A, Dahlhausen A, Keller D, Trautwein C, et al. The oral-gut axis: salivary and fecal microbiome dysbiosis in patients with inflammatory bowel disease. Front Cell Infect Microbiol. 2022;12: 1010853.<pub-id pub-id-type="pmid">36275026</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR39"><label>39.</label><citation-alternatives><element-citation id="ec-CR39" publication-type="journal"><person-group person-group-type="author"><name><surname>Tang</surname><given-names>X</given-names></name><name><surname>Wang</surname><given-names>W</given-names></name><name><surname>Hong</surname><given-names>G</given-names></name><name><surname>Duan</surname><given-names>C</given-names></name><name><surname>Zhu</surname><given-names>S</given-names></name><name><surname>Tian</surname><given-names>Y</given-names></name><etal/></person-group><article-title>Gut microbiota-mediated lysophosphatidylcholine generation promotes colitis in intestinal epithelium-specific Fut2 deficiency</article-title><source>J Biomed Sci</source><year>2021</year><volume>28</volume><fpage>20</fpage><?supplied-pmid 33722220?><pub-id pub-id-type="pmid">33722220</pub-id>
</element-citation><mixed-citation id="mc-CR39" publication-type="journal">Tang X, Wang W, Hong G, Duan C, Zhu S, Tian Y, et al. Gut microbiota-mediated lysophosphatidylcholine generation promotes colitis in intestinal epithelium-specific Fut2 deficiency. J Biomed Sci. 2021;28:20.<pub-id pub-id-type="pmid">33722220</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR40"><label>40.</label><citation-alternatives><element-citation id="ec-CR40" publication-type="journal"><person-group person-group-type="author"><name><surname>Liu</surname><given-names>B</given-names></name><name><surname>Ye</surname><given-names>D</given-names></name><name><surname>Yang</surname><given-names>H</given-names></name><name><surname>Song</surname><given-names>J</given-names></name><name><surname>Sun</surname><given-names>X</given-names></name><name><surname>Mao</surname><given-names>Y</given-names></name><etal/></person-group><article-title>Two-sample mendelian randomization analysis investigates causal associations between gut microbial genera and inflammatory bowel disease, and specificity causal associations in ulcerative colitis or crohn's disease</article-title><source>Front Immunol</source><year>2022</year><volume>13</volume><fpage>921546</fpage><?supplied-pmid 35860271?><pub-id pub-id-type="pmid">35860271</pub-id>
</element-citation><mixed-citation id="mc-CR40" publication-type="journal">Liu B, Ye D, Yang H, Song J, Sun X, Mao Y, et al. Two-sample mendelian randomization analysis investigates causal associations between gut microbial genera and inflammatory bowel disease, and specificity causal associations in ulcerative colitis or crohn&#x02019;s disease. Front Immunol. 2022;13: 921546.<pub-id pub-id-type="pmid">35860271</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR41"><label>41.</label><citation-alternatives><element-citation id="ec-CR41" publication-type="journal"><person-group person-group-type="author"><name><surname>Yin</surname><given-names>Y</given-names></name><name><surname>Cai</surname><given-names>J</given-names></name><name><surname>Zhou</surname><given-names>L</given-names></name><name><surname>Xing</surname><given-names>L</given-names></name><name><surname>Zhang</surname><given-names>W</given-names></name></person-group><article-title>Dietary oxidized beef protein alters gut microbiota and induces colonic inflammatory damage in C57BL/6 mice</article-title><source>Front Nutr</source><year>2022</year><volume>9</volume><fpage>980204</fpage><?supplied-pmid 36118776?><pub-id pub-id-type="pmid">36118776</pub-id>
</element-citation><mixed-citation id="mc-CR41" publication-type="journal">Yin Y, Cai J, Zhou L, Xing L, Zhang W. Dietary oxidized beef protein alters gut microbiota and induces colonic inflammatory damage in C57BL/6 mice. Front Nutr. 2022;9: 980204.<pub-id pub-id-type="pmid">36118776</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR42"><label>42.</label><citation-alternatives><element-citation id="ec-CR42" publication-type="journal"><person-group person-group-type="author"><name><surname>Clavel</surname><given-names>T</given-names></name><name><surname>Duck</surname><given-names>W</given-names></name><name><surname>Charrier</surname><given-names>C</given-names></name><name><surname>Wenning</surname><given-names>M</given-names></name><name><surname>Elson</surname><given-names>C</given-names></name><name><surname>Haller</surname><given-names>D</given-names></name><etal/></person-group><article-title>Enterorhabdus caecimuris sp. nov., a member of the family Coriobacteriaceae isolated from a mouse model of spontaneous colitis, and emended description of the genus Enterorhabdus Clavel et al. 2009</article-title><source>Int J Syst Evol Microbiol</source><year>2010</year><volume>60</volume><issue>7</issue><fpage>1527</fpage><lpage>31</lpage><?supplied-pmid 19684311?><pub-id pub-id-type="pmid">19684311</pub-id>
</element-citation><mixed-citation id="mc-CR42" publication-type="journal">Clavel T, Duck W, Charrier C, Wenning M, Elson C, Haller D, et al. Enterorhabdus caecimuris sp. nov., a member of the family Coriobacteriaceae isolated from a mouse model of spontaneous colitis, and emended description of the genus Enterorhabdus Clavel et al. 2009. Int J Syst Evol Microbiol. 2010;60(7):1527&#x02013;31.<pub-id pub-id-type="pmid">19684311</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR43"><label>43.</label><citation-alternatives><element-citation id="ec-CR43" publication-type="journal"><person-group person-group-type="author"><name><surname>Han</surname><given-names>M</given-names></name><name><surname>Liang</surname><given-names>J</given-names></name><name><surname>Hou</surname><given-names>M</given-names></name><name><surname>Liu</surname><given-names>Y</given-names></name><name><surname>Li</surname><given-names>H</given-names></name><name><surname>Gao</surname><given-names>Z</given-names></name></person-group><article-title>Bifidobacterium bifidum ameliorates DSS-induced colitis in mice by regulating microbial metabolome and targeting gut microbiota</article-title><source>J Agric Food Chem</source><year>2024</year><volume>72</volume><issue>24</issue><fpage>13593</fpage><lpage>609</lpage></element-citation><mixed-citation id="mc-CR43" publication-type="journal">Han M, Liang J, Hou M, Liu Y, Li H, Gao Z. Bifidobacterium bifidum ameliorates DSS-induced colitis in mice by regulating microbial metabolome and targeting gut microbiota. J Agric Food Chem. 2024;72(24):13593&#x02013;609.</mixed-citation></citation-alternatives></ref><ref id="CR44"><label>44.</label><citation-alternatives><element-citation id="ec-CR44" publication-type="journal"><person-group person-group-type="author"><name><surname>Zhong</surname><given-names>Y</given-names></name><name><surname>Xiao</surname><given-names>Q</given-names></name><name><surname>Huang</surname><given-names>J</given-names></name><name><surname>Yu</surname><given-names>S</given-names></name><name><surname>Chen</surname><given-names>L</given-names></name><name><surname>Wan</surname><given-names>Q</given-names></name><etal/></person-group><article-title>Ginsenoside Rg1 alleviates ulcerative colitis in obese mice by regulating the gut microbiota-lipid metabolism-Th1/Th2/Th17 cells axis</article-title><source>J Agric Food Chem</source><year>2023</year><volume>71</volume><issue>50</issue><fpage>20073</fpage><lpage>20091</lpage><?supplied-pmid 38064669?><pub-id pub-id-type="pmid">38064669</pub-id>
</element-citation><mixed-citation id="mc-CR44" publication-type="journal">Zhong Y, Xiao Q, Huang J, Yu S, Chen L, Wan Q, et al. Ginsenoside Rg1 alleviates ulcerative colitis in obese mice by regulating the gut microbiota-lipid metabolism-Th1/Th2/Th17 cells axis. J Agric Food Chem. 2023;71(50):20073&#x02013;91.<pub-id pub-id-type="pmid">38064669</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR45"><label>45.</label><citation-alternatives><element-citation id="ec-CR45" publication-type="journal"><person-group person-group-type="author"><name><surname>Liu</surname><given-names>Y</given-names></name><name><surname>Duan</surname><given-names>H</given-names></name><name><surname>Chen</surname><given-names>Y</given-names></name><name><surname>Zhang</surname><given-names>C</given-names></name><name><surname>Zhao</surname><given-names>J</given-names></name><name><surname>Narbad</surname><given-names>A</given-names></name><etal/></person-group><article-title>Intraspecific difference of <italic>Latilactobacillus sakei</italic> in inflammatory bowel diseases: insights into potential mechanisms through comparative genomics and metabolomics analyses</article-title><source>Imeta</source><year>2023</year><volume>2</volume><issue>4</issue><fpage>e136</fpage><?supplied-pmid 38868211?><pub-id pub-id-type="pmid">38868211</pub-id>
</element-citation><mixed-citation id="mc-CR45" publication-type="journal">Liu Y, Duan H, Chen Y, Zhang C, Zhao J, Narbad A, et al. Intraspecific difference of <italic>Latilactobacillus sakei</italic> in inflammatory bowel diseases: insights into potential mechanisms through comparative genomics and metabolomics analyses. Imeta. 2023;2(4): e136.<pub-id pub-id-type="pmid">38868211</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR46"><label>46.</label><citation-alternatives><element-citation id="ec-CR46" publication-type="journal"><person-group person-group-type="author"><name><surname>Wang</surname><given-names>M</given-names></name><name><surname>Xu</surname><given-names>X</given-names></name><name><surname>Sheng</surname><given-names>M</given-names></name><name><surname>Zhang</surname><given-names>M</given-names></name><name><surname>Wu</surname><given-names>F</given-names></name><name><surname>Zhao</surname><given-names>Z</given-names></name><etal/></person-group><article-title>Tannic acid protects against colitis by regulating the IL17-NF&#x003ba;B and microbiota-methylation pathways</article-title><source>Int J Biol Macromol</source><year>2024</year><volume>274</volume><issue>1</issue><fpage>133334</fpage><?supplied-pmid 38908626?><pub-id pub-id-type="pmid">38908626</pub-id>
</element-citation><mixed-citation id="mc-CR46" publication-type="journal">Wang M, Xu X, Sheng M, Zhang M, Wu F, Zhao Z, et al. Tannic acid protects against colitis by regulating the IL17-NF&#x003ba;B and microbiota-methylation pathways. Int J Biol Macromol. 2024;274(1): 133334.<pub-id pub-id-type="pmid">38908626</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR47"><label>47.</label><citation-alternatives><element-citation id="ec-CR47" publication-type="journal"><person-group person-group-type="author"><name><surname>Zeng</surname><given-names>Y</given-names></name><name><surname>Fadaak</surname><given-names>A</given-names></name><name><surname>Alomeir</surname><given-names>N</given-names></name><name><surname>Wu</surname><given-names>Y</given-names></name><name><surname>Wu</surname><given-names>TT</given-names></name><name><surname>Qing</surname><given-names>S</given-names></name><etal/></person-group><article-title>Effect of probiotic <italic>Lactobacillus plantarum</italic> on <italic>Streptococcus mutans</italic> and <italic>Candida albicans</italic> clinical isolates from children with early childhood caries</article-title><source>Int J Mol Sci</source><year>2023</year><volume>24</volume><fpage>2991</fpage><?supplied-pmid 36769313?><pub-id pub-id-type="pmid">36769313</pub-id>
</element-citation><mixed-citation id="mc-CR47" publication-type="journal">Zeng Y, Fadaak A, Alomeir N, Wu Y, Wu TT, Qing S, et al. Effect of probiotic <italic>Lactobacillus plantarum</italic> on <italic>Streptococcus mutans</italic> and <italic>Candida albicans</italic> clinical isolates from children with early childhood caries. Int J Mol Sci. 2023;24: 2991.<pub-id pub-id-type="pmid">36769313</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR48"><label>48.</label><citation-alternatives><element-citation id="ec-CR48" publication-type="journal"><person-group person-group-type="author"><name><surname>Lycke</surname><given-names>NY</given-names></name><name><surname>Bemark</surname><given-names>M</given-names></name></person-group><article-title>The regulation of gut mucosal IgA B-cell responses: recent developments</article-title><source>Mucosal Immunol</source><year>2017</year><volume>10</volume><fpage>1361</fpage><lpage>1374</lpage><?supplied-pmid 28745325?><pub-id pub-id-type="pmid">28745325</pub-id>
</element-citation><mixed-citation id="mc-CR48" publication-type="journal">Lycke NY, Bemark M. The regulation of gut mucosal IgA B-cell responses: recent developments. Mucosal Immunol. 2017;10:1361&#x02013;74.<pub-id pub-id-type="pmid">28745325</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR49"><label>49.</label><citation-alternatives><element-citation id="ec-CR49" publication-type="journal"><person-group person-group-type="author"><name><surname>Rogier</surname><given-names>EW</given-names></name><name><surname>Frantz</surname><given-names>AL</given-names></name><name><surname>Bruno</surname><given-names>ME</given-names></name><name><surname>Kaetzel</surname><given-names>CS</given-names></name></person-group><article-title>Secretory IgA is concentrated in the outer layer of colonic mucus along with gut bacteria</article-title><source>Pathogens</source><year>2014</year><volume>3</volume><fpage>390</fpage><lpage>403</lpage><?supplied-pmid 25437806?><pub-id pub-id-type="pmid">25437806</pub-id>
</element-citation><mixed-citation id="mc-CR49" publication-type="journal">Rogier EW, Frantz AL, Bruno ME, Kaetzel CS. Secretory IgA is concentrated in the outer layer of colonic mucus along with gut bacteria. Pathogens. 2014;3:390&#x02013;403.<pub-id pub-id-type="pmid">25437806</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR50"><label>50.</label><citation-alternatives><element-citation id="ec-CR50" publication-type="journal"><person-group person-group-type="author"><name><surname>Reikvam</surname><given-names>DH</given-names></name><name><surname>Derrien</surname><given-names>M</given-names></name><name><surname>Islam</surname><given-names>R</given-names></name><name><surname>Erofeev</surname><given-names>A</given-names></name><name><surname>Grcic</surname><given-names>V</given-names></name><name><surname>Sandvi</surname><given-names>A</given-names></name><etal/></person-group><article-title>Epithelial-microbial crosstalk in polymeric Ig receptor deficient mice</article-title><source>Eur J Immunol</source><year>2012</year><volume>42</volume><fpage>2959</fpage><lpage>2970</lpage><?supplied-pmid 22865203?><pub-id pub-id-type="pmid">22865203</pub-id>
</element-citation><mixed-citation id="mc-CR50" publication-type="journal">Reikvam DH, Derrien M, Islam R, Erofeev A, Grcic V, Sandvi A, et al. Epithelial-microbial crosstalk in polymeric Ig receptor deficient mice. Eur J Immunol. 2012;42:2959&#x02013;70.<pub-id pub-id-type="pmid">22865203</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR51"><label>51.</label><citation-alternatives><element-citation id="ec-CR51" publication-type="journal"><person-group person-group-type="author"><name><surname>Palm</surname><given-names>NW</given-names></name><name><surname>de Zoete</surname><given-names>MR</given-names></name><name><surname>Cullen</surname><given-names>TW</given-names></name><name><surname>Barry</surname><given-names>NA</given-names></name><name><surname>Stefanowski</surname><given-names>J</given-names></name><name><surname>Hao</surname><given-names>L</given-names></name><etal/></person-group><article-title>Immunoglobulin A coating identifies colitogenic bacteria in inflammatory bowel disease</article-title><source>Cell</source><year>2014</year><volume>158</volume><fpage>1000</fpage><lpage>1010</lpage><?supplied-pmid 25171403?><pub-id pub-id-type="pmid">25171403</pub-id>
</element-citation><mixed-citation id="mc-CR51" publication-type="journal">Palm NW, de Zoete MR, Cullen TW, Barry NA, Stefanowski J, Hao L, et al. Immunoglobulin A coating identifies colitogenic bacteria in inflammatory bowel disease. Cell. 2014;158:1000&#x02013;10.<pub-id pub-id-type="pmid">25171403</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR52"><label>52.</label><citation-alternatives><element-citation id="ec-CR52" publication-type="journal"><person-group person-group-type="author"><name><surname>Della Mina</surname><given-names>E</given-names></name><name><surname>Jackson</surname><given-names>KJL</given-names></name><name><surname>Crawford</surname><given-names>AJI</given-names></name><name><surname>Faulks</surname><given-names>ML</given-names></name><name><surname>Pathmanandavel</surname><given-names>K</given-names></name><name><surname>Acquarola</surname><given-names>N</given-names></name><etal/></person-group><article-title>A novel heterozygous variant in AICDA impairs Ig class switching and somatic hypermutation in human B cells and is associated with autosomal dominant HIGM2 syndrome</article-title><source>J Clin Immunol</source><year>2024</year><volume>44</volume><issue>3</issue><fpage>66</fpage><?supplied-pmid 38363477?><pub-id pub-id-type="pmid">38363477</pub-id>
</element-citation><mixed-citation id="mc-CR52" publication-type="journal">Della Mina E, Jackson KJL, Crawford AJI, Faulks ML, Pathmanandavel K, Acquarola N, et al. A novel heterozygous variant in AICDA impairs Ig class switching and somatic hypermutation in human B cells and is associated with autosomal dominant HIGM2 syndrome. J Clin Immunol. 2024;44(3):66.<pub-id pub-id-type="pmid">38363477</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR53"><label>53.</label><citation-alternatives><element-citation id="ec-CR53" publication-type="journal"><person-group person-group-type="author"><name><surname>Moll</surname><given-names>JM</given-names></name><name><surname>Myers</surname><given-names>PN</given-names></name><name><surname>Zhang</surname><given-names>C</given-names></name><name><surname>Eriksen</surname><given-names>C</given-names></name><name><surname>Wolf</surname><given-names>J</given-names></name><name><surname>Appelberg</surname><given-names>KS</given-names></name><etal/></person-group><article-title>Gut microbiota perturbation in IgA deficiency is influenced by IgA-autoantibody status</article-title><source>Gastroenterology</source><year>2021</year><volume>160</volume><issue>7</issue><fpage>2423</fpage><lpage>2434</lpage><?supplied-pmid 33662387?><pub-id pub-id-type="pmid">33662387</pub-id>
</element-citation><mixed-citation id="mc-CR53" publication-type="journal">Moll JM, Myers PN, Zhang C, Eriksen C, Wolf J, Appelberg KS, et al. Gut microbiota perturbation in IgA deficiency is influenced by IgA-autoantibody status. Gastroenterology. 2021;160(7):2423&#x02013;34.<pub-id pub-id-type="pmid">33662387</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR54"><label>54.</label><citation-alternatives><element-citation id="ec-CR54" publication-type="journal"><person-group person-group-type="author"><name><surname>Pabst</surname><given-names>O</given-names></name><name><surname>Cerovic</surname><given-names>V</given-names></name><name><surname>Hornef</surname><given-names>M</given-names></name></person-group><article-title>Secretory IgA in the coordination of establishment and maintenance of the microbiota</article-title><source>Trends Immunol</source><year>2016</year><volume>37</volume><issue>5</issue><fpage>287</fpage><lpage>296</lpage><?supplied-pmid 27066758?><pub-id pub-id-type="pmid">27066758</pub-id>
</element-citation><mixed-citation id="mc-CR54" publication-type="journal">Pabst O, Cerovic V, Hornef M. Secretory IgA in the coordination of establishment and maintenance of the microbiota. Trends Immunol. 2016;37(5):287&#x02013;96.<pub-id pub-id-type="pmid">27066758</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR55"><label>55.</label><citation-alternatives><element-citation id="ec-CR55" publication-type="journal"><person-group person-group-type="author"><name><surname>Pausder</surname><given-names>A</given-names></name><name><surname>Fricke</surname><given-names>J</given-names></name><name><surname>Schughart</surname><given-names>K</given-names></name><name><surname>Schreiber</surname><given-names>J</given-names></name><name><surname>Strowig</surname><given-names>T</given-names></name><name><surname>Bruder</surname><given-names>D</given-names></name><etal/></person-group><article-title>Exogenous and endogenous triggers differentially stimulate pIgR expression and antibacterial secretory immunity in the murine respiratory tract</article-title><source>Lung</source><year>2022</year><volume>200</volume><issue>1</issue><fpage>119</fpage><lpage>128</lpage><?supplied-pmid 34825965?><pub-id pub-id-type="pmid">34825965</pub-id>
</element-citation><mixed-citation id="mc-CR55" publication-type="journal">Pausder A, Fricke J, Schughart K, Schreiber J, Strowig T, Bruder D, et al. Exogenous and endogenous triggers differentially stimulate pIgR expression and antibacterial secretory immunity in the murine respiratory tract. Lung. 2022;200(1):119&#x02013;28.<pub-id pub-id-type="pmid">34825965</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR56"><label>56.</label><citation-alternatives><element-citation id="ec-CR56" publication-type="journal"><person-group person-group-type="author"><name><surname>Wei</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>JY</given-names></name></person-group><article-title>Role of polymeric immunoglobulin receptor in IgA and IgM transcytosis</article-title><source>Int J Mol Sci</source><year>2021</year><volume>22</volume><issue>5</issue><fpage>2284</fpage><?supplied-pmid 33668983?><pub-id pub-id-type="pmid">33668983</pub-id>
</element-citation><mixed-citation id="mc-CR56" publication-type="journal">Wei H, Wang JY. Role of polymeric immunoglobulin receptor in IgA and IgM transcytosis. Int J Mol Sci. 2021;22(5): 2284.<pub-id pub-id-type="pmid">33668983</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR57"><label>57.</label><citation-alternatives><element-citation id="ec-CR57" publication-type="journal"><person-group person-group-type="author"><name><surname>Medrano</surname><given-names>G</given-names></name><name><surname>Cailleux</surname><given-names>F</given-names></name><name><surname>Guan</surname><given-names>P</given-names></name><name><surname>Kuruvilla</surname><given-names>K</given-names></name><name><surname>Barlow-Anacker</surname><given-names>AJ</given-names></name><name><surname>Gosain</surname><given-names>A</given-names></name></person-group><article-title>B-lymphocyte-intrinsic and -extrinsic defects in secretory immunoglobulin A production in the neural crest-conditional deletion of endothelin receptor B model of Hirschsprung-associated enterocolitis</article-title><source>FASEB J</source><year>2019</year><volume>33</volume><issue>6</issue><fpage>7615</fpage><lpage>7624</lpage><?supplied-pmid 30908942?><pub-id pub-id-type="pmid">30908942</pub-id>
</element-citation><mixed-citation id="mc-CR57" publication-type="journal">Medrano G, Cailleux F, Guan P, Kuruvilla K, Barlow-Anacker AJ, Gosain A. B-lymphocyte-intrinsic and -extrinsic defects in secretory immunoglobulin A production in the neural crest-conditional deletion of endothelin receptor B model of Hirschsprung-associated enterocolitis. FASEB J. 2019;33(6):7615&#x02013;24.<pub-id pub-id-type="pmid">30908942</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR58"><label>58.</label><citation-alternatives><element-citation id="ec-CR58" publication-type="journal"><person-group person-group-type="author"><name><surname>Lima</surname><given-names>SF</given-names></name><name><surname>Gogokhia</surname><given-names>L</given-names></name><name><surname>Viladomiu</surname><given-names>M</given-names></name><name><surname>Chou</surname><given-names>L</given-names></name><name><surname>Putzel</surname><given-names>G</given-names></name><name><surname>Jin</surname><given-names>WB</given-names></name><etal/></person-group><article-title>Transferable immunoglobulin A-coated <italic>Odoribacter splanchnicus</italic> in responders to fecal microbiota transplantation for ulcerative colitis limits colonic inflammation</article-title><source>Gastroenterology</source><year>2022</year><volume>162</volume><fpage>166</fpage><lpage>178</lpage><?supplied-pmid 34606847?><pub-id pub-id-type="pmid">34606847</pub-id>
</element-citation><mixed-citation id="mc-CR58" publication-type="journal">Lima SF, Gogokhia L, Viladomiu M, Chou L, Putzel G, Jin WB, et al. Transferable immunoglobulin A-coated <italic>Odoribacter splanchnicus</italic> in responders to fecal microbiota transplantation for ulcerative colitis limits colonic inflammation. Gastroenterology. 2022;162:166&#x02013;78.<pub-id pub-id-type="pmid">34606847</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR59"><label>59.</label><citation-alternatives><element-citation id="ec-CR59" publication-type="journal"><person-group person-group-type="author"><name><surname>Miquel</surname><given-names>S</given-names></name><name><surname>Leclerc</surname><given-names>M</given-names></name><name><surname>Martin</surname><given-names>R</given-names></name><name><surname>Chain</surname><given-names>F</given-names></name><name><surname>Lenoir</surname><given-names>M</given-names></name><name><surname>Raguideau</surname><given-names>S</given-names></name><etal/></person-group><article-title>Identification of metabolic signatures linked to anti-inflammatory effects of Faecalibacterium prausnitzii</article-title><source>mBio</source><year>2015</year><volume>6</volume><fpage>e00300</fpage><lpage>15</lpage><?supplied-pmid 25900655?><pub-id pub-id-type="pmid">25900655</pub-id>
</element-citation><mixed-citation id="mc-CR59" publication-type="journal">Miquel S, Leclerc M, Martin R, Chain F, Lenoir M, Raguideau S, et al. Identification of metabolic signatures linked to anti-inflammatory effects of Faecalibacterium prausnitzii. mBio. 2015;6:e00300-15.<pub-id pub-id-type="pmid">25900655</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR60"><label>60.</label><citation-alternatives><element-citation id="ec-CR60" publication-type="journal"><person-group person-group-type="author"><name><surname>Benjamin</surname><given-names>S</given-names></name><name><surname>Planquette</surname><given-names>C</given-names></name><name><surname>Delansorne</surname><given-names>R</given-names></name></person-group><article-title>Inecalcitol induces CD38 expression in multiple myeloma and myeloid cell lines</article-title><source>Blood</source><year>2016</year><volume>128</volume><fpage>3521</fpage></element-citation><mixed-citation id="mc-CR60" publication-type="journal">Benjamin S, Planquette C, Delansorne R. Inecalcitol induces CD38 expression in multiple myeloma and myeloid cell lines. Blood. 2016;128: 3521.</mixed-citation></citation-alternatives></ref><ref id="CR61"><label>61.</label><citation-alternatives><element-citation id="ec-CR61" publication-type="journal"><person-group person-group-type="author"><name><surname>Garcia</surname><given-names>PM</given-names></name><name><surname>Moore</surname><given-names>J</given-names></name><name><surname>Kahan</surname><given-names>D</given-names></name><name><surname>Hong</surname><given-names>MY</given-names></name></person-group><article-title>Effects of vitamin D supplementation on inflammation, colonic cell kinetics, and microbiota in colitis: a review</article-title><source>Molecules</source><year>2020</year><volume>25</volume><fpage>2300</fpage><?supplied-pmid 32422882?><pub-id pub-id-type="pmid">32422882</pub-id>
</element-citation><mixed-citation id="mc-CR61" publication-type="journal">Garcia PM, Moore J, Kahan D, Hong MY. Effects of vitamin D supplementation on inflammation, colonic cell kinetics, and microbiota in colitis: a review. Molecules. 2020;25: 2300.<pub-id pub-id-type="pmid">32422882</pub-id>
</mixed-citation></citation-alternatives></ref></ref-list></back></article>