# Early prediction of gestational diabetes mellitus based on systematically selected multi-panel biomarkers and clinical accessibility-a longitudinal study of a multi-racial pregnant cohort.

<!DOCTYPE article PUBLIC "-//NLM//DTD JATS (Z39.96) Journal Archiving and Interchange DTD with MathML3 v1.3 20210610//EN" "JATS-archivearticle1-3-mathml3.dtd"> 
<article xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article" dtd-version="1.3"><?properties open_access?><?DTDIdentifier.IdentifierValue -//Springer-Verlag//DTD A++ V2.4//EN?><?DTDIdentifier.IdentifierType public?><?SourceDTD.DTDName A++V2.4.dtd?><?SourceDTD.Version 2.4?><?ConverterInfo.XSLTName springer2nlmx2.xsl?><?ConverterInfo.Version 1?><processing-meta base-tagset="archiving" mathml-version="3.0" table-model="xhtml" tagset-family="jats"><restricted-by>pmc</restricted-by></processing-meta><front><journal-meta><journal-id journal-id-type="nlm-ta">BMC Med</journal-id><journal-id journal-id-type="iso-abbrev">BMC Med</journal-id><journal-title-group><journal-title>BMC Medicine</journal-title></journal-title-group><issn pub-type="epub">1741-7015</issn><publisher><publisher-name>BioMed Central</publisher-name><publisher-loc>London</publisher-loc></publisher></journal-meta>
<article-meta><article-id pub-id-type="pmcid">12275388</article-id><article-id pub-id-type="publisher-id">4258</article-id><article-id pub-id-type="doi">10.1186/s12916-025-04258-w</article-id><article-categories><subj-group subj-group-type="heading"><subject>Research</subject></subj-group></article-categories><title-group><article-title>Early prediction of gestational diabetes mellitus based on systematically selected multi-panel biomarkers and clinical accessibility&#x02014;a longitudinal study of a multi-racial pregnant cohort</article-title></title-group><contrib-group><contrib contrib-type="author"><name><surname>Yang</surname><given-names>Jiaxi</given-names></name><xref ref-type="aff" rid="Aff1">1</xref><xref ref-type="aff" rid="Aff2">2</xref><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author"><name><surname>Cao</surname><given-names>Yaqi</given-names></name><xref ref-type="aff" rid="Aff4">4</xref><xref ref-type="aff" rid="Aff5">5</xref></contrib><contrib contrib-type="author"><name><surname>Qian</surname><given-names>Fang</given-names></name><xref ref-type="aff" rid="Aff6">6</xref><xref ref-type="aff" rid="Aff7">7</xref></contrib><contrib contrib-type="author"><name><surname>Grewal</surname><given-names>Jagteshwar</given-names></name><xref ref-type="aff" rid="Aff8">8</xref></contrib><contrib contrib-type="author"><name><surname>Sacks</surname><given-names>David B.</given-names></name><xref ref-type="aff" rid="Aff9">9</xref></contrib><contrib contrib-type="author"><name><surname>Chen</surname><given-names>Zhen</given-names></name><xref ref-type="aff" rid="Aff10">10</xref></contrib><contrib contrib-type="author"><name><surname>Tsai</surname><given-names>Michael Y.</given-names></name><xref ref-type="aff" rid="Aff11">11</xref></contrib><contrib contrib-type="author" equal-contrib="yes"><name><surname>Chen</surname><given-names>Jinbo</given-names></name><xref ref-type="aff" rid="Aff5">5</xref></contrib><contrib contrib-type="author" corresp="yes" equal-contrib="yes"><name><surname>Zhang</surname><given-names>Cuilin</given-names></name><address><email>obgzc@nus.edu.sg</email></address><xref ref-type="aff" rid="Aff1">1</xref><xref ref-type="aff" rid="Aff2">2</xref><xref ref-type="aff" rid="Aff3">3</xref><xref ref-type="aff" rid="Aff12">12</xref></contrib><aff id="Aff1"><label>1</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/01tgyzw49</institution-id><institution-id institution-id-type="GRID">grid.4280.e</institution-id><institution-id institution-id-type="ISNI">0000 0001 2180 6431</institution-id><institution>Global Center for Asian Women&#x02019;s Health, Yong Loo Lin School of Medicine, </institution><institution>National University of Singapore, </institution></institution-wrap>Singapore, Singapore </aff><aff id="Aff2"><label>2</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/01tgyzw49</institution-id><institution-id institution-id-type="GRID">grid.4280.e</institution-id><institution-id institution-id-type="ISNI">0000 0001 2180 6431</institution-id><institution>Department of Obstetrics and Gynecology, Yong Loo Lin School of Medicine, </institution><institution>National University of Singapore, </institution></institution-wrap>Singapore, Singapore </aff><aff id="Aff3"><label>3</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/01tgyzw49</institution-id><institution-id institution-id-type="GRID">grid.4280.e</institution-id><institution-id institution-id-type="ISNI">0000 0001 2180 6431</institution-id><institution>Bia-Echo Asia Center for Reproductive Longevity &#x00026; Equality (ACRLE), Yong Loo Lin School of Medicine, </institution><institution>National University of Singapore, </institution></institution-wrap>Singapore, Singapore </aff><aff id="Aff4"><label>4</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/0044e2g62</institution-id><institution-id institution-id-type="GRID">grid.411077.4</institution-id><institution-id institution-id-type="ISNI">0000 0004 0369 0529</institution-id><institution>Department of Statistics, School of Science, </institution><institution>Minzu University of China, </institution></institution-wrap>Beijing, China </aff><aff id="Aff5"><label>5</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/00b30xv10</institution-id><institution-id institution-id-type="GRID">grid.25879.31</institution-id><institution-id institution-id-type="ISNI">0000 0004 1936 8972</institution-id><institution>Department of Biostatistics, Epidemiology &#x00026; Informatics, </institution><institution>University of Pennsylvania Perelman School of Medicine, </institution></institution-wrap>Philadelphia, PA USA </aff><aff id="Aff6"><label>6</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/02e7b5302</institution-id><institution-id institution-id-type="GRID">grid.59025.3b</institution-id><institution-id institution-id-type="ISNI">0000 0001 2224 0361</institution-id><institution>Center for Sustainable Finance Innovation, Nanyang Business School, </institution><institution>Nanyang Technological University, </institution></institution-wrap>Singapore, Singapore </aff><aff id="Aff7"><label>7</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/01tgyzw49</institution-id><institution-id institution-id-type="GRID">grid.4280.e</institution-id><institution-id institution-id-type="ISNI">0000 0001 2180 6431</institution-id><institution>Department of Pediatrics, Yong Loo Lin School of Medicine, </institution><institution>National University of Singapore, </institution></institution-wrap>Singapore, Singapore </aff><aff id="Aff8"><label>8</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/01cwqze88</institution-id><institution-id institution-id-type="GRID">grid.94365.3d</institution-id><institution-id institution-id-type="ISNI">0000 0001 2297 5165</institution-id><institution>Division of Population Health Research, Eunice Kennedy Shriver National Institute of Child Health and Human Development, </institution><institution>National Institutes of Health, </institution></institution-wrap>Bethesda, MD USA </aff><aff id="Aff9"><label>9</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/01cwqze88</institution-id><institution-id institution-id-type="GRID">grid.94365.3d</institution-id><institution-id institution-id-type="ISNI">0000 0001 2297 5165</institution-id><institution>Department of Laboratory Medicine, Clinical Center, </institution><institution>National Institutes of Health, </institution></institution-wrap>Bethesda, MD USA </aff><aff id="Aff10"><label>10</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/01cwqze88</institution-id><institution-id institution-id-type="GRID">grid.94365.3d</institution-id><institution-id institution-id-type="ISNI">0000 0001 2297 5165</institution-id><institution>Biostatistics and Bioinformatics Branch, Eunice Kennedy Shriver National Institute of Child Health and Human Development, </institution><institution>National Institutes of Health, </institution></institution-wrap>Bethesda, MD USA </aff><aff id="Aff11"><label>11</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/017zqws13</institution-id><institution-id institution-id-type="GRID">grid.17635.36</institution-id><institution-id institution-id-type="ISNI">0000 0004 1936 8657</institution-id><institution>Department of Laboratory Medicine and Pathology, </institution><institution>University of Minnesota, </institution></institution-wrap>Minneapolis, MN USA </aff><aff id="Aff12"><label>12</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/03vek6s52</institution-id><institution-id institution-id-type="GRID">grid.38142.3c</institution-id><institution-id institution-id-type="ISNI">000000041936754X</institution-id><institution>Department of Nutrition, </institution><institution>Harvard T.H. Chan School of Public Health, </institution></institution-wrap>Boston, MA USA </aff></contrib-group><pub-date pub-type="epub"><day>18</day><month>7</month><year>2025</year></pub-date><pub-date pub-type="pmc-release"><day>18</day><month>7</month><year>2025</year></pub-date><pub-date pub-type="collection"><year>2025</year></pub-date><volume>23</volume><elocation-id>430</elocation-id><history><date date-type="received"><day>19</day><month>10</month><year>2024</year></date><date date-type="accepted"><day>4</day><month>7</month><year>2025</year></date></history><permissions><copyright-statement>&#x000a9; The Author(s) 2025</copyright-statement><copyright-year>2025</copyright-year><license><ali:license_ref specific-use="textmining" content-type="ccbyncndlicense">https://creativecommons.org/licenses/by-nc-nd/4.0/</ali:license_ref><license-p><bold>Open Access</bold> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article&#x02019;s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article&#x02019;s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <ext-link ext-link-type="uri" xlink:href="https://creativecommons.org/licenses/by-nc-nd/4.0/">http://creativecommons.org/licenses/by-nc-nd/4.0/</ext-link>.</license-p></license></permissions><abstract id="Abs1"><sec><title>Background</title><p id="Par1">Early identification of high-risk women is critical for preventing gestational diabetes mellitus (GDM). We aimed to improve early prediction of GDM using multiple panels of cardiometabolic biomarkers assessed in early and mid-pregnancy, considering clinical accessibility.</p></sec><sec><title>Methods</title><p id="Par2">In a US study of 2802 pregnant individuals, we assessed 91 cardiometabolic biomarkers at 10&#x02013;14 (random blood) and 15&#x02013;26 (fasting) gestational weeks (GW) in 107 GDM cases and 214 controls. Candidate biomarkers were categorized by clinical accessibility from high to low: group I (clinically accessible tests like HbA1c, lipids), group II (clinically accessible biomarkers upon request like insulin-like growth factor (IGF) axis markers, adipokines), and group III (specialty lab-required targeted metabolomics: amino acids (AAs) and phospholipid fatty acids (FAs)). At each visit, we constructed a full model incorporating all candidate biomarkers and conventional predictors. We built alternative models utilizing different groups of biomarkers considering clinical accessibility. Variable selection was performed to retain variables with a <italic>p</italic> value&#x02009;&#x0003c;&#x02009;0.10 for a parsimonious model. Model performance was evaluated by area under receiver operating characteristics curve (AUC), proportion of cases followed (PCF, %) and proportion needed to follow (PNF, %), and decision curve analysis.</p></sec><sec><title>Results</title><p id="Par3">A full model comprising conventional predictors, clinical and non-clinical cardiometabolic biomarkers, and metabolomic markers achieved the highest discriminative accuracy (AUC: 0.842 at 10&#x02013;14 GW, 0.829 at 15&#x02013;26 GW). The addition of novel biomarkers increased PCF and decreased PNF, suggesting increased clinical utility. For example, at 10&#x02013;14 GW, 69.5% of GDM cases are expected to be detected from women whose risk is above the 80% percentile estimated by the full model vs. 49.1% by the conventional model. Additionally, 46.1% of women identified as being at the highest risk by the full model are expected to account for 90.0% of GDM cases vs. 71.1% by the conventional model. Decision curve analysis showed that models incorporating novel biomarkers performed better than the conventional model including glucose, and the full model at 10&#x02013;14 GW had the highest net benefit, overall.</p></sec><sec><title>Conclusions</title><p id="Par4">This study suggested that a selected panel of cardiometabolic biomarkers using early-pregnancy random plasma samples predicted GDM comparably to those using mid-pregnancy fasting samples.</p></sec><sec><title>Supplementary Information</title><p>The online version contains supplementary material available at 10.1186/s12916-025-04258-w.</p></sec></abstract><kwd-group xml:lang="en"><title>Keywords</title><kwd>Prediction</kwd><kwd>Pregnancy</kwd><kwd>Gestational diabetes mellitus</kwd><kwd>Cardiometabolic biomarkers</kwd><kwd>Clinical accessibility</kwd><kwd>AUC</kwd><kwd>Decision curve analysis</kwd></kwd-group><funding-group><award-group><funding-source><institution-wrap><institution-id institution-id-type="FundRef">http://dx.doi.org/10.13039/100000002</institution-id><institution>National Institutes of Health</institution></institution-wrap></funding-source><award-id>HHSN275200800013C, HHSN275200800002I, HHSN27500006, HHSN275200800003IC, HHSN275200800014C, HHSN275200800012C, HHSN275200800028C, HHSN275201000009C, HHSN275201000001Z</award-id><award-id>HHSN275200800013C, HHSN275200800002I, HHSN27500006, HHSN275200800003IC, HHSN275200800014C, HHSN275200800012C, HHSN275200800028C, HHSN275201000009C, HHSN275201000001Z</award-id><award-id>HHSN275200800013C, HHSN275200800002I, HHSN27500006, HHSN275200800003IC, HHSN275200800014C, HHSN275200800012C, HHSN275200800028C, HHSN275201000009C, HHSN275201000001Z</award-id><award-id>HHSN275200800013C, HHSN275200800002I, HHSN27500006, HHSN275200800003IC, HHSN275200800014C, HHSN275200800012C, HHSN275200800028C, HHSN275201000009C, HHSN275201000001Z</award-id></award-group></funding-group><custom-meta-group><custom-meta><meta-name>issue-copyright-statement</meta-name><meta-value>&#x000a9; BioMed Central Ltd., part of Springer Nature 2025</meta-value></custom-meta></custom-meta-group></article-meta></front><body><sec id="Sec1"><title>Background</title><p id="Par25">Gestational diabetes mellitus (GDM) is a common condition in pregnancy that leads to maternal and fetal complications. It is associated with long-term cardiometabolic consequences for both the mother and the offspring, including higher risks of obesity, impaired glucose tolerance, and type 2 diabetes in the child [<xref ref-type="bibr" rid="CR1">1</xref>, <xref ref-type="bibr" rid="CR2">2</xref>]. Recent evidence supports the effectiveness of first-trimester diet and lifestyle modifications in managing maternal hyperglycemia and lowering the risks of GDM and subsequent adverse neonatal outcomes [<xref ref-type="bibr" rid="CR2">2</xref>&#x02013;<xref ref-type="bibr" rid="CR6">6</xref>]. Considering the health impacts of GDM, early GDM prediction offers unique opportunities and benefits for timely intervention beyond the prevention of GDM alone [<xref ref-type="bibr" rid="CR7">7</xref>]. The current screening recommendation at 24&#x02013;28 gestational weeks (GW) [<xref ref-type="bibr" rid="CR8">8</xref>], however, may miss early maternal hyperglycemia impacts, emphasizing the need for early identification [<xref ref-type="bibr" rid="CR9">9</xref>, <xref ref-type="bibr" rid="CR10">10</xref>].</p><p id="Par26">Developing prediction models for early identifying high-risk pregnancies for GDM has been a focus of recent efforts. Emerging studies suggest that first-trimester maternal cardiometabolic blood biomarkers [<xref ref-type="bibr" rid="CR11">11</xref>&#x02013;<xref ref-type="bibr" rid="CR16">16</xref>] and circulating metabolites [<xref ref-type="bibr" rid="CR17">17</xref>&#x02013;<xref ref-type="bibr" rid="CR20">20</xref>] enhance predictive accuracy beyond conventional, non-laboratory-based risk factors [<xref ref-type="bibr" rid="CR10">10</xref>]. Evaluating and comparing the performance of biomarkers assessed in early and later gestation can provide insights into their relative predictive values and importantly, inform the potential for early GDM identification [<xref ref-type="bibr" rid="CR10">10</xref>]. We are aware of only one blood-based study that assessed biomarkers in both early and mid-pregnancy [<xref ref-type="bibr" rid="CR19">19</xref>]. Evaluating biomarkers alongside their clinical accessibility and assessment burdens is critical for developing efficient GDM prediction models. Notably, we are unaware of studies that have evaluated biomarkers in conjunction with their clinical accessibility.</p><p id="Par27">Using data from a longitudinal pregnancy cohort of 2802 individuals, we aimed to develop clinically efficient GDM prediction models based on multi-panel cardiometabolic plasma biomarkers assessed at 10&#x02013;14 GW and 15&#x02013;26 GW prior to GDM diagnosis, using a targeted approach. We sequentially developed models guided by biomarkers&#x02019; clinical accessibility and evaluated their predictive accuracy and potential clinical utility.</p></sec><sec id="Sec2"><title>Methods</title><sec id="Sec3"><title>Study population and design</title><p id="Par28">This study was conducted in participants from the <italic>Eunice Kennedy Shriver</italic> National Institute of Child Health and Human Development (NICHD) Fetal Growth Studies&#x02014;Singleton Cohort, a longitudinal, multi-racial, low-risk US pregnancy cohort. Briefly, 2802 pregnant women aged 18&#x02013;40&#x000a0;years with a self-reported pre-pregnancy body mass index (BMI, kg/m<sup>2</sup>) ranging from 19.0 to 45.0 and without pre-existing diseases were enrolled before 13 GW across 12 US clinical sites (2009&#x02013;2013) [<xref ref-type="bibr" rid="CR21">21</xref>]. We established a GDM case&#x02013;control study nested within the original cohort. From medical records, 107 women from the original cohort were diagnosed with GDM based on a 100-g, 3-h oral glucose tolerance test at 27&#x02009;&#x000b1;&#x02009;4 GW (mean&#x02009;&#x000b1;&#x02009;standard deviation) according to the Carpenter and Coustan criteria [<xref ref-type="bibr" rid="CR22">22</xref>]. The 107 GDM cases were individually matched at a 1:2 ratio to 214 participants who did not develop GDM (i.e., non-GDM controls). The matching factors were age, race/ethnicity, and gestational age at blood collection. Written informed consent was obtained from all the participants.</p></sec><sec id="Sec4"><title>Covariate assessment</title><p id="Par29">At enrollment (before 13 GW), demographics, lifestyle, and clinical factors were collected from the 2802 participants using structured questionnaires. Trained study staff measured body weight (kg) and height (cm) to derive BMI. We used the BMI measured at enrollment as a proxy for pre-pregnancy BMI (Spearman correlation&#x02009;=&#x02009;0.98). Gestational age in weeks was determined by ultrasound during the study screening visit.</p></sec><sec id="Sec5"><title>Biomarker assessment</title><p id="Par30">Maternal blood samples were collected at 10&#x02013;14 GW and 15&#x02013;26 GW based on a standardized protocol [<xref ref-type="bibr" rid="CR23">23</xref>]. Random blood samples were collected at 10&#x02013;14 GW. Blood samples at 15&#x02013;26 GW were collected after an overnight fast of 8&#x02013;14&#x000a0;h, confirmed by a questionnaire. Of the 321 (107 GDM and 214 non-GDM) individuals in the GDM case&#x02013;control subset, 307 (100 GDM and 207 non-GDM) at 10&#x02013;14 GW and 304 (93 GDM and 211 non-GDM) at 15&#x02013;26 GW had available blood samples for analysis. A total of 294 individuals had biospecimen available from both visits. Baseline demographics and major clinical factors were comparable between those included in the analysis and the entire case&#x02013;control subset.</p><p id="Par31">We quantified plasma concentrations of 91 cardiometabolic biomarkers (Additional file <xref rid="MOESM1" ref-type="media">1</xref>: Table S1). Glucose and high-sensitivity C-reactive protein (hsCRP) were quantified by enzymatic assays on the Roche Modular P Chemistry analyzer (Roche Diagnostics, Indianapolis, IN) [<xref ref-type="bibr" rid="CR24">24</xref>]. HbA1c was assessed using a non-porous ion exchange high performance liquid chromatography assay, HLC-723G8 (Tosoh Bioscience, Inc., South San Francisco, CA and Tokyo, Japan) [<xref ref-type="bibr" rid="CR25">25</xref>]. Insulin and C-peptide were quantified by the Roche Elecsys 2010 Analyzer (Roche Diagnostics, Indianapolis, IN) [<xref ref-type="bibr" rid="CR26">26</xref>]. Plasma lipid markers, total cholesterol, high-density lipoprotein cholesterol (HDL-C), and triglycerides were assessed by enzymatic assays (Roche Diagnostic Indianapolis, IN) [<xref ref-type="bibr" rid="CR27">27</xref>]. Markers of thyroid function such as free T3 and T4 (fT3, fT4) were measured by electrochemiluminescence immunoassay (Roche Diagnostics, Indianapolis, IN) [<xref ref-type="bibr" rid="CR28">28</xref>]. Markers of iron status were assessed by ELISA for hepcidin (DRG Diagnostics, Marburg Germany) and Roche reagents for ferritin (Roche Diagnostics, Indianapolis, IN) [<xref ref-type="bibr" rid="CR29">29</xref>]. Concentrations of insulin-like growth factor (IGF) axis biomarkers were measured by ELISA (ALPCO Diagnostics, Salem, NH) [<xref ref-type="bibr" rid="CR24">24</xref>]. A panel of ten adipokines was assessed by ELISA or a quantitative sandwich enzyme immunoassay (BioVendor, R&#x00026;D System, and Mercodia, Sweden) [<xref ref-type="bibr" rid="CR23">23</xref>]. We quantified plasma concentrations of 24 individual amino acids (AAs) by an AA analyzer (Hitachi L-8900) [<xref ref-type="bibr" rid="CR30">30</xref>] and individual phospholipid FAs, including five even- and odd-chain saturated fatty acids (SFAs) and six mono- and 28 poly-unsaturated fatty acids (MUFAs, PUFAs), on the Hewlett Packard 5890 gas chromatography [<xref ref-type="bibr" rid="CR26">26</xref>, <xref ref-type="bibr" rid="CR31">31</xref>, <xref ref-type="bibr" rid="CR32">32</xref>]. All the assays were performed at the University of Minnesota Advanced Research and Diagnostics Laboratory (USA), with the investigators blinded to the GDM status.</p></sec><sec id="Sec6"><title>Statistical analysis</title><sec id="Sec7"><title>Model developments&#x02014;full model and alternative models considering clinical accessibility</title><p id="Par32">We built logistic regression models for predicting the development of GDM at 10&#x02013;14 GW and 15&#x02013;26 GW, respectively. Overall, we considered well-documented socio-demographic risk factors and cardiometabolic biomarkers significantly associated with the development of GDM in the original full cohort or the nested GDM case&#x02013;control subset as potential candidate predictors [<xref ref-type="bibr" rid="CR23">23</xref>&#x02013;<xref ref-type="bibr" rid="CR32">32</xref>]. Age, race/ethnicity, pre-pregnancy BMI, family history of diabetes, and plasma glucose (mg/dL) were considered as conventional predictors [<xref ref-type="bibr" rid="CR19">19</xref>, <xref ref-type="bibr" rid="CR33">33</xref>]. We categorized the candidate biomarkers into three groups (i.e., groups I, II, III) based on their clinical accessibility (Table&#x000a0;<xref rid="Tab1" ref-type="table">1</xref>). Group I comprised biomarkers routinely assessed at relatively affordable clinical costs, including HbA1c, hsCRP, C-peptide, insulin (fasting samples at 15&#x02013;26 GW only), lipids, ferritin, and markers of thyroid function. Group II comprised biomarkers that are clinically measurable upon request with reasonable costs but not routinely assessed, including IGF axis markers, adipokines, and hepcidin. Group III comprised targeted metabolomics: individual AAs and phospholipid FAs (i.e., SFAs, MUFAs, and PUFAs), which are not yet clinically accessible and require a specialty laboratory for assessment.
<table-wrap id="Tab1"><label>Table&#x000a0;1</label><caption><p>List of conventional risk factors and selected biomarkers<sup>a</sup> for the development of gestational diabetes mellitus, NICHD cohort (<italic>n</italic>&#x02009;=&#x02009;2802)</p></caption><table frame="hsides" rules="groups"><thead><tr><th align="left"/><th align="left">10&#x02013;14 gestational weeks</th><th align="left">15&#x02013;26 gestational weeks</th></tr></thead><tbody><tr><td align="left">Conventional risk factors</td><td align="left">Age (&#x0003c;&#x02009;30,&#x02009;&#x02265;&#x02009;30 years), pre-pregnancy BMI (&#x0003c;&#x02009;25.0, 25.0&#x02013;29.9, 30.0&#x02009;+&#x02009;kg/m<sup>2</sup>), race/ethnicity (non-Hispanic White, non-Hispanic Black, Hispanic, Asian/Pacific Islander), family history of diabetes (yes, no), random glucose (&#x0003e;&#x02009;94.0 mg/dL)</td><td align="left">Age (&#x0003c;&#x02009;30,&#x02009;&#x02265;&#x02009;30 years), pre-pregnancy BMI (&#x0003c;&#x02009;25.0, 25.0&#x02013;29.9, 30.0&#x02009;+&#x02009;kg/m<sup>2</sup>), race/ethnicity (non-Hispanic White, non-Hispanic Black, Hispanic, Asian/Pacific Islander), family history of diabetes (yes, no), fasting glucose (&#x0003e;&#x02009;88.0 mg/dL)</td></tr><tr><td align="left">Group I: candidate biomarkers commonly measured at clinical labs</td><td align="left">HbA1c, HDL-C, triglycerides, hsCRP, C-peptide, ferritin, fT3-to-fT4 ratio</td><td align="left">HbA1c, HDL-C, triglycerides, hsCRP, C-peptide, ferritin, fT3-to-fT4 ratio, fasting insulin</td></tr><tr><td align="left">Group II: candidate biomarkers measurable at clinical labs but not routinely measured</td><td align="left">IGF-I, IGFBP-2, HMW-adiponectin, FABP4, chemerin, IL-6, leptin, sOb-R</td><td align="left">IGF-I, IGFBP-2, HMW-adiponectin, FABP4, chemerin, IL-6, leptin, sOb-R, hepcidin</td></tr><tr><td align="left">Group III: selected targeted metabolomics markers requiring specialty lab for assessment</td><td align="left">Summed even-chained SFAs<sup>b</sup>, summed odd-chained SFAs<sup>c</sup>, PUFA 18:1n-7, PUFA 20:3n-6, PUFA 18:3n-6, MUFA 18:1n-7; glycine, alanine, aspartic acid, glutamic acid</td><td align="left">Summed even-chained SFAs, summed odd-chained SFAs, PUFA 18:1n-7, PUFA 20:3n-6, PUFA 22:4n-6, MUFA 18:1n-7; glycine, alanine, aspartic acid, glutamine-to-glutamic acid ratio, isoleucine, tyrosine</td></tr></tbody></table><table-wrap-foot><p><italic>Abbreviations</italic>: <italic>AA</italic> amino acid, <italic>AUC</italic> area under receiver operating characteristics curve, <italic>BMI</italic> body mass index, <italic>FA</italic> fatty acid, <italic>FABP4</italic> fatty acid binding protein-4, <italic>fT3-to-fT4 ratio</italic> free T3-to-free T4 ratio, <italic>GDM</italic> gestational diabetes mellitus, <italic>HbA1c</italic> hemoglobin A1c, <italic>HDL-C</italic> high-density lipoprotein cholesterol, <italic>HMW-adiponectin</italic> high-molecular-weight adiponectin, <italic>hsCRP</italic> high-sensitivity C-reactive protein, <italic>IGF-I</italic> insulin-like growth factor I, <italic>IGFBP-2</italic> insulin-like growth factor binding protein 2, <italic>IL-6</italic> interlukin-6, <italic>MUFA</italic> mono-unsaturated fatty acid, <italic>sOb-R</italic> soluble leptin receptor, <italic>PUFA</italic> poly-unsaturated fatty acid, <italic>SFA</italic> saturated fatty acid</p><p><sup>a</sup>Biomarkers were pre-selected from 91 individual cardiometabolic biomarkers assessed at 10&#x02013;14 GW and 16&#x02013;26 GW. These selected biomarkers were significantly related to the risk of GDM development from previous studies</p><p><sup>b</sup>Summed even-chained SFAs&#x02009;=&#x02009;sum of myristic acid (14:0), palmitic acid (16:0), and stearic acid (18:0)</p><p><sup>c</sup>Summed odd-chained SFAs&#x02009;=&#x02009;sum of pentadecanoic acid (15:0) and heptadecanoic acid (17:0)</p></table-wrap-foot></table-wrap></p><p id="Par33">Data for the conventional predictors except glucose were available for the full cohort of 2802 individuals, whereas plasma concentrations of glucose and candidate biomarkers at 10&#x02013;14 GW (random blood) and 15&#x02013;26 GW (fasting blood) were assessed only in the nested case&#x02013;control subset. These data were used to develop the visit-specific prediction models and to compute all the predictive measures. To maximize the use of available data, we adopted a two-phase design methodology to accommodate incomplete availability of biomarker data for fitting all regression models [<xref ref-type="bibr" rid="CR34">34</xref>]. Specifically, we applied a semiparametric maximum likelihood method [<xref ref-type="bibr" rid="CR35">35</xref>, <xref ref-type="bibr" rid="CR36">36</xref>] to estimate odds ratio association parameters and predictive performance measures. This approach enabled evaluation of the predictive values of matching factors and efficiently utilized both the full cohort data (&#x0201c;Phase I data&#x0201d;) and the biomarker data from the GDM case&#x02013;control subset (&#x0201c;Phase II data&#x0201d;), with high statistical efficiency for estimating predictive performance measures [<xref ref-type="bibr" rid="CR34">34</xref>].</p><p id="Par34">Taking this approach, we began with a full model including all the conventional predictors and candidate biomarkers from groups I, II, and III. However, not all candidate biomarkers in the full model are clinically accessible. Therefore, we developed alternative prediction models with descending clinical accessibility, using different groups of candidate biomarkers. Following the full model, we first built a model including group I and II candidate biomarkers, and then a model including group I candidate biomarkers, and lastly, a model containing only the conventional predictors including glucose (i.e., conventional model).</p><p id="Par35">To avoid model overfitting, we performed backward variable selection by sequentially removing variables of the candidate biomarkers with the largest <italic>p</italic> value (i.e., the least significant) until all the remaining variables had a <italic>p</italic> value&#x02009;&#x0003c;&#x02009;0.10 [<xref ref-type="bibr" rid="CR34">34</xref>, <xref ref-type="bibr" rid="CR37">37</xref>]. The conventional predictors were always retained. All the conventional predictors were categorized based on subject matter knowledge and previous prediction studies (Table&#x000a0;<xref rid="Tab1" ref-type="table">1</xref>) [<xref ref-type="bibr" rid="CR33">33</xref>]. Visit-specific concentration of plasma glucose at the 80% percentile was set as the binary cut-off for glucose. Variables of the candidate biomarkers were standardized (value minus the mean then divided by its standard deviation) and modeled as continuous variables given their linear associations with GDM risk observed in our prior work [<xref ref-type="bibr" rid="CR23">23</xref>&#x02013;<xref ref-type="bibr" rid="CR32">32</xref>]. Additivity was assumed in model developments, with no interaction terms included between predictors.</p><p id="Par36">Each model&#x02019;s predictive performance was assessed by plotting receiver operating characteristics curve, from which the area under the curve (AUC) was reported as the primary predictive measure. We considered two other measures to assess a model&#x02019;s potential public health utility: the proportion of cases followed (PCF[<italic>q</italic>]; the proportion of cases expected to be identified among women whose risk is above 100*(1&#x02009;&#x02212;&#x02009;<italic>q</italic>)% percentile) and the proportion needed to follow (PNF[<italic>p</italic>]; the proportion of high-risk sample that expects to yield 100*<italic>p</italic>% of cases) [<xref ref-type="bibr" rid="CR38">38</xref>, <xref ref-type="bibr" rid="CR39">39</xref>]. For demonstration purposes, we reported specific values of PCF and PNF by setting <italic>q</italic> to 0.2 and <italic>p</italic> to 0.9 [<xref ref-type="bibr" rid="CR38">38</xref>]. AUC, PCF, and PNF were estimated through resampling with 500 bootstrapping samples to address potential bias due to dual use of data for model development and validation [<xref ref-type="bibr" rid="CR40">40</xref>, <xref ref-type="bibr" rid="CR41">41</xref>]. Additionally, we performed decision curve analysis to estimate the net benefit of a model across possible predicted risk thresholds [<xref ref-type="bibr" rid="CR42">42</xref>, <xref ref-type="bibr" rid="CR43">43</xref>].</p></sec><sec id="Sec8"><title>Prediction model evaluation in clinical context</title><sec id="FPar1"><title>Full model predicting GDM risk among women at high-risk</title><p id="Par37">To demonstrate the clinical applications of the full model, we randomly selected eight individuals from those considered as &#x0201c;high-risk&#x0201d; based on non-laboratory, conventional risk factors (age&#x02009;&#x02265;&#x02009;30&#x000a0;years, BMI&#x02009;&#x02265;&#x02009;25.0&#x000a0;kg/m<sup>2</sup>, and having a family history of diabetes) through stratified sampling based on GDM status. We estimated individuals&#x02019; predicted risks of GDM at 10&#x02013;14 GW and 15&#x02013;26 GW, respectively, first according to the non-laboratory, conventional risk factors&#x02019; profile, next by adding glucose to the equation, and finally, according to the full model.</p></sec><sec id="FPar2"><title>Predictive performance of the full model vs. a clinically attainable model</title><p id="Par38">As group III metabolomic biomarkers are not clinically accessible, we considered the model comprising conventional predictors and group I and II candidate biomarkers, that is, a model with the biomarkers accessible either directly or upon request, as a clinically attainable alternative to the full model. We compared the predictive performance of the full model with that of the clinically attainable model.</p></sec><sec id="FPar3"><title>Prediction utilizing early-pregnancy non-fasting biomarkers vs. mid-pregnancy fasting biomarkers</title><p id="Par39">Recognizing the significance of early identification and the high burdens associated with biomarker assessment in pregnancy, particularly concerning the requirement for pregnant women to fast, we compared the performance of the full model at 10&#x02013;14 GW using random, non-fasting blood samples to that of the full model at 15&#x02013;26 GW using fasting blood samples in the GDM case&#x02013;control subset. We calculated predicted GDM risks for individuals using the full model at 10&#x02013;14 GW and the one at 15&#x02013;26 GW, respectively. Individuals with a predicted risk above the 80% percentile were classified as &#x0201c;predicted high-risk,&#x0201d; while others were classified as &#x0201c;predicted low-risk.&#x0201d; We then compared the predicted risk status at 10&#x02013;14 GW and 15&#x02013;26 GW, with the observed GDM status.</p><p>Figure&#x000a0;<xref rid="Fig1" ref-type="fig">1</xref> summarizes the workflow of our analysis. All the analyses were performed using R software (version 4.1.1), and &#x0201c;TwoPhaseAccuracy&#x0201d; and &#x0201c;osDesign&#x0201d; packages were used to build the models and estimate the predictive accuracy measures [<xref ref-type="bibr" rid="CR34">34</xref>].<fig id="Fig1"><label>Fig.&#x000a0;1</label><caption><p>Overview of study analysis workflow. Footnote: Candidate biomarkers utilized in the full model development were categorized into three groups based on their clinical accessibility from high to low. Group I biomarkers are routinely measured at clinical labs. Group II biomarkers can be measured at clinical labs but are not routinely measured. Group III biomarkers comprise targeted metabolomics which are not accessible in clinical labs and require specialty labs for assessment</p></caption><graphic xlink:href="12916_2025_4258_Fig1_HTML" id="MO1"/></fig></p></sec></sec></sec></sec><sec id="Sec9"><title>Results</title><p id="Par41">List of candidate biomarkers is summarized in Table&#x000a0;<xref rid="Tab1" ref-type="table">1</xref>. Additional file <xref rid="MOESM2" ref-type="media">2</xref>: Table S2 presents the baseline characteristics of the entire cohort and the nested GDM case&#x02013;control study according to the GDM status.</p><sec id="Sec10"><title>Prediction models for GDM</title><sec id="Sec11"><title>Full models for prediction</title><p id="Par42">Based on biomarkers assessed using random plasma samples at 10&#x02013;14 GW, we identified a prediction model (model 1&#x02014;full model) that included the conventional predictors, HbA1c from group I, IGF binding protein 2 (IGFBP-2) and leptin from group II, and summed odd-chain SFAs, summed even-chain SFAs, glycine, and aspartic acid from group III (Table&#x000a0;<xref rid="Tab2" ref-type="table">2</xref>). The model had an AUC (standard error, SE) of 0.842 (SE: 0.02). PCF (<italic>q</italic>&#x02009;=&#x02009;0.2) of the full model was 0.695 (SE: 0.04), that is, 69.5% of the GDM cases are expected to be detected from women whose risk is above the 80% percentile estimated by the full model. PNF (<italic>p</italic>&#x02009;=&#x02009;0.9) of the full model was 0.461 (0.05), that is, 46.1% of the women identified as being at the highest risk by the full model are expected to account for 90.0% of the GDM cases.
<table-wrap id="Tab2"><label>Table&#x000a0;2</label><caption><p>Prediction models of gestational diabetes mellitus and model&#x02019;s predictive performance at 10&#x02013;14 and 15&#x02013;26 gestational weeks, the NICHD Fetal Growth Studies&#x02014;Singleton Cohort (<italic>n</italic>&#x02009;=&#x02009;2802)</p></caption><table frame="hsides" rules="groups"><thead><tr><th align="left" rowspan="2"/><th align="left" colspan="4">10&#x02013;14 gestational weeks (<italic>n</italic> with available biomarkers: 100 GDM, 207 non-GDM<sup>a</sup>)</th><th align="left" colspan="4">15&#x02013;26 gestational weeks (<italic>n</italic> with available biomarkers: 93 GDM, 211 non-GDM<sup>a</sup>)</th></tr><tr><th align="left">Predictors</th><th align="left">AUC</th><th align="left">PCF (<italic>q</italic>&#x02009;=&#x02009;0.2)<sup>b</sup></th><th align="left">PNF (<italic>p</italic>&#x02009;=&#x02009;0.9)<sup>c</sup></th><th align="left">Predictors</th><th align="left">AUC</th><th align="left">PCF (<italic>q</italic>&#x02009;=&#x02009;0.2)</th><th align="left">PNF (<italic>p</italic>&#x02009;=&#x02009;0.9)</th></tr></thead><tbody><tr><td align="left">Model 1 (full model)<sup>d</sup></td><td align="left">Conventional risk factors, HbA1c, IGFBP-2, leptin, summed odd-chained SFAs, summed even-chained SFAs, glycine, aspartic acid</td><td align="left">0.842 (0.02)</td><td align="left">0.695 (0.04)</td><td align="left">0.461 (0.05)</td><td align="left">Conventional risk factors, HbA1c, sOb-R, summed odd-chain SFAs, PUFA 22:4n-6, glycine, aspartic acid</td><td align="left">0.829 (0.02)</td><td align="left">0.676 (0.05)</td><td align="left">0.480 (0.05)</td></tr><tr><td align="left"><p>Model 2</p><p>Conventional risk factors, group I, group II predictive biomarkers</p></td><td align="left">Conventional risk factors, HbA1c, fT3-to-fT4 ratio, IGFBP-2, leptin</td><td align="left">0.812 (0.03)</td><td align="left">0.638 (0.04)</td><td align="left">0.517 (0.05)</td><td align="left">Conventional risk factors, HbA1c, fT3-to-fT4 ratio, sOb-R</td><td align="left">0.794 (0.03)</td><td align="left">0.613 (0.05)</td><td align="left">0.578 (0.06)</td></tr><tr><td align="left"><p>Model 3</p><p>Conventional risk factors, group I predictive biomarkers</p></td><td align="left">Conventional risk factors, HbA1c, triglycerides, fT3-to-fT4 ratio</td><td align="left">0.786 (0.03)</td><td align="left">0.603 (0.07)</td><td align="left">0.592 (0.05)</td><td align="left">Conventional risk factors, HbA1c, triglycerides, fT3-to-fT4 ratio</td><td align="left">0.788 (0.03)</td><td align="left">0.614 (0.05)</td><td align="left">0.607 (0.05)</td></tr><tr><td align="left"><p>Model 4 (conventional model)</p><p>Conventional risk factors</p></td><td align="left">Age, pre-pregnancy BMI, race/ethnicity, family history of diabetes, random glucose<sup>e</sup></td><td align="left">0.720 (0.03)</td><td align="left">0.491 (0.04)</td><td align="left">0.711 (0.04)</td><td align="left">Age, pre-pregnancy BMI, race/ethnicity, family history of diabetes, fasting glucose<sup>e</sup></td><td align="left">0.741 (0.03)</td><td align="left">0.544 (0.07)</td><td align="left">0.696 (0.06)</td></tr></tbody></table><table-wrap-foot><p><italic>Abbreviations</italic>: <italic>AUC</italic> area under receiver operating characteristics curve, <italic>BMI</italic> body mass index, <italic>fT3-to-fT4 ratio</italic> free T3-to-free T4 ratio, <italic>GDM</italic> gestational diabetes mellitus, <italic>HbA1c</italic> hemoglobin A1c, <italic>IGFBP-2</italic> insulin-like growth factor binding protein 2, <italic>sOb-R</italic> soluble leptin receptor, <italic>PCF</italic> proportion of cases followed, <italic>PNF</italic> proportion needed to follow, <italic>PUFA</italic> poly-unsaturated fatty acid, <italic>SFA</italic> saturated fatty acid</p><p><sup>a</sup>Data for conventional predictors except glucose were available for the full cohort of 2802 individuals, whereas plasma concentrations of glucose and candidate biomarkers were assessed only in the nested case&#x02013;control subset. These data were used to develop visit-specific prediction models and to compute all the predictive measures</p><p><sup>b</sup>The proportion of cases followed, PCF(<italic>q</italic>): the proportion of cases expected to be identified among women whose risk is above 100*(1&#x02009;&#x02212;&#x02009;<italic>q</italic>)% percentile estimated by the model</p><p><sup>c</sup>The proportion needed to follow, PNF(<italic>p</italic>): the proportion of high-risk sample estimated by the model that one needs to follow to identify 100*<italic>p</italic>% of cases</p><p><sup>d</sup>Candidate biomarkers utilized in the full model development were categorized into three groups based on descending clinical accessibility. Group I biomarkers are routinely measured at clinical labs. Group II biomarkers can be measured at clinical labs but not routinely measured. Group III biomarkers comprise targeted metabolomics which are not accessible in clinical labs and require specialty labs for assessment</p><p><sup>e</sup>Binary cut-off for glucose was 0.94&#x000a0;mg/dL at 10&#x02013;14 GW and 0.88&#x000a0;mg/dL at 15&#x02013;26 GW</p></table-wrap-foot></table-wrap></p><p id="Par43">Similarly, we built a full model (model 1) at 15&#x02013;26 GW based on fasting blood samples. The model included HbA1c from group I, soluble leptin receptor (sOb-R) from group II, and summed odd-chain SFAs, PUFA 22:4n-6, glycine, and aspartic acid from group III, along with the conventional predictors. The model had an AUC of 0.829 (SE: 0.02), a PCF (<italic>q</italic>&#x02009;=&#x02009;0.2) of 0.676 (SE: 0.05), and a PNF (<italic>p</italic>&#x02009;=&#x02009;0.9) of 0.480 (SE: 0.05) (Table&#x000a0;<xref rid="Tab2" ref-type="table">2</xref>).</p></sec><sec id="Sec12"><title>Alternative prediction models considering clinical accessibility</title><p id="Par44">We developed alternative models, considering clinical accessibility. At 10&#x02013;14 GW, we first identified a prediction model (model 2) by incorporating group I and II candidate biomarkers, which are clinically accessible either directly or upon request. The model identified an additional predictor of free T3-to-free T4 (fT3-fT4), a thyroid function marker from group I, achieving an AUC of 0.812 (SE: 0.02), a PCF (<italic>q</italic>&#x02009;=&#x02009;0.2) of 0.638 (SE: 0.04), and a PNF (<italic>p</italic>&#x02009;=&#x02009;0.9) of 0.517 (SE: 0.05) (Table&#x000a0;<xref rid="Tab2" ref-type="table">2</xref>). We then identified a model (model 3) by utilizing group I candidate biomarkers which are routinely accessible at clinical labs. Triglycerides from group I was additionally included in the model, and the model had an AUC of 0.786 (SE: 0.03), a PCF (<italic>q</italic>&#x02009;=&#x02009;0.2) of 0.603 (SE: 0.07), and a PNF (<italic>p</italic>&#x02009;=&#x02009;0.9) of 0.592 (SE: 0.05). Lastly, we built a prediction model (conventional model, model 4) containing only conventional predictors. The model had an AUC of 0.720 (SE: 0.03), a PCF (<italic>q</italic>&#x02009;=&#x02009;0.2) of 0.491 (SE: 0.04), and a PNF (<italic>p</italic>&#x02009;=&#x02009;0.9) of 0.711 (SE: 0.04).</p><p id="Par45">We also identified alternative prediction models at 15&#x02013;26 GW. Model 2 included group I and II candidate biomarkers and identified fT3-to-fT4 as an additional predictor, achieving an AUC of 0.794 (SE: 0.03), a PCF (<italic>q</italic>&#x02009;=&#x02009;0.2) of 0.613 (SE: 0.05), and a PNF (<italic>p</italic>&#x02009;=&#x02009;0.9) of 0.578 (SE: 0.06) (Table&#x000a0;<xref rid="Tab2" ref-type="table">2</xref>). We next identified model 3, utilizing group I candidate biomarkers only. The model additionally included triglycerides from group I as a predictor and achieved an AUC of 0.788 (SE: 0.03), a PCF (<italic>q</italic>&#x02009;=&#x02009;0.2) of 0.614 (SE: 0.05), and a PNF (<italic>p</italic>&#x02009;=&#x02009;0.9) of 0.607 (SE: 0.05). Lastly, the conventional model (model 4) had an AUC (SE) of 0.741 (0.03), a PCF (<italic>q</italic>&#x02009;=&#x02009;0.2) of 0.544 (SE: 0.07), and a PNF (<italic>p</italic>&#x02009;=&#x02009;0.9) of 0.696 (SE: 0.06).</p><p id="Par46">At both 10&#x02013;14 GW and 15&#x02013;26 GW, compared to the full model, alternative prediction models had lower AUC and PCF values and higher PNF values at various cut-offs, as expected (Fig.&#x000a0;<xref rid="Fig2" ref-type="fig">2</xref>). Results from the decision curve analysis indicated that at both time points, the full model had the highest net benefit than all the alternative prediction models at various risk thresholds for defining high-risk group; conversely, the conventional model had the lowest net benefit (Fig.&#x000a0;<xref rid="Fig3" ref-type="fig">3</xref>).<fig id="Fig2"><label>Fig.&#x000a0;2</label><caption><p>Performance of prediction models of gestational diabetes mellitus at 10&#x02013;14 gestational weeks and 15&#x02013;26 gestational weeks, the NICHD Fetal Growth Studies&#x02014;Singleton Cohort (<italic>n</italic>&#x02009;=&#x02009;2802). Footnote: Receiver operating characteristics (ROC) curves (panel A, B)&#x000a0;and curves for the proportion of cases followed (PCF) (panel C, D)&#x000a0;and the proportion needed to follow (PNF) (panel E, F)&#x000a0;at various thresholds are presented for all models at 10&#x02013;14 gestational weeks and 15&#x02013;26 gestational weeks. Different colors represent models using different groups of biomarkers</p></caption><graphic xlink:href="12916_2025_4258_Fig2_HTML" id="MO2"/></fig><fig id="Fig3"><label>Fig.&#x000a0;3</label><caption><p>Decision curves of prediction models of gestational diabetes mellitus at 10&#x02013;14 gestational weeks and 15&#x02013;26 gestational weeks, the NICHD Fetal Growth Studies&#x02014;Singleton Cohort (<italic>n</italic>&#x02009;=&#x02009;2802). Footnote: Decision curves for all models at 10&#x02013;14 gestational weeks are represented by solid lines, and those for models at 15&#x02013;26 gestational weeks are represented by dashed lines. Different colors represent models using different groups of biomarkers</p></caption><graphic xlink:href="12916_2025_4258_Fig3_HTML" id="MO3"/></fig></p></sec></sec><sec id="Sec13"><title>Prediction model evaluation in clinical context</title><sec id="Sec14"><title>Full models predicting GDM risk among women at high-risk</title><p id="Par47">We calculated the predicted GDM risk for eight randomly selected high-risk women using the full models at 10&#x02013;14 GW and 15&#x02013;26 GW (Table&#x000a0;<xref rid="Tab3" ref-type="table">3</xref>). Conventional predictors (i.e., age, BMI, and family history of diabetes) yielded similar predicted risks (ranging from 0.05 to 0.26) for case and controls at 10&#x02013;14 GW and 15&#x02013;26 GW. The addition of glucose, particularly fasting glucose at 15&#x02013;26 GW, increased the accuracy of the predicted GDM risks. Predictive accuracy was further increased when estimating the risks using the full models for both GDM cases and the controls, as indicated by an increase in the predicted risk for cases and a decrease for controls, with the differences in predicted risks between cases and controls being greater when using the full models compared to conventional predictors. For example, at 10&#x02013;14 GW, the range of the predicted risks estimated by the full model was 0.20&#x02013;0.45 for cases and 0.02&#x02013;0.08 for controls (Table&#x000a0;<xref rid="Tab3" ref-type="table">3</xref>).
<table-wrap id="Tab3"><label>Table&#x000a0;3</label><caption><p>Predicted risks of gestational diabetes mellitus at 10&#x02013;14 gestational weeks and 15&#x02013;26 gestational weeks based on prediction models with conventional risk factors and the full models in a randomly selected high-risk sample (<italic>n</italic>&#x02009;=&#x02009;8)<sup>a</sup>, the NICHD Fetal Growth Studies&#x02014;Singleton Cohort</p></caption><table frame="hsides" rules="groups"><thead><tr><th align="left">Random ID</th><th align="left"/><th align="left">8732</th><th align="left">92,593</th><th align="left">56,484</th><th align="left">58,535</th><th align="left">90,378</th><th align="left">51,436</th><th align="left">66,901</th><th align="left">395</th></tr><tr><th align="left">GDM status</th><th align="left">Coefficient</th><th align="left">Yes</th><th align="left">Yes</th><th align="left">Yes</th><th align="left">Yes</th><th align="left">No</th><th align="left">No</th><th align="left">No</th><th align="left">No</th></tr></thead><tbody><tr><td align="left">Conventional risk factors; no biomarkers</td><td align="left"/><td align="left"/><td align="left"/><td align="left"/><td align="left"/><td align="left"/><td align="left"/><td align="left"/><td align="left"/></tr><tr><td align="left">Age, years</td><td align="left">Age&#x02009;&#x02265;&#x02009;30&#x000a0;years: 0.654</td><td align="left">32</td><td align="left">34</td><td align="left">33</td><td align="left">31</td><td align="left">37</td><td align="left">34</td><td align="left">31</td><td align="left">34</td></tr><tr><td align="left">BMI, kg/m<sup>2</sup></td><td align="left">BMI 24.0&#x02013;29.9: 0.876, BMI&#x02009;&#x0003e;&#x02009;29.9: 1.345</td><td align="left">33.1</td><td align="left">30.4</td><td align="left">28.3</td><td align="left">26.4</td><td align="left">28.5</td><td align="left">31.4</td><td align="left">30.2</td><td align="left">25.4</td></tr><tr><td align="left">Race/ethnicity</td><td align="left">Black:&#x02009;&#x02212;&#x02009;0.564; Hispanic 0.382; Asian/Pacific Islander: 0.806</td><td align="left">White</td><td align="left">Asian/Pacific Islander</td><td align="left">Black</td><td align="left">Hispanic</td><td align="left">Asian/Pacific Islander</td><td align="left">Black</td><td align="left">Black</td><td align="left">Asian/Pacific Islander</td></tr><tr><td align="left">Family history diabetes</td><td align="left">Yes: 0.631</td><td align="left">Yes</td><td align="left">Yes</td><td align="left">Yes</td><td align="left">Yes</td><td align="left">Yes</td><td align="left">Yes</td><td align="left">Yes</td><td align="left">Yes</td></tr><tr><td align="left">Predicted probability of GDM (95% CI)</td><td align="left"/><td align="left">0.14 (0.08, 0.22)</td><td align="left">0.26 (0.16, 0.41)</td><td align="left">0.05 (0.03, 0.10)</td><td align="left">0.13 (0.08, 0.20)</td><td align="left">0.18 (0.11, 0.29)</td><td align="left">0.08 (0.04, 0.15)</td><td align="left">0.19 (0.12, 0.28)</td><td align="left">0.18 (0.11, 0.29)</td></tr><tr><td align="left">10&#x02013;14 GW</td><td align="left"/><td align="left"/><td align="left"/><td align="left"/><td align="left"/><td align="left"/><td align="left"/><td align="left"/><td align="left"/></tr><tr><td align="left">Glucose, mg/dL</td><td align="left">0.002</td><td align="left">112</td><td align="left">90</td><td align="left">101</td><td align="left">83</td><td align="left">88</td><td align="left">84</td><td align="left">113</td><td align="left">81</td></tr><tr><td align="left">Predicted probability of GDM (95% CI)</td><td align="left"/><td align="left">0.17 (0.05, 0.47)</td><td align="left">0.15 (0.04, 0.45)</td><td align="left">0.09 (0.02, 0.37)</td><td align="left">0.08 (0.02, 0.27)</td><td align="left">0.11 (0.03, 0.35)</td><td align="left">0.05 (0.01, 0.21)</td><td align="left">0.25 (0.05, 0.65)</td><td align="left">0.11 (0.03, 0.35)</td></tr><tr><td align="left">HbA1c, %; mmol/mol</td><td align="left">0.005</td><td align="left">5.4</td><td align="left">5.4</td><td align="left">5.8</td><td align="left">5.4</td><td align="left">5.1</td><td align="left">5.3</td><td align="left">4.3</td><td align="left">5.2</td></tr><tr><td align="left">IGFBP-2, ng/mL</td><td align="left">0.013</td><td align="left">103.7</td><td align="left">82.5</td><td align="left">70.2</td><td align="left">54.3</td><td align="left">82.2</td><td align="left">76.5</td><td align="left">132</td><td align="left">122.7</td></tr><tr><td align="left">Leptin, ng/mL</td><td align="left">0.004</td><td align="left">72.6</td><td align="left">62.6</td><td align="left">61.9</td><td align="left">53.9</td><td align="left">26.1</td><td align="left">43</td><td align="left">38.1</td><td align="left">23.7</td></tr><tr><td align="left">Summed odd-chain SFAs, %</td><td align="left">0.010</td><td align="left">0.48</td><td align="left">0.59</td><td align="left">0.73</td><td align="left">0.6</td><td align="left">0.74</td><td align="left">0.72</td><td align="left">0.68</td><td align="left">0.56</td></tr><tr><td align="left">Summed even-chain SFAs, %</td><td align="left">0.007</td><td align="left">40.8</td><td align="left">41.6</td><td align="left">40.7</td><td align="left">41.4</td><td align="left">39.7</td><td align="left">40.9</td><td align="left">45.9</td><td align="left">39.3</td></tr><tr><td align="left">Glycine, &#x000b5;mol/dL</td><td align="left">0.009</td><td align="left">11.3</td><td align="left">11</td><td align="left">10.91</td><td align="left">12.5</td><td align="left">14.9</td><td align="left">11.4</td><td align="left">12.1</td><td align="left">12.8</td></tr><tr><td align="left">Aspartic acid, &#x000b5;mol/dL</td><td align="left">0.003</td><td align="left">0.99</td><td align="left">1.22</td><td align="left">0.53</td><td align="left">0.66</td><td align="left">1.08</td><td align="left">0.67</td><td align="left">0.21</td><td align="left">1.09</td></tr><tr><td align="left">Predicted probability of GDM (95% CI)</td><td align="left"/><td align="left">0.45 (0.12, 0.83)</td><td align="left">0.36 (0.07, 0.81)</td><td align="left">0.20 (0.03, 0.70)</td><td align="left">0.24 (0.07, 0.55)</td><td align="left">0.06 (0.01, 0.24)</td><td align="left">0.02 (0.01, 0.10)</td><td align="left">0.02 (0.01, 0.11)</td><td align="left">0.08 (0.02, 0.29)</td></tr><tr><td align="left">15&#x02013;26 GW</td><td align="left"/><td align="left"/><td align="left"/><td align="left"/><td align="left"/><td align="left"/><td align="left"/><td align="left"/><td align="left"/></tr><tr><td align="left">Fasting glucose, mg/dL</td><td align="left">0.0002</td><td align="left">97</td><td align="left">103</td><td align="left">96</td><td align="left">93</td><td align="left">95</td><td align="left">96</td><td align="left">70</td><td align="left">83</td></tr><tr><td align="left">Predicted probability of GDM (95% CI)</td><td align="left"/><td align="left">0.21 (0.05, 0.54)</td><td align="left">0.31 (0.07, 0.74)</td><td align="left">0.11 (0.02, 0.45)</td><td align="left">0.19 (0.04, 0.57)</td><td align="left">0.25 (0.05, 0.67)</td><td align="left">0.14 (0.02, 0.54)</td><td align="left">0.08 (0.02, 0.28)</td><td align="left">0.08 (0.05, 0.29)</td></tr><tr><td align="left">HbA1c, %; mmol/mol</td><td align="left">0.044</td><td align="left">5</td><td align="left">5</td><td align="left">5.9</td><td align="left">5.6</td><td align="left">4.7</td><td align="left">5.9</td><td align="left">4.1</td><td align="left">4.8</td></tr><tr><td align="left">sOb-R, ng/mL</td><td align="left">0.005</td><td align="left">13.9</td><td align="left">21.4</td><td align="left">25.2</td><td align="left">27.9</td><td align="left">4</td><td align="left">25.2</td><td align="left">19.6</td><td align="left">22.8</td></tr><tr><td align="left">Summed odd-chain SFAs, %</td><td align="left">0.003</td><td align="left">0.57</td><td align="left">0.51</td><td align="left">0.8</td><td align="left">0.68</td><td align="left">0.82</td><td align="left">0.61</td><td align="left">0.59</td><td align="left">0.63</td></tr><tr><td align="left">PUFA 22:4n-6, %</td><td align="left">0.024</td><td align="left">0.25</td><td align="left">0.2</td><td align="left">0.25</td><td align="left">0.18</td><td align="left">0.42</td><td align="left">0.22</td><td align="left">0.34</td><td align="left">0.28</td></tr><tr><td align="left">Glycine, &#x000b5;mol/dL</td><td align="left">0.004</td><td align="left">12.7</td><td align="left">12.5</td><td align="left">11.5</td><td align="left">13.4</td><td align="left">14.1</td><td align="left">11.5</td><td align="left">11.9</td><td align="left">12.5</td></tr><tr><td align="left">Aspartic acid, &#x000b5;mol/dL</td><td align="left">0.077</td><td align="left">1.76</td><td align="left">0.49</td><td align="left">0.43</td><td align="left">0.59</td><td align="left">0.98</td><td align="left">0.43</td><td align="left">0.1</td><td align="left">1.67</td></tr><tr><td align="left">Predicted probability of GDM (95% CI)</td><td align="left"/><td align="left">0.49 (0.20, 0.79)</td><td align="left">0.30 (0.09, 0.64)</td><td align="left">0.12 (0.01, 0.57)</td><td align="left">0.23 (0.04, 0.66)</td><td align="left">0.05 (0.01, 0.33)</td><td align="left">0.16 (0.03, 0.56)</td><td align="left">0.03 (0.01, 0.05)</td><td align="left">0.11 (0.03, 0.34)</td></tr></tbody></table><table-wrap-foot><p><italic>Abbreviations</italic>: <italic>BMI</italic> body mass index, <italic>CI</italic> confidence interval, <italic>GDM</italic> gestational diabetes mellitus, <italic>GW</italic> gestational week, <italic>HbA1c</italic> hemoglobin A1c, <italic>IGFBP-2</italic> insulin-like growth factor binding protein 2, <italic>sOb-R</italic> soluble leptin receptor, <italic>PUFA</italic> poly-unsaturated fatty acid, <italic>SFA</italic> saturated fatty acid</p><p><sup>a</sup>High-risk was defined as age&#x02009;&#x02265;&#x02009;30, BMI&#x02009;&#x02265;&#x02009;25.0&#x000a0;kg/m<sup>2</sup>, and having a family history of diabetes</p></table-wrap-foot></table-wrap></p></sec><sec id="Sec15"><title>Predictive performance of the full model vs. a clinically attainable model</title><p id="Par48">At both 10&#x02013;14 GW and 15&#x02013;26 GW, AUC, PCF and PNF values, and decision curve of the full model were more favorable than those of the clinically attainable model (model 2) (Figs. <xref rid="Fig2" ref-type="fig">2</xref>, <xref rid="Fig3" ref-type="fig">3</xref>). We re-calculated the predicted risks of GDM in the same high-risk subset using model 2 (Additional file <xref rid="MOESM3" ref-type="media">3</xref>: Table S3). For GDM cases, the predicted risk estimated by the full model was generally slightly higher than the risk estimated by model 2. For non-GDM controls, the predicted risks estimated by the two models were similar, with all estimated risks below 0.16 (Table&#x000a0;<xref rid="Tab3" ref-type="table">3</xref>; Additional file <xref rid="MOESM3" ref-type="media">3</xref>: Table S3).</p></sec><sec id="Sec16"><title>Prediction utilizing early-pregnancy non-fasting biomarkers vs. mid-pregnancy fasting biomarkers</title><p id="Par49">Using the full model at 10&#x02013;14 GW, 69.5% of predicted high-risk individuals (41 out of 59) were GDM cases, and 79.6% of predicted low-risk individuals (187 out of 235) were controls who did not develop GDM (high-risk defined as a predicted risk above the 80th percentile and low-risk otherwise) (Additional file <xref rid="MOESM4" ref-type="media">4</xref>: Table S4). Using the full model at 15&#x02013;26 GW, 71.2% (42 out of 59) of predicted high-risk individuals were GDM cases, while 80.0% (188 out of 235) of predicted low-risk individuals were controls. Decision curve analysis indicated that the full model at 10&#x02013;14 GW had overall greater net benefit than the full model at 15&#x02013;26 GW (Fig.&#x000a0;<xref rid="Fig3" ref-type="fig">3</xref>).</p></sec></sec></sec><sec id="Sec17"><title>Discussion</title><p id="Par50">In a multi-racial pregnancy cohort, based on biomarkers selected from more than 90 cardiometabolic markers measured using blood samples collected in early and mid-pregnancy, we identified prediction models for GDM comprising conventional risk factors, HbA1c, and selected biomarkers of IGF axis and adipokines, odd- and even-chain SFAs, and individual PUFAs and AAs, achieving an AUC of 0.84 at 10&#x02013;14 GW and 0.83 at 15&#x02013;26 GW. Notably, the models at 10&#x02013;14 GW based on random plasma samples performed comparably to those at 15&#x02013;26 GW based on fasting plasma samples. Findings from this study suggest the potential of early-pregnancy maternal cardiometabolic biomarkers in predicting GDM with clinical utility evidence.</p><p id="Par51">Prediction models from previous studies were generally developed based on conventional, non-laboratory risk factors. A systematic review published in 2016 [<xref ref-type="bibr" rid="CR33">33</xref>] validated 12 such models. The two models with the highest predictive performance included maternal age, pre-pregnancy BMI, race/ethnicity, family history of diabetes, and/or a history of GDM. We constructed a model incorporating these conventional factors but excluded history of GDM and observed an AUC of 0.72, comparable to the AUCs (0.74&#x02013;0.77) reported in the review [<xref ref-type="bibr" rid="CR33">33</xref>].</p><p id="Par52">Interventional studies have suggested the effectiveness of early prevention, particularly during the first trimester, in preventing GDM [<xref ref-type="bibr" rid="CR3">3</xref>, <xref ref-type="bibr" rid="CR5">5</xref>, <xref ref-type="bibr" rid="CR6">6</xref>] and associated pregnancy complications [<xref ref-type="bibr" rid="CR44">44</xref>, <xref ref-type="bibr" rid="CR45">45</xref>]. Such evidence underscores the significance of applying early and accurate GDM prediction to identify at-risk individuals for early prevention and management. Several prediction studies with maternal blood samples collected by 14 GW [<xref ref-type="bibr" rid="CR11">11</xref>&#x02013;<xref ref-type="bibr" rid="CR16">16</xref>] consistently indicated improved GDM prediction by including first-trimester cardiometabolic biomarkers beyond conventional risk factors. Commonly examined biomarkers include cardiometabolic markers of glucose and lipid metabolisms, insulin function, and inflammation [<xref ref-type="bibr" rid="CR10">10</xref>]. More recently, metabolomic studies suggested enhanced prediction by selected first-trimester metabolites involved in glucose, carbohydrate, lipid, and amino acid metabolism [<xref ref-type="bibr" rid="CR17">17</xref>&#x02013;<xref ref-type="bibr" rid="CR20">20</xref>]. One blood-based metabolomic study showed comparable predictive accuracies between a 17-metabolite panel at 10&#x02013;13 GW vs. a 13-metabolite panel at 16&#x02013;19 GW [<xref ref-type="bibr" rid="CR19">19</xref>]. In addition to blood-based biomarker studies, other biospecimens such as maternal urine involving metabolomic profiling have also been explored for GDM prediction [<xref ref-type="bibr" rid="CR9">9</xref>, <xref ref-type="bibr" rid="CR46">46</xref>, <xref ref-type="bibr" rid="CR47">47</xref>].</p><p id="Par53">In contrast to previous studies, we developed GDM prediction models with a pathway-specific approach, using a comprehensive list of selected cardiometabolic biomarkers, including targeted metabolomics, assessed at two specific time points, 10&#x02013;14 GW and 15&#x02013;26 GW. The biomarkers were selected from over 90 multi-panel maternal blood biomarkers based on their established individual associations with GDM and proposed biological pathways. Building on our previous findings regarding these associations, here, we fully investigated these biomarkers for their combined predictive power, aiming to identify the most efficient models. Since it is often challenging for pregnant women to fast, the collection of random blood samples at 10&#x02013;14 GW may, in fact, reflect a more realistic clinical practice than fasting blood samples, offering insights into early-pregnancy prediction with low-burden biomarker assessments. When evaluating a model&#x02019;s predictive performance, in addition to AUC, we estimated measures of PCF and PNF and performed decision curve analyses to evaluate potential clinical utility. PCF assesses a model&#x02019;s efficiency, and PNF assesses its feasibility, reflecting the potential clinical/public health applications for a risk prediction model [<xref ref-type="bibr" rid="CR39">39</xref>]. From the decision curve analysis, the net benefit at a given threshold can be interpreted as the number of true-positive classifications adjusted for the harmful effects of false-positive classifications, such that it incorporates both the predictive accuracy and tradeoffs between benefits and harms of using the model for identifying and potentially stratifying high-risks individuals for making clinical decisions [<xref ref-type="bibr" rid="CR48">48</xref>]. Results from the full models at 10&#x02013;14 GW and 15&#x02013;26 GW suggested that the inclusion of seven and six variables of individual or derived measures of maternal cardiometabolic biomarkers, respectively, improved the model&#x02019;s performance in discriminating between high-risk and low-risk individuals over conventional risk predictors and achieved superior clinical utility over all other considered models. Concentrations of these cardiometabolic biomarkers reflect several pathway-specific mechanisms proposed for the etiology of GDM. These mechanisms include glucose homeostasis and insulin signaling and sensitivity (HbA1c, glucose, thyroid markers, IGF family markers, adipokines, glycine, and aspartic acid), inflammatory pathways (adipokines), and lipid (triglycerides, phospholipid FAs) and amino acid metabolism [<xref ref-type="bibr" rid="CR49">49</xref>].</p><p id="Par54">As a novel contribution to the field, this study suggests the potential of using random non-fasting blood samples for predicting GDM in the first trimester. The full model using random blood samples at 10&#x02013;14 GW represents potentially an earlier and more convenient prediction tool than the full model using fasting blood samples at 15&#x02013;26 GW. The two full models demonstrated comparable discriminative accuracy and potential clinical utility. We further verified this observation in our exploratory analysis in the GDM case&#x02013;control set. These findings are promising and warrant further evaluation, considering the effectiveness of early interventions which, in turn, may reduce the risks of GDM and its associated short- and long-term complications.</p><p id="Par55">Clinical accessibility is crucial for real-world implementation of prediction models. We developed multiple alternative models, considering clinical accessibility of biomarkers. Specifically, we proposed a model more clinically attainable than the full model, comprising cardiometabolic biomarkers that are clinically accessible (group I) or accessible upon request (group II). Our results indicate that, in both early and mid-pregnancy, compared to the full model, the clinically attainable model showed lower discriminative performance and limited clinical utility and benefit. Nevertheless, it provided additional information beyond conventional predictors and clinically accessible cardiometabolic biomarkers (group I). We note that these models require validation in external studies, but they are promising as the biomarkers examined here are common and suitable for further evaluation and potential clinical implementation. Notably, we applied a novel statistical method that effectively uses data not only from the case&#x02013;control subset but also from the full cohort with available data on conventional risk predictors. As a result, the generalizability of our models is ensured, making them applicable to pregnant women similar to those included in our study. This study therefore provides valuable insights for future prediction research, especially under budget constraints. Upon future validations, evaluating the real-world applicability and feasibility of these prediction models is a critical next step. Practical implementation, including the selection of cut-offs to define high- vs. low-risk groups, should be assessed in clinical settings, considering factors such as real-world clinical accessibility, implementation costs, the extent to which the underlying population is at risk for GDM, and the intensity and costs of intervention strategies for identified high-risk individuals.</p><p id="Par56">This study has several strengths. The longitudinal design, with multiple distinct panels of cardiometabolic biomarkers assessed in early and mid-pregnancy, enabled a comprehensive analysis of GDM prediction. We systematically developed a series of prediction models based on these biomarkers to identify the most powerful model for GDM prediction and evaluated their predictive capacity while considering clinical accessibility of the biomarkers. Considering real-world clinical practice, we assessed the performance of the most powerful model (i.e., full model) in comparison with a more clinically attainable model. Alongside examining discriminative accuracy using AUC, we evaluated measures assessing clinical utility. Several limitations merit discussion. First, external validation of the prediction models was not conducted. Despite being one of the largest prospective investigations into GDM prediction, the sample size with available biomarker data was modest. Due to the limited sample size, we did not perform internal validation by splitting the data into training and testing datasets. Although the two-phase method that we applied for analyses ensured efficient utilization of both incomplete biomarker data and the fully available conventional predictor data, the power was lower than that if the biomarker data were fully available in the cohort. To address potential model overfitting and the lack of validation, we performed variable selection and bootstrapping to reduce the bias in the estimated predictive measures [<xref ref-type="bibr" rid="CR50">50</xref>]. Future external validation is needed to evaluate our findings. While we did not examine interaction terms between predictors, future well-powered studies should consider and investigate potential interactions between predictors, taking into account underlying biological mechanisms. Larger studies are warranted to assess maternal cardiometabolic biomarkers in predicting GDM across diverse racial/ethnic populations and, ideally, stratified by risk profile when possible. Differences in fasting duration among participants at 10&#x02013;14 GW may have introduced additional variation in levels of certain biomarkers, particularly those sensitive to fasting status, such as glucose and triglycerides. However, as fasting duration prior to biospecimen collection were non-differential by case&#x02013;control status, differential measurement error due to fasting status is unlikely.</p></sec><sec id="Sec18"><title>Conclusions</title><p id="Par57">We identified powerful prediction models for GDM using blood samples collected in early and mid-pregnancy in a longitudinal pregnancy cohort. The models at 10&#x02013;14 GW based on random plasma samples achieved predictive performance comparable to the models at 15&#x02013;26 GW based on fasting plasma samples. These findings suggest potential of early GDM prediction and warrant external validations.</p></sec><sec id="Sec22" sec-type="supplementary-material"><title>Supplementary Information</title><p>
<supplementary-material content-type="local-data" id="MOESM1"><media xlink:href="12916_2025_4258_MOESM1_ESM.docx"><caption><p>Additional file 1: Table S1. List of maternal plasma biomarkers assessed in the GDM case&#x02013;control study nested within the NICHD Fetal Growth Studies&#x02014;Singleton Cohort.</p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="MOESM2"><media xlink:href="12916_2025_4258_MOESM2_ESM.docx"><caption><p>Additional file 2: Table S2. Population characteristics overall and according to status of gestational diabetes mellitus&#x000a0;(i.e., cases and controls).</p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="MOESM3"><media xlink:href="12916_2025_4258_MOESM3_ESM.docx"><caption><p>Additional file 3: Table S3. Predicted risks of gestational diabetes mellitus at 10&#x02013;14 gestational weeks and 15&#x02013;26 gestational weeks based on prediction models of conventional risk factors and group I and II candidate biomarkers.</p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="MOESM4"><media xlink:href="12916_2025_4258_MOESM4_ESM.docx"><caption><p>Additional file 4: Table S4. Predicted GDM risk status at 10&#x02013;14 gestational weeks and 15&#x02013;26 gestational weeks according to the full models, using 80% percentile predicted risk as high-risk cut-off.</p></caption></media></supplementary-material></p></sec></body><back><glossary><title>Abbreviations</title><def-list><def-item><term>GDM</term><def><p id="Par5">Gestational diabetes mellitus</p></def></def-item><def-item><term>GW</term><def><p id="Par6">Gestational week</p></def></def-item><def-item><term>IGF</term><def><p id="Par7">Insulin-like growth factor</p></def></def-item><def-item><term>AA</term><def><p id="Par8">Amino acid</p></def></def-item><def-item><term>FA</term><def><p id="Par9">Fatty acid</p></def></def-item><def-item><term>AUC</term><def><p id="Par10">Area under receiver operating characteristics curve</p></def></def-item><def-item><term>PCF</term><def><p id="Par11">Proportion of cases followed</p></def></def-item><def-item><term>PNF</term><def><p id="Par12">Proportion needed to follow</p></def></def-item><def-item><term>NICHD</term><def><p id="Par13">National Institute of Child Health and Human Development</p></def></def-item><def-item><term>BMI</term><def><p id="Par14">Body mass index</p></def></def-item><def-item><term>hsCRP</term><def><p id="Par15">High-sensitivity C-reactive protein</p></def></def-item><def-item><term>HDL-C</term><def><p id="Par16">High-density lipoprotein cholesterol</p></def></def-item><def-item><term>fT3</term><def><p id="Par17">Free T3</p></def></def-item><def-item><term>fT4</term><def><p id="Par18">Free T4</p></def></def-item><def-item><term>SFA</term><def><p id="Par19">Saturated fatty acid</p></def></def-item><def-item><term>MUFA</term><def><p id="Par20">Mono-unsaturated fatty acid</p></def></def-item><def-item><term>PUFA</term><def><p id="Par21">Poly-unsaturated fatty acid</p></def></def-item><def-item><term>IGFBP-2</term><def><p id="Par22">Insulin-like growth factor binding protein 2</p></def></def-item><def-item><term>SE</term><def><p id="Par23">Standard error</p></def></def-item><def-item><term>sOb-R</term><def><p id="Par24">Soluble leptin receptor</p></def></def-item></def-list></glossary><fn-group><fn><p><bold>Publisher&#x02019;s Note</bold></p><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></fn><fn><p>Jinbo Chen and Cuilin Zhang contributed equally to this work.</p></fn></fn-group><ack><title>Acknowledgements</title><p>The authors would like to thank all the participants and staff at the participating clinical centers in the Eunice Kennedy Shriver National Institute of Child Health and Human Development Fetal Growth Studies&#x02013;Singleton Cohort, including Christina Care Health Systems; University of California, Irvine; Long Beach Memorial Medical Center; Northwestern University; Medical University of South Carolina; Columbia University; New York Hospital Queens; St Peter&#x02019;s University Hospital; University of Alabama at Birmingham; Women and Infants Hospital of Rhode Island; Fountain Valley Regional Hospital and Medical Center; and Tufts University. We also would like to acknowledge the C-TASC Corporation, which provided data coordination, and the Department of Laboratory Medicine and Pathology, University of Minnesota, which provided laboratory resources essential for biochemical analysis.</p><sec id="FPar4"><title>Prior presentation</title><p id="Par58">Parts of this study were presented as a poster presentation at the 84th Scientific Sessions of the American Diabetes Associations between June 21 and 24, 2024, in Orlando, FL, USA.</p></sec></ack><notes notes-type="author-contribution"><title>Authors&#x02019; contributions</title><p>JY: Methodology, software, formal analysis, investigation, writing &#x02013; original draft, writing &#x02013; review &#x00026; editing. YC: Methodology, software, formal analysis, investigation, writing &#x02013; review &#x00026; editing. FQ: Software, visualization, writing &#x02013; review &#x00026; editing. JG: Data curation, writing &#x02013; review &#x00026; editing. DBS: Methodology, writing &#x02013; review &#x00026; editing. ZC: Writing &#x02013; review &#x00026; editing. MYT: Data curation, resources, writing &#x02013; review &#x00026; editing. JC: Methodology, investigation, writing&#x02014;review &#x00026; editing, study supervision. CZ: Funding acquisition, data curation, conceptualization, methodology, investigation, writing&#x02014;reviewing &#x00026; editing, study supervision. CZ and JC are the senior authors of the study. JY and CZ have full access to the raw data. All authors read and approved the final manuscript.</p><sec id="FPar5"><title>Authors&#x02019; Twitter handles</title><p id="Par59">Twitter handles: @Cuilinchat (Cuilin Zhang).</p></sec></notes><notes notes-type="funding-information"><title>Funding</title><p>This research was supported by <italic>Eunice Kennedy Shriver</italic> National Institute of Child Health and Human Development intramural funding and American Recovery and Reinvestment Act funding (grants HHSN275200800013C, HHSN275200800002I, HHSN27500006, HHSN275200800003IC, HHSN275200800014C, HHSN275200800012C, HHSN275200800028C, HHSN275201000009C, and HHSN275201000001Z).</p></notes><notes notes-type="data-availability"><title>Data availability</title><p>The data used in this study are not publicly available due to privacy and confidentiality agreements. Access to the data is restricted to protect the personal and health information of the participants, in accordance with ethical guidelines and regulations. Researchers interested in accessing the data may contact the corresponding author with a detailed request and may be required to sign a data use agreement to ensure the protection of participant confidentiality.</p></notes><notes><title>Declarations</title><notes id="FPar6"><title>Ethics approval and consent to participate</title><p id="Par60">The study protocol was reviewed and approved by institutional review boards at NICHD and each of the clinical sites. Written informed consent was obtained from all the participants. The IRB approval number is NICHD (09-CH-N152).</p></notes><notes id="FPar7"><title>Consent for publication</title><p id="Par61">Not applicable.</p></notes><notes id="FPar8" notes-type="COI-statement"><title>Competing interests</title><p id="Par62">The authors declare no competing interests.</p></notes></notes><ref-list id="Bib1"><title>References</title><ref id="CR1"><label>1.</label><citation-alternatives><element-citation id="ec-CR1" publication-type="journal"><person-group person-group-type="author"><name><surname>Zhu</surname><given-names>Y</given-names></name><name><surname>Zhang</surname><given-names>C</given-names></name></person-group><article-title>Prevalence of gestational diabetes and risk of progression to type 2 diabetes: a global perspective</article-title><source>Curr Diab Rep</source><year>2016</year><volume>16</volume><fpage>7</fpage><?supplied-pmid 26742932?><pub-id pub-id-type="pmid">26742932</pub-id>
</element-citation><mixed-citation id="mc-CR1" publication-type="journal">Zhu Y, Zhang C. Prevalence of gestational diabetes and risk of progression to type 2 diabetes: a global perspective. Curr Diab Rep. 2016;16:7.<pub-id pub-id-type="pmid">26742932</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR2"><label>2.</label><citation-alternatives><element-citation id="ec-CR2" publication-type="journal"><person-group person-group-type="author"><name><surname>McIntyre</surname><given-names>HD</given-names></name><name><surname>Catalano</surname><given-names>P</given-names></name><name><surname>Zhang</surname><given-names>C</given-names></name><name><surname>Desoye</surname><given-names>G</given-names></name><name><surname>Mathiesen</surname><given-names>ER</given-names></name><name><surname>Damm</surname><given-names>P</given-names></name></person-group><article-title>Gestational diabetes mellitus</article-title><source>Nat Rev Dis Primers</source><year>2019</year><volume>5</volume><fpage>47</fpage><?supplied-pmid 31296866?><pub-id pub-id-type="pmid">31296866</pub-id>
</element-citation><mixed-citation id="mc-CR2" publication-type="journal">McIntyre HD, Catalano P, Zhang C, Desoye G, Mathiesen ER, Damm P. Gestational diabetes mellitus. Nat Rev Dis Primers. 2019;5:47.<pub-id pub-id-type="pmid">31296866</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR3"><label>3.</label><citation-alternatives><element-citation id="ec-CR3" publication-type="journal"><person-group person-group-type="author"><name><surname>Assaf-Balut</surname><given-names>C</given-names></name><name><surname>Garc&#x000ed;a de la Torre</surname><given-names>N</given-names></name><name><surname>Dur&#x000e1;n</surname><given-names>A</given-names></name><name><surname>Fuentes</surname><given-names>M</given-names></name><name><surname>Bordi&#x000fa;</surname><given-names>E</given-names></name><name><surname>Del Valle</surname><given-names>L</given-names></name><etal/></person-group><article-title>A Mediterranean diet with additional extra virgin olive oil and pistachios reduces the incidence of gestational diabetes mellitus (GDM): a randomized controlled trial: the St. Carlos GDM prevention study</article-title><source>PLoS ONE</source><year>2017</year><volume>12</volume><fpage>e0185873</fpage><?supplied-pmid 29049303?><pub-id pub-id-type="pmid">29049303</pub-id>
</element-citation><mixed-citation id="mc-CR3" publication-type="journal">Assaf-Balut C, Garc&#x000ed;a de la Torre N, Dur&#x000e1;n A, Fuentes M, Bordi&#x000fa; E, Del Valle L, et al. A Mediterranean diet with additional extra virgin olive oil and pistachios reduces the incidence of gestational diabetes mellitus (GDM): a randomized controlled trial: the St. Carlos GDM prevention study. PLoS ONE. 2017;12:e0185873.<pub-id pub-id-type="pmid">29049303</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR4"><label>4.</label><citation-alternatives><element-citation id="ec-CR4" publication-type="journal"><person-group person-group-type="author"><name><surname>Harrison</surname><given-names>CL</given-names></name><name><surname>Lombard</surname><given-names>CB</given-names></name><name><surname>Strauss</surname><given-names>BJ</given-names></name><name><surname>Teede</surname><given-names>HJ</given-names></name></person-group><article-title>Optimizing healthy gestational weight gain in women at high risk of gestational diabetes: a randomized controlled trial</article-title><source>Obesity (Silver Spring)</source><year>2013</year><volume>21</volume><fpage>904</fpage><lpage>909</lpage><?supplied-pmid 23784892?><pub-id pub-id-type="pmid">23784892</pub-id>
</element-citation><mixed-citation id="mc-CR4" publication-type="journal">Harrison CL, Lombard CB, Strauss BJ, Teede HJ. Optimizing healthy gestational weight gain in women at high risk of gestational diabetes: a randomized controlled trial. Obesity (Silver Spring). 2013;21:904&#x02013;9.<pub-id pub-id-type="pmid">23784892</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR5"><label>5.</label><citation-alternatives><element-citation id="ec-CR5" publication-type="journal"><person-group person-group-type="author"><name><surname>Sadiya</surname><given-names>A</given-names></name><name><surname>Jakapure</surname><given-names>V</given-names></name><name><surname>Shaar</surname><given-names>G</given-names></name><name><surname>Adnan</surname><given-names>R</given-names></name><name><surname>Tesfa</surname><given-names>Y</given-names></name></person-group><article-title>Lifestyle intervention in early pregnancy can prevent gestational diabetes in high-risk pregnant women in the UAE: a randomized controlled trial</article-title><source>BMC Pregnancy Childbirth</source><year>2022</year><volume>22</volume><fpage>668</fpage><?supplied-pmid 36042401?><pub-id pub-id-type="pmid">36042401</pub-id>
</element-citation><mixed-citation id="mc-CR5" publication-type="journal">Sadiya A, Jakapure V, Shaar G, Adnan R, Tesfa Y. Lifestyle intervention in early pregnancy can prevent gestational diabetes in high-risk pregnant women in the UAE: a randomized controlled trial. BMC Pregnancy Childbirth. 2022;22:668.<pub-id pub-id-type="pmid">36042401</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR6"><label>6.</label><citation-alternatives><element-citation id="ec-CR6" publication-type="journal"><person-group person-group-type="author"><name><surname>Wang</surname><given-names>C</given-names></name><name><surname>Wei</surname><given-names>Y</given-names></name><name><surname>Zhang</surname><given-names>X</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Xu</surname><given-names>Q</given-names></name><name><surname>Sun</surname><given-names>Y</given-names></name><etal/></person-group><article-title>A randomized clinical trial of exercise during pregnancy to prevent gestational diabetes mellitus and improve pregnancy outcome in overweight and obese pregnant women</article-title><source>Am J Obstet Gynecol</source><year>2017</year><volume>216</volume><fpage>340</fpage><lpage>351</lpage><?supplied-pmid 28161306?><pub-id pub-id-type="pmid">28161306</pub-id>
</element-citation><mixed-citation id="mc-CR6" publication-type="journal">Wang C, Wei Y, Zhang X, Zhang Y, Xu Q, Sun Y, et al. A randomized clinical trial of exercise during pregnancy to prevent gestational diabetes mellitus and improve pregnancy outcome in overweight and obese pregnant women. Am J Obstet Gynecol. 2017;216:340&#x02013;51.<pub-id pub-id-type="pmid">28161306</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR7"><label>7.</label><citation-alternatives><element-citation id="ec-CR7" publication-type="journal"><person-group person-group-type="author"><name><surname>Saravanan</surname><given-names>P</given-names></name></person-group><article-title>Gestational diabetes: opportunities for improving maternal and child health</article-title><source>Lancet Diabetes Endocrinol</source><year>2020</year><volume>8</volume><fpage>793</fpage><lpage>800</lpage><?supplied-pmid 32822601?><pub-id pub-id-type="pmid">32822601</pub-id>
</element-citation><mixed-citation id="mc-CR7" publication-type="journal">Saravanan P. Gestational diabetes: opportunities for improving maternal and child health. Lancet Diabetes Endocrinol. 2020;8:793&#x02013;800.<pub-id pub-id-type="pmid">32822601</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR8"><label>8.</label><citation-alternatives><element-citation id="ec-CR8" publication-type="journal"><person-group person-group-type="author"><name><surname>Davidson</surname><given-names>KW</given-names></name><name><surname>Barry</surname><given-names>MJ</given-names></name><name><surname>Mangione</surname><given-names>CM</given-names></name><name><surname>Cabana</surname><given-names>M</given-names></name><name><surname>Caughey</surname><given-names>AB</given-names></name><name><surname>Davis</surname><given-names>EM</given-names></name><etal/></person-group><article-title>Screening for gestational diabetes: US Preventive Services Task Force recommendation statement</article-title><source>JAMA</source><year>2021</year><volume>326</volume><fpage>531</fpage><lpage>538</lpage><?supplied-pmid 34374716?><pub-id pub-id-type="pmid">34374716</pub-id>
</element-citation><mixed-citation id="mc-CR8" publication-type="journal">Davidson KW, Barry MJ, Mangione CM, Cabana M, Caughey AB, Davis EM, et al. Screening for gestational diabetes: US Preventive Services Task Force recommendation statement. JAMA. 2021;326:531&#x02013;8.<pub-id pub-id-type="pmid">34374716</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR9"><label>9.</label><citation-alternatives><element-citation id="ec-CR9" publication-type="journal"><person-group person-group-type="author"><name><surname>Koos</surname><given-names>BJ</given-names></name><name><surname>Gornbein</surname><given-names>JA</given-names></name></person-group><article-title>Early pregnancy metabolites predict gestational diabetes mellitus: implications for fetal programming</article-title><source>Am J Obstet Gynecol</source><year>2021</year><volume>224</volume><fpage>215.e1</fpage><lpage>215.e7</lpage><?supplied-pmid 32739399?><pub-id pub-id-type="pmid">32739399</pub-id>
</element-citation><mixed-citation id="mc-CR9" publication-type="journal">Koos BJ, Gornbein JA. Early pregnancy metabolites predict gestational diabetes mellitus: implications for fetal programming. Am J Obstet Gynecol. 2021;224:215.e1-215.e7.<pub-id pub-id-type="pmid">32739399</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR10"><label>10.</label><citation-alternatives><element-citation id="ec-CR10" publication-type="journal"><person-group person-group-type="author"><name><surname>Powe</surname><given-names>CE</given-names></name></person-group><article-title>Early pregnancy biochemical predictors of gestational diabetes mellitus</article-title><source>Curr Diab Rep</source><year>2017</year><volume>17</volume><fpage>12</fpage><?supplied-pmid 28229385?><pub-id pub-id-type="pmid">28229385</pub-id>
</element-citation><mixed-citation id="mc-CR10" publication-type="journal">Powe CE. Early pregnancy biochemical predictors of gestational diabetes mellitus. Curr Diab Rep. 2017;17:12.<pub-id pub-id-type="pmid">28229385</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR11"><label>11.</label><citation-alternatives><element-citation id="ec-CR11" publication-type="journal"><person-group person-group-type="author"><name><surname>Sweeting</surname><given-names>AN</given-names></name><name><surname>Wong</surname><given-names>J</given-names></name><name><surname>Appelblom</surname><given-names>H</given-names></name><name><surname>Ross</surname><given-names>GP</given-names></name><name><surname>Kouru</surname><given-names>H</given-names></name><name><surname>Williams</surname><given-names>PF</given-names></name><etal/></person-group><article-title>A novel early pregnancy risk prediction model for gestational diabetes mellitus</article-title><source>Fetal Diagn Ther</source><year>2019</year><volume>45</volume><fpage>76</fpage><lpage>84</lpage><?supplied-pmid 29898442?><pub-id pub-id-type="pmid">29898442</pub-id>
</element-citation><mixed-citation id="mc-CR11" publication-type="journal">Sweeting AN, Wong J, Appelblom H, Ross GP, Kouru H, Williams PF, et al. A novel early pregnancy risk prediction model for gestational diabetes mellitus. Fetal Diagn Ther. 2019;45:76&#x02013;84.<pub-id pub-id-type="pmid">29898442</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR12"><label>12.</label><citation-alternatives><element-citation id="ec-CR12" publication-type="journal"><person-group person-group-type="author"><name><surname>Tenenbaum-Gavish</surname><given-names>K</given-names></name><name><surname>Sharabi-Nov</surname><given-names>A</given-names></name><name><surname>Binyamin</surname><given-names>D</given-names></name><name><surname>M&#x000f8;ller</surname><given-names>HJ</given-names></name><name><surname>Danon</surname><given-names>D</given-names></name><name><surname>Rothman</surname><given-names>L</given-names></name><etal/></person-group><article-title>First trimester biomarkers for prediction of gestational diabetes mellitus</article-title><source>Placenta</source><year>2020</year><volume>101</volume><fpage>80</fpage><lpage>89</lpage><?supplied-pmid 32937245?><pub-id pub-id-type="pmid">32937245</pub-id>
</element-citation><mixed-citation id="mc-CR12" publication-type="journal">Tenenbaum-Gavish K, Sharabi-Nov A, Binyamin D, M&#x000f8;ller HJ, Danon D, Rothman L, et al. First trimester biomarkers for prediction of gestational diabetes mellitus. Placenta. 2020;101:80&#x02013;9.<pub-id pub-id-type="pmid">32937245</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR13"><label>13.</label><citation-alternatives><element-citation id="ec-CR13" publication-type="journal"><person-group person-group-type="author"><name><surname>Savvidou</surname><given-names>M</given-names></name><name><surname>Nelson</surname><given-names>SM</given-names></name><name><surname>Makgoba</surname><given-names>M</given-names></name><name><surname>Messow</surname><given-names>C-M</given-names></name><name><surname>Sattar</surname><given-names>N</given-names></name><name><surname>Nicolaides</surname><given-names>K</given-names></name></person-group><article-title>First-trimester prediction of gestational diabetes mellitus: examining the potential of combining maternal characteristics and laboratory measures</article-title><source>Diabetes</source><year>2010</year><volume>59</volume><fpage>3017</fpage><lpage>3022</lpage><?supplied-pmid 20876721?><pub-id pub-id-type="pmid">20876721</pub-id>
</element-citation><mixed-citation id="mc-CR13" publication-type="journal">Savvidou M, Nelson SM, Makgoba M, Messow C-M, Sattar N, Nicolaides K. First-trimester prediction of gestational diabetes mellitus: examining the potential of combining maternal characteristics and laboratory measures. Diabetes. 2010;59:3017&#x02013;22.<pub-id pub-id-type="pmid">20876721</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR14"><label>14.</label><citation-alternatives><element-citation id="ec-CR14" publication-type="journal"><person-group person-group-type="author"><name><surname>Abell</surname><given-names>SK</given-names></name><name><surname>Shorakae</surname><given-names>S</given-names></name><name><surname>Boyle</surname><given-names>JA</given-names></name><name><surname>De Courten</surname><given-names>B</given-names></name><name><surname>Stepto</surname><given-names>NK</given-names></name><name><surname>Teede</surname><given-names>HJ</given-names></name><etal/></person-group><article-title>Role of serum biomarkers to optimise a validated clinical risk prediction tool for gestational diabetes</article-title><source>Aust N Z J Obstet Gynaecol</source><year>2019</year><volume>59</volume><fpage>251</fpage><lpage>257</lpage><?supplied-pmid 29900538?><pub-id pub-id-type="pmid">29900538</pub-id>
</element-citation><mixed-citation id="mc-CR14" publication-type="journal">Abell SK, Shorakae S, Boyle JA, De Courten B, Stepto NK, Teede HJ, et al. Role of serum biomarkers to optimise a validated clinical risk prediction tool for gestational diabetes. Aust N Z J Obstet Gynaecol. 2019;59:251&#x02013;7.<pub-id pub-id-type="pmid">29900538</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR15"><label>15.</label><citation-alternatives><element-citation id="ec-CR15" publication-type="journal"><person-group person-group-type="author"><name><surname>Nanda</surname><given-names>S</given-names></name><name><surname>Savvidou</surname><given-names>M</given-names></name><name><surname>Syngelaki</surname><given-names>A</given-names></name><name><surname>Akolekar</surname><given-names>R</given-names></name><name><surname>Nicolaides</surname><given-names>KH</given-names></name></person-group><article-title>Prediction of gestational diabetes mellitus by maternal factors and biomarkers at 11 to 13 weeks</article-title><source>Prenat Diagn</source><year>2011</year><volume>31</volume><fpage>135</fpage><lpage>141</lpage><?supplied-pmid 21268030?><pub-id pub-id-type="pmid">21268030</pub-id>
</element-citation><mixed-citation id="mc-CR15" publication-type="journal">Nanda S, Savvidou M, Syngelaki A, Akolekar R, Nicolaides KH. Prediction of gestational diabetes mellitus by maternal factors and biomarkers at 11 to 13 weeks. Prenat Diagn. 2011;31:135&#x02013;41.<pub-id pub-id-type="pmid">21268030</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR16"><label>16.</label><citation-alternatives><element-citation id="ec-CR16" publication-type="journal"><person-group person-group-type="author"><name><surname>Niu</surname><given-names>ZR</given-names></name><name><surname>Bai</surname><given-names>LW</given-names></name><name><surname>Lu</surname><given-names>Q</given-names></name></person-group><article-title>Establishment of gestational diabetes risk prediction model and clinical verification</article-title><source>J Endocrinol Invest</source><year>2024</year><volume>47</volume><fpage>1281</fpage><lpage>1287</lpage><?supplied-pmid 38085430?><pub-id pub-id-type="pmid">38085430</pub-id>
</element-citation><mixed-citation id="mc-CR16" publication-type="journal">Niu ZR, Bai LW, Lu Q. Establishment of gestational diabetes risk prediction model and clinical verification. J Endocrinol Invest. 2024;47:1281&#x02013;7.<pub-id pub-id-type="pmid">38085430</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR17"><label>17.</label><citation-alternatives><element-citation id="ec-CR17" publication-type="journal"><person-group person-group-type="author"><name><surname>Nevalainen</surname><given-names>J</given-names></name><name><surname>Sairanen</surname><given-names>M</given-names></name><name><surname>Appelblom</surname><given-names>H</given-names></name><name><surname>Gissler</surname><given-names>M</given-names></name><name><surname>Timonen</surname><given-names>S</given-names></name><name><surname>Ryyn&#x000e4;nen</surname><given-names>M</given-names></name></person-group><article-title>First-trimester maternal serum amino acids and acylcarnitines are significant predictors of gestational diabetes</article-title><source>Rev Diabet Stud</source><year>2016</year><volume>13</volume><fpage>236</fpage><lpage>245</lpage><?supplied-pmid 28278310?><pub-id pub-id-type="pmid">28278310</pub-id>
</element-citation><mixed-citation id="mc-CR17" publication-type="journal">Nevalainen J, Sairanen M, Appelblom H, Gissler M, Timonen S, Ryyn&#x000e4;nen M. First-trimester maternal serum amino acids and acylcarnitines are significant predictors of gestational diabetes. Rev Diabet Stud. 2016;13:236&#x02013;45.<pub-id pub-id-type="pmid">28278310</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR18"><label>18.</label><citation-alternatives><element-citation id="ec-CR18" publication-type="journal"><person-group person-group-type="author"><name><surname>Jiang</surname><given-names>R</given-names></name><name><surname>Wu</surname><given-names>S</given-names></name><name><surname>Fang</surname><given-names>C</given-names></name><name><surname>Wang</surname><given-names>C</given-names></name><name><surname>Yang</surname><given-names>Y</given-names></name><name><surname>Liu</surname><given-names>C</given-names></name><etal/></person-group><article-title>Amino acids levels in early pregnancy predict subsequent gestational diabetes</article-title><source>J Diabetes</source><year>2020</year><volume>12</volume><fpage>503</fpage><lpage>511</lpage><?supplied-pmid 31883199?><pub-id pub-id-type="pmid">31883199</pub-id>
</element-citation><mixed-citation id="mc-CR18" publication-type="journal">Jiang R, Wu S, Fang C, Wang C, Yang Y, Liu C, et al. Amino acids levels in early pregnancy predict subsequent gestational diabetes. J Diabetes. 2020;12:503&#x02013;11.<pub-id pub-id-type="pmid">31883199</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR19"><label>19.</label><citation-alternatives><element-citation id="ec-CR19" publication-type="journal"><person-group person-group-type="author"><name><surname>Zhu</surname><given-names>Y</given-names></name><name><surname>Barupal</surname><given-names>DK</given-names></name><name><surname>Ngo</surname><given-names>AL</given-names></name><name><surname>Quesenberry</surname><given-names>CP</given-names></name><name><surname>Feng</surname><given-names>J</given-names></name><name><surname>Fiehn</surname><given-names>O</given-names></name><etal/></person-group><article-title>Predictive metabolomic markers in early to mid-pregnancy for gestational diabetes mellitus: a prospective test and validation study</article-title><source>Diabetes</source><year>2022</year><volume>71</volume><fpage>1807</fpage><lpage>1817</lpage><?supplied-pmid 35532743?><pub-id pub-id-type="pmid">35532743</pub-id>
</element-citation><mixed-citation id="mc-CR19" publication-type="journal">Zhu Y, Barupal DK, Ngo AL, Quesenberry CP, Feng J, Fiehn O, et al. Predictive metabolomic markers in early to mid-pregnancy for gestational diabetes mellitus: a prospective test and validation study. Diabetes. 2022;71:1807&#x02013;17.<pub-id pub-id-type="pmid">35532743</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR20"><label>20.</label><citation-alternatives><element-citation id="ec-CR20" publication-type="journal"><person-group person-group-type="author"><name><surname>Liu</surname><given-names>J</given-names></name><name><surname>Li</surname><given-names>J</given-names></name><name><surname>Li</surname><given-names>W</given-names></name><name><surname>Li</surname><given-names>N</given-names></name><name><surname>Huo</surname><given-names>X</given-names></name><name><surname>Wang</surname><given-names>H</given-names></name><etal/></person-group><article-title>Predictive values of serum metabolites in early pregnancy and their possible pathways for gestational diabetes: a nested case-control study in Tianjin, China</article-title><source>J Diabetes Complications</source><year>2021</year><volume>35</volume><fpage>108048</fpage><?supplied-pmid 34563440?><pub-id pub-id-type="pmid">34563440</pub-id>
</element-citation><mixed-citation id="mc-CR20" publication-type="journal">Liu J, Li J, Li W, Li N, Huo X, Wang H, et al. Predictive values of serum metabolites in early pregnancy and their possible pathways for gestational diabetes: a nested case-control study in Tianjin, China. J Diabetes Complications. 2021;35:108048.<pub-id pub-id-type="pmid">34563440</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR21"><label>21.</label><citation-alternatives><element-citation id="ec-CR21" publication-type="journal"><person-group person-group-type="author"><name><surname>Grewal</surname><given-names>J</given-names></name><name><surname>Grantz</surname><given-names>KL</given-names></name><name><surname>Zhang</surname><given-names>C</given-names></name><name><surname>Sciscione</surname><given-names>A</given-names></name><name><surname>Wing</surname><given-names>DA</given-names></name><name><surname>Grobman</surname><given-names>WA</given-names></name><etal/></person-group><article-title>Cohort profile: NICHD Fetal Growth Studies-Singletons and twins</article-title><source>Int J Epidemiol</source><year>2018</year><volume>47</volume><fpage>25</fpage><lpage>25l</lpage><?supplied-pmid 29025016?><pub-id pub-id-type="pmid">29025016</pub-id>
</element-citation><mixed-citation id="mc-CR21" publication-type="journal">Grewal J, Grantz KL, Zhang C, Sciscione A, Wing DA, Grobman WA, et al. Cohort profile: NICHD Fetal Growth Studies-Singletons and twins. Int J Epidemiol. 2018;47:25&#x02013;25l.<pub-id pub-id-type="pmid">29025016</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR22"><label>22.</label><mixed-citation publication-type="other">American College of Obstetricians and Gynecologists Committee on Practice Bulletins-Obstetrics. Practice Bulletin No. 137: gestational diabetes mellitus. Clinical management guidelines for obstetrician-gynecologists. Obstet Gynecol. 2013;122:406&#x02013;16.</mixed-citation></ref><ref id="CR23"><label>23.</label><citation-alternatives><element-citation id="ec-CR23" publication-type="journal"><person-group person-group-type="author"><name><surname>Francis</surname><given-names>EC</given-names></name><name><surname>Li</surname><given-names>M</given-names></name><name><surname>Hinkle</surname><given-names>SN</given-names></name><name><surname>Cao</surname><given-names>Y</given-names></name><name><surname>Chen</surname><given-names>J</given-names></name><name><surname>Wu</surname><given-names>J</given-names></name><etal/></person-group><article-title>Adipokines in early and mid-pregnancy and subsequent risk of gestational diabetes: a longitudinal study in a multiracial cohort</article-title><source>BMJ Open Diabetes Res Care</source><year>2020</year><volume>8</volume><fpage>e001333</fpage><?supplied-pmid 32747382?><pub-id pub-id-type="pmid">32747382</pub-id>
</element-citation><mixed-citation id="mc-CR23" publication-type="journal">Francis EC, Li M, Hinkle SN, Cao Y, Chen J, Wu J, et al. Adipokines in early and mid-pregnancy and subsequent risk of gestational diabetes: a longitudinal study in a multiracial cohort. BMJ Open Diabetes Res Care. 2020;8:e001333.<pub-id pub-id-type="pmid">32747382</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR24"><label>24.</label><citation-alternatives><element-citation id="ec-CR24" publication-type="journal"><person-group person-group-type="author"><name><surname>Zhu</surname><given-names>Y</given-names></name><name><surname>Mendola</surname><given-names>P</given-names></name><name><surname>Albert</surname><given-names>PS</given-names></name><name><surname>Bao</surname><given-names>W</given-names></name><name><surname>Hinkle</surname><given-names>SN</given-names></name><name><surname>Tsai</surname><given-names>MY</given-names></name><etal/></person-group><article-title>Insulin-like growth factor axis and gestational diabetes mellitus: a longitudinal study in a multiracial cohort</article-title><source>Diabetes</source><year>2016</year><volume>65</volume><fpage>3495</fpage><lpage>3504</lpage><?supplied-pmid 27468747?><pub-id pub-id-type="pmid">27468747</pub-id>
</element-citation><mixed-citation id="mc-CR24" publication-type="journal">Zhu Y, Mendola P, Albert PS, Bao W, Hinkle SN, Tsai MY, et al. Insulin-like growth factor axis and gestational diabetes mellitus: a longitudinal study in a multiracial cohort. Diabetes. 2016;65:3495&#x02013;504.<pub-id pub-id-type="pmid">27468747</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR25"><label>25.</label><citation-alternatives><element-citation id="ec-CR25" publication-type="journal"><person-group person-group-type="author"><name><surname>Hinkle</surname><given-names>SN</given-names></name><name><surname>Tsai</surname><given-names>MY</given-names></name><name><surname>Rawal</surname><given-names>S</given-names></name><name><surname>Albert</surname><given-names>PS</given-names></name><name><surname>Zhang</surname><given-names>C</given-names></name></person-group><article-title>HbA(1c) measured in the first trimester of pregnancy and the association with gestational diabetes</article-title><source>Sci Rep</source><year>2018</year><volume>8</volume><fpage>12249</fpage><?supplied-pmid 30116010?><pub-id pub-id-type="pmid">30116010</pub-id>
</element-citation><mixed-citation id="mc-CR25" publication-type="journal">Hinkle SN, Tsai MY, Rawal S, Albert PS, Zhang C. HbA(1c) measured in the first trimester of pregnancy and the association with gestational diabetes. Sci Rep. 2018;8:12249.<pub-id pub-id-type="pmid">30116010</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR26"><label>26.</label><citation-alternatives><element-citation id="ec-CR26" publication-type="journal"><person-group person-group-type="author"><name><surname>Zhu</surname><given-names>Y</given-names></name><name><surname>Li</surname><given-names>M</given-names></name><name><surname>Rahman</surname><given-names>ML</given-names></name><name><surname>Hinkle</surname><given-names>SN</given-names></name><name><surname>Wu</surname><given-names>J</given-names></name><name><surname>Weir</surname><given-names>NL</given-names></name><etal/></person-group><article-title>Plasma phospholipid n-3 and n-6 polyunsaturated fatty acids in relation to cardiometabolic markers and gestational diabetes: a longitudinal study within the prospective NICHD fetal growth studies</article-title><source>PLoS Med</source><year>2019</year><volume>16</volume><fpage>e1002910</fpage><?supplied-pmid 31518348?><pub-id pub-id-type="pmid">31518348</pub-id>
</element-citation><mixed-citation id="mc-CR26" publication-type="journal">Zhu Y, Li M, Rahman ML, Hinkle SN, Wu J, Weir NL, et al. Plasma phospholipid n-3 and n-6 polyunsaturated fatty acids in relation to cardiometabolic markers and gestational diabetes: a longitudinal study within the prospective NICHD Fetal Growth Studies. PLoS Med. 2019;16:e1002910.<pub-id pub-id-type="pmid">31518348</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR27"><label>27.</label><citation-alternatives><element-citation id="ec-CR27" publication-type="journal"><person-group person-group-type="author"><name><surname>Bao</surname><given-names>W</given-names></name><name><surname>Dar</surname><given-names>S</given-names></name><name><surname>Zhu</surname><given-names>Y</given-names></name><name><surname>Wu</surname><given-names>J</given-names></name><name><surname>Rawal</surname><given-names>S</given-names></name><name><surname>Li</surname><given-names>S</given-names></name><etal/></person-group><article-title>Plasma concentrations of lipids during pregnancy and the risk of gestational diabetes mellitus: a longitudinal study</article-title><source>J Diabetes</source><year>2018</year><volume>10</volume><fpage>487</fpage><lpage>495</lpage><?supplied-pmid 28436169?><pub-id pub-id-type="pmid">28436169</pub-id>
</element-citation><mixed-citation id="mc-CR27" publication-type="journal">Bao W, Dar S, Zhu Y, Wu J, Rawal S, Li S, et al. Plasma concentrations of lipids during pregnancy and the risk of gestational diabetes mellitus: a longitudinal study. J Diabetes. 2018;10:487&#x02013;95.<pub-id pub-id-type="pmid">28436169</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR28"><label>28.</label><citation-alternatives><element-citation id="ec-CR28" publication-type="journal"><person-group person-group-type="author"><name><surname>Rawal</surname><given-names>S</given-names></name><name><surname>Tsai</surname><given-names>MY</given-names></name><name><surname>Hinkle</surname><given-names>SN</given-names></name><name><surname>Zhu</surname><given-names>Y</given-names></name><name><surname>Bao</surname><given-names>W</given-names></name><name><surname>Lin</surname><given-names>Y</given-names></name><etal/></person-group><article-title>A longitudinal study of thyroid markers across pregnancy and the risk of gestational diabetes</article-title><source>J Clin Endocrinol Metab</source><year>2018</year><volume>103</volume><fpage>2447</fpage><lpage>2456</lpage><?supplied-pmid 29889229?><pub-id pub-id-type="pmid">29889229</pub-id>
</element-citation><mixed-citation id="mc-CR28" publication-type="journal">Rawal S, Tsai MY, Hinkle SN, Zhu Y, Bao W, Lin Y, et al. A longitudinal study of thyroid markers across pregnancy and the risk of gestational diabetes. J Clin Endocrinol Metab. 2018;103:2447&#x02013;56.<pub-id pub-id-type="pmid">29889229</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR29"><label>29.</label><citation-alternatives><element-citation id="ec-CR29" publication-type="journal"><person-group person-group-type="author"><name><surname>Rawal</surname><given-names>S</given-names></name><name><surname>Hinkle</surname><given-names>SN</given-names></name><name><surname>Bao</surname><given-names>W</given-names></name><name><surname>Zhu</surname><given-names>Y</given-names></name><name><surname>Grewal</surname><given-names>J</given-names></name><name><surname>Albert</surname><given-names>PS</given-names></name><etal/></person-group><article-title>A longitudinal study of iron status during pregnancy and the risk of gestational diabetes: findings from a prospective, multiracial cohort</article-title><source>Diabetologia</source><year>2017</year><volume>60</volume><fpage>249</fpage><lpage>257</lpage><?supplied-pmid 27830277?><pub-id pub-id-type="pmid">27830277</pub-id>
</element-citation><mixed-citation id="mc-CR29" publication-type="journal">Rawal S, Hinkle SN, Bao W, Zhu Y, Grewal J, Albert PS, et al. A longitudinal study of iron status during pregnancy and the risk of gestational diabetes: findings from a prospective, multiracial cohort. Diabetologia. 2017;60:249&#x02013;57.<pub-id pub-id-type="pmid">27830277</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR30"><label>30.</label><citation-alternatives><element-citation id="ec-CR30" publication-type="journal"><person-group person-group-type="author"><name><surname>Yang</surname><given-names>J</given-names></name><name><surname>Wu</surname><given-names>J</given-names></name><name><surname>Tekola-Ayele</surname><given-names>F</given-names></name><name><surname>Li</surname><given-names>L-J</given-names></name><name><surname>Bremer</surname><given-names>AA</given-names></name><name><surname>Lu</surname><given-names>R</given-names></name><etal/></person-group><article-title>Plasma amino acids in early pregnancy and midpregnancy and their interplay with phospholipid fatty acids in association with the risk of gestational diabetes mellitus: results from a longitudinal prospective cohort</article-title><source>Diabetes Care</source><year>2023</year><volume>46</volume><fpage>722</fpage><lpage>732</lpage><?supplied-pmid 36701229?><pub-id pub-id-type="pmid">36701229</pub-id>
</element-citation><mixed-citation id="mc-CR30" publication-type="journal">Yang J, Wu J, Tekola-Ayele F, Li L-J, Bremer AA, Lu R, et al. Plasma amino acids in early pregnancy and midpregnancy and their interplay with phospholipid fatty acids in association with the risk of gestational diabetes mellitus: results from a longitudinal prospective cohort. Diabetes Care. 2023;46:722&#x02013;32.<pub-id pub-id-type="pmid">36701229</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR31"><label>31.</label><citation-alternatives><element-citation id="ec-CR31" publication-type="journal"><person-group person-group-type="author"><name><surname>Zhu</surname><given-names>Y</given-names></name><name><surname>Tsai</surname><given-names>MY</given-names></name><name><surname>Sun</surname><given-names>Q</given-names></name><name><surname>Hinkle</surname><given-names>SN</given-names></name><name><surname>Rawal</surname><given-names>S</given-names></name><name><surname>Mendola</surname><given-names>P</given-names></name><etal/></person-group><article-title>A prospective and longitudinal study of plasma phospholipid saturated fatty acid profile in relation to cardiometabolic biomarkers and the risk of gestational diabetes</article-title><source>Am J Clin Nutr</source><year>2018</year><volume>107</volume><fpage>1017</fpage><lpage>1026</lpage><?supplied-pmid 29868913?><pub-id pub-id-type="pmid">29868913</pub-id>
</element-citation><mixed-citation id="mc-CR31" publication-type="journal">Zhu Y, Tsai MY, Sun Q, Hinkle SN, Rawal S, Mendola P, et al. A prospective and longitudinal study of plasma phospholipid saturated fatty acid profile in relation to cardiometabolic biomarkers and the risk of gestational diabetes. Am J Clin Nutr. 2018;107:1017&#x02013;26.<pub-id pub-id-type="pmid">29868913</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR32"><label>32.</label><citation-alternatives><element-citation id="ec-CR32" publication-type="journal"><person-group person-group-type="author"><name><surname>Tsoi</surname><given-names>KY</given-names></name><name><surname>Zhu</surname><given-names>Y</given-names></name><name><surname>Wu</surname><given-names>J</given-names></name><name><surname>Sun</surname><given-names>Q</given-names></name><name><surname>Hinkle</surname><given-names>SN</given-names></name><name><surname>Li</surname><given-names>LJ</given-names></name><etal/></person-group><article-title>Plasma phospholipid monounsaturated fatty acids and gestational diabetes mellitus: a longitudinal study in the NICHD Fetal Growth Studies-Singletons Cohort</article-title><source>Diabetes</source><year>2022</year><volume>71</volume><fpage>2707</fpage><lpage>2715</lpage><?supplied-pmid 36095049?><pub-id pub-id-type="pmid">36095049</pub-id>
</element-citation><mixed-citation id="mc-CR32" publication-type="journal">Tsoi KY, Zhu Y, Wu J, Sun Q, Hinkle SN, Li LJ, et al. Plasma phospholipid monounsaturated fatty acids and gestational diabetes mellitus: a longitudinal study in the NICHD Fetal Growth Studies-Singletons Cohort. Diabetes. 2022;71:2707&#x02013;15.<pub-id pub-id-type="pmid">36095049</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR33"><label>33.</label><citation-alternatives><element-citation id="ec-CR33" publication-type="journal"><person-group person-group-type="author"><name><surname>Lamain-de Ruiter</surname><given-names>M</given-names></name><name><surname>Kwee</surname><given-names>A</given-names></name><name><surname>Naaktgeboren</surname><given-names>CA</given-names></name><name><surname>de Groot</surname><given-names>I</given-names></name><name><surname>Evers</surname><given-names>IM</given-names></name><name><surname>Groenendaal</surname><given-names>F</given-names></name><etal/></person-group><article-title>External validation of prognostic models to predict risk of gestational diabetes mellitus in one Dutch cohort: prospective multicentre cohort study</article-title><source>BMJ</source><year>2016</year><volume>354</volume><fpage>i4338</fpage><?supplied-pmid 27576867?><pub-id pub-id-type="pmid">27576867</pub-id>
</element-citation><mixed-citation id="mc-CR33" publication-type="journal">Lamain-de Ruiter M, Kwee A, Naaktgeboren CA, de Groot I, Evers IM, Groenendaal F, et al. External validation of prognostic models to predict risk of gestational diabetes mellitus in one Dutch cohort: prospective multicentre cohort study. BMJ. 2016;354:i4338.<pub-id pub-id-type="pmid">27576867</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR34"><label>34.</label><citation-alternatives><element-citation id="ec-CR34" publication-type="journal"><person-group person-group-type="author"><name><surname>Cao</surname><given-names>Y</given-names></name><name><surname>Haneuse</surname><given-names>S</given-names></name><name><surname>Zheng</surname><given-names>Y</given-names></name><name><surname>Chen</surname><given-names>J</given-names></name></person-group><article-title>Two-phase stratified sampling and analysis for predicting binary outcomes</article-title><source>Biostatistics</source><year>2023</year><volume>24</volume><fpage>585</fpage><lpage>602</lpage><?supplied-pmid 34923588?><pub-id pub-id-type="pmid">34923588</pub-id>
</element-citation><mixed-citation id="mc-CR34" publication-type="journal">Cao Y, Haneuse S, Zheng Y, Chen J. Two-phase stratified sampling and analysis for predicting binary outcomes. Biostatistics. 2023;24:585&#x02013;602.<pub-id pub-id-type="pmid">34923588</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR35"><label>35.</label><citation-alternatives><element-citation id="ec-CR35" publication-type="journal"><person-group person-group-type="author"><name><surname>Scott</surname><given-names>AJ</given-names></name><name><surname>Wild</surname><given-names>CJ</given-names></name></person-group><article-title>Fitting regression models to case-control data by maximum likelihood</article-title><source>Biometrika</source><year>1997</year><volume>84</volume><fpage>57</fpage><lpage>71</lpage></element-citation><mixed-citation id="mc-CR35" publication-type="journal">Scott AJ, Wild CJ. Fitting regression models to case-control data by maximum likelihood. Biometrika. 1997;84:57&#x02013;71.</mixed-citation></citation-alternatives></ref><ref id="CR36"><label>36.</label><citation-alternatives><element-citation id="ec-CR36" publication-type="journal"><person-group person-group-type="author"><name><surname>Breslow</surname><given-names>NE</given-names></name><name><surname>Holubkov</surname><given-names>R</given-names></name></person-group><article-title>Maximum likelihood estimation of logistic regression parameters under two-phase, outcome-dependent sampling</article-title><source>J R Stat Soc Series B Stat Methodol</source><year>1997</year><volume>59</volume><fpage>447</fpage><lpage>461</lpage></element-citation><mixed-citation id="mc-CR36" publication-type="journal">Breslow NE, Holubkov R. Maximum likelihood estimation of logistic regression parameters under two-phase, outcome-dependent sampling. J R Stat Soc Series B Stat Methodol. 1997;59:447&#x02013;61.</mixed-citation></citation-alternatives></ref><ref id="CR37"><label>37.</label><citation-alternatives><element-citation id="ec-CR37" publication-type="journal"><person-group person-group-type="author"><name><surname>Chowdhury</surname><given-names>MZI</given-names></name><name><surname>Turin</surname><given-names>TC</given-names></name></person-group><article-title>Variable selection strategies and its importance in clinical prediction modelling</article-title><source>Fam Med Community Health</source><year>2020</year><volume>8</volume><fpage>e000262</fpage><?supplied-pmid 32148735?><pub-id pub-id-type="pmid">32148735</pub-id>
</element-citation><mixed-citation id="mc-CR37" publication-type="journal">Chowdhury MZI, Turin TC. Variable selection strategies and its importance in clinical prediction modelling. Fam Med Community Health. 2020;8:e000262.<pub-id pub-id-type="pmid">32148735</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR38"><label>38.</label><citation-alternatives><element-citation id="ec-CR38" publication-type="journal"><person-group person-group-type="author"><name><surname>Pfeiffer</surname><given-names>RM</given-names></name><name><surname>Gail</surname><given-names>MH</given-names></name></person-group><article-title>Two criteria for evaluating risk prediction models</article-title><source>Biometrics</source><year>2011</year><volume>67</volume><fpage>1057</fpage><lpage>1065</lpage><?supplied-pmid 21155746?><pub-id pub-id-type="pmid">21155746</pub-id>
</element-citation><mixed-citation id="mc-CR38" publication-type="journal">Pfeiffer RM, Gail MH. Two criteria for evaluating risk prediction models. Biometrics. 2011;67:1057&#x02013;65.<pub-id pub-id-type="pmid">21155746</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR39"><label>39.</label><citation-alternatives><element-citation id="ec-CR39" publication-type="journal"><person-group person-group-type="author"><name><surname>Pfeiffer</surname><given-names>RM</given-names></name></person-group><article-title>Extensions of criteria for evaluating risk prediction models for public health applications</article-title><source>Biostatistics</source><year>2013</year><volume>14</volume><fpage>366</fpage><lpage>381</lpage><?supplied-pmid 23087412?><pub-id pub-id-type="pmid">23087412</pub-id>
</element-citation><mixed-citation id="mc-CR39" publication-type="journal">Pfeiffer RM. Extensions of criteria for evaluating risk prediction models for public health applications. Biostatistics. 2013;14:366&#x02013;81.<pub-id pub-id-type="pmid">23087412</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR40"><label>40.</label><citation-alternatives><element-citation id="ec-CR40" publication-type="book"><person-group person-group-type="author"><name><surname>Harrell</surname><given-names>FE</given-names></name></person-group><source>Regression modeling strategies: with applications to linear models, logistic regression, and survival analysis</source><year>2001</year><publisher-loc>New York</publisher-loc><publisher-name>Springer</publisher-name></element-citation><mixed-citation id="mc-CR40" publication-type="book">Harrell FE. Regression modeling strategies: with applications to linear models, logistic regression, and survival analysis. New York: Springer; 2001.</mixed-citation></citation-alternatives></ref><ref id="CR41"><label>41.</label><citation-alternatives><element-citation id="ec-CR41" publication-type="book"><person-group person-group-type="author"><name><surname>Efron</surname><given-names>B</given-names></name><name><surname>Tibshirani</surname><given-names>R</given-names></name></person-group><source>An introduction to the bootstrap</source><year>1994</year><publisher-loc>New York</publisher-loc><publisher-name>CRC Press</publisher-name></element-citation><mixed-citation id="mc-CR41" publication-type="book">Efron B, Tibshirani R. An introduction to the bootstrap. New York: CRC Press; 1994.</mixed-citation></citation-alternatives></ref><ref id="CR42"><label>42.</label><citation-alternatives><element-citation id="ec-CR42" publication-type="journal"><person-group person-group-type="author"><name><surname>Vickers</surname><given-names>AJ</given-names></name><name><surname>Elkin</surname><given-names>EB</given-names></name></person-group><article-title>Decision curve analysis: a novel method for evaluating prediction models</article-title><source>Med Decis Making</source><year>2006</year><volume>26</volume><fpage>565</fpage><lpage>574</lpage><?supplied-pmid 17099194?><pub-id pub-id-type="pmid">17099194</pub-id>
</element-citation><mixed-citation id="mc-CR42" publication-type="journal">Vickers AJ, Elkin EB. Decision curve analysis: a novel method for evaluating prediction models. Med Decis Making. 2006;26:565&#x02013;74.<pub-id pub-id-type="pmid">17099194</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR43"><label>43.</label><citation-alternatives><element-citation id="ec-CR43" publication-type="journal"><person-group person-group-type="author"><name><surname>Kerr</surname><given-names>KF</given-names></name><name><surname>Brown</surname><given-names>MD</given-names></name><name><surname>Zhu</surname><given-names>K</given-names></name><name><surname>Janes</surname><given-names>H</given-names></name></person-group><article-title>Assessing the clinical impact of risk prediction models with decision curves: guidance for correct interpretation and appropriate use</article-title><source>J Clin Oncol</source><year>2016</year><volume>34</volume><fpage>2534</fpage><lpage>2540</lpage><?supplied-pmid 27247223?><pub-id pub-id-type="pmid">27247223</pub-id>
</element-citation><mixed-citation id="mc-CR43" publication-type="journal">Kerr KF, Brown MD, Zhu K, Janes H. Assessing the clinical impact of risk prediction models with decision curves: guidance for correct interpretation and appropriate use. J Clin Oncol. 2016;34:2534&#x02013;40.<pub-id pub-id-type="pmid">27247223</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR44"><label>44.</label><citation-alternatives><element-citation id="ec-CR44" publication-type="journal"><person-group person-group-type="author"><name><surname>Simmons</surname><given-names>D</given-names></name><name><surname>Immanuel</surname><given-names>J</given-names></name><name><surname>Hague</surname><given-names>WM</given-names></name><name><surname>Teede</surname><given-names>H</given-names></name><name><surname>Nolan</surname><given-names>CJ</given-names></name><name><surname>Peek</surname><given-names>MJ</given-names></name><etal/></person-group><article-title>Treatment of gestational diabetes mellitus diagnosed early in pregnancy</article-title><source>N Engl J Med</source><year>2023</year><volume>388</volume><fpage>2132</fpage><lpage>2144</lpage><?supplied-pmid 37144983?><pub-id pub-id-type="pmid">37144983</pub-id>
</element-citation><mixed-citation id="mc-CR44" publication-type="journal">Simmons D, Immanuel J, Hague WM, Teede H, Nolan CJ, Peek MJ, et al. Treatment of gestational diabetes mellitus diagnosed early in pregnancy. N Engl J Med. 2023;388:2132&#x02013;44.<pub-id pub-id-type="pmid">37144983</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR45"><label>45.</label><citation-alternatives><element-citation id="ec-CR45" publication-type="journal"><person-group person-group-type="author"><name><surname>Yang</surname><given-names>J</given-names></name><name><surname>Song</surname><given-names>Y</given-names></name><name><surname>Gaskins</surname><given-names>AJ</given-names></name><name><surname>Li</surname><given-names>L-J</given-names></name><name><surname>Huang</surname><given-names>Z</given-names></name><name><surname>Eriksson</surname><given-names>JG</given-names></name><etal/></person-group><article-title>Mediterranean diet and female reproductive health over lifespan: a systematic review and meta-analysis</article-title><source>Am J Obstet Gynecol</source><year>2023</year><volume>229</volume><fpage>617</fpage><lpage>631</lpage><?supplied-pmid 37506751?><pub-id pub-id-type="pmid">37506751</pub-id>
</element-citation><mixed-citation id="mc-CR45" publication-type="journal">Yang J, Song Y, Gaskins AJ, Li L-J, Huang Z, Eriksson JG, et al. Mediterranean diet and female reproductive health over lifespan: a systematic review and meta-analysis. Am J Obstet Gynecol. 2023;229:617&#x02013;31.<pub-id pub-id-type="pmid">37506751</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR46"><label>46.</label><citation-alternatives><element-citation id="ec-CR46" publication-type="journal"><person-group person-group-type="author"><name><surname>Law</surname><given-names>KP</given-names></name><name><surname>Han</surname><given-names>TL</given-names></name><name><surname>Mao</surname><given-names>X</given-names></name><name><surname>Zhang</surname><given-names>H</given-names></name></person-group><article-title>Tryptophan and purine metabolites are consistently upregulated in the urinary metabolome of patients diagnosed with gestational diabetes mellitus throughout pregnancy: a longitudinal metabolomics study of Chinese pregnant women part 2</article-title><source>Clin Chim Acta</source><year>2017</year><volume>468</volume><fpage>126</fpage><lpage>139</lpage><?supplied-pmid 28238935?><pub-id pub-id-type="pmid">28238935</pub-id>
</element-citation><mixed-citation id="mc-CR46" publication-type="journal">Law KP, Han TL, Mao X, Zhang H. Tryptophan and purine metabolites are consistently upregulated in the urinary metabolome of patients diagnosed with gestational diabetes mellitus throughout pregnancy: a longitudinal metabolomics study of Chinese pregnant women part 2. Clin Chim Acta. 2017;468:126&#x02013;39.<pub-id pub-id-type="pmid">28238935</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR47"><label>47.</label><citation-alternatives><element-citation id="ec-CR47" publication-type="journal"><person-group person-group-type="author"><name><surname>Law</surname><given-names>KP</given-names></name><name><surname>Zhang</surname><given-names>H</given-names></name></person-group><article-title>The pathogenesis and pathophysiology of gestational diabetes mellitus: deductions from a three-part longitudinal metabolomics study in China</article-title><source>Clin Chim Acta</source><year>2017</year><volume>468</volume><fpage>60</fpage><lpage>70</lpage><?supplied-pmid 28213010?><pub-id pub-id-type="pmid">28213010</pub-id>
</element-citation><mixed-citation id="mc-CR47" publication-type="journal">Law KP, Zhang H. The pathogenesis and pathophysiology of gestational diabetes mellitus: deductions from a three-part longitudinal metabolomics study in China. Clin Chim Acta. 2017;468:60&#x02013;70.<pub-id pub-id-type="pmid">28213010</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR48"><label>48.</label><citation-alternatives><element-citation id="ec-CR48" publication-type="journal"><person-group person-group-type="author"><name><surname>Raji</surname><given-names>OY</given-names></name><name><surname>Duffy</surname><given-names>SW</given-names></name><name><surname>Agbaje</surname><given-names>OF</given-names></name><name><surname>Baker</surname><given-names>SG</given-names></name><name><surname>Christiani</surname><given-names>DC</given-names></name><name><surname>Cassidy</surname><given-names>A</given-names></name><etal/></person-group><article-title>Predictive accuracy of the Liverpool Lung Project risk model for stratifying patients for computed tomography screening for lung cancer: a case-control and cohort validation study</article-title><source>Ann Intern Med</source><year>2012</year><volume>157</volume><fpage>242</fpage><lpage>250</lpage><?supplied-pmid 22910935?><pub-id pub-id-type="pmid">22910935</pub-id>
</element-citation><mixed-citation id="mc-CR48" publication-type="journal">Raji OY, Duffy SW, Agbaje OF, Baker SG, Christiani DC, Cassidy A, et al. Predictive accuracy of the Liverpool Lung Project risk model for stratifying patients for computed tomography screening for lung cancer: a case-control and cohort validation study. Ann Intern Med. 2012;157:242&#x02013;50.<pub-id pub-id-type="pmid">22910935</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR49"><label>49.</label><citation-alternatives><element-citation id="ec-CR49" publication-type="journal"><person-group person-group-type="author"><name><surname>Plows</surname><given-names>JF</given-names></name><name><surname>Stanley</surname><given-names>JL</given-names></name><name><surname>Baker</surname><given-names>PN</given-names></name><name><surname>Reynolds</surname><given-names>CM</given-names></name><name><surname>Vickers</surname><given-names>MH</given-names></name></person-group><article-title>The pathophysiology of gestational diabetes mellitus</article-title><source>Int J Mol Sci</source><year>2018</year><volume>19</volume><fpage>3342</fpage><?supplied-pmid 30373146?><pub-id pub-id-type="pmid">30373146</pub-id>
</element-citation><mixed-citation id="mc-CR49" publication-type="journal">Plows JF, Stanley JL, Baker PN, Reynolds CM, Vickers MH. The pathophysiology of gestational diabetes mellitus. Int J Mol Sci. 2018;19:3342.<pub-id pub-id-type="pmid">30373146</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR50"><label>50.</label><citation-alternatives><element-citation id="ec-CR50" publication-type="journal"><person-group person-group-type="author"><name><surname>Molinaro</surname><given-names>AM</given-names></name><name><surname>Simon</surname><given-names>R</given-names></name><name><surname>Pfeiffer</surname><given-names>RM</given-names></name></person-group><article-title>Prediction error estimation: a comparison of resampling methods</article-title><source>Bioinformatics</source><year>2005</year><volume>21</volume><fpage>3301</fpage><lpage>3307</lpage><?supplied-pmid 15905277?><pub-id pub-id-type="pmid">15905277</pub-id>
</element-citation><mixed-citation id="mc-CR50" publication-type="journal">Molinaro AM, Simon R, Pfeiffer RM. Prediction error estimation: a comparison of resampling methods. Bioinformatics. 2005;21:3301&#x02013;7.<pub-id pub-id-type="pmid">15905277</pub-id>
</mixed-citation></citation-alternatives></ref></ref-list></back></article>