# The causal effects of remnant cholesterol on increased risk of cardiovascular diseases in East Asians.

<!DOCTYPE article PUBLIC "-//NLM//DTD JATS (Z39.96) Journal Archiving and Interchange DTD with MathML3 v1.3 20210610//EN" "JATS-archivearticle1-3-mathml3.dtd"> 
<article xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article" dtd-version="1.3"><?properties open_access?><?DTDIdentifier.IdentifierValue -//Springer-Verlag//DTD A++ V2.4//EN?><?DTDIdentifier.IdentifierType public?><?SourceDTD.DTDName A++V2.4.dtd?><?SourceDTD.Version 2.4?><?ConverterInfo.XSLTName springer2nlmx2.xsl?><?ConverterInfo.Version 1?><processing-meta base-tagset="archiving" mathml-version="3.0" table-model="xhtml" tagset-family="jats"><restricted-by>pmc</restricted-by></processing-meta><front><journal-meta><journal-id journal-id-type="nlm-ta">BMC Med</journal-id><journal-id journal-id-type="iso-abbrev">BMC Med</journal-id><journal-title-group><journal-title>BMC Medicine</journal-title></journal-title-group><issn pub-type="epub">1741-7015</issn><publisher><publisher-name>BioMed Central</publisher-name><publisher-loc>London</publisher-loc></publisher></journal-meta>
<article-meta><article-id pub-id-type="pmcid">12382207</article-id><article-id pub-id-type="pmid">40859268</article-id>
<article-id pub-id-type="publisher-id">4329</article-id><article-id pub-id-type="doi">10.1186/s12916-025-04329-y</article-id><article-categories><subj-group subj-group-type="heading"><subject>Research</subject></subj-group></article-categories><title-group><article-title>The causal effects of remnant cholesterol on increased risk of cardiovascular diseases in East Asians</article-title></title-group><contrib-group><contrib contrib-type="author" equal-contrib="yes"><name><surname>Lin</surname><given-names>Chenhao</given-names></name><xref ref-type="aff" rid="Aff1">1</xref><xref ref-type="aff" rid="Aff2">2</xref></contrib><contrib contrib-type="author" equal-contrib="yes"><name><surname>Wen</surname><given-names>Haolong</given-names></name><xref ref-type="aff" rid="Aff1">1</xref></contrib><contrib contrib-type="author" equal-contrib="yes"><name><surname>Yu</surname><given-names>Mengyao</given-names></name><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author"><name><surname>Huang</surname><given-names>Qingxia</given-names></name><xref ref-type="aff" rid="Aff1">1</xref></contrib><contrib contrib-type="author"><name><surname>Wang</surname><given-names>Qi</given-names></name><xref ref-type="aff" rid="Aff1">1</xref></contrib><contrib contrib-type="author"><name><surname>Lin</surname><given-names>Jinran</given-names></name><xref ref-type="aff" rid="Aff1">1</xref></contrib><contrib contrib-type="author"><name><surname>Yang</surname><given-names>Wenjun</given-names></name><xref ref-type="aff" rid="Aff1">1</xref><xref ref-type="aff" rid="Aff4">4</xref></contrib><contrib contrib-type="author"><name><surname>Wong</surname><given-names>Jennifer Ming Jen</given-names></name><xref ref-type="aff" rid="Aff5">5</xref></contrib><contrib contrib-type="author"><name><surname>Xia</surname><given-names>Mingfeng</given-names></name><xref ref-type="aff" rid="Aff6">6</xref><xref ref-type="aff" rid="Aff7">7</xref></contrib><contrib contrib-type="author"><name><surname>Tang</surname><given-names>Huiru</given-names></name><xref ref-type="aff" rid="Aff1">1</xref></contrib><contrib contrib-type="author"><name><surname>Jin</surname><given-names>Li</given-names></name><xref ref-type="aff" rid="Aff1">1</xref></contrib><contrib contrib-type="author"><name><surname>Wang</surname><given-names>Sijia</given-names></name><xref ref-type="aff" rid="Aff8">8</xref></contrib><contrib contrib-type="author"><name><surname>Wang</surname><given-names>Xiaofeng</given-names></name><xref ref-type="aff" rid="Aff1">1</xref><xref ref-type="aff" rid="Aff9">9</xref></contrib><contrib contrib-type="author" corresp="yes"><name><surname>Dai</surname><given-names>Yuxiang</given-names></name><address><email>dai.yuxiang@hotmail.com</email></address><xref ref-type="aff" rid="Aff5">5</xref></contrib><contrib contrib-type="author" corresp="yes"><name><surname>Chen</surname><given-names>Guo-Chong</given-names></name><address><email>gcchen@suda.edu.cn</email></address><xref ref-type="aff" rid="Aff10">10</xref></contrib><contrib contrib-type="author" corresp="yes"><name><surname>Zheng</surname><given-names>Yan</given-names></name><address><email>yan_zheng@fudan.edu.cn</email></address><xref ref-type="aff" rid="Aff1">1</xref><xref ref-type="aff" rid="Aff5">5</xref></contrib><aff id="Aff1"><label>1</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/013q1eq08</institution-id><institution-id institution-id-type="GRID">grid.8547.e</institution-id><institution-id institution-id-type="ISNI">0000 0001 0125 2443</institution-id><institution>State Key Laboratory of Genetic Engineering, Human Phenome Institute, and School of Life Sciences, Zhongshan Hospital, </institution><institution>Fudan University, </institution></institution-wrap>Shanghai, China </aff><aff id="Aff2"><label>2</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/04kx2sy84</institution-id><institution-id institution-id-type="GRID">grid.256111.0</institution-id><institution-id institution-id-type="ISNI">0000 0004 1760 2876</institution-id><institution>College of Food Science, </institution><institution>Fujian Agriculture and Forestry University, </institution></institution-wrap>Fuzhou, China </aff><aff id="Aff3"><label>3</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/013q1eq08</institution-id><institution-id institution-id-type="GRID">grid.8547.e</institution-id><institution-id institution-id-type="ISNI">0000 0001 0125 2443</institution-id><institution>Human Phenome Institute, Shanghai Pudong Hospital, Fudan University Pudong Medical Center, Zhangjiang Fudan International Innovation Center, </institution><institution>Fudan University, </institution></institution-wrap>Shanghai, China </aff><aff id="Aff4"><label>4</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/013q1eq08</institution-id><institution-id institution-id-type="GRID">grid.8547.e</institution-id><institution-id institution-id-type="ISNI">0000 0001 0125 2443</institution-id><institution>Ministry of Education Key Laboratory of Contemporary Anthropology, </institution><institution>Fudan University, </institution></institution-wrap>Shanghai, 200433 China </aff><aff id="Aff5"><label>5</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/013q1eq08</institution-id><institution-id institution-id-type="GRID">grid.8547.e</institution-id><institution-id institution-id-type="ISNI">0000 0001 0125 2443</institution-id><institution>Department of Cardiology, Zhongshan Hospital, </institution><institution>Fudan University, Shanghai Institute of Cardiovascular Disease, </institution></institution-wrap>Shanghai, China </aff><aff id="Aff6"><label>6</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/013q1eq08</institution-id><institution-id institution-id-type="GRID">grid.8547.e</institution-id><institution-id institution-id-type="ISNI">0000 0001 0125 2443</institution-id><institution>Department of Endocrinology and Metabolism, Zhongshan Hospital and Fudan Institute for Metabolic Diseases, </institution><institution>Fudan University, </institution></institution-wrap>Shanghai, China </aff><aff id="Aff7"><label>7</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/013q1eq08</institution-id><institution-id institution-id-type="GRID">grid.8547.e</institution-id><institution-id institution-id-type="ISNI">0000 0001 0125 2443</institution-id><institution>Department of Endocrinology and Metabolism, Wusong Branch of Zhongshan Hospital, </institution><institution>Fudan University, </institution></institution-wrap>Shanghai, China </aff><aff id="Aff8"><label>8</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/034t30j35</institution-id><institution-id institution-id-type="GRID">grid.9227.e</institution-id><institution-id institution-id-type="ISNI">0000000119573309</institution-id><institution>CAS Key Laboratory of Computational Biology, Shanghai Institute of Nutrition and Health, University of Chinese Academy of Sciences, </institution><institution>Chinese Academy of Sciences, </institution></institution-wrap>Shanghai, China </aff><aff id="Aff9"><label>9</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/013q1eq08</institution-id><institution-id institution-id-type="GRID">grid.8547.e</institution-id><institution-id institution-id-type="ISNI">0000 0001 0125 2443</institution-id><institution>Fudan University-the People&#x02019;s hospital of Rugao Joint Research Institute of Longevity and Aging, </institution></institution-wrap>Rugao Jiangsu, China </aff><aff id="Aff10"><label>10</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/05t8y2r12</institution-id><institution-id institution-id-type="GRID">grid.263761.7</institution-id><institution-id institution-id-type="ISNI">0000 0001 0198 0694</institution-id><institution>Department of Nutrition and Food Hygiene, </institution><institution>Suzhou Medical College of Soochow University, </institution></institution-wrap>Suzhou, China </aff></contrib-group><pub-date pub-type="epub"><day>26</day><month>8</month><year>2025</year></pub-date><pub-date pub-type="pmc-release"><day>26</day><month>8</month><year>2025</year></pub-date><pub-date pub-type="collection"><year>2025</year></pub-date><volume>23</volume><elocation-id>495</elocation-id><history><date date-type="received"><day>17</day><month>4</month><year>2025</year></date><date date-type="accepted"><day>6</day><month>8</month><year>2025</year></date></history><permissions><copyright-statement>&#x000a9; The Author(s) 2025</copyright-statement><copyright-year>2025</copyright-year><license><ali:license_ref specific-use="textmining" content-type="ccbyncndlicense">https://creativecommons.org/licenses/by-nc-nd/4.0/</ali:license_ref><license-p><bold>Open Access</bold> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article&#x02019;s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article&#x02019;s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <ext-link ext-link-type="uri" xlink:href="https://creativecommons.org/licenses/by-nc-nd/4.0/">http://creativecommons.org/licenses/by-nc-nd/4.0/</ext-link>.</license-p></license></permissions><abstract id="Abs1"><sec><title>Background</title><p id="Par15">Remnant cholesterol (RC) has been implicated in cardiovascular diseases (CVDs) in populations of European ancestry, yet its causal role remains underexplored in populations of East Asian ancestry, which are underrepresented in genetic studies. We sought to investigate the causal association between circulating RC levels and CVD risk in East Asian populations.</p></sec><sec><title>Methods</title><p id="Par16">We first conducted observational analyses of RC and multiple CVD outcomes in Chinese populations. We then conducted genome-wide association studies (GWAS) of RC in 14,939 Chinese individuals and assessed its causal associations with CVD risk using two-sample Mendelian randomization (MR) with Biobank Japan data. Replication analyses in European ancestry populations utilized summary statistics from the UK Biobank and FinnGen.</p></sec><sec><title>Results</title><p id="Par17">Circulating RC levels were significantly associated with multiple CVD outcomes in Chinese individuals. Our GWAS identified seven significant loci associated with circulating RC levels in the Chinese population. In the MR analyses, we found that genetically predicted higher RC levels were significantly associated with increased risks of aortic aneurysm (odds ratio per standard deviation increase, 1.82; 95% CI, 1.53&#x02013;2.17; <italic>P</italic>&#x02009;=&#x02009;1.17&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;11</sup>) and ischemic heart diseases, such as myocardial infarction (1.22, 1.13&#x02013;1.32; <italic>P</italic>&#x02009;=&#x02009;1.81&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;7</sup>) and stable angina pectoris (1.17, 1.11&#x02013;1.23; <italic>P</italic>&#x02009;=&#x02009;1.18&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;8</sup>). Notably, these associations persisted after adjustment for total cholesterol, low-density and high-density lipoprotein cholesterols, Apolipoprotein A1 and Apolipoprotein B, respectively. Replication in European ancestry populations confirmed consistent causal effects for aortic aneurysm and ischemic heart diseases. A suggestive East Asian-specific association was identified between genetically predicted higher RC levels and an increased risk of peripheral artery disease, whereas a suggestive association with a higher risk of atrial flutter/fibrillation and ventricular arrhythmia was only observed in populations of European ancestry.</p></sec><sec><title>Conclusions</title><p id="Par18">Our findings establish RC as an independent causal CVD risk factor in East Asian ancestry individuals and highlight population-specific differences in CVD risk profiles, emphasizing the need for ethnicity-tailored prevention strategies.</p></sec><sec><title>Supplementary Information</title><p>The online version contains supplementary material available at 10.1186/s12916-025-04329-y.</p></sec></abstract><kwd-group xml:lang="en"><title>Keywords</title><kwd>Remnant cholesterol</kwd><kwd>Cardiovascular diseases</kwd><kwd>Genome-wide association study</kwd><kwd>Mendelian randomization</kwd></kwd-group><funding-group><award-group><funding-source><institution-wrap><institution-id institution-id-type="FundRef">https://doi.org/10.13039/501100012166</institution-id><institution>National Key Research and Development Program of China</institution></institution-wrap></funding-source><award-id>2022YFC3400700</award-id></award-group></funding-group><funding-group><award-group><funding-source><institution>Shanghai Municipal Science and Technology Major Project</institution></funding-source><award-id>2023SHZDZX02</award-id></award-group></funding-group><funding-group><award-group><funding-source><institution>Grant of Shanghai Municipal Health Commission</institution></funding-source><award-id>2022JC012</award-id></award-group></funding-group><funding-group><award-group><funding-source><institution-wrap><institution-id institution-id-type="FundRef">https://doi.org/10.13039/501100012247</institution-id><institution>Program of Shanghai Academic Research Leader</institution></institution-wrap></funding-source><award-id>22XD1423300</award-id></award-group></funding-group><funding-group><award-group><funding-source><institution>Shanghai Oriental Youth Elite Talents Program</institution></funding-source><award-id>QNZH2024009</award-id></award-group></funding-group><funding-group><award-group><funding-source><institution-wrap><institution-id institution-id-type="FundRef">https://doi.org/10.13039/501100013285</institution-id><institution>Program for Professor of Special Appointment (Eastern Scholar) at Shanghai Institutions of Higher Learning</institution></institution-wrap></funding-source></award-group></funding-group><custom-meta-group><custom-meta><meta-name>issue-copyright-statement</meta-name><meta-value>&#x000a9; BioMed Central Ltd., part of Springer Nature 2025</meta-value></custom-meta></custom-meta-group></article-meta></front><body><sec id="Sec1"><title>Background</title><p id="Par48">The cardiovascular impact of circulating lipids is complex and remains a debate[<xref ref-type="bibr" rid="CR1">1</xref>]. While elevated low-density lipoprotein cholesterol (LDL-c) is well-established as a causal cardiovascular risk factor through both genetic studies and successful LDL-c-lowering therapies [<xref ref-type="bibr" rid="CR2">2</xref>, <xref ref-type="bibr" rid="CR3">3</xref>], the protective role of high-density lipoprotein cholesterol (HDL-c) has been challenged by inconclusive results from interventional trials [<xref ref-type="bibr" rid="CR4">4</xref>, <xref ref-type="bibr" rid="CR5">5</xref>]. On the other hand, despite the widespread use of LDL-c-lowering treatments, such as statins, a significant residual cardiovascular risk persists among the treated individuals [<xref ref-type="bibr" rid="CR6">6</xref>, <xref ref-type="bibr" rid="CR7">7</xref>]. Remnant cholesterol (RC), which primarily refers to the content of triglyceride-rich lipoproteins (e.g., chylomicrons, very-low-density lipoproteins [VLDL], and intermediate-density lipoproteins [IDL]), has emerged as an independent and important contributor to the risk of cardiovascular disease (CVD) beyond LDL-c and HDL-c [<xref ref-type="bibr" rid="CR8">8</xref>, <xref ref-type="bibr" rid="CR9">9</xref>]. Emerging studies have demonstrated that higher RC levels are associated with an increased risk of CVD, even in the context of optimal LDL-c levels [<xref ref-type="bibr" rid="CR10">10</xref>&#x02013;<xref ref-type="bibr" rid="CR12">12</xref>].
</p><p id="Par49">The effects of lipids on CVD risk are further modulated by metabolic conditions, genetic predispositions, and population-specific factors, complicating the understanding of direct causality [<xref ref-type="bibr" rid="CR13">13</xref>&#x02013;<xref ref-type="bibr" rid="CR15">15</xref>]. Recently, genetically predicted RC levels have been associated with CVD risk in populations of European ancestry [<xref ref-type="bibr" rid="CR16">16</xref>&#x02013;<xref ref-type="bibr" rid="CR18">18</xref>], while evidence in population of East&#x000a0;Asian ancestry remains limited. East&#x000a0;Asian ancestry individuals exhibit distinct lipid profiles, a rising burden of metabolic disorders, and genetic differences compared to population of European ancestry, underscoring the need for targeted investigations to understand the causal role of RC in this group.
</p><p id="Par50">In this study, we performed a genome-wide association study (GWAS) of RC in Chinese populations and employed two-sample Mendelian randomization (MR) to examine the potential causal association between RC and CVD in East Asian ancestry individuals, integrating the summary statistics from the Biobank Japan [<xref ref-type="bibr" rid="CR19">19</xref>]. Furthermore, we compared the causal effects of RC on CVD between populations of East Asian ancestry and European ancestry using GWAS summary statistics from the UK Biobank and FinnGen [<xref ref-type="bibr" rid="CR20">20</xref>]. We also carried out an observational study in a Chinese population to further validate our findings. Our findings may provide novel insights into the population-specific role of RC in CVD risk, with implications for prevention and therapeutic strategies.</p></sec><sec id="Sec2"><title>Methods</title><sec id="Sec3"><title>Study participants</title><p id="Par51">The present study included 14,939 Han Chinese from four populations: the Shanghai Changfeng Study [<xref ref-type="bibr" rid="CR21">21</xref>&#x02013;<xref ref-type="bibr" rid="CR23">23</xref>], the Rugao Longevity and Aging Study [<xref ref-type="bibr" rid="CR24">24</xref>], the National Survey of Physical Traits cohort [<xref ref-type="bibr" rid="CR25">25</xref>] and the Genetic characteristics of coRonary Artery disease in ChiNese young aDults study (GRAND) [<xref ref-type="bibr" rid="CR26">26</xref>, <xref ref-type="bibr" rid="CR27">27</xref>]. Details of the participants&#x02019; characteristics and genotyping methodologies are provided in Additional file 1: Table S1. Notably, among the four Chinese populations, participants were instructed to fast overnight prior to blood sample collection the following morning. For each population, written informed consent was obtained from all participants before being enrolled, and the studies were approved by the institutional review boards of the participating institutions.</p><p id="Par52">For the European study, we included approximately 280,000 individuals from the UK Biobank with genotyping data and nuclear magnetic resonance (NMR)-based&#x000a0;metabolomic data [<xref ref-type="bibr" rid="CR28">28</xref>, <xref ref-type="bibr" rid="CR29">29</xref>]. The blood samples were collected during the participants&#x02019; visit to the assessment center, and the duration of fasting prior to blood draw was recorded. Following the central quality control procedures of the UK Biobank [<xref ref-type="bibr" rid="CR28">28</xref>], we excluded: (1) individuals who had withdrawn consent, (2) non-Caucasian participants, (3) those with sex chromosome aneuploidy, (4) those with a mismatch between genetically inferred sex and self-reported sex, and (5) outliers for heterozygosity or missing rate. Ultimately, 213,397 participants were included in the present analyses.</p><sec id="Sec4"><title>Measurement of circulating remnant cholesterol levels</title><p id="Par53">In this study, the levels of RC in each Chinese population were directly measured using the same NMR-based metabolomics platform. Briefly, blood metabolomics profiling was performed on a 600&#x000a0;MHz AVANCE III NMR spectrometer (Bruker Biospin GmbH, Germany), following previously established protocols (Additional file 2: Supplementary Method) [<xref ref-type="bibr" rid="CR22">22</xref>, <xref ref-type="bibr" rid="CR30">30</xref>, <xref ref-type="bibr" rid="CR31">31</xref>]. For European ancestry participants from the UK Biobank, metabolites in EDTA plasma samples were measured using Nightingale Health&#x02019;s high-throughput NMR-based platform [<xref ref-type="bibr" rid="CR29">29</xref>]. Notably, to account for the effects of lipid-lowering medications, we corrected LDL-c and total cholesterol (TC) levels in statin users by dividing LDL-c by 0.70 and TC by 0.80, and subsequently recalculated RC levels [<xref ref-type="bibr" rid="CR32">32</xref>]. The RC levels were then normalized using rank-based inverse normal transformation and analyzed in the observational studies and GWAS.</p></sec></sec><sec id="Sec5"><title>Observational study of remnant cholesterol and coronary artery disease severity&#x000a0;in GRAND study</title><p id="Par54">We investigated the association between RC levels and coronary artery disease (CAD) and CAD severity in the GRAND cohort. This study included 3779 participants who underwent coronary angiography with physician-confirmed CAD diagnoses and subtype classification. 2296 participants were diagnosed with CAD, among whom 2289 and 2277 underwent Gensini score assessment and circulating cardiac troponin T (cTnT) level measurement, respectively (Additional file 1: Table S2). Serum cTnT was measured using venous blood samples that were obtained at admission and by an automated analyzer using a high-sensitivity assay (Roche Diagnostics, Indianapolis, IN). Gensini score was used to quantify atherosclerotic burden by evaluating both the degree of luminal narrowing and lesion location (with higher scores indicating more severe proximal lesions) through a standardized scoring system that sums segmental scores for individual coronary stenoses. We obtained self-reported data on age, sex, height, weight, smoking status, and alcohol consumption through electronic medical records, along with information on the use of glucose-lowering medications, antihypertensive drugs, and lipid-lowering agents. The diagnosis of hypertension and type 2 diabetes was determined through electronic medical records and physician assessments. Body mass index (BMI) was computed by dividing the weight (in kilograms) by the square of height (in meters). Logistic regression models were used to evaluate the odds ratio (OR) with 95% confidence interval (CI) of RC levels and CAD and its subtype classification. Linear regression models were used to evaluate the estimate &#x003b2; (logOR) with 95% CI between RC levels and CAD severity.</p><p id="Par55">To further investigate the association between RC levels and CAD prognosis in the Chinese population, we conducted a three-year longitudinal study involving 1,990 CAD patients from the GRAND cohort (Additional file 1: Table S3). The baseline coronary atherosclerotic burden was quantified using the Gensini score. Clinical follow-ups were conducted through either clinic visits or telephone interviews to document the occurrence of major adverse cardiovascular event (MACE) during the 3-year follow-up period. MACE was defined as cardiovascular death, unplanned revascularization, non-fatal myocardial infarction, and stroke. Cox proportional hazards models were used to evaluate the hazard ratios (HRs) with 95% CIs of the baseline RC levels with the risk of MACE.</p></sec><sec id="Sec6"><title>Genome-wide association studies for remnant cholesterol</title><p id="Par56">A GWAS for RC was conducted in each Chinese population using a linear mixed model implemented in BOLT-LMM software (2.4.1) [<xref ref-type="bibr" rid="CR33">33</xref>] (Fig.&#x000a0;<xref rid="Fig1" ref-type="fig">1</xref>). Single nucleotide polymorphisms (SNPs) were excluded from the analyses if they failed to meet the following criteria: deviation from Hardy&#x02013;Weinberg equilibrium (HWE, <italic>P</italic>&#x02009;&#x0003c;&#x02009;10<sup>&#x02212;5</sup>), minor allele frequency (MAF)&#x02009;&#x0003c;&#x02009;1%, or low imputation quality (INFO&#x02009;&#x0003c;&#x02009;0.4). The analysis was adjusted for age, sex, and the top five genetic ancestry principal components. Results from each population were meta-analyzed using a fixed-effect model based on effect sizes and standard errors in METAL software [<xref ref-type="bibr" rid="CR34">34</xref>].<fig id="Fig1"><label>Fig.&#x000a0;1</label><caption><p>The analysis workflow of the study</p></caption><graphic xlink:href="12916_2025_4329_Fig1_HTML" id="MO1"/></fig></p><p id="Par57">Due to the larger sample size and more stringent imputation strategy employed in the UK Biobank (e.g., multiple imputation reference panels) [<xref ref-type="bibr" rid="CR28">28</xref>], variants with INFO&#x02009;&#x0003e;&#x02009;0.8 were retained. Other quality control thresholds, including HWE&#x02009;&#x0003e;&#x02009;10<sup>&#x02212;5</sup> and MAF&#x02009;&#x0003e;&#x02009;1%, were applied consistently across both the Chinese and UK Biobank cohorts. GWAS was performed using PLINK (2.0) software [<xref ref-type="bibr" rid="CR35">35</xref>], with adjustments of age, sex, fasting time, genotyping chips, assessment centers, metabolomic profiling batch, use of lipid-lowering medications, and the top ten genetic ancestry principal components.</p></sec><sec id="Sec7"><title>Data sources for cardiovascular diseases</title><p id="Par58">Genetic variants associated with CVDs in East Asian ancestry individuals were obtained from BioBank Japan (Additional file 1: Table S4) [<xref ref-type="bibr" rid="CR19">19</xref>]. Briefly, GWASs were conducted in over 179,000 Japanese individuals using a generalized linear mixed model in SAIGE (v.0.37), with adjustment of age, age<sup>2</sup>, sex, age&#x02009;&#x000d7;&#x02009;sex, age<sup>2</sup>&#x02009;&#x000d7;&#x02009;sex, and the top 20 principal components [<xref ref-type="bibr" rid="CR19">19</xref>]. Based on ICD-10 codes, we selected circulatory system diseases (I00-I99) with a prevalence greater than 0.5% (hum0197.v3). This resulted in 13 cardiovascular outcomes, including aortic aneurysm, peripheral arterial disease, myocardial infarction, angina pectoris, stable angina pectoris, unstable angina pectoris, cerebral aneurysm, intracerebral hemorrhage, ischemic stroke, subarachnoid hemorrhage, chronic heart failure, atrial flutter/fibrillation, and ventricular arrhythmia (Additional file 1: Table S4).</p><p id="Par59">For replication, we matched these 13 cardiovascular outcomes with 12 diseases from the latest release of the FinnGen database (DF12, Additional file 1: Table S4) [<xref ref-type="bibr" rid="CR20">20</xref>]. GWASs for these disease traits in FinnGen were performed in over 453,000 Finnish biobank participants using regenie (v.2.2.4), with adjustment of age, sex, top 10 principal components, and genotyping batch as covariates [<xref ref-type="bibr" rid="CR20">20</xref>]. Further details on the data sources are provided in Additional file 1: Table S4.</p></sec><sec id="Sec8"><title>Instrumental variables selection</title><p id="Par60">To ensure the validity of instrumental variables (IVs), three fundamental assumptions must be met: (1) the IV must be strongly associated with the exposure of interest, (2) the IV must be independent of any confounders affecting the relationship between the exposure and the outcome, and (3) the IV should affect the outcome exclusively through the exposure pathway&#x000a0;[<xref ref-type="bibr" rid="CR36">36</xref>]. In this study, SNPs significantly associated with RC at the genome-wide significance threshold (<italic>P</italic>&#x02009;&#x0003c;&#x02009;5&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;8</sup>) were selected as candidate IVs. Linkage disequilibrium (LD) was assessed between SNPs to select independent genetic variants using PLINK (1.9) software (window size&#x02009;=&#x02009;10,000&#x000a0;kb, <italic>R</italic><sup>2</sup>&#x02009;&#x0003c;&#x02009;0.1 for East Asians, with the 1000 Genome Project Phase 3 v5 East Asian population as the reference panel)&#x000a0;[<xref ref-type="bibr" rid="CR35">35</xref>]. For SNPs in LD, we selected the variant with the lowest <italic>P</italic>-value. The strength of each IV was evaluated by calculating the F statistic, and SNPs with an F statistic&#x02009;&#x0003c;&#x02009;10 were excluded [<xref ref-type="bibr" rid="CR37">37</xref>].</p></sec><sec id="Sec9"><title>Two-sample Mendelian randomization analysis in East Asian ancestry individuals</title><p id="Par61">Before each MR analysis, the selected IVs were harmonized across exposure and outcome summary statistics to ensure that the effects of SNPs corresponded to the same allele. Palindromic SNPs with intermediate allele frequencies (MAF&#x02009;&#x0003e;&#x02009;0.42) were excluded (Additional file 2: Figure S1). Importantly, there was no overlap between the samples used for the exposure and outcome GWAS.</p><p id="Par62">For the main analysis, we used the fixed-effect inverse variance weighted (IVW) method to estimate the causal effect of exposure on the outcome. This method assumes no (unbalanced) horizontal pleiotropy. Given that pleiotropic bias could potentially violate this assumption, we conducted sensitivity analyses using the weighted median method, MR-Egger method, and MR-PRESSO method to assess the robustness and reliability of the findings [<xref ref-type="bibr" rid="CR38">38</xref>]. A causal association was considered robust when all sensitivity analysis methods yielded results that were consistent with the effect direction estimated by the IVW method.</p><p id="Par63">The results are presented as OR (95% CI), reflecting the assessed CVD risk associated with one standard deviation (SD) increase in genetically predicted circulating levels of RC. Directional pleiotropy was tested using the hypothesis test for the MR-Egger regression&#x02019;s intercept, and heterogeneity was assessed via Cochran&#x02019;s <italic>Q</italic>-test. If heterogeneity was detected, a multiplicative random-effects IVW method was applied [<xref ref-type="bibr" rid="CR38">38</xref>]. Furthermore, leave-one-out analyses were performed to evaluate whether the observed effects were driven by any single SNP.</p><p id="Par64">To address the multiple testing issues, we applied a threshold of statistical significance at <italic>P</italic>&#x02009;&#x0003c;&#x02009;0.0038 (0.05/13 cardiovascular outcomes) using Bonferroni correction. Results with 0.0038&#x02009;&#x02264;&#x02009;<italic>P</italic>&#x02009;&#x0003c;&#x02009;0.05 were considered suggestive associations. All MR analyses were performed using R software (4.2.2) with the TwoSampleMR (0.5.7) and MRPRESSO (1.0) packages. This study adhered to the guidelines for strengthening the reporting of observational studies in Epidemiology using Mendelian randomization studies (STROBE-MR; Additional file 3) [<xref ref-type="bibr" rid="CR39">39</xref>, <xref ref-type="bibr" rid="CR40">40</xref>].</p></sec><sec id="Sec10"><title>Multivariable Mendelian randomization analysis</title><p id="Par65">To control for potential confounding effects from other lipid traits, including LDL-c, TC, HDL-c, Apolipoprotein A1 (ApoA1), and Apolipoprotein B (ApoB), we conducted multivariable Mendelian randomization (MVMR) to estimate the independent causal effect of RC on each CVD outcome (Additional file 2: Figure S1). Because the inclusion of highly correlated exposures can reduce statistical power, weaken instrument strength, and introduce model instability [<xref ref-type="bibr" rid="CR41">41</xref>], we designed three biologically informed models based on the distinct roles of cholesterol-carrying lipoproteins and apolipoproteins. In Model 1, we adjusted for TC alone, as it represents total circulating cholesterol. In Model 2, we decomposed TC into its components&#x02014;LDL-c, HDL-c, and RC&#x02014;to investigate whether RC is independently associated with CVD beyond other cholesterol fractions. In Model 3, we adjusted for ApoB and ApoA1, the main structural proteins of LDL-c/VLDL-c and HDL-c, respectively. Notably, ApoB has recently been reported as an independent CVD risk factor beyond LDL-c&#x000a0;[<xref ref-type="bibr" rid="CR42">42</xref>]. This modeling approach allowed us to account for the biological relevance of each lipid trait while minimizing multicollinearity (e.g., between ApoB and LDL-c), which could otherwise lead to unstable estimates.</p><p id="Par66">The IVW method was used as the primary MVMR approach. As a sensitivity analysis, the MR-Egger method was also applied. The joint instrument strength for the exposures in the MVMR analysis was evaluated using the Sanderson&#x02013;Windmeijer conditional F statistic&#x000a0;[<xref ref-type="bibr" rid="CR43">43</xref>]. IVs with a conditional F statistic below 10 were considered weak. The <italic>P</italic>&#x02009;&#x0003c;&#x02009;0.05 was interpreted as statistically significant, indicating strong evidence for a causal association. All MVMR analyses were performed using the R packages TwoSampleMR (0.5.7), MVMR (0.4), and MendelianRandomization (0.9.0).</p></sec></sec><sec id="Sec11"><title>Replication analysis in population of European ancestry</title><p id="Par67">To assess the generalizability and robustness of our results, we replicated the two-sample MR and MVMR analyses within populations of European ancestry using data from the UK Biobank and FinnGen databases (Additional file 1: Table S4). Briefly, SNPs significantly associated with RC (<italic>P</italic>&#x02009;&#x0003c;&#x02009;5&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;8</sup>) were selected as IVs for the MR analysis. For the selected SNPs, LD was examined to ensure independence (<italic>R</italic><sup>2</sup>&#x02009;&#x0003c;&#x02009;0.001 within a 10,000&#x000a0;kb window for European ancestry, justifying the larger sample size for Europeans). The remaining procedures were consistent with those described in East Asian ancestry individuals.</p><sec id="Sec12"><title>Reverse Mendelian randomization analysis</title><p id="Par68">To estimate the causal effects of genetically predicted CVD risks on the RC level, we further performed reverse MR analyses. For each CVD, we included SNPs that reached the genome-wide significance threshold (<italic>P</italic>&#x02009;&#x0003c;&#x02009;5&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;8</sup>; or <italic>P</italic>&#x02009;&#x0003c;&#x02009;1&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;5</sup> if there were not enough SNPs) as candidates for IVs. Independent SNPs with an LD <italic>r</italic><sup>2</sup>&#x02009;&#x0003c;&#x02009;0.001 and an F statistic&#x02009;&#x0003e;&#x02009;10 were retained. The specific MR methods were consistent with those described above. Statistical significance was defined using the same threshold as in the forward MR, with a Bonferroni-corrected threshold of <italic>P</italic>&#x02009;&#x0003c;&#x02009;0.0038 (0.05/13 CVDs).</p></sec></sec><sec id="Sec13"><title>Results</title><sec id="Sec14"><title>Remnant cholesterol as a biomarker of coronary artery diseases in Chinese population</title><p id="Par69">We conducted an observational study in the GRAND cohort to investigate the association between RC levels and CAD-related outcomes. Of the 3779 individuals included in GRAND, 2296 had CAD, comprising 453 with myocardial infarction, 815 with stable angina, and 967 with unstable angina (Additional file 1: Table S2). In univariable logistic regression analyses, higher RC levels were significantly associated with increased odds of all five CAD subtypes (Table&#x000a0;<xref rid="Tab1" ref-type="table">1</xref>). After multivariable adjustment, higher RC levels remained independently associated with increased odds of myocardial infarction (adjusted OR [95%CI], 1.352 [1.176, 1.556]; <italic>P</italic>&#x02009;=&#x02009;2.45&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;5</sup>), unstable angina (1.130 [1.027, 1.244], <italic>P</italic>&#x02009;=&#x02009;0.013), angina pectoris (1.115 [1.027, 1.210], <italic>P</italic>&#x02009;=&#x02009;9.10&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;3</sup>), and overall CAD (1.145 [1.059, 1.238], <italic>P</italic>&#x02009;=&#x02009;6.82&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;4</sup>), but not with stable angina (1.085 [0.980, 1.203], <italic>P</italic>&#x02009;=&#x02009;0.117; Table&#x000a0;<xref rid="Tab1" ref-type="table">1</xref>). Meanwhile, we also performed linear regression analyses to evaluate the relationship between RC levels and CAD severity (Table&#x000a0;<xref rid="Tab1" ref-type="table">1</xref>). In univariable models, higher RC levels were significantly associated with increased cTnT levels and Gensini score (Table&#x000a0;<xref rid="Tab1" ref-type="table">1</xref>), and the multivariable adjusted association remained significant with cTnT levels (<italic>&#x003b2;</italic>&#x02009;=&#x02009;0.059, [95%CI, 0.020&#x02013;0.098], <italic>P</italic>&#x02009;=&#x02009;3.29&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;3</sup>),&#x000a0;but was attenuated to be insignificant with Gensini score 0.028 [&#x02212;&#x02009;0.013 to 0.069], <italic>P</italic>&#x02009;=&#x02009;0.184), although the direction of association remained positive (Table&#x000a0;<xref rid="Tab1" ref-type="table">1</xref>). Additionally, during a follow-up of 36&#x000a0;months, 268 MACE cases were recorded. Per SD increase in baseline RC levels was significantly associated with an elevated risk of MACE (HR&#x02009;=&#x02009;1.140, [95% CI, 1.012&#x02013;1.286], <italic>P</italic>&#x02009;=&#x02009;0.031), and this association remained significant with multivariable adjustment of baseline Gensini score, age, sex, BMI, smoking status, alcohol consumption, type 2 diabetes, hypertension, and relevant medication use (HR&#x02009;=&#x02009;1.137, [95% CI, 1.009&#x02013;1.282], <italic>P</italic>&#x02009;=&#x02009;0.036; Table&#x000a0;<xref rid="Tab1" ref-type="table">1</xref> and Additional file 1: Table S3).
<table-wrap id="Tab1"><label>Table&#x000a0;1</label><caption><p>Association between baseline remnant cholesterol levels and coronary artery disease status, severity and major adverse cardiovascular events</p></caption><table frame="hsides" rules="groups"><thead><tr><th align="left">Outcome</th><th align="left">Sample size</th><th align="left" colspan="2">Univariable model</th><th align="left" colspan="2">Multivariable model<sup>b</sup>
</th></tr></thead><tbody><tr><td align="left"><bold>Cross-sectional analysis</bold></td><td align="left"/><td align="left"><bold>ORs or </bold><bold><italic>&#x003b2;</italic></bold><bold> [95% CI]</bold><sup><bold>a</bold></sup></td><td align="left"><bold><italic>P</italic></bold></td><td align="left"><bold>ORs or </bold><bold><italic>&#x003b2;</italic></bold><bold> [95% CI]</bold></td><td align="left"><bold><italic>P</italic></bold></td></tr><tr><td align="left" colspan="6"><bold>Disease status</bold></td></tr><tr><td align="left">Myocardial infarction</td><td align="left">N<sub>case</sub>&#x02009;=&#x02009;453; N<sub>control</sub>&#x02009;=&#x02009;1483</td><td align="left">1.508 [1.352, 1.685]</td><td align="left">2.44&#x02009;&#x000d7;&#x02009;10<sup>&#x02013;13</sup></td><td align="left">1.352 [1.176, 1.556]</td><td align="left">2.45&#x02009;&#x000d7;&#x02009;10<sup>&#x02013;5</sup></td></tr><tr><td align="left">Stable angina pectoris</td><td align="left">N<sub>case</sub>&#x02009;=&#x02009;816; N<sub>control</sub>&#x02009;=&#x02009;1483</td><td align="left">1.149 [1.054, 1.252]</td><td align="left">1.57&#x02009;&#x000d7;&#x02009;10<sup>&#x02013;3</sup></td><td align="left">1.085 [0.980, 1.203]</td><td align="left">0.117</td></tr><tr><td align="left">Unstable angina pectoris</td><td align="left">N<sub>case</sub>&#x02009;=&#x02009;967; N<sub>control</sub>&#x02009;=&#x02009;1483</td><td align="left">1.174 [1.082, 1.274]</td><td align="left">1.24&#x02009;&#x000d7;&#x02009;10<sup>&#x02013;4</sup></td><td align="left">1.130 [1.027, 1.244]</td><td align="left">0.013</td></tr><tr><td align="left">Angina pectoris</td><td align="left">N<sub>case</sub>&#x02009;=&#x02009;1783; N<sub>control</sub>&#x02009;=&#x02009;1483</td><td align="left">1.164 [1.086, 1.248]</td><td align="left">1.83&#x02009;&#x000d7;&#x02009;10<sup>&#x02013;5</sup></td><td align="left">1.115 [1.027, 1.210]</td><td align="left">9.10&#x02009;&#x000d7;&#x02009;10<sup>&#x02013;3</sup></td></tr><tr><td align="left">Coronary artery disease</td><td align="left">N<sub>case</sub>&#x02009;=&#x02009;2296; N<sub>control</sub>&#x02009;=&#x02009;1483</td><td align="left">1.219 [1.141, 1.302]</td><td align="left">4.74&#x02009;&#x000d7;&#x02009;10<sup>&#x02013;9</sup></td><td align="left">1.145 [1.059, 1.238]</td><td align="left">6.82&#x02009;&#x000d7;&#x02009;10<sup>&#x02013;4</sup></td></tr><tr><td align="left" colspan="6"><bold>Disease severity</bold></td></tr><tr><td align="left">Gensini Score</td><td align="left">2289</td><td align="left">0.045 [0.004, 0.086]</td><td align="left">0.032</td><td align="left">0.028 [&#x02212;0.013, 0.069]</td><td align="left">0.184</td></tr><tr><td align="left">cTnT</td><td align="left">2277</td><td align="left">0.045 [0.004, 0.085]</td><td align="left">0.032</td><td align="left">0.059 [0.020, 0.098]</td><td align="left">3.29&#x02009;&#x000d7;&#x02009;10<sup>&#x02013;3</sup></td></tr><tr><td align="left"><bold>Longitudinal analysis in CAD patients</bold></td><td align="left"/><td align="left"><bold>HR [95% CI]</bold></td><td align="left"><bold><italic>P</italic></bold></td><td align="left"><bold>HR [95% CI]</bold></td><td align="left"><bold><italic>P</italic></bold></td></tr><tr><td align="left" colspan="6"><bold>Disease prognosis</bold></td></tr><tr><td align="left">MACE</td><td align="left">N<sub>MACE</sub>&#x02009;=&#x02009;268; N<sub>non-MACE</sub>&#x02009;=&#x02009;1722</td><td align="left">1.140 [1.012, 1.286]</td><td align="left">0.031</td><td align="left">1.137 [1.009, 1.282]</td><td align="left">0.036</td></tr></tbody></table><table-wrap-foot><p><sup>a</sup>Odds ratios (ORs) with 95% CI were calculated for myocardial infarction, stable angina pectoris, unstable angina pectoris, angina pectoris, and coronary artery disease. Estimate &#x003b2; (logORs) with 95% CI were calculated for Gensini score and cTnT</p><p><sup>b</sup>For the association between RC levels and myocardial infarction, stable angina pectoris, unstable angina pectoris, angina pectoris, coronary artery diseases, Gensini score, and cTnT, we adjusted age, sex, BMI, smoking, alcohol consumption, type 2 diabetes, hypertension, antidiabetic drugs use, and antihypertensive drugs use in the multivariable models; for the association between baseline RC levels and the 36-month cumulative incidence of MACE, we adjusted baseline Gensini score, age, sex, BMI, smoking, alcohol consumption, type 2 diabetes, hypertension, antidiabetic drugs use, and antihypertensive drugs use in the multivariable models</p></table-wrap-foot></table-wrap></p></sec><sec id="Sec15"><title>Seven loci of remnant cholesterol identified through GWAS in the&#x000a0;Chinese&#x000a0;Population</title><p id="Par70">We further investigated the genetic architecture of RC in the Chinese population by conducting a GWAS. The characteristics of Chinese population are summarized in Additional file 1: Table S1. The mean (SD) age was 61.1 (14.1) years, the mean (SD) BMI was 24.5 (3.4) kg/m<sup>2</sup>, and 49.6% were male. In the RC GWAS, the genomic inflation factor indicated minimal risk of excessive population stratification (<italic>&#x003bb;</italic><sub>gc-meta</sub>&#x02009;=&#x02009;1.027, Additional file 2: Figure S2). The SNP-based heritability (<italic>h</italic><sup>2</sup>) for RC in Chinese was estimated to be 18.8% using LDSC regression. We identified seven genetic loci significantly associated with RC (<italic>DOCK7</italic>, <italic>GCKR</italic>, <italic>BCL7B</italic>, <italic>TRIB1</italic>, <italic>LPL</italic>, the&#x000a0;<italic>APOA5/A4/C3/A1</italic> gene&#x000a0;cluster, and <italic>APOC1</italic>; <italic>P</italic>&#x02009;&#x0003c;&#x02009;5&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;8</sup>), collectively harboring 17 independent associations (LD <italic>R</italic><sup>2</sup>&#x02009;&#x0003c;&#x02009;0.1 within 1&#x000a0;Mb distance; Fig.&#x000a0;<xref rid="Fig2" ref-type="fig">2</xref> and Additional file 1: Table S5). Among these, <italic>DOCK7</italic>, <italic>LPL</italic>, the&#x000a0;<italic>APOA5/A4/C3/A1</italic> gene&#x000a0;cluster, and <italic>APOC1</italic> were previously implicated in lipid metabolism [<xref ref-type="bibr" rid="CR44">44</xref>&#x02013;<xref ref-type="bibr" rid="CR46">46</xref>]. The <italic>GCKR</italic> gene encodes the glucokinase regulatory protein, which is involved in various metabolic processes [<xref ref-type="bibr" rid="CR47">47</xref>]. <italic>BCL7B</italic> and <italic>TRIB1</italic> have also been associated with lipid levels [<xref ref-type="bibr" rid="CR48">48</xref>, <xref ref-type="bibr" rid="CR49">49</xref>]<italic>.</italic> No novel loci were identified beyond those reported in prior lipid-related GWAS, likely due to the smaller sample size in the Chinese populations. Meanwhile, to assess the potential bias introduced by lipid-lowering medications, we performed sensitivity GWAS analyses after excluding medication users. The lead SNPs remained directionally consistent, with slightly attenuated significance, supporting the robustness of our results (Additional file 2: Figure S3).<fig id="Fig2"><label>Fig.&#x000a0;2</label><caption><p>Results of the meta-analysis of GWAS for remnant cholesterol (RC) levels in 14,939 Han Chinese. The Manhattan plot showed seven significantly associated loci for RC levels. Variants in the seven loci were highlighted in red. The genome-wide threshold for significant (<italic>P</italic>&#x02009;=&#x02009;5.0&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;8</sup>) associations was indicated by the horizontal dashed line</p></caption><graphic xlink:href="12916_2025_4329_Fig2_HTML" id="MO2"/></fig></p><p id="Par71">Subsequently, we performed cross-ancestry comparisons focusing on the seven lead loci identified in the Chinese population by conducting a GWAS of RC in the UK Biobank population (Additional file 1: Table S6). Notably, except for the variant rs12539351 (<italic>P</italic>&#x02009;=&#x02009;1.0&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;3</sup> in UK Biobank), all other&#x000a0;six lead SNPs reached genome-wide significance (<italic>P</italic>&#x02009;&#x0003c;&#x02009;5&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;8</sup>) in UK Biobank with consistent effect directions. The rs12539351, located near the <italic>BCL7B</italic> gene, shows different LD structures between East Asian and European populations (Additional file 2: Figure S4), with a larger LD block in East Asians. SNPs in high LD with it have been reported to be associated with lipid traits in both populations [<xref ref-type="bibr" rid="CR49">49</xref>, <xref ref-type="bibr" rid="CR50">50</xref>].</p><p id="Par72">In addition, we observed notable differences in effect sizes and allele frequencies between the two populations. Among the six lead SNPs (out of seven identified in Chinese RC GWAS) reaching genome-wide significance in both ancestries, effect sizes tend to be larger in the Chinese population. Moreover, aside from the seven lead SNPs, six out of 12 additional independent SNPs identified in the Chinese cohort were excluded from the European ancestry analysis due to low allele frequencies (MAF&#x02009;&#x0003c;&#x02009;0.01).</p></sec><sec id="Sec16"><title>Genetically predicted remnant cholesterol and cardiovascular diseases in East Asian ancestry individuals</title><p id="Par73">We integrated the GWAS summary statistics for CVDs from Biobank Japan and conducted two-sample MR analyses. Detailed information, including the IVs used, is provided in Additional file 1: Table S7. One palindromic variant, rs2001945 (MAF<sub>exposure</sub>&#x02009;=&#x02009;0.42 and MAF<sub>outcome</sub>&#x02009;=&#x02009;0.43), was excluded from the analysis during the harmonization process. The F statistics for all IVs in this study were&#x02009;&#x0003e;&#x02009;10, indicating no weak instrument bias.</p><p id="Par74">Our analysis revealed several potential causal effects of RC on different cardiovascular outcomes in East Asian ancestry individuals (Fig.&#x000a0;<xref rid="Fig3" ref-type="fig">3</xref>A and Additional file 1: Table S8). Genetically predicted higher levels of RC were significantly associated with a higher risk of aortic aneurysm (OR per SD increase in RC, 1.83 [95% CI, 1.54&#x02013;2.17], <italic>P</italic>&#x02009;=&#x02009;1.16&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;11</sup>), myocardial infarction (1.23 [1.13&#x02013;1.33], <italic>P</italic>&#x02009;=&#x02009;4.71&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;7</sup>), angina pectoris (1.17 [1.08&#x02013;1.26], <italic>P</italic>&#x02009;=&#x02009;1.34&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;4</sup>), and stable angina pectoris (1.17 [1.10&#x02013;1.24], <italic>P</italic>&#x02009;=&#x02009;1.74&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;6</sup>). Additionally, genetically predicted higher levels of RC were suggestively associated with an increased risk of unstable angina pectoris (1.15 [1.01&#x02013;1.30], <italic>P</italic>&#x02009;=&#x02009;0.03) and peripheral arterial disease (1.14 [1.04&#x02013;1.26], <italic>P</italic>&#x02009;=&#x02009;0.005). Sensitivity analyses confirmed that the directions of these associations were consistent with the results from the IVW method (Fig.&#x000a0;<xref rid="Fig3" ref-type="fig">3</xref>A and Additional file 1: Table S8). No significant associations were found for other cardiovascular outcomes, including atrial flutter/fibrillation, cerebral aneurysm, chronic heart failure, intracerebral hemorrhage, ischemic stroke, subarachnoid hemorrhage, and ventricular arrhythmia (Additional file 1: Table S8). Furthermore, the MR-Egger method showed no evidence of unbalanced pleiotropy affecting these findings (all <italic>P</italic> for MR-Egger intercept&#x02009;&#x0003e;&#x02009;0.05; Additional file 1: Table S8). Leave-one-out analyses also indicated that no individual SNP significantly influenced the outcomes, further supporting the robustness of the results (Additional file 2: Figure S5). In addition, to explore the effects of individual RC components on CVD, we further conducted separate MR analyses for IDL-c and VLDL-c on those CVD outcomes where a significant causal association with RC was observed. The results indicated that the effects of both IDL-c and VLDL-c on CVD risk were directionally consistent with those observed for overall RC (Additional file 2: Figure S6).<fig id="Fig3"><label>Fig.&#x000a0;3</label><caption><p>Causal effects of genetically predicted circulating levels of remnant cholesterol on cardiovascular diseases using univariate Mendelian randomization in a population of <bold>A</bold> East-Asian ancestry and <bold>B</bold> European ancestry. The inverse variance weighted method was considered the primary result. Sensitivity analyses were conducted using the MR-Egger method, weighted median method, weighted mode method, and MR-PRESSO method. The X-axes are shown on a log-transformed scale</p></caption><graphic xlink:href="12916_2025_4329_Fig3_HTML" id="MO3"/></fig></p><p id="Par75">Next, we conducted MVMR analyses to assess the independent associations between RC and CVDs across three models (Fig.&#x000a0;<xref rid="Fig4" ref-type="fig">4</xref>A and Additional file 1: Table S9). Notably, after adjusting for TC (Model 1), LDL-c and HDL-c (Model 2), and ApoA1 and ApoB (Model 3), the associations between RC and aortic aneurysm, myocardial infarction, angina pectoris, and stable angina pectoris remained statistically significant (all <italic>P</italic>&#x02009;&#x0003c;&#x02009;0.05), suggesting that RC exerts a causal effect on these cardiovascular outcomes independent of other lipoproteins (Fig.&#x000a0;<xref rid="Fig4" ref-type="fig">4</xref>A). In contrast, the suggestively significant association with unstable angina pectoris was no longer observed in model 1, and the association with peripheral arterial disease disappeared in model 1 and model 2 (Fig.&#x000a0;<xref rid="Fig4" ref-type="fig">4</xref>A). However, the direction of association remained consistent with those observed in the primary analyses.<fig id="Fig4"><label>Fig.&#x000a0;4</label><caption><p>Causal effects of genetically predicted circulating remnant cholesterol levels on cardiovascular diseases in populations of A East Asian ancestry and <bold>B</bold> European ancestry, using univariable and multivariable Mendelian randomization (MR). The primary results were derived using the inverse&#x000a0;variance weighted (IVW) method under the univariable MR framework. Sensitivity analyses were performed using multivariable MR with Model 1: total cholesterol (TC); Model 2: low-density lipoprotein cholesterol (LDL-c) and high-density lipoprotein cholesterol (HDL-c); Model 3: apolipoprotein A1 (ApoA1) and apolipoprotein B (ApoB). X-axes are shown on a log-transformed scale</p></caption><graphic xlink:href="12916_2025_4329_Fig4_HTML" id="MO4"/></fig></p></sec><sec id="Sec17"><title>Two-sample Mendelian randomization in population of European ancestry</title><p id="Par76">To replicate the associations between genetically predicted RC and CVDs in a population of European ancestry, we first conducted a GWAS for RC using data from Europeans of the UK Biobank. We identified a total of 26,905 significant variants (<italic>P</italic>&#x02009;&#x0003c;&#x02009;5&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;8</sup>), including 134 lead SNPs (<italic>R</italic><sup>2</sup>&#x02009;&#x0003c;&#x02009;0.001, justifying the larger sample size of Europeans, Additional file 2: Figure S7). Using LDSC regression, we estimated the SNP-based heritability of RC in a population of European ancestry to be 14.4%, which is comparable to a previous estimate of 12.6% reported in an NMR-based metabolomics GWAS [<xref ref-type="bibr" rid="CR51">51</xref>]. We then integrated the CVD GWAS summary statistics from FinnGen (R12, the latest release) and conducted two-sample MR in European ancestry population. We analyzed 12 CVDs, excluding stable angina pectoris, which was unavailable from the FinnGen database based on the ICD-10 codes (Additional file 1: Table S4).</p><p id="Par77">The summary information for the 134 lead SNPs used as candidate IVs for RC in European ancestry individuals is provided in Additional file 1: Table S10. Three palindromic variants (rs6721748, rs7544869, rs9848779, and rs9388530) with an MAF&#x02009;&#x0003e;&#x02009;0.42 were removed from the analyses. The F statistics for all IVs were&#x02009;&#x0003e;&#x02009;10, indicating no weak instrument bias.</p><p id="Par78">Consistent with the findings in East Asian ancestry individuals, genetically predicted higher levels of RC were significantly associated with an increased risk of aortic aneurysm (1.31 [1.19&#x02013;1.44], <italic>P</italic>&#x02009;=&#x02009;4.75&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;8</sup>), myocardial infarction (1.45 [1.33&#x02013;1.59], <italic>P</italic>&#x02009;=&#x02009;5.37&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;16</sup>), angina pectoris (1.46 [1.35&#x02013;1.58], <italic>P</italic>&#x02009;=&#x02009;1.45&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;20</sup>), unstable angina pectoris (1.48 [1.34&#x02013;1.63], <italic>P</italic>&#x02009;=&#x02009;1.37&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;15</sup>), and atrial flutter/fibrillation (1.11 [1.04&#x02013;1.19], <italic>P</italic>&#x02009;=&#x02009;1.99&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;3</sup>) in European ancestry individuals (Fig.&#x000a0;<xref rid="Fig3" ref-type="fig">3</xref>B and Additional file 1: Table S11). Notably, the causal effect of RC on atrial flutter/fibrillation was not observed in East Asian ancestry individuals (0.97 [0.88&#x02013;1.07], <italic>P</italic>&#x02009;=&#x02009;0.57; Fig.&#x000a0;<xref rid="Fig3" ref-type="fig">3</xref>B). Meanwhile, the magnitude of RC-associated risk for aortic aneurysm was higher in East Asian ancestry individuals than in European ancestry individuals (OR<sub>EAS</sub> = 1.83 vs. OR<sub>EUR</sub>&#x02009;=&#x02009;1.31), whereas the magnitude of RC-associated risk for ischemic heart diseases was greater in European ancestry individuals (e.g., for myocardial infarction OR<sub>EAS</sub>&#x02009;=&#x02009;1.23 vs. OR<sub>EUR</sub>&#x02009;=&#x02009;1.45, Fig.&#x000a0;<xref rid="Fig3" ref-type="fig">3</xref>). Moreover, genetically predicted higher levels of RC were suggestively associated with an increased risk of ventricular arrhythmia (1.06 [1.01&#x02013;1.11], <italic>P</italic>&#x02009;=&#x02009;0.009). However, no significant association was found between genetically predicted RC levels and peripheral arterial disease in European ancestry individuals (0.98 [0.87&#x02013;1.11] in European ancestry individuals versus 1.14 [1.04&#x02013;1.26] in East Asian ancestry individuals).</p><p id="Par79">Although we observed potentially unbalanced pleiotropy in the analyses for angina pectoris and myocardial infarction (<italic>P</italic> for intercept&#x02009;&#x0003c;&#x02009;0.05 based on the MR-Egger&#x02019;s intercept test; Additional file 1: Table S11), the MR-Egger method still supported a causal association between RC and these outcomes (1.64 [1.44&#x02013;1.86], <italic>P</italic>&#x02009;=&#x02009;3.33&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;12</sup>) and myocardial infarction (1.64 [1.42&#x02013;1.89], <italic>P</italic>&#x02009;=&#x02009;2.65&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;10</sup>). Leave-one-out analyses showed that no single SNP had a significant impact on the results (Additional file 2: Figure S8). As in East Asian ancestry individuals, genetically predicted RC levels were not associated with cerebral aneurysm, chronic heart failure, ischemic stroke, intracerebral hemorrhage, or subarachnoid hemorrhage in European ancestry individuals (Additional file 1: Table S11). Similarly, in the population of European ancestry, IDL-c and VLDL-c demonstrated causal effect directions aligned with those of RC, with no evidence of heterogeneity (Additional file 2: Figure S9).</p><p id="Par80">We also conducted the MVMR analyses in European ancestry population. After adjusting for TC, the causal associations between RC and aortic aneurysm, myocardial infarction, angina pectoris, unstable angina pectoris, atrial flutter/fibrillation, and ventricular arrhythmia remained statistically significant (Fig.&#x000a0;<xref rid="Fig4" ref-type="fig">4</xref>B and Additional file 1: Table S12, all <italic>P</italic>&#x02009;&#x0003c;&#x02009;0.05). However, in Model 2, which adjusted for LDL-c and HDL-c, and in Model 3, which adjusted for ApoA1 and ApoB, the statistical significance of several associations was no longer observed. This may be attributed to a drop in the conditional F statistic below 10, suggesting that these analyses may be affected by weak instrument bias (Additional file 1: Table S12). Nonetheless, the associations retained the same directional trend as observed in the primary analyses.</p></sec><sec id="Sec18"><title>Reverse MR analyses assessing the causal effects of cardiovascular diseases on remnant cholesterol</title><p id="Par81">To evaluate the possibility of reverse causality and to support the robustness of our primary findings, we conducted reverse MR analyses to assess the causal effects of CVDs on the levels of RC both in East Asian and European ancestry individuals (Fig.&#x000a0;<xref rid="Fig5" ref-type="fig">5</xref>). Detailed information on the IVs for each CVD outcome is provided in Additional file 1: Table S13 for East Asian ancestry individuals and Additional file 1: Table S14 for European ancestry individuals.<fig id="Fig5"><label>Fig.&#x000a0;5</label><caption><p>Causal effects of genetically predicted cardiovascular diseases on circulating remnant cholesterol levels in East Asian ancestry individuals (inner circle) and European ancestry individuals (outer circle). The potential causal association was evaluated using two-sample MR analyses with the IVW method. Sensitivity analyses were conducted using the MR-Egger method, weighted median method, weighted mode method, and MR-PRESSO method. The results are presented as causal estimates with Beta (SE) for each disease status</p></caption><graphic xlink:href="12916_2025_4329_Fig5_HTML" id="MO5"/></fig></p><p id="Par82">No significant associations were observed between genetically predicted CVDs and RC after correction for multiple testing (<italic>P</italic>&#x02009;&#x0003c;&#x02009;0.0038) in either populations of East Asian ancestry or European ancestry, though genetically predicted higher risks of ischemic heart diseases showed suggestive associations in both populations (all 0.0038&#x02009;&#x0003c;&#x02009;<italic>P</italic>&#x02009;&#x0003c;&#x02009;0.05; Fig.&#x000a0;<xref rid="Fig5" ref-type="fig">5</xref>, Additional file 1: Tables S15&#x02013;S16). No causal relationships were observed between other CVDs and RC. Furthermore, the removal of any individual SNP did not significantly affect the results (Figures S10 and S11).</p></sec></sec><sec id="Sec19"><title>Discussion</title><p id="Par83">In the population of East Asian ancestry, our study revealed elevated RC as an independent causal risk factor for aortic aneurysm and ischemic heart diseases, including myocardial infarction and angina pectoris. Notably, these causal associations persisted after rigorous adjustment for traditional lipid measures (TC, LDL-c, HDL-c, ApoA1, or ApoB) using MVMR, underscoring the unique role of RC in cardiovascular pathophysiology. While these results are in line with previous findings based on European ancestry populations supporting the broad causal relevance of RC to CVD, we identified population-specific risk gradients. Although the direction of effect is consistent, there is heterogeneity in effect sizes between individuals of East Asian and European ancestry in the MR analyses (e.g., risk for aortic aneurysm and ischemic heart disease seems higher for East Asian ancestry population). However, caution is warranted in interpreting these differences, as different IVs were constructed for each population. This heterogeneity highlights divergent cardiovascular consequences of RC across ethnicities, likely reflecting differences in genetic architecture, lipid metabolism, or environmental modifiers.</p><p id="Par84">Observational studies across diverse populations revealed that RC levels are positively associated with CVD risks [<xref ref-type="bibr" rid="CR10">10</xref>&#x02013;<xref ref-type="bibr" rid="CR12">12</xref>]. For example, in the Copenhagen General Population Study, the HR for the risk of ischemic heart disease was 2.3 (95% CI, 1.7&#x02013;3.1) comparing participants with the highest versus the lowest quintile of RC [<xref ref-type="bibr" rid="CR10">10</xref>]. MR studies in European ancestry population further validated RC&#x02019;s causal role [<xref ref-type="bibr" rid="CR16">16</xref>&#x02013;<xref ref-type="bibr" rid="CR18">18</xref>]. In populations of East Asian ancestry, such as Lee et al.&#x02019;s nationwide Korean study, elevated RC was linked to CVD risk even after LDL-c adjustment [<xref ref-type="bibr" rid="CR52">52</xref>]. However, causal inference in East Asian ancestry individuals has been hindered by the absence of large-scale RC GWAS. By performing the first RC GWAS in&#x02009;&#x0003e;&#x02009;14,000 East Asian ancestry individuals and integrating MR, our study bridges this critical gap, enabling robust causal assessment in this underrepresented population. Moreover, our reverse MR analyses found no evidence for a causal effect of CVD on RC levels, helping to rule out reverse causality. This directional confirmation not only suggests that previous observational associations were unlikely to have been driven by reverse causation&#x000a0;but also strengthens the robustness of the observed RC-to-CVD causal relationship and adds confidence to the inference that lowering RC may be an effective strategy for CVD prevention.</p><p id="Par85">RC primarily refers to the cholesterol content within triglyceride-rich lipoproteins, including chylomicrons, VLDL, and IDL, which can enter the arterial intima and become preferentially trapped due to their relatively large size [<xref ref-type="bibr" rid="CR8">8</xref>, <xref ref-type="bibr" rid="CR9">9</xref>]. Recent studies have shown that RC is emerging as an important residual risk factor beyond LDL-c in the development of CVD [<xref ref-type="bibr" rid="CR53">53</xref>], through mechanisms such as endothelial dysfunction, inflammation, and plaque destabilization [<xref ref-type="bibr" rid="CR54">54</xref>]. Our results extend prior observational and MR evidence linking RC to CVD risk in both Europeans and East Asian ancestry populations, but uniquely highlight aortic aneurysm as a priority outcome in East Asian ancestry populations. While aortic aneurysm is often considered a late-stage complication of atherosclerosis [<xref ref-type="bibr" rid="CR55">55</xref>], our MR findings suggest RC may directly drive vascular remodeling and aneurysm formation, independent of classical atherosclerotic pathways [<xref ref-type="bibr" rid="CR56">56</xref>]. This observation is consistent with recent multi-ancestry studies implicating dyslipidemia in aneurysm pathogenesis [<xref ref-type="bibr" rid="CR16">16</xref>, <xref ref-type="bibr" rid="CR57">57</xref>] and emphasizes the outsized role of RC in East Asian ancestry individuals, where non-HDL cholesterol levels are rising disproportionately [<xref ref-type="bibr" rid="CR58">58</xref>]. Current lipid-lowering therapies predominantly target LDL-c, yet our data suggest RC contributes to residual CVD risk unmitigated by statins. This is particularly relevant in East Asia, where aortic aneurysm prevalence is increasing but underrecognized [<xref ref-type="bibr" rid="CR59">59</xref>, <xref ref-type="bibr" rid="CR60">60</xref>]. Conversely, the higher ischemic heart disease risks observed in European ancestry population may reflect population-specific interactions between RC and other metabolic risk factors (e.g., insulin resistance) [<xref ref-type="bibr" rid="CR61">61</xref>, <xref ref-type="bibr" rid="CR62">62</xref>], warranting tailored interventions.</p><p id="Par86">Strengths of our study include the direct measurement of RC using NMR technology, a method shown to be more accurate than calculations based on traditional clinical lipid measurements [<xref ref-type="bibr" rid="CR63">63</xref>]. We employed two-sample MR analysis to mitigate confounding and replicated our findings across ethnic populations using large-scale biobanks, including Biobank Japan, FinnGen, and the UK Biobank. Sensitivity analyses, such as MR-Egger and MVMR, further demonstrated the robustness of our results against pleiotropy and adjustments for lipid covariates.</p><p id="Par87">Limitations of this study should also be noted. First, the modest sample size of the Chinese RC GWAS (<italic>N</italic>&#x02009;=&#x02009;14,939) limited statistical power to detect novel RC-associated loci, though this did not compromise the MR findings. As a result, no novel loci were identified beyond those previously reported in European ancestry GWAS. Nevertheless, our study provides the first ancestry-specific genetic evidence for remnant cholesterol in East Asian ancestry populations, filling a critical gap in the literature. Second, genetic variants associated with metabolic traits may impact CVD risk through pleiotropic pathways involving unmeasured confounders. To address this, multiple MR models were applied in sensitivity analyses, which reinforced the reliability of our conclusions despite potential violations of MR assumptions. Third, while MVMR suggested independence of our results from LDL-c and ApoB, weak instrument bias in European ancestry population analyses precluded definitive conclusions regarding these associations.</p></sec><sec id="Sec20"><title>Conclusions</title><p id="Par88">In conclusion, our study reveals that, in populations of East Asian ancestry, genetically determined higher levels of RC are associated with various adverse cardiovascular outcomes, even independently of other lipid traits. Our findings bridge a critical gap in understanding the causal role of RC in populations of East Asian ancestry and highlight aortic aneurysm as a key outcome for targeted intervention. Future research should elucidate mechanisms linking RC to vascular remodeling and evaluate RC-lowering therapies (e.g., fibrates, omega-3 fatty acids) in diverse populations. Addressing these gaps will advance precision strategies to reduce the global burden of RC-driven CVDs.</p></sec><sec id="Sec22" sec-type="supplementary-material"><title>Supplementary Information</title><p>
<supplementary-material content-type="local-data" id="MOESM1"><media xlink:href="12916_2025_4329_MOESM1_ESM.xlsx"><caption><p>Additional File 1: Tables S1-S16. Table S1 - Characteristics of the including study populations. Table S2 - The characteristics of GRAND cohort in the cross-section study. Table S3 - The characteristics of GRAND cohort in the longitudinal study. Table S4 - Data sources of GWASs for remnant cholesterol and cardiovascular diseases. Table S5- Summary of seven loci associated with remnant cholesterol in Meta-analysis of four Chinese studies. Table S6 - Comparison of seven loci associated with remnant cholesterol identified in the meta-analysis of four Chinese studies with corresponding results in the UK Biobank population. Table S7 - Effect estimates of the associations between genetic instrumental variables for remnant cholesterol and cardiovascular diseases in East Asians. Table S8 - Causal effects of remnant cholesterol on cardiovascular diseases in East Asians. Table S9 - Multivariable Mendelian randomization results of remnant cholesterol and each lipid profile component on cardiovascular diseases in East Asians. Table S10 - Effect estimates of the associations between genetic instrumental variables for remnant cholesterol and cardiovascular diseases in Europeans. Table S11 - Causal effects of remnant cholesterol on cardiovascular diseases in Europeans. Table S12 - Multivariable Mendelian randomization results of remnant cholesterol and each lipid profile component on cardiovascular diseases in Europeans. Table S13 - Effect estimates of the associations between genetic instrumental variables for cardiovascular diseases and remnant cholesterol in East Asians. Table S14 - Effect estimates of the associations between genetic instrumental variables for cardiovascular diseases and remnant cholesterol in Europeans. Table S15 - Association of genetically predicted risk of cardiovascular diseases with remnant cholesterol using univariable inverse-variance weighted method and sensitivity analyses in East Asians. Table S16 - Association of genetic predisposition to cardiovascular diseases with remnant cholesterol in univariable inverse-variance weighted analyses and in sensitivity analyses in Europeans.</p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="MOESM2"><media xlink:href="12916_2025_4329_MOESM2_ESM.docx"><caption><p>Additional File 2: Supplementary method and Figures S1-S11. Supplementary method 1- Nuclear magnetic resonance-based metabolomics profiling and data processing in four Chinses populations. Figure S1 - The workflow of Mendelian randomization in the current study. Figure S2 - Quantile-quantile (Q-Q) plots of GWAS results. Figure S3 - Comparison of effect sizes and P values for 17 identified independent significant associations between the full Chinese population sample and the subset excluding lipid-lowering drug users. Figure S4 - Comparison of linkage disequilibrium pattern of rs12539351 in East Asian ancestry population and European ancestry population. Figure S5 - The leave-one-out analysis for the identified significant casual effects of RC on cardiovascular diseases in East Asians. Figure S6 - Mendelian randomization estimates of RC and its components (IDL-c and VLDL-c) on CVDs in East-Asian ancestry population. Figure S7 - Results of GWAS for RC levels in Europeans of UK Biobank. Figure S8 - The leave-one-out analysis for the identified significant casual effect of RC on cardiovascular diseases in Europeans. Figure S9 - Mendelian randomization estimates of RC and its components (IDL-c and VLDL-c) on CVDs in European ancestry population. Figure S10 - The leave-one-out analyses for the casual effect of cardiovascular diseases on RC in East Asians. Figure S11 - The leave-one-out analyses for the casual effect of cardiovascular diseases on RC in Europeans. </p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="MOESM3"><media xlink:href="12916_2025_4329_MOESM3_ESM.docx"><caption><p>Additional File 3: STROBE-MR checklist of recommended items to address in reports of Mendelian randomization studies.</p></caption></media></supplementary-material></p></sec></body><back><glossary><title>Abbreviations</title><def-list><def-item><term>ApoA1</term><def><p id="Par19">Apolipoprotein A1</p></def></def-item><def-item><term>ApoB</term><def><p id="Par20">Apolipoprotein B</p></def></def-item><def-item><term>BMI</term><def><p id="Par21">Body mass index</p></def></def-item><def-item><term>CAD</term><def><p id="Par22">Coronary artery disease</p></def></def-item><def-item><term>CI</term><def><p id="Par23">Confidence interval</p></def></def-item><def-item><term>cTnT</term><def><p id="Par24">Cardiac troponin T</p></def></def-item><def-item><term>CVD</term><def><p id="Par25">Cardiovascular disease</p></def></def-item><def-item><term>GRAND</term><def><p id="Par26">The Genetic characteristics of coRonary Artery disease in ChiNese young aDults study</p></def></def-item><def-item><term>GWAS</term><def><p id="Par27">Genome-wide association study</p></def></def-item><def-item><term>HDL-c</term><def><p id="Par28">High-density lipoprotein cholesterol</p></def></def-item><def-item><term>HR</term><def><p id="Par29">Hazard ratio</p></def></def-item><def-item><term>HWE</term><def><p id="Par30">Hardy-Weinberg equilibrium</p></def></def-item><def-item><term>IDL</term><def><p id="Par31">Intermediate-density lipoprotein</p></def></def-item><def-item><term>INFO</term><def><p id="Par32">Imputation quality</p></def></def-item><def-item><term>IV</term><def><p id="Par33">Instrumental variable</p></def></def-item><def-item><term>IVW</term><def><p id="Par34">Inverse variance weighted</p></def></def-item><def-item><term>LD</term><def><p id="Par35">Linkage disequilibrium</p></def></def-item><def-item><term>LDL-c</term><def><p id="Par36">Low-density lipoprotein cholesterol</p></def></def-item><def-item><term>MACE</term><def><p id="Par37">Major adverse cardiovascular event</p></def></def-item><def-item><term>MAF</term><def><p id="Par38">Minor allele frequency</p></def></def-item><def-item><term>MR</term><def><p id="Par39">Mendelian randomization</p></def></def-item><def-item><term>MVMR</term><def><p id="Par40">Multivariable Mendelian randomization</p></def></def-item><def-item><term>NMR</term><def><p id="Par41">Nuclear magnetic resonance</p></def></def-item><def-item><term>OR</term><def><p id="Par42">Odds ratio</p></def></def-item><def-item><term>RC</term><def><p id="Par43">Remnant cholesterol</p></def></def-item><def-item><term>SD</term><def><p id="Par44">Standard deviation</p></def></def-item><def-item><term>SNP</term><def><p id="Par45">Single nucleotide polymorphism</p></def></def-item><def-item><term>TC</term><def><p id="Par46">Total cholesterol</p></def></def-item><def-item><term>VLDL</term><def><p id="Par47">Very-low-density lipoprotein</p></def></def-item></def-list></glossary><fn-group><fn><p><bold>Publisher's Note</bold></p><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></fn><fn><p>Chenhao Lin, Haolong Wen and Mengyao Yu contributed equally to this work.</p></fn></fn-group><ack><title>Acknowledgements</title><p>The data analysis server is supported by the Human Phenome Data Center of Fudan University, and we thank the center staff for their support. UK Biobank approved this present research under application number 54294. We gratefully acknowledge the contributors to the data used in this work.</p></ack><notes notes-type="author-contribution"><title>Authors&#x02019; contributions</title><p>C.L., H.W., and Y.Z. drafted the manuscript. C.L. and H.W. performed formal statistical analysis. J.L., W.Y., and J.M.J.W. assisted with statistical analysis. Q.X.H., Q.W., and H.T. provided metabolite measurements and quality control in the study. Y.Z., Y.D., M.X., L.J, X.W, S.W., and H.T. organized data collection and/or preparation. G.C. and M.Y. provided critical comments on the manuscript. G.C. and Y.Z. conceived the idea and approach, and supervised the research. All authors contributed to the interpretation of results and critically reviewed the manuscript.</p></notes><notes notes-type="funding-information"><title>Funding</title><p>This research was supported by the National Key R&#x00026;D Program of China (2022YFC3400700, 2022YFA0806400), Shanghai Municipal Science and Technology Major Project (Grant No. 2023SHZDZX02 and 2017SHZDZX01), the Program for Professor of Special Appointment (Eastern Scholar) at Shanghai Institutions of Higher Learning, Grant of Shanghai Municipal Health Commission (2022JC012), National Natural Science Foundation of China (No. 82370357), grant from the Shanghai Municipal Health Commission (2022JC012), the Program of Shanghai Academic Research Leader (22XD1423300) and Shanghai Oriental Youth Elite Talents Program (QNZH2024009).</p></notes><notes notes-type="data-availability"><title>Data availability
</title><p>The UK Biobank data are available from the UK Biobank application (<ext-link ext-link-type="uri" xlink:href="https://www.ukbiobank.ac.uk/">www.ukbiobank.ac.uk/</ext-link>). The GWAS summary statistics of Biobank Japan are available from the website of Biobank Japan (<ext-link ext-link-type="uri" xlink:href="https://pheweb.jp/">https://pheweb.jp/</ext-link>) [<xref ref-type="bibr" rid="CR64">64</xref>]. The GWAS summary statistics of FinnGen (R12) are available from the website of FinnGen <ext-link ext-link-type="uri" xlink:href="https://www.finngen.fi/en/access_results">https://www.finngen.fi/en/access_results</ext-link>) [<xref ref-type="bibr" rid="CR65">65</xref>]. The data required for the analysis of the article are provided in the Supplementary Tables, and further data can be obtained by a reasonable request to the corresponding author.</p></notes><notes><title>Declarations</title><notes id="FPar2"><title>Ethics approval and consent to participate</title><p id="Par89">The National Survey of Physical Traits is the subproject of the National Science &#x00026; Technology Basic Research Project which was approved by the Ethics Committee of Human Genetic Resources of the School of Life Sciences, Fudan University (14117). The Shanghai Changfeng study was approved by the Ethical Committee of Zhongshan Hospital, Fudan University (2013&#x02013;132). The Rugao Longevity and Aging study was approved by the Human Ethics Committee of the School of Life Sciences, Fudan University (BE1815). The GRAND study was approved by the central Ethics Committee at Zhongshan Hospital, Fudan University (B2017-051). All participants or their legal representatives provided informed consent before joining the study.</p></notes><notes id="FPar3"><title>Consent for publication</title><p id="Par90">All authors have consent for publication.</p></notes><notes id="FPar4" notes-type="COI-statement"><title>Competing interests
</title><p id="Par91">The authors declare no competing interests.</p></notes></notes><ref-list id="Bib1"><title>References</title><ref id="CR1"><label>1.</label><citation-alternatives><element-citation id="ec-CR1" publication-type="journal"><person-group person-group-type="author"><name><surname>Goldstein</surname><given-names>JL</given-names></name><name><surname>Brown</surname><given-names>MS</given-names></name></person-group><article-title>A century of cholesterol and coronaries: from plaques to genes to statins</article-title><source>Cell</source><year>2015</year><volume>161</volume><issue>1</issue><fpage>161</fpage><lpage>172</lpage><pub-id pub-id-type="pmid">25815993</pub-id>
</element-citation><mixed-citation id="mc-CR1" publication-type="journal">Goldstein JL, Brown MS. A century of cholesterol and coronaries: from plaques to genes to statins. Cell. 2015;161(1):161&#x02013;72.<pub-id pub-id-type="pmid">25815993</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR2"><label>2.</label><citation-alternatives><element-citation id="ec-CR2" publication-type="journal"><person-group person-group-type="author"><name><surname>Ference</surname><given-names>BA</given-names></name><name><surname>Braunwald</surname><given-names>E</given-names></name><name><surname>Catapano</surname><given-names>AL</given-names></name></person-group><article-title>The LDL cumulative exposure hypothesis: evidence and practical applications</article-title><source>Nat Rev Cardiol</source><year>2024</year><volume>21</volume><issue>10</issue><fpage>701</fpage><lpage>716</lpage><pub-id pub-id-type="pmid">38969749</pub-id>
</element-citation><mixed-citation id="mc-CR2" publication-type="journal">Ference BA, Braunwald E, Catapano AL. The LDL cumulative exposure hypothesis: evidence and practical applications. Nat Rev Cardiol. 2024;21(10):701&#x02013;16.<pub-id pub-id-type="pmid">38969749</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR3"><label>3.</label><citation-alternatives><element-citation id="ec-CR3" publication-type="journal"><person-group person-group-type="author"><name><surname>Ference</surname><given-names>BA</given-names></name><name><surname>Ginsberg</surname><given-names>HN</given-names></name><name><surname>Graham</surname><given-names>I</given-names></name><name><surname>Ray</surname><given-names>KK</given-names></name><name><surname>Packard</surname><given-names>CJ</given-names></name><name><surname>Bruckert</surname><given-names>E</given-names></name><name><surname>Hegele</surname><given-names>RA</given-names></name><name><surname>Krauss</surname><given-names>RM</given-names></name><name><surname>Raal</surname><given-names>FJ</given-names></name><name><surname>Schunkert</surname><given-names>H</given-names></name><etal/></person-group><article-title>Low-density lipoproteins cause atherosclerotic cardiovascular disease. 1. Evidence from genetic, epidemiologic, and clinical studies. A consensus statement from the European Atherosclerosis Society Consensus Panel</article-title><source>Eur Heart J</source><year>2017</year><volume>38</volume><issue>32</issue><fpage>2459</fpage><lpage>72</lpage><pub-id pub-id-type="pmid">28444290</pub-id>
</element-citation><mixed-citation id="mc-CR3" publication-type="journal">Ference BA, Ginsberg HN, Graham I, Ray KK, Packard CJ, Bruckert E, Hegele RA, Krauss RM, Raal FJ, Schunkert H, et al. Low-density lipoproteins cause atherosclerotic cardiovascular disease. 1. Evidence from genetic, epidemiologic, and clinical studies. A consensus statement from the European Atherosclerosis Society Consensus Panel. Eur Heart J. 2017;38(32):2459&#x02013;72.<pub-id pub-id-type="pmid">28444290</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR4"><label>4.</label><citation-alternatives><element-citation id="ec-CR4" publication-type="journal"><person-group person-group-type="author"><name><surname>Barter</surname><given-names>PJ</given-names></name><name><surname>Caulfield</surname><given-names>M</given-names></name><name><surname>Eriksson</surname><given-names>M</given-names></name><name><surname>Grundy</surname><given-names>SM</given-names></name><name><surname>Kastelein</surname><given-names>JJ</given-names></name><name><surname>Komajda</surname><given-names>M</given-names></name><name><surname>Lopez-Sendon</surname><given-names>J</given-names></name><name><surname>Mosca</surname><given-names>L</given-names></name><name><surname>Tardif</surname><given-names>JC</given-names></name><name><surname>Waters</surname><given-names>DD</given-names></name><etal/></person-group><article-title>Effects of torcetrapib in patients at high risk for coronary events</article-title><source>N Engl J Med</source><year>2007</year><volume>357</volume><issue>21</issue><fpage>2109</fpage><lpage>22</lpage><pub-id pub-id-type="pmid">17984165</pub-id>
</element-citation><mixed-citation id="mc-CR4" publication-type="journal">Barter PJ, Caulfield M, Eriksson M, Grundy SM, Kastelein JJ, Komajda M, Lopez-Sendon J, Mosca L, Tardif JC, Waters DD, et al. Effects of torcetrapib in patients at high risk for coronary events. N Engl J Med. 2007;357(21):2109&#x02013;22.<pub-id pub-id-type="pmid">17984165</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR5"><label>5.</label><citation-alternatives><element-citation id="ec-CR5" publication-type="journal"><person-group person-group-type="author"><name><surname>Lincoff</surname><given-names>AM</given-names></name><name><surname>Nicholls</surname><given-names>SJ</given-names></name><name><surname>Riesmeyer</surname><given-names>JS</given-names></name><name><surname>Barter</surname><given-names>PJ</given-names></name><name><surname>Brewer</surname><given-names>HB</given-names></name><name><surname>Fox</surname><given-names>KAA</given-names></name><name><surname>Gibson</surname><given-names>CM</given-names></name><name><surname>Granger</surname><given-names>C</given-names></name><name><surname>Menon</surname><given-names>V</given-names></name><name><surname>Montalescot</surname><given-names>G</given-names></name><etal/></person-group><article-title>Evacetrapib and cardiovascular outcomes in high-risk vascular disease</article-title><source>N Engl J Med</source><year>2017</year><volume>376</volume><issue>20</issue><fpage>1933</fpage><lpage>1942</lpage><pub-id pub-id-type="pmid">28514624</pub-id>
</element-citation><mixed-citation id="mc-CR5" publication-type="journal">Lincoff AM, Nicholls SJ, Riesmeyer JS, Barter PJ, Brewer HB, Fox KAA, Gibson CM, Granger C, Menon V, Montalescot G, et al. Evacetrapib and cardiovascular outcomes in high-risk vascular disease. N Engl J Med. 2017;376(20):1933&#x02013;42.<pub-id pub-id-type="pmid">28514624</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR6"><label>6.</label><citation-alternatives><element-citation id="ec-CR6" publication-type="journal"><person-group person-group-type="author"><name><surname>Dhindsa</surname><given-names>DS</given-names></name><name><surname>Sandesara</surname><given-names>PB</given-names></name><name><surname>Shapiro</surname><given-names>MD</given-names></name><name><surname>Wong</surname><given-names>ND</given-names></name></person-group><article-title>The evolving understanding and approach to residual cardiovascular risk management</article-title><source>Front Cardiovasc Med</source><year>2020</year><volume>7</volume><fpage>88</fpage><pub-id pub-id-type="pmid">32478100</pub-id>
</element-citation><mixed-citation id="mc-CR6" publication-type="journal">Dhindsa DS, Sandesara PB, Shapiro MD, Wong ND. The evolving understanding and approach to residual cardiovascular risk management. Front Cardiovasc Med. 2020;7: 88.<pub-id pub-id-type="pmid">32478100</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR7"><label>7.</label><citation-alternatives><element-citation id="ec-CR7" publication-type="journal"><person-group person-group-type="author"><name><surname>Chapman</surname><given-names>MJ</given-names></name><name><surname>Ginsberg</surname><given-names>HN</given-names></name><name><surname>Amarenco</surname><given-names>P</given-names></name><name><surname>Andreotti</surname><given-names>F</given-names></name><name><surname>Boren</surname><given-names>J</given-names></name><name><surname>Catapano</surname><given-names>AL</given-names></name><name><surname>Descamps</surname><given-names>OS</given-names></name><name><surname>Fisher</surname><given-names>E</given-names></name><name><surname>Kovanen</surname><given-names>PT</given-names></name><name><surname>Kuivenhoven</surname><given-names>JA</given-names></name><etal/></person-group><article-title>Triglyceride-rich lipoproteins and high-density lipoprotein cholesterol in patients at high risk of cardiovascular disease: evidence and guidance for management</article-title><source>Eur Heart J</source><year>2011</year><volume>32</volume><issue>11</issue><fpage>1345</fpage><lpage>61</lpage><pub-id pub-id-type="pmid">21531743</pub-id>
</element-citation><mixed-citation id="mc-CR7" publication-type="journal">Chapman MJ, Ginsberg HN, Amarenco P, Andreotti F, Boren J, Catapano AL, Descamps OS, Fisher E, Kovanen PT, Kuivenhoven JA, et al. Triglyceride-rich lipoproteins and high-density lipoprotein cholesterol in patients at high risk of cardiovascular disease: evidence and guidance for management. Eur Heart J. 2011;32(11):1345&#x02013;61.<pub-id pub-id-type="pmid">21531743</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR8"><label>8.</label><citation-alternatives><element-citation id="ec-CR8" publication-type="journal"><person-group person-group-type="author"><name><surname>Boren</surname><given-names>J</given-names></name><name><surname>Taskinen</surname><given-names>MR</given-names></name><name><surname>Bjornson</surname><given-names>E</given-names></name><name><surname>Packard</surname><given-names>CJ</given-names></name></person-group><article-title>Metabolism of triglyceride-rich lipoproteins in health and dyslipidaemia</article-title><source>Nat Rev Cardiol</source><year>2022</year><volume>19</volume><issue>9</issue><fpage>577</fpage><lpage>592</lpage><pub-id pub-id-type="pmid">35318466</pub-id>
</element-citation><mixed-citation id="mc-CR8" publication-type="journal">Boren J, Taskinen MR, Bjornson E, Packard CJ. Metabolism of triglyceride-rich lipoproteins in health and dyslipidaemia. Nat Rev Cardiol. 2022;19(9):577&#x02013;92.<pub-id pub-id-type="pmid">35318466</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR9"><label>9.</label><citation-alternatives><element-citation id="ec-CR9" publication-type="journal"><person-group person-group-type="author"><name><surname>Nordestgaard</surname><given-names>BG</given-names></name></person-group><article-title>Triglyceride-rich lipoproteins and atherosclerotic cardiovascular disease: new insights from epidemiology, genetics, and biology</article-title><source>Circ Res</source><year>2016</year><volume>118</volume><issue>4</issue><fpage>547</fpage><lpage>563</lpage><pub-id pub-id-type="pmid">26892957</pub-id>
</element-citation><mixed-citation id="mc-CR9" publication-type="journal">Nordestgaard BG. Triglyceride-rich lipoproteins and atherosclerotic cardiovascular disease: new insights from epidemiology, genetics, and biology. Circ Res. 2016;118(4):547&#x02013;63.<pub-id pub-id-type="pmid">26892957</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR10"><label>10.</label><citation-alternatives><element-citation id="ec-CR10" publication-type="journal"><person-group person-group-type="author"><name><surname>Varbo</surname><given-names>A</given-names></name><name><surname>Benn</surname><given-names>M</given-names></name><name><surname>Tybjaerg-Hansen</surname><given-names>A</given-names></name><name><surname>Jorgensen</surname><given-names>AB</given-names></name><name><surname>Frikke-Schmidt</surname><given-names>R</given-names></name><name><surname>Nordestgaard</surname><given-names>BG</given-names></name></person-group><article-title>Remnant cholesterol as a causal risk factor for ischemic heart disease</article-title><source>J Am Coll Cardiol</source><year>2013</year><volume>61</volume><issue>4</issue><fpage>427</fpage><lpage>436</lpage><pub-id pub-id-type="pmid">23265341</pub-id>
</element-citation><mixed-citation id="mc-CR10" publication-type="journal">Varbo A, Benn M, Tybjaerg-Hansen A, Jorgensen AB, Frikke-Schmidt R, Nordestgaard BG. Remnant cholesterol as a causal risk factor for ischemic heart disease. J Am Coll Cardiol. 2013;61(4):427&#x02013;36.<pub-id pub-id-type="pmid">23265341</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR11"><label>11.</label><citation-alternatives><element-citation id="ec-CR11" publication-type="journal"><person-group person-group-type="author"><name><surname>Wadstrom</surname><given-names>BN</given-names></name><name><surname>Wulff</surname><given-names>AB</given-names></name><name><surname>Pedersen</surname><given-names>KM</given-names></name><name><surname>Jensen</surname><given-names>GB</given-names></name><name><surname>Nordestgaard</surname><given-names>BG</given-names></name></person-group><article-title>Elevated remnant cholesterol increases the risk of peripheral artery disease, myocardial infarction, and ischaemic stroke: a cohort-based study</article-title><source>Eur Heart J</source><year>2022</year><volume>43</volume><issue>34</issue><fpage>3258</fpage><lpage>3269</lpage><pub-id pub-id-type="pmid">34661640</pub-id>
</element-citation><mixed-citation id="mc-CR11" publication-type="journal">Wadstrom BN, Wulff AB, Pedersen KM, Jensen GB, Nordestgaard BG. Elevated remnant cholesterol increases the risk of peripheral artery disease, myocardial infarction, and ischaemic stroke: a cohort-based study. Eur Heart J. 2022;43(34):3258&#x02013;69.<pub-id pub-id-type="pmid">34661640</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR12"><label>12.</label><citation-alternatives><element-citation id="ec-CR12" publication-type="journal"><person-group person-group-type="author"><name><surname>Quispe</surname><given-names>R</given-names></name><name><surname>Martin</surname><given-names>SS</given-names></name><name><surname>Michos</surname><given-names>ED</given-names></name><name><surname>Lamba</surname><given-names>I</given-names></name><name><surname>Blumenthal</surname><given-names>RS</given-names></name><name><surname>Saeed</surname><given-names>A</given-names></name><name><surname>Lima</surname><given-names>J</given-names></name><name><surname>Puri</surname><given-names>R</given-names></name><name><surname>Nomura</surname><given-names>S</given-names></name><name><surname>Tsai</surname><given-names>M</given-names></name><etal/></person-group><article-title>Remnant cholesterol predicts cardiovascular disease beyond LDL and ApoB: a primary prevention study</article-title><source>Eur Heart J</source><year>2021</year><volume>42</volume><issue>42</issue><fpage>4324</fpage><lpage>32</lpage><pub-id pub-id-type="pmid">34293083</pub-id>
</element-citation><mixed-citation id="mc-CR12" publication-type="journal">Quispe R, Martin SS, Michos ED, Lamba I, Blumenthal RS, Saeed A, Lima J, Puri R, Nomura S, Tsai M, et al. Remnant cholesterol predicts cardiovascular disease beyond LDL and ApoB: a primary prevention study. Eur Heart J. 2021;42(42):4324&#x02013;32.<pub-id pub-id-type="pmid">34293083</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR13"><label>13.</label><citation-alternatives><element-citation id="ec-CR13" publication-type="journal"><person-group person-group-type="author"><name><surname>Arsenault</surname><given-names>BJ</given-names></name><name><surname>Boekholdt</surname><given-names>SM</given-names></name><name><surname>Kastelein</surname><given-names>JJ</given-names></name></person-group><article-title>Lipid parameters for measuring risk of cardiovascular disease</article-title><source>Nat Rev Cardiol</source><year>2011</year><volume>8</volume><issue>4</issue><fpage>197</fpage><lpage>206</lpage><pub-id pub-id-type="pmid">21283149</pub-id>
</element-citation><mixed-citation id="mc-CR13" publication-type="journal">Arsenault BJ, Boekholdt SM, Kastelein JJ. Lipid parameters for measuring risk of cardiovascular disease. Nat Rev Cardiol. 2011;8(4):197&#x02013;206.<pub-id pub-id-type="pmid">21283149</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR14"><label>14.</label><citation-alternatives><element-citation id="ec-CR14" publication-type="journal"><person-group person-group-type="author"><name><surname>Holmes</surname><given-names>MV</given-names></name><name><surname>Asselbergs</surname><given-names>FW</given-names></name><name><surname>Palmer</surname><given-names>TM</given-names></name><name><surname>Drenos</surname><given-names>F</given-names></name><name><surname>Lanktree</surname><given-names>MB</given-names></name><name><surname>Nelson</surname><given-names>CP</given-names></name><name><surname>Dale</surname><given-names>CE</given-names></name><name><surname>Padmanabhan</surname><given-names>S</given-names></name><name><surname>Finan</surname><given-names>C</given-names></name><name><surname>Swerdlow</surname><given-names>DI</given-names></name><etal/></person-group><article-title>Mendelian randomization of blood lipids for coronary heart disease</article-title><source>Eur Heart J</source><year>2015</year><volume>36</volume><issue>9</issue><fpage>539</fpage><lpage>50</lpage><pub-id pub-id-type="pmid">24474739</pub-id>
</element-citation><mixed-citation id="mc-CR14" publication-type="journal">Holmes MV, Asselbergs FW, Palmer TM, Drenos F, Lanktree MB, Nelson CP, Dale CE, Padmanabhan S, Finan C, Swerdlow DI, et al. Mendelian randomization of blood lipids for coronary heart disease. Eur Heart J. 2015;36(9):539&#x02013;50.<pub-id pub-id-type="pmid">24474739</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR15"><label>15.</label><citation-alternatives><element-citation id="ec-CR15" publication-type="journal"><person-group person-group-type="author"><name><surname>Levin</surname><given-names>MG</given-names></name><name><surname>Burgess</surname><given-names>S</given-names></name></person-group><article-title>Mendelian randomization as a tool for cardiovascular research: a review</article-title><source>JAMA Cardiol</source><year>2024</year><volume>9</volume><issue>1</issue><fpage>79</fpage><lpage>89</lpage><pub-id pub-id-type="pmid">37938820</pub-id>
</element-citation><mixed-citation id="mc-CR15" publication-type="journal">Levin MG, Burgess S. Mendelian randomization as a tool for cardiovascular research: a review. JAMA Cardiol. 2024;9(1):79&#x02013;89.<pub-id pub-id-type="pmid">37938820</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR16"><label>16.</label><citation-alternatives><element-citation id="ec-CR16" publication-type="journal"><person-group person-group-type="author"><name><surname>Guan</surname><given-names>B</given-names></name><name><surname>Wang</surname><given-names>A</given-names></name><name><surname>Xu</surname><given-names>H</given-names></name></person-group><article-title>Causal associations of remnant cholesterol with cardiometabolic diseases and risk factors: a mendelian randomization analysis</article-title><source>Cardiovasc Diabetol</source><year>2023</year><volume>22</volume><issue>1</issue><fpage>207</fpage><pub-id pub-id-type="pmid">37563569</pub-id>
</element-citation><mixed-citation id="mc-CR16" publication-type="journal">Guan B, Wang A, Xu H. Causal associations of remnant cholesterol with cardiometabolic diseases and risk factors: a mendelian randomization analysis. Cardiovasc Diabetol. 2023;22(1):207.<pub-id pub-id-type="pmid">37563569</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR17"><label>17.</label><citation-alternatives><element-citation id="ec-CR17" publication-type="journal"><person-group person-group-type="author"><name><surname>Navarese</surname><given-names>EP</given-names></name><name><surname>Vine</surname><given-names>D</given-names></name><name><surname>Proctor</surname><given-names>S</given-names></name><name><surname>Grzelakowska</surname><given-names>K</given-names></name><name><surname>Berti</surname><given-names>S</given-names></name><name><surname>Kubica</surname><given-names>J</given-names></name><name><surname>Raggi</surname><given-names>P</given-names></name></person-group><article-title>Independent causal effect of remnant cholesterol on atherosclerotic cardiovascular outcomes: a Mendelian randomization study</article-title><source>Arterioscler Thromb Vasc Biol</source><year>2023</year><volume>43</volume><issue>9</issue><fpage>e373</fpage><lpage>e380</lpage><pub-id pub-id-type="pmid">37439258</pub-id>
</element-citation><mixed-citation id="mc-CR17" publication-type="journal">Navarese EP, Vine D, Proctor S, Grzelakowska K, Berti S, Kubica J, Raggi P. Independent causal effect of remnant cholesterol on atherosclerotic cardiovascular outcomes: a Mendelian randomization study. Arterioscler Thromb Vasc Biol. 2023;43(9):e373&#x02013;80.<pub-id pub-id-type="pmid">37439258</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR18"><label>18.</label><citation-alternatives><element-citation id="ec-CR18" publication-type="journal"><person-group person-group-type="author"><name><surname>Zhao</surname><given-names>Y</given-names></name><name><surname>Zhuang</surname><given-names>Z</given-names></name><name><surname>Li</surname><given-names>Y</given-names></name><name><surname>Xiao</surname><given-names>W</given-names></name><name><surname>Song</surname><given-names>Z</given-names></name><name><surname>Huang</surname><given-names>N</given-names></name><name><surname>Wang</surname><given-names>W</given-names></name><name><surname>Dong</surname><given-names>X</given-names></name><name><surname>Jia</surname><given-names>J</given-names></name><name><surname>Clarke</surname><given-names>R</given-names></name><etal/></person-group><article-title>Elevated blood remnant cholesterol and triglycerides are causally related to the risks of cardiometabolic multimorbidity</article-title><source>Nat Commun</source><year>2024</year><volume>15</volume><issue>1</issue><fpage>2451</fpage><pub-id pub-id-type="pmid">38503751</pub-id>
</element-citation><mixed-citation id="mc-CR18" publication-type="journal">Zhao Y, Zhuang Z, Li Y, Xiao W, Song Z, Huang N, Wang W, Dong X, Jia J, Clarke R, et al. Elevated blood remnant cholesterol and triglycerides are causally related to the risks of cardiometabolic multimorbidity. Nat Commun. 2024;15(1):2451.<pub-id pub-id-type="pmid">38503751</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR19"><label>19.</label><citation-alternatives><element-citation id="ec-CR19" publication-type="journal"><person-group person-group-type="author"><name><surname>Sakaue</surname><given-names>S</given-names></name><name><surname>Kanai</surname><given-names>M</given-names></name><name><surname>Tanigawa</surname><given-names>Y</given-names></name><name><surname>Karjalainen</surname><given-names>J</given-names></name><name><surname>Kurki</surname><given-names>M</given-names></name><name><surname>Koshiba</surname><given-names>S</given-names></name><name><surname>Narita</surname><given-names>A</given-names></name><name><surname>Konuma</surname><given-names>T</given-names></name><name><surname>Yamamoto</surname><given-names>K</given-names></name><name><surname>Akiyama</surname><given-names>M</given-names></name><etal/></person-group><article-title>A cross-population atlas of genetic associations for 220 human phenotypes</article-title><source>Nat Genet</source><year>2021</year><volume>53</volume><issue>10</issue><fpage>1415</fpage><lpage>1424</lpage><pub-id pub-id-type="pmid">34594039</pub-id>
</element-citation><mixed-citation id="mc-CR19" publication-type="journal">Sakaue S, Kanai M, Tanigawa Y, Karjalainen J, Kurki M, Koshiba S, Narita A, Konuma T, Yamamoto K, Akiyama M, et al. A cross-population atlas of genetic associations for 220 human phenotypes. Nat Genet. 2021;53(10):1415&#x02013;24.<pub-id pub-id-type="pmid">34594039</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR20"><label>20.</label><citation-alternatives><element-citation id="ec-CR20" publication-type="journal"><person-group person-group-type="author"><name><surname>Kurki</surname><given-names>MI</given-names></name><name><surname>Karjalainen</surname><given-names>J</given-names></name><name><surname>Palta</surname><given-names>P</given-names></name><name><surname>Sipila</surname><given-names>TP</given-names></name><name><surname>Kristiansson</surname><given-names>K</given-names></name><name><surname>Donner</surname><given-names>KM</given-names></name><name><surname>Reeve</surname><given-names>MP</given-names></name><name><surname>Laivuori</surname><given-names>H</given-names></name><name><surname>Aavikko</surname><given-names>M</given-names></name><name><surname>Kaunisto</surname><given-names>MA</given-names></name><etal/></person-group><article-title>Finngen provides genetic insights from a well-phenotyped isolated population</article-title><source>Nature</source><year>2023</year><volume>613</volume><issue>7944</issue><fpage>508</fpage><lpage>18</lpage><pub-id pub-id-type="pmid">36653562</pub-id>
</element-citation><mixed-citation id="mc-CR20" publication-type="journal">Kurki MI, Karjalainen J, Palta P, Sipila TP, Kristiansson K, Donner KM, Reeve MP, Laivuori H, Aavikko M, Kaunisto MA, et al. Finngen provides genetic insights from a well-phenotyped isolated population. Nature. 2023;613(7944):508&#x02013;18.<pub-id pub-id-type="pmid">36653562</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR21"><label>21.</label><citation-alternatives><element-citation id="ec-CR21" publication-type="journal"><person-group person-group-type="author"><name><surname>Gao</surname><given-names>X</given-names></name><name><surname>Hofman</surname><given-names>A</given-names></name><name><surname>Hu</surname><given-names>Y</given-names></name><name><surname>Lin</surname><given-names>H</given-names></name><name><surname>Zhu</surname><given-names>C</given-names></name><name><surname>Jeekel</surname><given-names>J</given-names></name><name><surname>Jin</surname><given-names>X</given-names></name><name><surname>Wang</surname><given-names>J</given-names></name><name><surname>Gao</surname><given-names>J</given-names></name><name><surname>Yin</surname><given-names>Y</given-names></name><etal/></person-group><article-title>The Shanghai Changfeng study: a community-based prospective cohort study of chronic diseases among middle-aged and elderly: objectives and design</article-title><source>Eur J Epidemiol</source><year>2010</year><volume>25</volume><issue>12</issue><fpage>885</fpage><lpage>893</lpage><pub-id pub-id-type="pmid">21120588</pub-id>
</element-citation><mixed-citation id="mc-CR21" publication-type="journal">Gao X, Hofman A, Hu Y, Lin H, Zhu C, Jeekel J, Jin X, Wang J, Gao J, Yin Y, et al. The Shanghai Changfeng study: a community-based prospective cohort study of chronic diseases among middle-aged and elderly: objectives and design. Eur J Epidemiol. 2010;25(12):885&#x02013;93.<pub-id pub-id-type="pmid">21120588</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR22"><label>22.</label><citation-alternatives><element-citation id="ec-CR22" publication-type="journal"><person-group person-group-type="author"><name><surname>Wu</surname><given-names>Q</given-names></name><name><surname>Huang</surname><given-names>QX</given-names></name><name><surname>Zeng</surname><given-names>HL</given-names></name><name><surname>Ma</surname><given-names>S</given-names></name><name><surname>Lin</surname><given-names>HD</given-names></name><name><surname>Xia</surname><given-names>MF</given-names></name><name><surname>Tang</surname><given-names>HR</given-names></name><name><surname>Gao</surname><given-names>X</given-names></name></person-group><article-title>Prediction of metabolic disorders using NMR-based metabolomics: the Shanghai Changfeng study</article-title><source>Phenomics</source><year>2021</year><volume>1</volume><issue>4</issue><fpage>186</fpage><lpage>198</lpage><pub-id pub-id-type="pmid">36939780</pub-id>
</element-citation><mixed-citation id="mc-CR22" publication-type="journal">Wu Q, Huang QX, Zeng HL, Ma S, Lin HD, Xia MF, Tang HR, Gao X. Prediction of metabolic disorders using NMR-based metabolomics: the Shanghai Changfeng study. Phenomics. 2021;1(4):186&#x02013;98.<pub-id pub-id-type="pmid">36939780</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR23"><label>23.</label><mixed-citation publication-type="other"> Huang Q, Qadri SF, Bian H, Yi X, Lin C, Yang X, Zhu X, Lin H, Yan H, Chang X et al: A metabolome-derived score predicts metabolic dysfunction-associated steatohepatitis and mortality from liver disease. J Hepatol 2024.</mixed-citation></ref><ref id="CR24"><label>24.</label><citation-alternatives><element-citation id="ec-CR24" publication-type="journal"><person-group person-group-type="author"><name><surname>Liu</surname><given-names>Z</given-names></name><name><surname>Wang</surname><given-names>Y</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Chu</surname><given-names>X</given-names></name><name><surname>Wang</surname><given-names>Z</given-names></name><name><surname>Qian</surname><given-names>D</given-names></name><name><surname>Chen</surname><given-names>F</given-names></name><name><surname>Xu</surname><given-names>J</given-names></name><name><surname>Li</surname><given-names>S</given-names></name><name><surname>Jin</surname><given-names>L</given-names></name><etal/></person-group><article-title>Cohort profile: the Rugao Longevity and Ageing Study (RuLAS)</article-title><source>Int J Epidemiol</source><year>2016</year><volume>45</volume><issue>4</issue><fpage>1064</fpage><lpage>1073</lpage><pub-id pub-id-type="pmid">26066327</pub-id>
</element-citation><mixed-citation id="mc-CR24" publication-type="journal">Liu Z, Wang Y, Zhang Y, Chu X, Wang Z, Qian D, Chen F, Xu J, Li S, Jin L, et al. Cohort profile: the Rugao Longevity and Ageing Study (RuLAS). Int J Epidemiol. 2016;45(4):1064&#x02013;73.<pub-id pub-id-type="pmid">26066327</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR25"><label>25.</label><mixed-citation publication-type="other"> Wang F, Luo Q, Chen Y, Liu Y, Xu K, Adhikari K, Cai X, Liu J, Li Y, Liu X et al: A genome-wide scan on individual typology angle found variants at SLC24A2 associated with skin color variation in Chinese populations. J Invest Dermatol 2022, 142(4):1223&#x02013;1227 e1214.</mixed-citation></ref><ref id="CR26"><label>26.</label><citation-alternatives><element-citation id="ec-CR26" publication-type="journal"><person-group person-group-type="author"><name><surname>Shalaimaiti</surname><given-names>S</given-names></name><name><surname>Dai</surname><given-names>Y-X</given-names></name><name><surname>Wu</surname><given-names>H-Y</given-names></name><name><surname>Qian</surname><given-names>J-Y</given-names></name><name><surname>Zheng</surname><given-names>Y</given-names></name><name><surname>Yao</surname><given-names>K</given-names></name><name><surname>Ge</surname><given-names>J-B</given-names></name></person-group><article-title>Clinical and genetic characteristics of coronary artery disease in Chinese young adults: rationale and design of the prospectivegenetic characteristics of coronaryartery disease in ChiNese young aDults (GRAND) Study</article-title><source>Cardiology Plus</source><year>2021</year><volume>6</volume><issue>1</issue><fpage>65</fpage><lpage>73</lpage></element-citation><mixed-citation id="mc-CR26" publication-type="journal">Shalaimaiti S, Dai Y-X, Wu H-Y, Qian J-Y, Zheng Y, Yao K, Ge J-B. Clinical and genetic characteristics of coronary artery disease in Chinese young adults: rationale and design of the prospectivegenetic characteristics of coronaryartery disease in ChiNese young aDults (GRAND) Study. Cardiology Plus. 2021;6(1):65&#x02013;73.</mixed-citation></citation-alternatives></ref><ref id="CR27"><label>27.</label><citation-alternatives><element-citation id="ec-CR27" publication-type="journal"><person-group person-group-type="author"><name><surname>Mei</surname><given-names>Z</given-names></name><name><surname>Xu</surname><given-names>L</given-names></name><name><surname>Huang</surname><given-names>Q</given-names></name><name><surname>Lin</surname><given-names>C</given-names></name><name><surname>Yu</surname><given-names>M</given-names></name><name><surname>Shali</surname><given-names>S</given-names></name><name><surname>Wu</surname><given-names>H</given-names></name><name><surname>Lu</surname><given-names>Y</given-names></name><name><surname>Wu</surname><given-names>R</given-names></name><name><surname>Wang</surname><given-names>Z</given-names></name><etal/></person-group><article-title>Metabonomic biomarkers of plaque burden and instability in patients with coronary atherosclerotic disease after moderate lipid-lowering therapy</article-title><source>J Am Heart Assoc</source><year>2024</year><volume>13</volume><issue>24</issue><fpage>e036906</fpage><pub-id pub-id-type="pmid">39655754</pub-id>
</element-citation><mixed-citation id="mc-CR27" publication-type="journal">Mei Z, Xu L, Huang Q, Lin C, Yu M, Shali S, Wu H, Lu Y, Wu R, Wang Z, et al. Metabonomic biomarkers of plaque burden and instability in patients with coronary atherosclerotic disease after moderate lipid-lowering therapy. J Am Heart Assoc. 2024;13(24): e036906.<pub-id pub-id-type="pmid">39655754</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR28"><label>28.</label><citation-alternatives><element-citation id="ec-CR28" publication-type="journal"><person-group person-group-type="author"><name><surname>Bycroft</surname><given-names>C</given-names></name><name><surname>Freeman</surname><given-names>C</given-names></name><name><surname>Petkova</surname><given-names>D</given-names></name><name><surname>Band</surname><given-names>G</given-names></name><name><surname>Elliott</surname><given-names>LT</given-names></name><name><surname>Sharp</surname><given-names>K</given-names></name><name><surname>Motyer</surname><given-names>A</given-names></name><name><surname>Vukcevic</surname><given-names>D</given-names></name><name><surname>Delaneau</surname><given-names>O</given-names></name><name><surname>O&#x02019;Connell</surname><given-names>J</given-names></name><etal/></person-group><article-title>The UK biobank resource with deep phenotyping and genomic data</article-title><source>Nature</source><year>2018</year><volume>562</volume><issue>7726</issue><fpage>203</fpage><lpage>9</lpage><pub-id pub-id-type="pmid">30305743</pub-id>
</element-citation><mixed-citation id="mc-CR28" publication-type="journal">Bycroft C, Freeman C, Petkova D, Band G, Elliott LT, Sharp K, Motyer A, Vukcevic D, Delaneau O, O&#x02019;Connell J, et al. The UK biobank resource with deep phenotyping and genomic data. Nature. 2018;562(7726):203&#x02013;9.<pub-id pub-id-type="pmid">30305743</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR29"><label>29.</label><citation-alternatives><element-citation id="ec-CR29" publication-type="journal"><person-group person-group-type="author"><name><surname>Julkunen</surname><given-names>H</given-names></name><name><surname>Cichonska</surname><given-names>A</given-names></name><name><surname>Tiainen</surname><given-names>M</given-names></name><name><surname>Koskela</surname><given-names>H</given-names></name><name><surname>Nybo</surname><given-names>K</given-names></name><name><surname>Makela</surname><given-names>V</given-names></name><name><surname>Nokso-Koivisto</surname><given-names>J</given-names></name><name><surname>Kristiansson</surname><given-names>K</given-names></name><name><surname>Perola</surname><given-names>M</given-names></name><name><surname>Salomaa</surname><given-names>V</given-names></name><etal/></person-group><article-title>Atlas of plasma NMR biomarkers for health and disease in 118,461 individuals from the UK Biobank</article-title><source>Nat Commun</source><year>2023</year><volume>14</volume><issue>1</issue><fpage>604</fpage><pub-id pub-id-type="pmid">36737450</pub-id>
</element-citation><mixed-citation id="mc-CR29" publication-type="journal">Julkunen H, Cichonska A, Tiainen M, Koskela H, Nybo K, Makela V, Nokso-Koivisto J, Kristiansson K, Perola M, Salomaa V, et al. Atlas of plasma NMR biomarkers for health and disease in 118,461 individuals from the UK Biobank. Nat Commun. 2023;14(1):604.<pub-id pub-id-type="pmid">36737450</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR30"><label>30.</label><citation-alternatives><element-citation id="ec-CR30" publication-type="journal"><person-group person-group-type="author"><name><surname>Jimenez</surname><given-names>B</given-names></name><name><surname>Holmes</surname><given-names>E</given-names></name><name><surname>Heude</surname><given-names>C</given-names></name><name><surname>Tolson</surname><given-names>RF</given-names></name><name><surname>Harvey</surname><given-names>N</given-names></name><name><surname>Lodge</surname><given-names>SL</given-names></name><name><surname>Chetwynd</surname><given-names>AJ</given-names></name><name><surname>Cannet</surname><given-names>C</given-names></name><name><surname>Fang</surname><given-names>F</given-names></name><name><surname>Pearce</surname><given-names>JTM</given-names></name><etal/></person-group><article-title>Quantitative lipoprotein subclass and low molecular weight metabolite analysis in human serum and plasma by (1)H NMR spectroscopy in a multilaboratory trial</article-title><source>Anal Chem</source><year>2018</year><volume>90</volume><issue>20</issue><fpage>11962</fpage><lpage>71</lpage><pub-id pub-id-type="pmid">30211542</pub-id>
</element-citation><mixed-citation id="mc-CR30" publication-type="journal">Jimenez B, Holmes E, Heude C, Tolson RF, Harvey N, Lodge SL, Chetwynd AJ, Cannet C, Fang F, Pearce JTM, et al. Quantitative lipoprotein subclass and low molecular weight metabolite analysis in human serum and plasma by (1)H NMR spectroscopy in a multilaboratory trial. Anal Chem. 2018;90(20):11962&#x02013;71.<pub-id pub-id-type="pmid">30211542</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR31"><label>31.</label><mixed-citation publication-type="other"> Huang Q, Chen Q, Yi X, Wang H, Wang Q, Zhi H, Wu J, Wang DW, Tang H: Reproducibility of NMR-based quantitative metabolomics and HBV-caused changes in human serum lipoprotein subclasses and small metabolites. Journal of Pharmaceutical Analysis 2024:101180.</mixed-citation></ref><ref id="CR32"><label>32.</label><citation-alternatives><element-citation id="ec-CR32" publication-type="journal"><person-group person-group-type="author"><name><surname>Graham</surname><given-names>SE</given-names></name><name><surname>Clarke</surname><given-names>SL</given-names></name><name><surname>Wu</surname><given-names>KH</given-names></name><name><surname>Kanoni</surname><given-names>S</given-names></name><name><surname>Zajac</surname><given-names>GJM</given-names></name><name><surname>Ramdas</surname><given-names>S</given-names></name><name><surname>Surakka</surname><given-names>I</given-names></name><name><surname>Ntalla</surname><given-names>I</given-names></name><name><surname>Vedantam</surname><given-names>S</given-names></name><name><surname>Winkler</surname><given-names>TW</given-names></name><etal/></person-group><article-title>The power of genetic diversity in genome-wide association studies of lipids</article-title><source>Nature</source><year>2021</year><volume>600</volume><issue>7890</issue><fpage>675</fpage><lpage>9</lpage><pub-id pub-id-type="pmid">34887591</pub-id>
</element-citation><mixed-citation id="mc-CR32" publication-type="journal">Graham SE, Clarke SL, Wu KH, Kanoni S, Zajac GJM, Ramdas S, Surakka I, Ntalla I, Vedantam S, Winkler TW, et al. The power of genetic diversity in genome-wide association studies of lipids. Nature. 2021;600(7890):675&#x02013;9.<pub-id pub-id-type="pmid">34887591</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR33"><label>33.</label><citation-alternatives><element-citation id="ec-CR33" publication-type="journal"><person-group person-group-type="author"><name><surname>Loh</surname><given-names>PR</given-names></name><name><surname>Tucker</surname><given-names>G</given-names></name><name><surname>Bulik-Sullivan</surname><given-names>BK</given-names></name><name><surname>Vilhjalmsson</surname><given-names>BJ</given-names></name><name><surname>Finucane</surname><given-names>HK</given-names></name><name><surname>Salem</surname><given-names>RM</given-names></name><name><surname>Chasman</surname><given-names>DI</given-names></name><name><surname>Ridker</surname><given-names>PM</given-names></name><name><surname>Neale</surname><given-names>BM</given-names></name><name><surname>Berger</surname><given-names>B</given-names></name><etal/></person-group><article-title>Efficient bayesian mixed-model analysis increases association power in large cohorts</article-title><source>Nat Genet</source><year>2015</year><volume>47</volume><issue>3</issue><fpage>284</fpage><lpage>90</lpage><pub-id pub-id-type="pmid">25642633</pub-id>
</element-citation><mixed-citation id="mc-CR33" publication-type="journal">Loh PR, Tucker G, Bulik-Sullivan BK, Vilhjalmsson BJ, Finucane HK, Salem RM, Chasman DI, Ridker PM, Neale BM, Berger B, et al. Efficient bayesian mixed-model analysis increases association power in large cohorts. Nat Genet. 2015;47(3):284&#x02013;90.<pub-id pub-id-type="pmid">25642633</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR34"><label>34.</label><citation-alternatives><element-citation id="ec-CR34" publication-type="journal"><person-group person-group-type="author"><name><surname>Willer</surname><given-names>CJ</given-names></name><name><surname>Li</surname><given-names>Y</given-names></name><name><surname>Abecasis</surname><given-names>GR</given-names></name></person-group><article-title>METAL: fast and efficient meta-analysis of genomewide association scans</article-title><source>Bioinformatics</source><year>2010</year><volume>26</volume><issue>17</issue><fpage>2190</fpage><lpage>2191</lpage><pub-id pub-id-type="pmid">20616382</pub-id>
</element-citation><mixed-citation id="mc-CR34" publication-type="journal">Willer CJ, Li Y, Abecasis GR. METAL: fast and efficient meta-analysis of genomewide association scans. Bioinformatics. 2010;26(17):2190&#x02013;1.<pub-id pub-id-type="pmid">20616382</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR35"><label>35.</label><mixed-citation publication-type="other">Chang CC, Chow CC, Tellier LC, Vattikuti S, Purcell SM, Lee JJ. Second-generation PLINK: rising to the challenge of larger and richer datasets. Gigascience. 2015;4.</mixed-citation></ref><ref id="CR36"><label>36.</label><citation-alternatives><element-citation id="ec-CR36" publication-type="journal"><person-group person-group-type="author"><name><surname>Davies</surname><given-names>NM</given-names></name><name><surname>Holmes</surname><given-names>MV</given-names></name><name><surname>Davey Smith</surname><given-names>G</given-names></name></person-group><article-title>Reading Mendelian randomisation studies: a guide, glossary, and checklist for clinicians</article-title><source>BMJ</source><year>2018</year><volume>362</volume><fpage>k601</fpage><pub-id pub-id-type="pmid">30002074</pub-id>
</element-citation><mixed-citation id="mc-CR36" publication-type="journal">Davies NM, Holmes MV, Davey Smith G. Reading Mendelian randomisation studies: a guide, glossary, and checklist for clinicians. BMJ. 2018;362: k601.<pub-id pub-id-type="pmid">30002074</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR37"><label>37.</label><citation-alternatives><element-citation id="ec-CR37" publication-type="journal"><person-group person-group-type="author"><name><surname>Pierce</surname><given-names>BL</given-names></name><name><surname>Ahsan</surname><given-names>H</given-names></name><name><surname>Vanderweele</surname><given-names>TJ</given-names></name></person-group><article-title>Power and instrument strength requirements for Mendelian randomization studies using multiple genetic variants</article-title><source>Int J Epidemiol</source><year>2011</year><volume>40</volume><issue>3</issue><fpage>740</fpage><lpage>752</lpage><pub-id pub-id-type="pmid">20813862</pub-id>
</element-citation><mixed-citation id="mc-CR37" publication-type="journal">Pierce BL, Ahsan H, Vanderweele TJ. Power and instrument strength requirements for Mendelian randomization studies using multiple genetic variants. Int J Epidemiol. 2011;40(3):740&#x02013;52.<pub-id pub-id-type="pmid">20813862</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR38"><label>38.</label><citation-alternatives><element-citation id="ec-CR38" publication-type="journal"><person-group person-group-type="author"><name><surname>Bowden</surname><given-names>J</given-names></name><name><surname>Holmes</surname><given-names>MV</given-names></name></person-group><article-title>Meta-analysis and Mendelian randomization: a review</article-title><source>Res Synth Methods</source><year>2019</year><volume>10</volume><issue>4</issue><fpage>486</fpage><lpage>496</lpage><pub-id pub-id-type="pmid">30861319</pub-id>
</element-citation><mixed-citation id="mc-CR38" publication-type="journal">Bowden J, Holmes MV. Meta-analysis and Mendelian randomization: a review. Res Synth Methods. 2019;10(4):486&#x02013;96.<pub-id pub-id-type="pmid">30861319</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR39"><label>39.</label><citation-alternatives><element-citation id="ec-CR39" publication-type="journal"><person-group person-group-type="author"><name><surname>Skrivankova</surname><given-names>VW</given-names></name><name><surname>Richmond</surname><given-names>RC</given-names></name><name><surname>Woolf</surname><given-names>BAR</given-names></name><name><surname>Yarmolinsky</surname><given-names>J</given-names></name><name><surname>Davies</surname><given-names>NM</given-names></name><name><surname>Swanson</surname><given-names>SA</given-names></name><name><surname>VanderWeele</surname><given-names>TJ</given-names></name><name><surname>Higgins</surname><given-names>JPT</given-names></name><name><surname>Timpson</surname><given-names>NJ</given-names></name><name><surname>Dimou</surname><given-names>N</given-names></name><etal/></person-group><article-title>Strengthening the reporting of observational studies in epidemiology using mendelian randomization: the STROBE-MR statement</article-title><source>JAMA</source><year>2021</year><volume>326</volume><issue>16</issue><fpage>1614</fpage><lpage>1621</lpage><pub-id pub-id-type="pmid">34698778</pub-id>
</element-citation><mixed-citation id="mc-CR39" publication-type="journal">Skrivankova VW, Richmond RC, Woolf BAR, Yarmolinsky J, Davies NM, Swanson SA, VanderWeele TJ, Higgins JPT, Timpson NJ, Dimou N, et al. Strengthening the reporting of observational studies in epidemiology using mendelian randomization: the STROBE-MR statement. JAMA. 2021;326(16):1614&#x02013;21.<pub-id pub-id-type="pmid">34698778</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR40"><label>40.</label><citation-alternatives><element-citation id="ec-CR40" publication-type="journal"><person-group person-group-type="author"><name><surname>Skrivankova</surname><given-names>VW</given-names></name><name><surname>Richmond</surname><given-names>RC</given-names></name><name><surname>Woolf</surname><given-names>BAR</given-names></name><name><surname>Davies</surname><given-names>NM</given-names></name><name><surname>Swanson</surname><given-names>SA</given-names></name><name><surname>VanderWeele</surname><given-names>TJ</given-names></name><name><surname>Timpson</surname><given-names>NJ</given-names></name><name><surname>Higgins</surname><given-names>JPT</given-names></name><name><surname>Dimou</surname><given-names>N</given-names></name><name><surname>Langenberg</surname><given-names>C</given-names></name><etal/></person-group><article-title>Strengthening the reporting of observational studies in epidemiology using mendelian randomisation (STROBE-MR): explanation and elaboration</article-title><source>BMJ</source><year>2021</year><volume>375</volume><fpage>n2233</fpage><pub-id pub-id-type="pmid">34702754</pub-id>
</element-citation><mixed-citation id="mc-CR40" publication-type="journal">Skrivankova VW, Richmond RC, Woolf BAR, Davies NM, Swanson SA, VanderWeele TJ, Timpson NJ, Higgins JPT, Dimou N, Langenberg C, et al. Strengthening the reporting of observational studies in epidemiology using mendelian randomisation (STROBE-MR): explanation and elaboration. BMJ. 2021;375: n2233.<pub-id pub-id-type="pmid">34702754</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR41"><label>41.</label><citation-alternatives><element-citation id="ec-CR41" publication-type="journal"><person-group person-group-type="author"><name><surname>Sanderson</surname><given-names>E</given-names></name><name><surname>Glymour</surname><given-names>MM</given-names></name><name><surname>Holmes</surname><given-names>MV</given-names></name><name><surname>Kang</surname><given-names>H</given-names></name><name><surname>Morrison</surname><given-names>J</given-names></name><name><surname>Munaf&#x000f2;</surname><given-names>MR</given-names></name><name><surname>Palmer</surname><given-names>T</given-names></name><name><surname>Schooling</surname><given-names>CM</given-names></name><name><surname>Wallace</surname><given-names>C</given-names></name><name><surname>Zhao</surname><given-names>Q</given-names></name><etal/></person-group><article-title>Mendelian randomization</article-title><source>Nat Rev Methods Primers</source><year>2022</year><volume>2</volume><issue>1</issue><fpage>6</fpage><pub-id pub-id-type="pmid">37325194</pub-id>
</element-citation><mixed-citation id="mc-CR41" publication-type="journal">Sanderson E, Glymour MM, Holmes MV, Kang H, Morrison J, Munaf&#x000f2; MR, Palmer T, Schooling CM, Wallace C, Zhao Q, et al. Mendelian randomization. Nat Rev Methods Primers. 2022;2(1): 6.<pub-id pub-id-type="pmid">37325194</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR42"><label>42.</label><citation-alternatives><element-citation id="ec-CR42" publication-type="journal"><person-group person-group-type="author"><name><surname>De Oliveira-Gomes</surname><given-names>D</given-names></name><name><surname>Joshi</surname><given-names>PH</given-names></name><name><surname>Peterson</surname><given-names>ED</given-names></name><name><surname>Rohatgi</surname><given-names>A</given-names></name><name><surname>Khera</surname><given-names>A</given-names></name><name><surname>Navar</surname><given-names>AM</given-names></name></person-group><article-title>Apolipoprotein B: bridging the gap between evidence and clinical practice</article-title><source>Circulation</source><year>2024</year><volume>150</volume><issue>1</issue><fpage>62</fpage><lpage>79</lpage><pub-id pub-id-type="pmid">38950110</pub-id>
</element-citation><mixed-citation id="mc-CR42" publication-type="journal">De Oliveira-Gomes D, Joshi PH, Peterson ED, Rohatgi A, Khera A, Navar AM. Apolipoprotein B: bridging the gap between evidence and clinical practice. Circulation. 2024;150(1):62&#x02013;79.<pub-id pub-id-type="pmid">38950110</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR43"><label>43.</label><citation-alternatives><element-citation id="ec-CR43" publication-type="journal"><person-group person-group-type="author"><name><surname>Sanderson</surname><given-names>E</given-names></name><name><surname>Davey Smith</surname><given-names>G</given-names></name><name><surname>Windmeijer</surname><given-names>F</given-names></name><name><surname>Bowden</surname><given-names>J</given-names></name></person-group><article-title>An examination of multivariable Mendelian randomization in the single-sample and two-sample summary data settings</article-title><source>Int J Epidemiol</source><year>2019</year><volume>48</volume><issue>3</issue><fpage>713</fpage><lpage>727</lpage><pub-id pub-id-type="pmid">30535378</pub-id>
</element-citation><mixed-citation id="mc-CR43" publication-type="journal">Sanderson E, Davey Smith G, Windmeijer F, Bowden J. An examination of multivariable Mendelian randomization in the single-sample and two-sample summary data settings. Int J Epidemiol. 2019;48(3):713&#x02013;27.<pub-id pub-id-type="pmid">30535378</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR44"><label>44.</label><citation-alternatives><element-citation id="ec-CR44" publication-type="journal"><person-group person-group-type="author"><name><surname>Nagy</surname><given-names>R</given-names></name><name><surname>Boutin</surname><given-names>TS</given-names></name><name><surname>Marten</surname><given-names>J</given-names></name><name><surname>Huffman</surname><given-names>JE</given-names></name><name><surname>Kerr</surname><given-names>SM</given-names></name><name><surname>Campbell</surname><given-names>A</given-names></name><name><surname>Evenden</surname><given-names>L</given-names></name><name><surname>Gibson</surname><given-names>J</given-names></name><name><surname>Amador</surname><given-names>C</given-names></name><name><surname>Howard</surname><given-names>DM</given-names></name><etal/></person-group><article-title>Exploration of haplotype research consortium imputation for genome-wide association studies in 20,032 Generation Scotland participants</article-title><source>Genome Med</source><year>2017</year><volume>9</volume><issue>1</issue><fpage>23</fpage><pub-id pub-id-type="pmid">28270201</pub-id>
</element-citation><mixed-citation id="mc-CR44" publication-type="journal">Nagy R, Boutin TS, Marten J, Huffman JE, Kerr SM, Campbell A, Evenden L, Gibson J, Amador C, Howard DM, et al. Exploration of haplotype research consortium imputation for genome-wide association studies in 20,032 Generation Scotland participants. Genome Med. 2017;9(1):23.<pub-id pub-id-type="pmid">28270201</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR45"><label>45.</label><citation-alternatives><element-citation id="ec-CR45" publication-type="journal"><person-group person-group-type="author"><name><surname>Sinnott-Armstrong</surname><given-names>N</given-names></name><name><surname>Tanigawa</surname><given-names>Y</given-names></name><name><surname>Amar</surname><given-names>D</given-names></name><name><surname>Mars</surname><given-names>N</given-names></name><name><surname>Benner</surname><given-names>C</given-names></name><name><surname>Aguirre</surname><given-names>M</given-names></name><name><surname>Venkataraman</surname><given-names>GR</given-names></name><name><surname>Wainberg</surname><given-names>M</given-names></name><name><surname>Ollila</surname><given-names>HM</given-names></name><name><surname>Kiiskinen</surname><given-names>T</given-names></name><etal/></person-group><article-title>Genetics of 35 blood and urine biomarkers in the UK Biobank</article-title><source>Nat Genet</source><year>2021</year><volume>53</volume><issue>2</issue><fpage>185</fpage><lpage>194</lpage><pub-id pub-id-type="pmid">33462484</pub-id>
</element-citation><mixed-citation id="mc-CR45" publication-type="journal">Sinnott-Armstrong N, Tanigawa Y, Amar D, Mars N, Benner C, Aguirre M, Venkataraman GR, Wainberg M, Ollila HM, Kiiskinen T, et al. Genetics of 35 blood and urine biomarkers in the UK Biobank. Nat Genet. 2021;53(2):185&#x02013;94.<pub-id pub-id-type="pmid">33462484</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR46"><label>46.</label><citation-alternatives><element-citation id="ec-CR46" publication-type="journal"><person-group person-group-type="author"><name><surname>Waterworth</surname><given-names>DM</given-names></name><name><surname>Ricketts</surname><given-names>SL</given-names></name><name><surname>Song</surname><given-names>K</given-names></name><name><surname>Chen</surname><given-names>L</given-names></name><name><surname>Zhao</surname><given-names>JH</given-names></name><name><surname>Ripatti</surname><given-names>S</given-names></name><name><surname>Aulchenko</surname><given-names>YS</given-names></name><name><surname>Zhang</surname><given-names>W</given-names></name><name><surname>Yuan</surname><given-names>X</given-names></name><name><surname>Lim</surname><given-names>N</given-names></name><etal/></person-group><article-title>Genetic variants influencing circulating lipid levels and risk of coronary artery disease</article-title><source>Arterioscler Thromb Vasc Biol</source><year>2010</year><volume>30</volume><issue>11</issue><fpage>2264</fpage><lpage>76</lpage><pub-id pub-id-type="pmid">20864672</pub-id>
</element-citation><mixed-citation id="mc-CR46" publication-type="journal">Waterworth DM, Ricketts SL, Song K, Chen L, Zhao JH, Ripatti S, Aulchenko YS, Zhang W, Yuan X, Lim N, et al. Genetic variants influencing circulating lipid levels and risk of coronary artery disease. Arterioscler Thromb Vasc Biol. 2010;30(11):2264&#x02013;76.<pub-id pub-id-type="pmid">20864672</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR47"><label>47.</label><citation-alternatives><element-citation id="ec-CR47" publication-type="journal"><person-group person-group-type="author"><name><surname>Teslovich</surname><given-names>TM</given-names></name><name><surname>Musunuru</surname><given-names>K</given-names></name><name><surname>Smith</surname><given-names>AV</given-names></name><name><surname>Edmondson</surname><given-names>AC</given-names></name><name><surname>Stylianou</surname><given-names>IM</given-names></name><name><surname>Koseki</surname><given-names>M</given-names></name><name><surname>Pirruccello</surname><given-names>JP</given-names></name><name><surname>Ripatti</surname><given-names>S</given-names></name><name><surname>Chasman</surname><given-names>DI</given-names></name><name><surname>Willer</surname><given-names>CJ</given-names></name><etal/></person-group><article-title>Biological, clinical and population relevance of 95 loci for blood lipids</article-title><source>Nature</source><year>2010</year><volume>466</volume><issue>7307</issue><fpage>707</fpage><lpage>713</lpage><pub-id pub-id-type="pmid">20686565</pub-id>
</element-citation><mixed-citation id="mc-CR47" publication-type="journal">Teslovich TM, Musunuru K, Smith AV, Edmondson AC, Stylianou IM, Koseki M, Pirruccello JP, Ripatti S, Chasman DI, Willer CJ, et al. Biological, clinical and population relevance of 95 loci for blood lipids. Nature. 2010;466(7307):707&#x02013;13.<pub-id pub-id-type="pmid">20686565</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR48"><label>48.</label><citation-alternatives><element-citation id="ec-CR48" publication-type="journal"><person-group person-group-type="author"><name><surname>Richardson</surname><given-names>TG</given-names></name><name><surname>Sanderson</surname><given-names>E</given-names></name><name><surname>Palmer</surname><given-names>TM</given-names></name><name><surname>Ala-Korpela</surname><given-names>M</given-names></name><name><surname>Ference</surname><given-names>BA</given-names></name><name><surname>Davey Smith</surname><given-names>G</given-names></name><name><surname>Holmes</surname><given-names>MV</given-names></name></person-group><article-title>Evaluating the relationship between circulating lipoprotein lipids and apolipoproteins with risk of coronary heart disease: a multivariable mendelian randomisation analysis</article-title><source>PLoS Med</source><year>2020</year><volume>17</volume><issue>3</issue><fpage>e1003062</fpage><pub-id pub-id-type="pmid">32203549</pub-id>
</element-citation><mixed-citation id="mc-CR48" publication-type="journal">Richardson TG, Sanderson E, Palmer TM, Ala-Korpela M, Ference BA, Davey Smith G, Holmes MV. Evaluating the relationship between circulating lipoprotein lipids and apolipoproteins with risk of coronary heart disease: a multivariable mendelian randomisation analysis. PLoS Med. 2020;17(3): e1003062.<pub-id pub-id-type="pmid">32203549</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR49"><label>49.</label><citation-alternatives><element-citation id="ec-CR49" publication-type="journal"><person-group person-group-type="author"><name><surname>Karjalainen</surname><given-names>MK</given-names></name><name><surname>Karthikeyan</surname><given-names>S</given-names></name><name><surname>Oliver-Williams</surname><given-names>C</given-names></name><name><surname>Sliz</surname><given-names>E</given-names></name><name><surname>Allara</surname><given-names>E</given-names></name><name><surname>Fung</surname><given-names>WT</given-names></name><name><surname>Surendran</surname><given-names>P</given-names></name><name><surname>Zhang</surname><given-names>W</given-names></name><name><surname>Jousilahti</surname><given-names>P</given-names></name><name><surname>Kristiansson</surname><given-names>K</given-names></name><etal/></person-group><article-title>Genome-wide characterization of circulating metabolic biomarkers</article-title><source>Nature</source><year>2024</year><volume>628</volume><issue>8006</issue><fpage>130</fpage><lpage>138</lpage><pub-id pub-id-type="pmid">38448586</pub-id>
</element-citation><mixed-citation id="mc-CR49" publication-type="journal">Karjalainen MK, Karthikeyan S, Oliver-Williams C, Sliz E, Allara E, Fung WT, Surendran P, Zhang W, Jousilahti P, Kristiansson K, et al. Genome-wide characterization of circulating metabolic biomarkers. Nature. 2024;628(8006):130&#x02013;8.<pub-id pub-id-type="pmid">38448586</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR50"><label>50.</label><citation-alternatives><element-citation id="ec-CR50" publication-type="journal"><person-group person-group-type="author"><name><surname>Kim</surname><given-names>YJ</given-names></name><name><surname>Go</surname><given-names>MJ</given-names></name><name><surname>Hu</surname><given-names>C</given-names></name><name><surname>Hong</surname><given-names>CB</given-names></name><name><surname>Kim</surname><given-names>YK</given-names></name><name><surname>Lee</surname><given-names>JY</given-names></name><name><surname>Hwang</surname><given-names>JY</given-names></name><name><surname>Oh</surname><given-names>JH</given-names></name><name><surname>Kim</surname><given-names>DJ</given-names></name><name><surname>Kim</surname><given-names>NH</given-names></name><etal/></person-group><article-title>Large-scale genome-wide association studies in East Asians identify new genetic loci influencing metabolic traits</article-title><source>Nat Genet</source><year>2011</year><volume>43</volume><issue>10</issue><fpage>990</fpage><lpage>995</lpage><pub-id pub-id-type="pmid">21909109</pub-id>
</element-citation><mixed-citation id="mc-CR50" publication-type="journal">Kim YJ, Go MJ, Hu C, Hong CB, Kim YK, Lee JY, Hwang JY, Oh JH, Kim DJ, Kim NH, et al. Large-scale genome-wide association studies in East Asians identify new genetic loci influencing metabolic traits. Nat Genet. 2011;43(10):990&#x02013;5.<pub-id pub-id-type="pmid">21909109</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR51"><label>51.</label><citation-alternatives><element-citation id="ec-CR51" publication-type="journal"><person-group person-group-type="author"><name><surname>Hagenbeek</surname><given-names>FA</given-names></name><name><surname>Pool</surname><given-names>R</given-names></name><name><surname>van Dongen</surname><given-names>J</given-names></name><name><surname>Draisma</surname><given-names>HHM</given-names></name><name><surname>Jan Hottenga</surname><given-names>J</given-names></name><name><surname>Willemsen</surname><given-names>G</given-names></name><name><surname>Abdellaoui</surname><given-names>A</given-names></name><name><surname>Fedko</surname><given-names>IO</given-names></name><name><surname>den Braber</surname><given-names>A</given-names></name><name><surname>Visser</surname><given-names>PJ</given-names></name><etal/></person-group><article-title>Heritability estimates for 361 blood metabolites across 40 genome-wide association studies</article-title><source>Nat Commun</source><year>2020</year><volume>11</volume><issue>1</issue><fpage>39</fpage><pub-id pub-id-type="pmid">31911595</pub-id>
</element-citation><mixed-citation id="mc-CR51" publication-type="journal">Hagenbeek FA, Pool R, van Dongen J, Draisma HHM, Jan Hottenga J, Willemsen G, Abdellaoui A, Fedko IO, den Braber A, Visser PJ, et al. Heritability estimates for 361 blood metabolites across 40 genome-wide association studies. Nat Commun. 2020;11(1):39.<pub-id pub-id-type="pmid">31911595</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR52"><label>52.</label><citation-alternatives><element-citation id="ec-CR52" publication-type="journal"><person-group person-group-type="author"><name><surname>Lee</surname><given-names>SJ</given-names></name><name><surname>Kim</surname><given-names>S-E</given-names></name><name><surname>Go</surname><given-names>T-H</given-names></name><name><surname>Kang</surname><given-names>DR</given-names></name><name><surname>Jeon</surname><given-names>H-S</given-names></name><name><surname>Kim</surname><given-names>Y-I</given-names></name><name><surname>Cho</surname><given-names>D-H</given-names></name><name><surname>Park</surname><given-names>YJ</given-names></name><name><surname>Lee</surname><given-names>J-H</given-names></name><name><surname>Lee</surname><given-names>J-W</given-names></name><etal/></person-group><article-title>Remnant cholesterol, low-density lipoprotein cholesterol, and incident cardiovascular disease among Koreans: a national population-based study</article-title><source>Eur J Prev Cardiol</source><year>2023</year><volume>30</volume><issue>11</issue><fpage>1142</fpage><lpage>1150</lpage><pub-id pub-id-type="pmid">36753060</pub-id>
</element-citation><mixed-citation id="mc-CR52" publication-type="journal">Lee SJ, Kim S-E, Go T-H, Kang DR, Jeon H-S, Kim Y-I, Cho D-H, Park YJ, Lee J-H, Lee J-W, et al. Remnant cholesterol, low-density lipoprotein cholesterol, and incident cardiovascular disease among Koreans: a national population-based study. Eur J Prev Cardiol. 2023;30(11):1142&#x02013;50.<pub-id pub-id-type="pmid">36753060</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR53"><label>53.</label><citation-alternatives><element-citation id="ec-CR53" publication-type="journal"><person-group person-group-type="author"><name><surname>Burnett</surname><given-names>JR</given-names></name><name><surname>Hooper</surname><given-names>AJ</given-names></name><name><surname>Hegele</surname><given-names>RA</given-names></name></person-group><article-title>Remnant cholesterol and atherosclerotic cardiovascular disease risk</article-title><source>J Am Coll Cardiol</source><year>2020</year><volume>76</volume><issue>23</issue><fpage>2736</fpage><lpage>2739</lpage><pub-id pub-id-type="pmid">33272367</pub-id>
</element-citation><mixed-citation id="mc-CR53" publication-type="journal">Burnett JR, Hooper AJ, Hegele RA. Remnant cholesterol and atherosclerotic cardiovascular disease risk. J Am Coll Cardiol. 2020;76(23):2736&#x02013;9.<pub-id pub-id-type="pmid">33272367</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR54"><label>54.</label><citation-alternatives><element-citation id="ec-CR54" publication-type="journal"><person-group person-group-type="author"><name><surname>Schwartz</surname><given-names>EA</given-names></name><name><surname>Reaven</surname><given-names>PD</given-names></name></person-group><article-title>Lipolysis of triglyceride-rich lipoproteins, vascular inflammation, and atherosclerosis</article-title><source>Biochim Biophys Acta</source><year>2012</year><volume>1821</volume><issue>5</issue><fpage>858</fpage><lpage>866</lpage><pub-id pub-id-type="pmid">22001233</pub-id>
</element-citation><mixed-citation id="mc-CR54" publication-type="journal">Schwartz EA, Reaven PD. Lipolysis of triglyceride-rich lipoproteins, vascular inflammation, and atherosclerosis. Biochim Biophys Acta. 2012;1821(5):858&#x02013;66.<pub-id pub-id-type="pmid">22001233</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR55"><label>55.</label><citation-alternatives><element-citation id="ec-CR55" publication-type="journal"><person-group person-group-type="author"><name><surname>Golledge</surname><given-names>J</given-names></name><name><surname>Norman</surname><given-names>PE</given-names></name></person-group><article-title>Atherosclerosis and abdominal aortic aneurysm: cause, response, or common risk factors?</article-title><source>Arterioscler Thromb Vasc Biol</source><year>2010</year><volume>30</volume><issue>6</issue><fpage>1075</fpage><lpage>1077</lpage><pub-id pub-id-type="pmid">20484703</pub-id>
</element-citation><mixed-citation id="mc-CR55" publication-type="journal">Golledge J, Norman PE. Atherosclerosis and abdominal aortic aneurysm: cause, response, or common risk factors? Arterioscler Thromb Vasc Biol. 2010;30(6):1075&#x02013;7.<pub-id pub-id-type="pmid">20484703</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR56"><label>56.</label><citation-alternatives><element-citation id="ec-CR56" publication-type="journal"><person-group person-group-type="author"><name><surname>Golledge</surname><given-names>J</given-names></name></person-group><article-title>Abdominal aortic aneurysm: update on pathogenesis and medical treatments</article-title><source>Nat Rev Cardiol</source><year>2019</year><volume>16</volume><issue>4</issue><fpage>225</fpage><lpage>242</lpage><pub-id pub-id-type="pmid">30443031</pub-id>
</element-citation><mixed-citation id="mc-CR56" publication-type="journal">Golledge J. Abdominal aortic aneurysm: update on pathogenesis and medical treatments. Nat Rev Cardiol. 2019;16(4):225&#x02013;42.<pub-id pub-id-type="pmid">30443031</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR57"><label>57.</label><citation-alternatives><element-citation id="ec-CR57" publication-type="journal"><person-group person-group-type="author"><name><surname>Zheng</surname><given-names>S</given-names></name><name><surname>Tsao</surname><given-names>PS</given-names></name><name><surname>Pan</surname><given-names>C</given-names></name></person-group><article-title>Abdominal aortic aneurysm and cardiometabolic traits share strong genetic susceptibility to lipid metabolism and inflammation</article-title><source>Nat Commun</source><year>2024</year><volume>15</volume><issue>1</issue><fpage>5652</fpage><pub-id pub-id-type="pmid">38969659</pub-id>
</element-citation><mixed-citation id="mc-CR57" publication-type="journal">Zheng S, Tsao PS, Pan C. Abdominal aortic aneurysm and cardiometabolic traits share strong genetic susceptibility to lipid metabolism and inflammation. Nat Commun. 2024;15(1):5652.<pub-id pub-id-type="pmid">38969659</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR58"><label>58.</label><mixed-citation publication-type="other"> Taddei C, Zhou B, Bixby H, Carrillo-Larco RM, Danaei G, Jackson R, Farzadfar F, Sophiea MK, Di Cesare M, Iurilli MLC et al: Repositioning of the global epicentre of non-optimal cholesterol. Nature 2020, 582(7810):73.</mixed-citation></ref><ref id="CR59"><label>59.</label><citation-alternatives><element-citation id="ec-CR59" publication-type="journal"><person-group person-group-type="author"><name><surname>Song</surname><given-names>P</given-names></name><name><surname>He</surname><given-names>Y</given-names></name><name><surname>Adeloye</surname><given-names>D</given-names></name><name><surname>Zhu</surname><given-names>Y</given-names></name><name><surname>Ye</surname><given-names>X</given-names></name><name><surname>Yi</surname><given-names>Q</given-names></name><name><surname>Rahimi</surname><given-names>K</given-names></name><name><surname>Rudan</surname><given-names>I</given-names></name><collab>Global Health Epidemiology Research G</collab></person-group><article-title>The global and regional prevalence of abdominal aortic aneurysms: a systematic review and modeling analysis</article-title><source>Ann Surg</source><year>2023</year><volume>277</volume><issue>6</issue><fpage>912</fpage><lpage>9</lpage><pub-id pub-id-type="pmid">36177847</pub-id>
</element-citation><mixed-citation id="mc-CR59" publication-type="journal">Song P, He Y, Adeloye D, Zhu Y, Ye X, Yi Q, Rahimi K, Rudan I, Global Health Epidemiology Research G. The global and regional prevalence of abdominal aortic aneurysms: a systematic review and modeling analysis. Ann Surg. 2023;277(6):912&#x02013;9.<pub-id pub-id-type="pmid">36177847</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR60"><label>60.</label><citation-alternatives><element-citation id="ec-CR60" publication-type="journal"><person-group person-group-type="author"><name><surname>Goh</surname><given-names>RSJ</given-names></name><name><surname>Chong</surname><given-names>B</given-names></name><name><surname>Jayabaskaran</surname><given-names>J</given-names></name><name><surname>Jauhari</surname><given-names>SM</given-names></name><name><surname>Chan</surname><given-names>SP</given-names></name><name><surname>Kueh</surname><given-names>MTW</given-names></name><name><surname>Shankar</surname><given-names>K</given-names></name><name><surname>Li</surname><given-names>H</given-names></name><name><surname>Chin</surname><given-names>YH</given-names></name><name><surname>Kong</surname><given-names>G</given-names></name><etal/></person-group><article-title>The burden of cardiovascular disease in Asia from 2025 to 2050: a forecast analysis for East Asia, South Asia, South-East Asia, Central Asia, and high-income Asia Pacific regions</article-title><source>Lancet Reg Health</source><year>2024</year><volume>49</volume><fpage>101138</fpage></element-citation><mixed-citation id="mc-CR60" publication-type="journal">Goh RSJ, Chong B, Jayabaskaran J, Jauhari SM, Chan SP, Kueh MTW, Shankar K, Li H, Chin YH, Kong G, et al. The burden of cardiovascular disease in Asia from 2025 to 2050: a forecast analysis for East Asia, South Asia, South-East Asia, Central Asia, and high-income Asia Pacific regions. Lancet Reg Health. 2024;49: 101138.</mixed-citation></citation-alternatives></ref><ref id="CR61"><label>61.</label><citation-alternatives><element-citation id="ec-CR61" publication-type="journal"><person-group person-group-type="author"><name><surname>Bhatnagar</surname><given-names>A</given-names></name></person-group><article-title>Environmental determinants of cardiovascular disease</article-title><source>Circ Res</source><year>2017</year><volume>121</volume><issue>2</issue><fpage>162</fpage><lpage>180</lpage><pub-id pub-id-type="pmid">28684622</pub-id>
</element-citation><mixed-citation id="mc-CR61" publication-type="journal">Bhatnagar A. Environmental determinants of cardiovascular disease. Circ Res. 2017;121(2):162&#x02013;80.<pub-id pub-id-type="pmid">28684622</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR62"><label>62.</label><citation-alternatives><element-citation id="ec-CR62" publication-type="journal"><person-group person-group-type="author"><name><surname>Bays</surname><given-names>HE</given-names></name><name><surname>Kulkarni</surname><given-names>A</given-names></name><name><surname>German</surname><given-names>C</given-names></name><name><surname>Satish</surname><given-names>P</given-names></name><name><surname>Iluyomade</surname><given-names>A</given-names></name><name><surname>Dudum</surname><given-names>R</given-names></name><name><surname>Thakkar</surname><given-names>A</given-names></name><name><surname>Rifai</surname><given-names>MA</given-names></name><name><surname>Mehta</surname><given-names>A</given-names></name><name><surname>Thobani</surname><given-names>A</given-names></name><etal/></person-group><article-title>Ten things to know about ten cardiovascular disease risk factors - 2022</article-title><source>Am J Prev Cardiol</source><year>2022</year><volume>10</volume><fpage>100342</fpage><pub-id pub-id-type="pmid">35517870</pub-id>
</element-citation><mixed-citation id="mc-CR62" publication-type="journal">Bays HE, Kulkarni A, German C, Satish P, Iluyomade A, Dudum R, Thakkar A, Rifai MA, Mehta A, Thobani A, et al. Ten things to know about ten cardiovascular disease risk factors - 2022. Am J Prev Cardiol. 2022;10: 100342.<pub-id pub-id-type="pmid">35517870</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR63"><label>63.</label><citation-alternatives><element-citation id="ec-CR63" publication-type="journal"><person-group person-group-type="author"><name><surname>Sturzebecher</surname><given-names>PE</given-names></name><name><surname>Katzmann</surname><given-names>JL</given-names></name><name><surname>Laufs</surname><given-names>U</given-names></name></person-group><article-title>What is &#x02018;remnant cholesterol&#x02019;?</article-title><source>Eur Heart J</source><year>2023</year><volume>44</volume><issue>16</issue><fpage>1446</fpage><lpage>8</lpage><pub-id pub-id-type="pmid">36650915</pub-id>
</element-citation><mixed-citation id="mc-CR63" publication-type="journal">Sturzebecher PE, Katzmann JL, Laufs U. What is &#x02018;remnant cholesterol&#x02019;? Eur Heart J. 2023;44(16):1446&#x02013;8.<pub-id pub-id-type="pmid">36650915</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR64"><label>64.</label><mixed-citation publication-type="other"> Biobank Japan access results. Available from: <ext-link ext-link-type="uri" xlink:href="https://pheweb.jp/">https://pheweb.jp/</ext-link>. Cited 2025 January 2</mixed-citation></ref><ref id="CR65"><label>65.</label><mixed-citation publication-type="other"> FinnGen access results. Available from: <ext-link ext-link-type="uri" xlink:href="https://www.finngen.fi/en/access_results">https://www.finngen.fi/en/access_results</ext-link>. Cited 2025 January 2</mixed-citation></ref></ref-list></back></article>