# Development and recalibration of a multivariable type 1 diabetes prediction model for type 1 diabetes across multiple screening studies.

<!DOCTYPE article PUBLIC "-//NLM//DTD JATS (Z39.96) Journal Archiving and Interchange DTD with MathML3 v1.3 20210610//EN" "JATS-archivearticle1-3-mathml3.dtd"> 
<article xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article" dtd-version="1.3"><?properties open_access?><?DTDIdentifier.IdentifierValue -//Springer-Verlag//DTD A++ V2.4//EN?><?DTDIdentifier.IdentifierType public?><?SourceDTD.DTDName A++V2.4.dtd?><?SourceDTD.Version 2.4?><?ConverterInfo.XSLTName springer2nlmx2.xsl?><?ConverterInfo.Version 1?><processing-meta base-tagset="archiving" mathml-version="3.0" table-model="xhtml" tagset-family="jats"><restricted-by>pmc</restricted-by></processing-meta><front><journal-meta><journal-id journal-id-type="nlm-ta">BMC Med</journal-id><journal-id journal-id-type="iso-abbrev">BMC Med</journal-id><journal-title-group><journal-title>BMC Medicine</journal-title></journal-title-group><issn pub-type="epub">1741-7015</issn><publisher><publisher-name>BioMed Central</publisher-name><publisher-loc>London</publisher-loc></publisher></journal-meta>
<article-meta><article-id pub-id-type="pmcid">12281708</article-id><article-id pub-id-type="pmid">40691800</article-id>
<article-id pub-id-type="publisher-id">4225</article-id><article-id pub-id-type="doi">10.1186/s12916-025-04225-5</article-id><article-categories><subj-group subj-group-type="heading"><subject>Research</subject></subj-group></article-categories><title-group><article-title>Development and recalibration&#x000a0;of a&#x000a0;multivariable&#x000a0;type 1 diabetes prediction model for&#x000a0;type 1 diabetes across multiple screening studies</article-title></title-group><contrib-group><contrib contrib-type="author"><name><surname>Templeman</surname><given-names>Erin L.</given-names></name><xref ref-type="aff" rid="Aff1">1</xref></contrib><contrib contrib-type="author"><name><surname>Ferrat</surname><given-names>Lauric A.</given-names></name><xref ref-type="aff" rid="Aff1">1</xref><xref ref-type="aff" rid="Aff2">2</xref></contrib><contrib contrib-type="author"><name><surname>Parikh</surname><given-names>Hemang M.</given-names></name><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author"><name><surname>You</surname><given-names>Lu</given-names></name><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author"><name><surname>Triolo</surname><given-names>Taylor M.</given-names></name><xref ref-type="aff" rid="Aff4">4</xref></contrib><contrib contrib-type="author"><name><surname>Steck</surname><given-names>Andrea K.</given-names></name><xref ref-type="aff" rid="Aff4">4</xref></contrib><contrib contrib-type="author"><name><surname>Hagopian</surname><given-names>William A.</given-names></name><xref ref-type="aff" rid="Aff5">5</xref></contrib><contrib contrib-type="author"><name><surname>Vehik</surname><given-names>Kendra</given-names></name><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author"><name><surname>Onengut-Gumuscu</surname><given-names>Suna</given-names></name><xref ref-type="aff" rid="Aff6">6</xref></contrib><contrib contrib-type="author"><name><surname>Gottlieb</surname><given-names>Peter A.</given-names></name><xref ref-type="aff" rid="Aff7">7</xref></contrib><contrib contrib-type="author"><name><surname>Rich</surname><given-names>Stephen S.</given-names></name><xref ref-type="aff" rid="Aff6">6</xref></contrib><contrib contrib-type="author"><name><surname>Krischer</surname><given-names>Jeffery P.</given-names></name><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author" equal-contrib="yes"><name><surname>Redondo</surname><given-names>Maria J.</given-names></name><xref ref-type="aff" rid="Aff8">8</xref></contrib><contrib contrib-type="author" corresp="yes" equal-contrib="yes"><name><surname>Oram</surname><given-names>Richard A.</given-names></name><address><email>R.Oram@exeter.ac.uk</email></address><xref ref-type="aff" rid="Aff1">1</xref></contrib><contrib contrib-type="author" id="IA1"><collab>the Type 1 diabetes TrialNet Study Group</collab></contrib><aff id="Aff1"><label>1</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/03yghzc09</institution-id><institution-id institution-id-type="GRID">grid.8391.3</institution-id><institution-id institution-id-type="ISNI">0000 0004 1936 8024</institution-id><institution>Department of Clinical and Biomedical Sciences, </institution><institution>University of Exeter, </institution></institution-wrap>Exeter, UK </aff><aff id="Aff2"><label>2</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/01swzsf04</institution-id><institution-id institution-id-type="GRID">grid.8591.5</institution-id><institution-id institution-id-type="ISNI">0000 0001 2175 2154</institution-id><institution>Department of Genetic Medicine and Development, </institution><institution>Univerisity of Geneva, </institution></institution-wrap>Geneva, Switzerland </aff><aff id="Aff3"><label>3</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/032db5x82</institution-id><institution-id institution-id-type="GRID">grid.170693.a</institution-id><institution-id institution-id-type="ISNI">0000 0001 2353 285X</institution-id><institution>Health Informatics Institute, </institution><institution>Morsani College of Medicine, University of South Florida, </institution></institution-wrap>Tampa, FL USA </aff><aff id="Aff4"><label>4</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/03wmf1y16</institution-id><institution-id institution-id-type="GRID">grid.430503.1</institution-id><institution-id institution-id-type="ISNI">0000 0001 0703 675X</institution-id><institution>Davis Center for Diabetes, </institution><institution>University of Colorado Anschutz Medical Campus, </institution></institution-wrap>Aurora, CO USA </aff><aff id="Aff5"><label>5</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/02ets8c94</institution-id><institution-id institution-id-type="ISNI">0000 0001 2296 1126</institution-id><institution>Indiana University School of Medicine, </institution></institution-wrap>Indiana, IN USA </aff><aff id="Aff6"><label>6</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/0153tk833</institution-id><institution-id institution-id-type="GRID">grid.27755.32</institution-id><institution-id institution-id-type="ISNI">0000 0000 9136 933X</institution-id><institution> Department of Genome Sciences, </institution><institution>University of Virginia School of Medicine, </institution></institution-wrap>Charlottesville, VA USA </aff><aff id="Aff7"><label>7</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/03x0d4x24</institution-id><institution-id institution-id-type="GRID">grid.280838.9</institution-id><institution-id institution-id-type="ISNI">0000 0000 9212 4713</institution-id><institution>Pacific Northwest Research Institute, </institution></institution-wrap>Seattle, WA USA </aff><aff id="Aff8"><label>8</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/05cz92x43</institution-id><institution-id institution-id-type="GRID">grid.416975.8</institution-id><institution-id institution-id-type="ISNI">0000 0001 2200 2638</institution-id><institution>Baylor College of Medicine, Texas Children&#x02019;s Hospital, </institution></institution-wrap>Houston, TX USA </aff></contrib-group><pub-date pub-type="epub"><day>21</day><month>7</month><year>2025</year></pub-date><pub-date pub-type="pmc-release"><day>21</day><month>7</month><year>2025</year></pub-date><pub-date pub-type="collection"><year>2025</year></pub-date><volume>23</volume><elocation-id>433</elocation-id><history><date date-type="received"><day>6</day><month>1</month><year>2025</year></date><date date-type="accepted"><day>17</day><month>6</month><year>2025</year></date></history><permissions><copyright-statement>&#x000a9; The Author(s) 2025</copyright-statement><copyright-year>2025</copyright-year><license><ali:license_ref specific-use="textmining" content-type="ccbyncndlicense">https://creativecommons.org/licenses/by-nc-nd/4.0/</ali:license_ref><license-p><bold>Open Access</bold> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article&#x02019;s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article&#x02019;s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <ext-link ext-link-type="uri" xlink:href="https://creativecommons.org/licenses/by-nc-nd/4.0/">http://creativecommons.org/licenses/by-nc-nd/4.0/</ext-link>.</license-p></license></permissions><abstract id="Abs1"><sec><title>Background</title><p id="Par1">Accurate type 1 diabetes prediction is important to facilitate screening for pre-clinical type 1 diabetes to enable potential early disease-modifying interventions and to reduce the risk of severe presentation with diabetic ketoacidosis. We aimed to assess the generalisability of a prediction model developed in children followed from birth. Additionally, we sought to create an application for easy calculation and visualisation of individualised risk prediction.</p></sec><sec><title>Methods</title><p id="Par2">We developed and internally validated a stratified prediction model combining a genetic risk score, age, islet autoantibodies, and family history using data from children followed since birth by The Environmental Determinants of Diabetes in the Young (TEDDY) study. We tested the validity of the model through external validation in the Type 1 Diabetes TrialNet Pathway to Prevention study, which conducts cross-sectional screening in relatives of people with type 1 diabetes. We recalibrated the model by adjusting for baseline risk and selection criteria in TrialNet using logistic recalibration to improve calibration across all ages.</p></sec><sec><title>Results</title><p id="Par3">The study included 7798 TEDDY and 4068 TrialNet participants, with 305 (4%) and 1373 (34%) developing type 1 diabetes, respectively. The stratified model showed similar discriminative ability in autoantibody-positive participants across TEDDY and TrialNet, but inferior calibration in TrialNet (Brier score 0.40 [95% CI 0.38,0.43]). Adjustment for baseline risk and selection criteria in TrialNet using logistic recalibration improved calibration across all ages (Brier score 0.16 [0.14,0.17]; <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.001). A web calculator was developed to visualise individual risk estimates (<ext-link ext-link-type="uri" xlink:href="https://t1dpredictor.diabetesgenes.org">https://t1dpredictor.diabetesgenes.org</ext-link>).</p></sec><sec><title>Conclusions</title><p id="Par4">A stratified model incorporating the type 1 diabetes genetic risk score, family history, age, and autoantibody status can predict type 1 diabetes risk with improved accuracy, but may need recalibration depending on the screening strategy.</p></sec><sec><title>Supplementary Information</title><p>The online version contains supplementary material available at 10.1186/s12916-025-04225-5.</p></sec></abstract><kwd-group xml:lang="en"><title>Keywords</title><kwd>Type 1 diabetes</kwd><kwd>Prediction</kwd><kwd>Recalibration</kwd><kwd>Adjustment</kwd><kwd>Genetics</kwd><kwd>Autoantibody</kwd><kwd>FDR</kwd></kwd-group><funding-group><award-group><funding-source><institution>Randox Laboratories</institution></funding-source></award-group></funding-group><funding-group><award-group><funding-source><institution-wrap><institution-id institution-id-type="FundRef">https://doi.org/10.13039/100008664</institution-id><institution>Juvenile Diabetes Research Foundation United Kingdom</institution></institution-wrap></funding-source><award-id>3-SRA-2019-827-S-B</award-id><award-id>3-SRA-2019-827-S-B</award-id></award-group></funding-group><funding-group><award-group><funding-source><institution-wrap><institution-id institution-id-type="FundRef">https://doi.org/10.13039/100000002</institution-id><institution>National Institutes of Health</institution></institution-wrap></funding-source><award-id>R01 DK121843</award-id><award-id>K23 DK136931</award-id><award-id>R01 DK121843</award-id><award-id>R01 DK121843</award-id><award-id>R01 DK121843</award-id><award-id>R01 DK121843</award-id><award-id>R01 DK121843</award-id><award-id>R01 DK121843</award-id><award-id>R01 DK121843</award-id></award-group></funding-group><custom-meta-group><custom-meta><meta-name>issue-copyright-statement</meta-name><meta-value>&#x000a9; BioMed Central Ltd., part of Springer Nature 2025</meta-value></custom-meta></custom-meta-group></article-meta></front><body><sec id="Sec1"><title>Background</title><p id="Par27">Teplizumab, which is approved by the U.S. Food and Drug Administration (FDA) for stage 2 pre-clinical type 1 diabetes, has been shown to delay the onset of stage 3 (clinical) disease by a median of 3 years [<xref ref-type="bibr" rid="CR1">1</xref>, <xref ref-type="bibr" rid="CR2">2</xref>]. The need to identify individuals eligible for therapy, coupled with growing evidence for lower risk of diabetic ketoacidosis in individuals identified in early stages of pre-clinical type 1 diabetes [<xref ref-type="bibr" rid="CR3">3</xref>], has driven heightened interest in screening and risk prediction for type 1 diabetes. The stages of pre-clinical type 1 diabetes include individuals with multiple islet autoantibodies (stage 1), and individuals with evidence of dysglycaemia in addition to islet autoantibodies (stage 2) [<xref ref-type="bibr" rid="CR4">4</xref>]. A combination of variables, including genetics, age, number and specificity of islet autoantibodies, and metabolic measures, affect the risk of progression to clinical stage 3 type 1 diabetes and account for the majority of type 1 diabetes risk [<xref ref-type="bibr" rid="CR5">5</xref>, <xref ref-type="bibr" rid="CR6">6</xref>].
</p><p id="Par28">Autoantibody screening for pre-clinical type 1 diabetes has been performed in natural history studies with a variety of designs [<xref ref-type="bibr" rid="CR7">7</xref>]. These designs include family studies, such as the Type 1 Diabetes TrialNet Pathway to Prevention Study (TrialNet) [<xref ref-type="bibr" rid="CR8">8</xref>, <xref ref-type="bibr" rid="CR9">9</xref>], birth cohort studies of genetically high-risk infants, e.g. The Environmental Determinants of Diabetes in the Young (TEDDY) study [<xref ref-type="bibr" rid="CR10">10</xref>&#x02013;<xref ref-type="bibr" rid="CR12">12</xref>], and cross-sectional population screening of children, such as Fr1DA [<xref ref-type="bibr" rid="CR13">13</xref>]. Several methods to stratify individuals for future type 1 diabetes risk have been developed and tested in these studies. However, there are few assessments of whether models developed in one screening setting (e.g. longitudinal natural history studies) can be used in other approaches (e.g. cross-sectional studies of relatives at risk). The transferability of a predictive model would support its validity, robustness, and generalizability and facilitate accurate risk prediction in a range of settings. Furthermore, the translation of predictive models into clinical research and practice requires freely accessible, user-friendly tools that provide individual risk prediction. Characteristics such as initial risk, age of screening, monitoring intensity and methods, genetic selection, and other study-specific variables may further influence the accuracy, calibration, and overall performance of prediction models. The availability of tools that incorporate multiple predictive variables for type 1 diabetes is currently limited but is common in other diseases (e.g. Q-risk&#x000a0;for cardiovascular disease risk) [<xref ref-type="bibr" rid="CR14">14</xref>] and is likely to play a significant role as screening for type 1 diabetes exapands.</p><p id="Par29">We aimed to assess the generalisability of a type 1 diabetes prediction model developed in TEDDY, a birth cohort study that incorporates genetics, autoantibody status, age, and family history. We validated this model externally in the TrialNet study, a cohort of autoantibody positive relatives of individuals with type 1 diabetes. All TEDDY participants were included in model development to ensure diverse risk profiles. In TrialNet, we focused on autoantibody-positive participants to assess the generalisability of the model in this high-risk population, where autoantibody positivity is a key marker of progression to type 1 diabetes. Furthermore, we developed a type 1 diabetes risk prediction web application to quantify and visualise individual risk and allow use by healthcare professionals and researchers to support risk stratification and decision-making regarding follow-up and potential early interventions.</p></sec><sec id="Sec2"><title>Methods</title><p id="Par30">In summary, we developed a type 1 diabetes prediction model using data from the TEDDY study. For external validation, we used the TrialNet dataset. To adjust for the different selection criteria between the two studies, we used logistic recalibration. The steps taken are illustrated in Fig. <xref rid="Fig1" ref-type="fig">1</xref>.<fig id="Fig1"><label>Fig. 1</label><caption><p>Flow chart for training, validation, and recalibration of type 1 diabetes prediction model using the TEDDY and TrialNet cohorts. This figure illustrates the methods and results of a type 1 diabetes (T1D) prediction model evaluated across three stages: training, validation, and recalibration. Each model included the variables: age of the individual, GRS2, number of autoantibodies, and first-degree relative status; however, each differs in modelling approach (original model: linear cox model with landmarking approach, stratified model: stratified cox model with landmarking approach, recalibrated model: stratified cox model, recalibrated in three age groups within TrialNet, screened age 1&#x02013;7, screened age 8&#x02013;17, and screened age 18&#x02009;+). The equations of each model are detailed in Additional File 1&#x02014;Supplementary Methods &#x02013; Model Structure. Each stage&#x02019;s results are summarised in the right-hand column, with green arrows indicating satisfactory performance and orange arrows marking an area needing improvement</p></caption><graphic xlink:href="12916_2025_4225_Fig1_HTML" id="MO1"/></fig></p><sec id="Sec3"><title>Participants</title><sec id="Sec4"><title>Discovery cohort: TEDDY</title><p id="Par31">The TEDDY study followed participants from birth who carried a type 1 diabetes genetic risk based upon their HLA DR-DQ haplotype, as previously described [<xref ref-type="bibr" rid="CR6">6</xref>]. A total of 424,047 children were screened from the USA, Sweden, Germany, and Finland, with 8676 children enrolled from September 2004 to February 2010. Of the 8676 TEDDY enrollees, 7798 were analysed on the basis of having available results for islet autoantibody testing and genotyping, as previously described [<xref ref-type="bibr" rid="CR11">11</xref>, <xref ref-type="bibr" rid="CR15">15</xref>]. Risk at baseline can be defined from each landmark age.</p></sec><sec id="Sec5"><title>External validation cohort: TrialNet Pathway to Prevention</title><p id="Par32">TrialNet Pathway to Prevention is an observational study that prospectively follows autoantibody-positive relatives of people living with type 1 diabetes. Our analysis included 4680 participants enrolled between April 2004 and April 2023 who had at least one confirmed (positive on 2 consecutive tests) islet autoantibody and had a baseline oral glucose tolerance test (OGTT) within a year after testing autoantibody positive. Due to small number of adults (aged&#x02009;&#x0003e;&#x02009;18 years) with 3 autoantibodies these people were grouped with 2 autoantibody positive participants into a&#x02009;&#x0003e;&#x02009;2 autoantibody group for calculation of individual risk prediction analyses. Risk at baseline for TrialNet participants is defined as the risk from their first recorded visit.</p></sec><sec id="Sec6"><title>Choice of external validation cohort</title><p id="Par33">TrialNet was selected as a validation cohort due to its large number of T1D cases, broad age range, and intensive follow-up. This cohort selected by family history and autoantibody-positivity has a higher risk for T1D than TEDDY. However, the risk factors studied tend to consistently impact risk of type 1 diabetes independent of study design, and major differences include information accounted for in the models we developed (age, family history, genetic risk, and antibody status).</p><p id="Par34">To assess the applicability of our model despite these underlying differences, we examined whether the original model would remain predictive when applied in TrialNet, or whether a simple recalibration would enhance accuracy. The recalibration approach may allow a range of smaller studies or screening programmes to take advantage of a refined version of existing prediction models. This method aims to preserve the model&#x02019;s generalisability, maintaining broader applicability to lower-risk groups while improving prediction for high-risk populations.</p><p id="Par35">All study participants gave informed consent, and the studies were approved by the ethics committee at each site. The proportion of missing data was small in both dataset (&#x0003c;5%), thus we did a complete case analysis as the loss of information was minimal. Research ethics board approval was not required for this analysis as it involves de-identified and anonymised data. </p></sec></sec><sec id="Sec7"><title>Procedures</title><sec id="Sec8"><title>Study protocol</title><p id="Par36">TEDDY participants were monitored every 3 months until 4 years of age with autoantibody tests, and oral glucose tolerance tests (OGTTs) in autoantibody positive participants, and then either 3- or 6-month intervals depending on autoantibody status. After age 4, autoantibody-negative children were assessed every 6 months while children with any positive autoantibody remained on quarterly visits until 15 years of age or until the onset of clinical type 1 diabetes [<xref ref-type="bibr" rid="CR15">15</xref>, <xref ref-type="bibr" rid="CR16">16</xref>].</p><p id="Par37">TrialNet participants were monitored at 6- or 12-month intervals (with autoantibody and OGTT testing) depending on their estimated risk. Single autoantibody-positive participants were followed on an annual basis. Multiple autoantibody-positive were usually followed on a semi-annual basis [<xref ref-type="bibr" rid="CR8">8</xref>]. All participants were monitored until progression to type 1 diabetes or entered into a prevention clinical trial.</p></sec><sec id="Sec9"><title>Autoantibody assays</title><p id="Par38">TEDDY and TrialNet participants were tested for autoantibodies to glutamic acid decarboxylase (GADA), insulin (micro-insulin antibody assay [mIAA]), and insulinoma associated antigen 2 (IA-2A). GADA, mIAA, and IA-2A autoantibodies were used in the development of the original prediction model, and therefore to accurately assess generalisability across datasets, we assessed the positivity of these autoantibodies. GADA, mIAA, and IA-2A were measured by radioimmunoassay in the TrialNet Core Laboratory at the Barbara Davis Center for TrialNet participants. Autoantibodies for TEDDY participants were measured in two separate core laboratories, as previously described [<xref ref-type="bibr" rid="CR17">17</xref>].</p></sec><sec id="Sec10"><title>Diagnosis of type 1 diabetes</title><p id="Par39">Type 1 diabetes was diagnosed based on hyperglycaemia, including fasting plasma glucose&#x02009;&#x02265;&#x02009;7.0 mmol/L (&#x02265;&#x02009;126 mg/dL), 2-h plasma glucose&#x02009;&#x02265;&#x02009;11.1 mmol/L (&#x02265;&#x02009;200 mg/dL) after 75 g oral glucose, or HbA1c&#x02009;&#x02265;&#x02009;6.5% (&#x02265;&#x02009;48 mmol/mol), with diagnoses confirmed if symptomatic type 1 diabetes or testing confirmed on two separate occasions if asymptomatic.</p></sec><sec id="Sec11"><title>SNP Genotyping and Genetic Risk Score (GRS) computation</title><p id="Par40">TEDDY participants were genotyped on an Illumina Infinium ImmunoChip SNP array. Prior to imputation, quality control filtering was conducted on SNP variants based on missing genotype rates (&#x0003c;&#x02009;1%), Hardy&#x02013;Weinberg equilibrium (<italic>p</italic>&#x02009;&#x0003c;&#x02009;1&#x02009;&#x000d7;&#x02009;10<sup>&#x02212;6</sup>), and minor allele frequency (&#x0003c;&#x02009;1%). The ImmunoChip data was imputed using the 1000 Genomes reference panel. Sharp et al. [<xref ref-type="bibr" rid="CR18">18</xref>] include 42 SNPs which were directly genotyped, and we imputed 25 SNPs, 6 non-HLA, and 19 HLA. Twenty-one SNPs had an <italic>r</italic><sup>2</sup>&#x02009;&#x02265;&#x02009;0.75 and four SNPs were imputed with <italic>r</italic><sup>2</sup>&#x02009;=&#x02009;0.358&#x02013;0.544.</p><p id="Par41">TrialNet participants were genotyped on an Illumina Infinium T1DExomeChip SNP array. TrialNet participants genotyping data was imputed using the TOPMed reference panel as previously described [<xref ref-type="bibr" rid="CR19">19</xref>]. Sharp et al. [<xref ref-type="bibr" rid="CR18">18</xref>] include 30 SNPs which were directly genotyped, and we imputed the remaining 37 SNPs (see Additional File 1&#x02014;Supplementary Methods &#x02013; TrialNet Genotyping), using the TOPMed reference panel [<xref ref-type="bibr" rid="CR20">20</xref>, <xref ref-type="bibr" rid="CR21">21</xref>] and Michigan Imputation Server [<xref ref-type="bibr" rid="CR22">22</xref>], respectively. The median <italic>r</italic><sup>2</sup> of the non-HLA SNPs was 0.997 (min&#x02009;=&#x02009;0.858, max&#x02009;=&#x02009;0.999), and the median <italic>r</italic><sup>2</sup> of the HLA SNPs was 0.998 (min&#x02009;=&#x02009;0.925, max&#x02009;=&#x02009;0.999). GRS values computed using different high-quality imputation reference panels (including 1000 Genomes and TOPMed) are highly correlated (Pearson <italic>r</italic>&#x02009;&#x0003e;&#x02009;0.98), supporting the robustness of cross-panel comparisons [<xref ref-type="bibr" rid="CR23">23</xref>].</p><p id="Par42">Code to generate the HLA interaction part of the type 1 diabetes GRS is freely available online (<ext-link ext-link-type="uri" xlink:href="https://github.com/sethsh7/hla-prs-toolkit">https://github.com/sethsh7/hla-prs-toolkit</ext-link>). To facilitate simpler GRS generation and comparison, especially when imputation and reference panel selection present challenges, we have developed a streamlined version of the GRS code. This code aims to mitigate these complexities and is freely available online (<ext-link ext-link-type="uri" xlink:href="https://github.com/sethsh7/prsedm">https://github.com/sethsh7/prsedm</ext-link>) [<xref ref-type="bibr" rid="CR24">24</xref>].</p></sec></sec><sec id="Sec12"><title>Statistical analysis</title><sec id="Sec13"><title>Model training</title><p id="Par43">We aimed to refine a previously published prediction model [<xref ref-type="bibr" rid="CR6">6</xref>], utilising the variables and methodology. The original prediction model combined the GRS (specifically GRS2 [<xref ref-type="bibr" rid="CR18">18</xref>]), number of positive autoantibodies, and the presence of a first-degree relative (FDR) with type 1 diabetes.</p><p id="Par44">The updated stratified model, uses a stratified Cox proportional hazards model with a landmarking approach, allowing scores to be updated as new information becomes available [<xref ref-type="bibr" rid="CR6">6</xref>, <xref ref-type="bibr" rid="CR25">25</xref>]. The stratified prediction model accounts for individual-specific variables that proportionally change the mean risk of type 1 diabetes, with the mean risk representing the average risk across the entire cohort without considering these individual variables (see Additional File 1&#x02014;Supplementary Methods &#x02013; Model Structure). To explore potential differential effects, we included interaction terms between the number of autoantibodies and the GRS, as well as between the number of autoantibodies and FDR status. This approach allows us to assess whether heritable risk varies according to autoantibody status.</p><p id="Par45">Predictors of disease progression can change over time, which can violate the proportional hazards (PH) assumption in a Cox proportional hazards model. Prior to seroconversion, genetics and family history are the most predictive variables of type 1 diabetes and, therefore, have the most influence on diabetes risk [<xref ref-type="bibr" rid="CR5">5</xref>]. However, when an individual has stage 1 or 2 type 1 diabetes, metabolic and autoantibody features are most predictive variables [<xref ref-type="bibr" rid="CR5">5</xref>]. The shift in the weight of predictive variables depending on stage of type 1 diabetes violates the PH assumption, potentially producing misleading results [<xref ref-type="bibr" rid="CR26">26</xref>]. To address this, we incorporated stratification by the number of autoantibodies to account for heterogeneity between predictors by stage and ensure more accurate modelling of the disease progression [<xref ref-type="bibr" rid="CR27">27</xref>, <xref ref-type="bibr" rid="CR28">28</xref>].</p></sec><sec id="Sec14"><title>External validation</title><p id="Par46">We used Kaplan&#x02013;Meier curves and estimates to demonstrate the disease-free survival probabilities, and used the log-rank test to assess the difference between studies, TEDDY and TrialNet. Due to the large differences within the study design, TrialNet (the validation dataset) included participants over a larger age range than TEDDY (the discovery dataset). We divided TrialNet participants into three categories characterised by their age at screening. The first category included participants most similar to TEDDY, i.e. aged 7 years and younger (<italic>n</italic>&#x02009;=&#x02009;1310). The second category included TrialNet children screened between (and including) 8 and 17 years of age (<italic>n</italic>&#x02009;=&#x02009;1794). The third category included all TrialNet autoantibody positive adults (i.e. 18 years of age and older) (<italic>n</italic>&#x02009;=&#x02009;963). We used a landmark approach as previously described [<xref ref-type="bibr" rid="CR6">6</xref>] (for details, see Additional File 1&#x02014;Supplementary Method &#x02013; Statistical Analysis) and one landmark model was applied within each age group (Additional File 2&#x02014;Figure S1).</p></sec><sec id="Sec15"><title>Evaluation of predictive performance</title><p id="Par47">We measured two aspects of predictive performance: discrimination and calibration. Discrimination, as measured by the area under the time-dependent receiver operating characteristic curve (ROC-AUC) [<xref ref-type="bibr" rid="CR29">29</xref>], measures the extent to which the model can predict an event at future horizons. We computed ROC-AUC using three-fold cross-validation, repeated ten-fold to reduce overestimation of model performance. ROC-AUC was evaluated at 1, 3, and 5 years in both cohorts and due to the extended follow-up in TEDDY, ROC-AUC was additionally evaluated at 8 years in TEDDY. Calibration measures the agreement between the predicted risk and the observed outcomes. We use smoothed calibration curves [<xref ref-type="bibr" rid="CR30">30</xref>] and survival Brier score [<xref ref-type="bibr" rid="CR31">31</xref>] to assess model calibration. We calculated the Brier score also using three-fold cross-validation, repeated ten-fold, the same internal validation methodology as ROC-AUC.</p><p id="Par48">We assessed the predictive performance of GRS and FDR (as a post hoc analysis) as a univariate predictor calculating ROC-AUC values for all TEDDY participants, autoantibody-positive TEDDY participants, and TrialNet participants. Additionally, we evaluated predictive performance of the predictive models across these groups at all landmark ages and time horizons. To compare AUC values, we assessed model performance at each landmark age and time horizon in one dataset and compared it to the corresponding values in the other dataset, repeating this process for all landmark ages and time horizons, ensuring consistent comparison of predictive performance. We further evaluated the model performance within pre-defined TrialNet age groups, using paired <italic>t</italic>-tests to determine the significance between models.</p></sec><sec id="Sec16"><title>Recalibration for cohort differences</title><p id="Par49">As expected, differences in risk at baseline are apparent between the discovery (TEDDY) and validation (TrialNet) datasets, consistent across the landmark ages in TEDDY. These two studies have differing eligibility criteria, and proportions of participants progressing to disease. For example, since we only selected autoantibody positive TrialNet participants, they are at higher risk of progressing to clinical type 1 diabetes than TEDDY participants. This difference causes the original and stratified model to underestimate the risks in TrialNet participants. Therefore, to adjust for differences in both the baseline risk and rate of progression between the discovery (TEDDY) and external validation (TrialNet) cohort, we performed logistic recalibration to the stratified prediction model. Logistic recalibration fits the linear predictors from the stratified model as the only covariate per strata in a Cox proportional hazards model [<xref ref-type="bibr" rid="CR32">32</xref>], adjusting baseline risk and coefficients by a common factor. The data will be split into a 75:25 training and testing ratio within each age group, with discrimination metrics and calibration graphs generated based on the testing cohorts. Recalibration was performed separately for each age group to account for distinct risk profiles and the varying effects of predictive variables across these groups.</p></sec><sec id="Sec17"><title>Sensitivity analyses</title><p id="Par50">To assess the robustness of the recalibrated model, we performed subgroup analyses within TrialNet, stratified by age group (&#x02264;&#x02009;7 years, 8&#x02013;17 years,&#x02009;&#x02265;&#x02009;18 years) and by number of autoantibodies present at screening (single vs. multiple). For each subgroup, we computed the Brier score to quantify predictive accuracy using the original, stratified, and recalibrated models.</p><p id="Par51">Results were computed with R version 4.3.1 (2023&#x02013;06&#x02013;16) [<xref ref-type="bibr" rid="CR33">33</xref>] with the packages survival [<xref ref-type="bibr" rid="CR34">34</xref>] and timeROC [<xref ref-type="bibr" rid="CR29">29</xref>]. <italic>P</italic>-values&#x02009;&#x0003c;&#x02009;0.05 were considered statistically significant.</p></sec></sec></sec><sec id="Sec18"><title>Results</title><sec id="Sec19"><title>Participant baseline characteristics and progression to stage 3 type 1 diabetes</title><p id="Par52">The analysis included 7798 TEDDY participants with a median follow-up period of 9.3 years, and 4086 TrialNet participants (mean&#x02009;&#x000b1;&#x02009;SD age 14.9&#x02009;&#x000b1;&#x02009;12.2 years; 32%&#x02009;&#x02264;&#x02009;7 years of age) with a median follow-up period of 4.8 years. Type 1 diabetes developed in 305 TEDDY participants (4%) and 1373 TrialNet participants (34%) during the follow-up. Further characteristics of the TEDDY and TrialNet participants, categorised by age-group, are detailed in Table <xref rid="Tab1" ref-type="table">1</xref>.
<table-wrap id="Tab1"><label>Table 1</label><caption><p>Table of summary characteristics of TEDDY participants and TrialNet participants, split by age-group.&#x000a0;Values are presented as counts numbers (percentage of group) unless otherwise stated. Genetic risk score is abbreviated to &#x0201c;GRS&#x0201d;. Age groups are defined as &#x0201c;Age 1&#x02013;7&#x0201d; which includes all screened individuals from 1 years old and up to 8 years, &#x0201c;Age 8&#x02013;17&#x0201d; includes all screened individuals aged 8 years and up to individuals screened 18 years old, &#x0201c;Age 18+&#x0201d; includes all screened individuals aged 18 years and older</p></caption><table frame="hsides" rules="groups"><thead><tr><th align="left" rowspan="2" colspan="2"/><th align="left" rowspan="2">TEDDY<break/><italic>n</italic> = 7798</th><th align="left" colspan="3">TrialNet, split by age at screening (years)</th></tr><tr><th align="left">Age 1&#x02013;7<break/><italic>n </italic>= 1287</th><th align="left">Age 8&#x02013;17<break/><italic>n</italic> = 1818</th><th align="left">Age 18+<break/><italic>n</italic> = 981</th></tr></thead><tbody><tr><td align="left" colspan="2">Cohort selection</td><td align="left">High genetic risk participants followed</td><td align="left" colspan="3">Cross-sectional screening of autoantibody-positive relatives of participants with type 1 diabetes</td></tr><tr><td align="left" colspan="2">Developed type 1 diabetes</td><td align="left">305 (3.9%)</td><td align="left">600 (46.6%)</td><td align="left">601 (33.1%)</td><td align="left">172 (17.5%)</td></tr><tr><td align="left" colspan="2">Sex (% female)</td><td align="left">3832 (49.1%)</td><td align="left">598 (46.5%)</td><td align="left">811 (44.6%)</td><td align="left">629 (64.1%)</td></tr><tr><td align="left" colspan="2">First-degree relative with type 1 diabetes</td><td align="left">886 (11.4%)</td><td align="left">1101 (85.5%)</td><td align="left">1587 (87.3%)</td><td align="left">950 (96.8%)</td></tr><tr><td align="left" colspan="2">Progression to multiple autoantibodies</td><td align="left">452 (5.8%)</td><td align="left">989 (76.8%)</td><td align="left">1207 (66.4%)</td><td align="left">369 (37.6%)</td></tr><tr><td align="left" colspan="2">GRS [Mean (SD)]</td><td align="left">14.3 (1.30)</td><td align="left">13.5 (2.09)</td><td align="left">13.2 (2.10)</td><td align="left">12.7 (2.21)</td></tr><tr><td align="left" colspan="2">Follow-up [median years (IQR)]</td><td align="left">9.5 (5.2; 11.1)</td><td align="left">5.8 (3.0; 9.3)</td><td align="left">4.5 (2.0; 7.6)</td><td align="left">4.4 (1.9; 8.0)</td></tr><tr><td align="left" rowspan="5">Race</td><td align="left">Hispanic, regardless of race</td><td align="left">571 (7.3%)</td><td align="left">102 (7.9%)</td><td align="left">203 (11.2%)</td><td align="left">94 (9.6%)</td></tr><tr><td align="left">White, non-Hispanic</td><td align="left">4671 (59.9%)</td><td align="left">1043 (81.0%)</td><td align="left">1413 (77.7%)</td><td align="left">790 (80.5%)</td></tr><tr><td align="left">African American, non-Hispanic</td><td align="left">67 (0.9%)</td><td align="left">37 (2.9%)</td><td align="left">45 (2.5%)</td><td align="left">23 (2.3%)</td></tr><tr><td align="left">All other races, non-Hispanic</td><td align="left">151 (1.9%)</td><td align="left">21 (1.6%)</td><td align="left">25 (1.4%)</td><td align="left">16 (1.6%)</td></tr><tr><td align="left">Unknown<sup>a</sup></td><td align="left">2338 (30.0%)</td><td align="left">84 (6.5%)</td><td align="left">132 (7.3%)</td><td align="left">58 (5.9%)</td></tr></tbody></table><table-wrap-foot><p><sup>a</sup>indicates missing Race or Ethnicity data; the percentage is higher in TEDDY due to a differential collection and reporting within Europe</p></table-wrap-foot></table-wrap></p><p id="Par53">Survival analysis demonstrated differences in absolute risk of progression between the two cohorts. As expected, TrialNet participants demonstrated a higher risk of progression to type 1 diabetes risk than TEDDY participants (5-year type 1 diabetes risk; TrialNet 35% [95% CI 33,37]; TEDDY: 2% [95% CI 2,3]; <italic>p</italic>&#x02009;&#x0003c;&#x02009;0.001).</p><p id="Par54">Significant differences persisted when the analysis was conducted with stratification by autoantibody status compared to participants from TEDDY. Among single autoantibody-positive participants, those in TrialNet had around a 10% higher risk of progressing to stage 3 clinical disease than those in TEDDY (TrialNet: 19% [<xref ref-type="bibr" rid="CR17">17</xref>, <xref ref-type="bibr" rid="CR21">21</xref>]]; TEDDY: 12% [<xref ref-type="bibr" rid="CR7">7</xref>, <xref ref-type="bibr" rid="CR17">17</xref>]; <italic>p</italic>&#x02009;=&#x02009;0.08). The risk of progression of participants present with two autoantibodies is also higher in TrialNet, by around 15% (TrialNet: 48% [<xref ref-type="bibr" rid="CR44">44</xref>, <xref ref-type="bibr" rid="CR51">51</xref>]; TEDDY: 33% [<xref ref-type="bibr" rid="CR23">23</xref>, <xref ref-type="bibr" rid="CR42">42</xref>]; <italic>p</italic>&#x02009;=&#x02009;0.04). However, we observed no difference in probability of the progression between the two studies for participants with three positive autoantibodies (TrialNet: 61% [56,65]; TEDDY: 67% [56,75]; <italic>p</italic>&#x02009;=&#x02009;0.40) (Additional File 2&#x02014;Figure S2 and Additional File 3&#x02014;Table S1).</p><p id="Par55">The GRS was a stronger discriminator as an individual variable in TEDDY compared to TrialNet, with higher predictive performance in TEDDY (ROC-AUC: 0.73 [95% CI 0.71,0.76]) than in TrialNet (0.64 [95% CI 0.62,0.67]). However, when comparing participants with similar characteristics (e.g. autoantibody positive participants), the discrimination was more comparable between the two cohorts (ROC-AUC TEDDY: 0.58 [0.46,0.71]; TrialNet: 0.64 [0.62,0.67]) though confidence intervals were wide. Additionally, FDR status did not provide meaningful discrimination in TrialNet (ROC-AUC TrialNet: 0.50 [95% CI 0.49, 0.51]).</p></sec><sec id="Sec20"><title>Validation of stratified prediction model in autoantibody positive TrialNet participants</title><p id="Par56">We stratified the original model by autoantibody status to address the imbalance of autoantibody-negative to autoantibody-positive participants between cohorts (Additional File 3&#x02014;Table S2A and Table S2B).</p><p id="Par57">We first assessed the stratified model&#x02019;s performance and calibration in the TEDDY cohort. The model demonstrated strong discrimination (ROC-AUC: 0.75&#x02013;0.98 across all time horizons; Additional File 2&#x02014;Figure S3A) and showed consistent performance across all landmark ages for time horizons 3&#x02013;8 years (Additional File 3&#x02014;Table S3). Calibration was notably improved in the stratified model, compared to the original model, for participants with multiple autoantibodies (Additional File 2&#x02014;Figure S4).</p><p id="Par58">We then assessed discrimination in TrialNet and found the model had comparable discriminative ability when applied to participants with similar characteristics from TEDDY (e.g. autoantibody positive) (mean 3-year ROC-AUC TrialNet: 0.712, TEDDY: 0.756; Additional File 2&#x02014;Figure S3B and Figure S3C). As expected, given differences in cohort selection criteria, we observed consistent underestimation of the risk of type 1 diabetes in TrialNet (Additional File 2&#x02014;Figure S5). With an observed probability of 50%, predicted risk estimates are underestimated by approximately five-fold in children under 8 years old (predicted risk: 10%) and in those aged 8&#x02013;17 years (predicted risk: 9%), while understimation was approximately 2.5-fold in adults (predicted risk: 20%).</p></sec><sec id="Sec21"><title>Recalibration of prediction model in autoantibody positive TrialNet participants</title><p id="Par59">Next, we asked the question of whether a simple recalibration could take advantage of cohort similarities and modify the model for use across studies such as TEDDY and TrialNet. We observed that logisitic recalibration significantly improved the model&#x02019;s performance, removing the underestimation of risk and resulting in more accurate and calibrated predictions (Fig. <xref rid="Fig2" ref-type="fig">2</xref>, Additional File 3&#x02014;Table S2C).<fig id="Fig2"><label>Fig. 2</label><caption><p>Recalibration of stratified model improves accuracy of disease prediction among TrialNet cohort.&#x000a0;Smoothed calibration curves and histograms of estimates from stratified and recalibrated models in TrialNet, split by age group. Calibration curves show the 5-year predicted probabilities on the <italic>x</italic>-axis and the 5-year observed probabilities (determined from observed frequencies) on the <italic>y</italic>-axis. The smoothed calibration curves represent the mean, with the shaded area indicating the 95% confidence interval. The dotted line represents perfect calibration, where predicted probabilities align with the observed outcomes. Histograms of the predicted estimates are displayed above the calibration curves to illustrate the distribution of the estimates. Panel<bold> A</bold> Estimates for TrialNet participants aged 1 to 7 years (training: <italic>n</italic> = 965, testing: <italic>n</italic> = 322). Panel<bold> B</bold> Estimates for TrialNet participants aged 8 to 17 years (training: <italic>n</italic> = 1363, testing: <italic>n</italic> = 455). Panel<bold> C</bold> Estimates for TrialNet participants aged 18 years and over (training: <italic>n</italic> = 735, testing:
<italic>n</italic> = 246). Each age group was split into a 75/25 training and testing dataset</p></caption><graphic xlink:href="12916_2025_4225_Fig2_HTML" id="MO2"/></fig></p><p id="Par60">We observed significantly reduced Brier score values indiciating improved calibration in the recalibrated model compared to either the stratified or original prediction models (Fig.&#x000a0;<xref rid="Fig3" ref-type="fig">3</xref>). <fig id="Fig3"><label>Fig. 3</label><caption><p>Logistic Recalibration adjusts for differences between TEDDY and TrialNet, improving Brier score and calibration of estimates. Bar chart displaying Brier scores (with 95% confidence intervals) for the original, the stratified model, and the recalibrated model when applied to TrialNet. The Brier score, a measure of prediction accuracy, incorporates both calibration and discrimination components. Lower brier scores indicate better overall model performance, but due to the score being influenced by event prevalence, direct comparisons between age groups are not meaningful. To address this, the Brier score metrics have been grouped by age group to allow for comparison of models within each age group. To ensure robust validation, a three-fold cross-validation approach was used, splitting each age group into three, and as a result, the Brier score was assessed in one-third of each age group (Age 1&#x02013;7 years 1287 participants; Age 8&#x02013;17 years: 1818 participants; Age 18+ years: 981 participants)</p></caption><graphic xlink:href="12916_2025_4225_Fig3_HTML" id="MO3"/></fig></p><p id="Par61">The difference between the Brier Score in the recalibrated model compared to the stratified prediction model was largest in participants aged between 8 and 17 years old (stratified model Brier Score = 0.403 [95% CI 0.375,0.430]; recalibrated model Brier Score = 0.161 [95% CI 0.144,0.166]; difference = 0.2422). A lower Brier score indicates better model accuracy, where here there is an 85% decrease representing signficant prediction reliability. Discrimination of the recalibrated model across age groups was not significantly different (<italic>p</italic>=0.29) (Additional File 2 - Figure S6).</p><p id="Par62">Sensitivity analysis demonstrated reduced Brier scores in the recalibrated model across age and autoantibody subgroup (Additional File 2 - Figure S7) showing improved predictive performance regardless of risk group.</p></sec><sec id="Sec23"><title>Individualised risk prediction: development of a risk calculator tool</title><p id="Par63">We generated individualised type 1 diabetes risk estimates using the predictors of the recalibrated model (number of autoantibodies, GRS, and presence of an affected first-degree relative) and the age group of the individual (Additional File 3&#x02014;Table S4). We observed a strong impact of GRS on disease prediction, even in TrialNet, where all participants are positive for autoantibodies and have a positive family history of type 1 diabetes. For a 7-year-old child with a single autoantibody, having a 5th percentile GRS predicts an 18% 5-year risk of developing type 1 diabetes, while a 95th percentile GRS would substantially raise the 5-year risk to 56%.</p><p id="Par64">To facilitate individualised risk prediction, we developed an application where an individual&#x02019;s features can be entered to generate their type 1 diabetes risk estimate using the recalibrated model [<xref ref-type="bibr" rid="CR35">35</xref>, <xref ref-type="bibr" rid="CR36">36</xref>]. Type 1 diabetes risk is displayed through a variety of visualisations, estimated risk with a confidence interval, icon array plot, and a line graph displaying the increasing risk over time (Fig. <xref rid="Fig4" ref-type="fig">4</xref>).<fig id="Fig4"><label>Fig. 4</label><caption><p>Online risk calculator visualises type 1 diabetes risk estimates across multiple horizons (1-year, 3-year, and 5-year). An example output from the online type 1 diabetes risk calculator [<xref ref-type="bibr" rid="CR36">36</xref>], which provides visulaisations of risk estimates for three different time horizons (1-year, 3-year, and 5-year). The scenario inputs required are shown, and users can view the corresponding risk predictions for each horizon. The tool is accessible online offering an interactive experience for individals and healthcare professionals to estimate furutre risk based on personalised input data</p></caption><graphic xlink:href="12916_2025_4225_Fig4_HTML" id="MO4"/></fig></p></sec></sec><sec id="Sec24"><title>Discussion</title><p id="Par65">We tested a type 1 diabetes prediction model developed in a birth cohort and found the model was discriminative in a cohort of cross-sectionally ascertained, autoantibody-positive relatives of participants with type 1 diabetes. As expected, the model required recalibration. We observed differences by age that were accounted for by a landmark approach. However, predictions in adults were less well calibrated than in children under 18. We developed a type 1 diabetes prediction web application that allows application of the stratified and recalibrated models to participants currently being monitored in screening studies. Translation of validated models into routine care will require easy-to-use interfaces such as this, linked to electronic health record biomarkers and clinical data.</p><p id="Par66">Our study builds on previous work that showed accurate discrimination and internal validation of a combined risk model in TEDDY, one of the largest and most comprehensive natural history studies of infants followed from birth. We stratified the original model by autoantibody status to address the imbalance between the larger number of autoantibody negative participants and the smaller number of autoantibody positive participants in the discovery dataset, compared with the validation dataset (TrialNet), where inclusion required the presence of one or more autoantibodies in this cohort. This was important because a model mainly trained on autoantibody negative participants would likely not work well in autoantibody positive participants, and predictive variables have varying impact across diabetes stages. Our stratified prediction model had higher weights for family history and GRS in single autoantibody positive participants compared to multiple autoantibody positive participants, highlighting that genetic risk has a greater influence before the onset of stage 1 type 1 diabetes, which is consistent with our previous findings in TrialNet [<xref ref-type="bibr" rid="CR5">5</xref>]. We used the same variables for all models, with different coefficients dependent on autoantibody status; however, others have shown the importance of metabolic variables once participants have stage 1 type 1 diabetes [<xref ref-type="bibr" rid="CR37">37</xref>].</p><p id="Par67">The good discrimination of the original prediction model in the TrialNet dataset highlights that, in keeping with existing knowledge, the main and strongest predictors of type 1 diabetes are consistent across many study designs [<xref ref-type="bibr" rid="CR38">38</xref>&#x02013;<xref ref-type="bibr" rid="CR40">40</xref>]. We note, as previously described [<xref ref-type="bibr" rid="CR6">6</xref>], that the level of discrimination is less when compared to screening or classification settings where there has been no genetic pre-selection to rule out large numbers of low-risk participants. However, the similar discrimination between TrialNet participants and TEDDY participants with similar characteristics highlights that the features of the model have validity despite the different age ranges and selection of the discovery and validation cohorts.</p><p id="Par68">We recalibrated the stratified model for the TrialNet study to enhance its performance across various screening settings. Given the differences in baseline risk, logistic recalibration allowed us to significantly improve calibration by using a straightforward adjustment. While developing a new model in one screening setting might achieve similar results, it requires substantially more data and time. In contrast, recalibration leverages existing work, yielding accurate predictions with a more robust and transferable model that can be effectively applied. As screening for type 1 diabetes becomes more widespread, recalibration of existing models offers a practical and timely solution for ensuring accurate predictions in the wave of screening studies now being initiated worldwide.</p><p id="Par69">Stratification by autoantibody status and interactions with heritable risk factors improved the TEDDY model&#x02019;s discrimination when applied to the TrialNet cohort, but recalibration remained necessary to address residual differences in baseline risk. This suggests the potential for further stratification by additional factors to resolve more of the calibration issues in younger children. To achieve stable predictions, much more data, particularly from FDR in TEDDY or other similar studies, would be required, which is currently not available. Although metabolic variables from oral glucose tolerance tests could capture some variability in risk, persistent calibration issues indicate the need for more flexible modelling approaches [<xref ref-type="bibr" rid="CR41">41</xref>].</p><p id="Par70">Age of assessment is critical to the prediction of type 1 diabetes; we accounted for age in the original prediction model, by taking a landmark approach to modelling [<xref ref-type="bibr" rid="CR6">6</xref>], as others have done [<xref ref-type="bibr" rid="CR42">42</xref>]. The age landmark approach allows for differences in risk of progression depending on age, with a higher likelihood of rapid progression in younger children [<xref ref-type="bibr" rid="CR43">43</xref>]. Good discrimination and accurate calibration were achieved once differences in baseline risk had been accounted for; our results imply accurate prediction using the recalibrated model in screening studies for children up to age 18. We had less data over the age of 18 and found model estimates to be discriminative but calibrated slightly less well. We have limited data for this age group, and further studies are needed to fully understand adult-onset type 1 diabetes and the variables that can help refine predictive models.</p><p id="Par71">Our model collapsed information from autoantibodies into a categorical variable of autoantibody number (0,&#x000a0;1, 2, or 3), due to the size of the discovery dataset and the relative importance of autoantibody number in risk prediction and staging. Other features, such as autoantibody type and titre, may be important for risk prediction and warrant further investigation. An important consideration is the ease of assessment; for example, autoantibody testing assays across the world are compared by the Islet Autoantibody Standardisation Programme (IASP) [<xref ref-type="bibr" rid="CR44">44</xref>]. Assay performance against a standard set of positive and negative samples is compared across commercial and academic assays. However, titre is not harmonised, and not all studies measure the same autoantibodies. Therefore, this is a trade-off of precision of assessment against the generalizability and practicality of testing in the public health setting.</p><p id="Par72">The generation of the application represents an easy-to-use tool to help understand an individual&#x02019;s risk and assist in decision making. This creates an interface for clinical researchers, clinicians, families, or participants themselves to generate type 1 diabetes risk estimates. This may be helpful for communication of risk, and for decisions on the frequency of individual follow-up. This approach of generating an app or calculator has been successful in many other settings, maturity-onset diabetes in the young (MODY) and cardiovascular disease, which are both used to make clinical decisions [<xref ref-type="bibr" rid="CR45">45</xref>, <xref ref-type="bibr" rid="CR46">46</xref>].</p><sec id="Sec25"><title>Limitations</title><p id="Par73">Recalibration of the TEDDY-derived stratified prediction model improved the calibration of estimates and resulted in well-calibrated estimates in all age groups. Discrimination could be improved by additional predictors and their interaction [<xref ref-type="bibr" rid="CR47">47</xref>], such as the inclusion of metabolic variables particularly for the later stages.</p><p id="Par74">Most of the participants in both cohorts are of European ancestry; in the future, it is important to address this imbalance by including participants from diverse ancestries and geographies to ensure prediction models are valid for all. We used T1D GRS percentiles derived from a UK population for individualised risk predictions, assuming that the GRS distributions are similar across the UK and USA. However, it has been previously demonstrated that GRS distributions can vary across populations [<xref ref-type="bibr" rid="CR48">48</xref>]. Validating these differences and adjusting percentiles accordingly are necessary to ensure accurate and equitable predictions.</p><p id="Par75">Recalibration is common within disease prediction, making adjustments to improve prediction accuracy and reliability as patient populations evolve which have been utilised in many other disease settings including colon cancer and COVID-19 [<xref ref-type="bibr" rid="CR49">49</xref>, <xref ref-type="bibr" rid="CR50">50</xref>]. Besides logistic recalibration, other methodologies include temporal recalibration, which adjusts prediction over time, Bayesian updating which updates the parameters with new data while retaining prior information, and dynamic updating that continuously updates predictions as new data becomes available. However, these methodologies were not applied in this study as we aimed to implement a simpler update to the model due to available data constraints.</p><p id="Par76">The recalibration of the stratified prediction model uses baseline variables, as described in the methods. This allows for simplification of type 1 diabetes risk calculation, making risk accessible when screening participants at a single timepoint when no historical data is available. Additional risk information, from longitudinal follow-up, can be considered to further improve discrimination and calibration of prediction models, such as joint modelling or models utilising historical data, like haemoglobin A1c [<xref ref-type="bibr" rid="CR51">51</xref>, <xref ref-type="bibr" rid="CR52">52</xref>]. However, these approaches require data to be obtained from multiple visits and therefore require more information and potentially more computing resource [<xref ref-type="bibr" rid="CR53">53</xref>].</p><p id="Par77">TrialNet participants, being autoantibody-positive and having a higher proportion of family history of T1D, represent a higher-risk group than the TEDDY cohort, and therefore by using TrialNet as a validation cohort could introduce a potential spectrum bias. This could affect the generalizability of the recalibrated model to other screening populations with lower baseline risk for T1D. While the recalibration process helps adjust for differences between the cohorts, the findings from this study are most relevant to higher-risk populations similar to TrialNet. Our study also highlights the need to consider calibration in addition to discrimination when considering a prediction model, this is relevant to the T1D prediction field in general where differences in screening approach and T1D incidence may influence calibration of any prediciton model. Future studies with larger, more diverse cohorts will help further validate and refine the model&#x02019;s performance across a broader range of T1D risk levels.</p></sec></sec><sec id="Sec26"><title>Conclusions</title><p id="Par78">We externally validated a type 1 diabetes prediction model with recalibration of risk to account for a different screening scenario. Our study highlights that models of disease risk are likely to be broadly generalisable with modest recalibration required depending on the screening setting. Importantly, we developed an interactive tool that allows clinicians and researchers to visualise risk, and this may be important for accurate communication of future type 1 diabetes risk in screening studies.</p></sec><sec id="Sec118" sec-type="supplementary-material"><title>Supplementary Information</title><p>
<supplementary-material content-type="local-data" id="MOESM1"><media xlink:href="12916_2025_4225_MOESM1_ESM.docx"><caption><p>Additional File 1: Supplementary Methods Model Structure. Supplementary Methods - TrialNet Genotyping. Supplementary Methods - Statistical Analysis.</p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="MOESM2"><media xlink:href="12916_2025_4225_MOESM2_ESM.docx"><caption><p>Additional File 2: Figures S1 &#x02013; S7. Figure S1: Calibration Declines with Increasing Landmark Age in Stratified Models Applied to TrialNet Participants Aged 7 Years and Younger. Figure S2: Kaplan-Meier survival curves of TEDDY and TrialNet individuals, stratified by number of autoantibodies displaying sustained increased survival risk in TEDDY individuals. Figure S3: Discrimination performance of stratified landmark models are higher when considering all TEDDY individuals compared to TrialNet indivduals, but similar when comparing the subset of autoantibody positive TEDDY individuals to TrialNet indivduals across time horizons. Figure S4: Smoothed calibration curves of the 5-year predicted probabilities from the stratified models improved calibration specfically for multiple autoantibody positive TEDDY individuals. Figure S5: The stratified model underestimates the risk of progression to T1D clinical onset among all TrialNet age groups. Figure S6: Discrimination of all modelsare similar across age groups in TrialNet. Figure S7: Logistic Recalibration adjusts for differences between TEDDY and TrialNet, improving Brier score and calibration of estimates in both individuals positive for a single autoantibody or multiple autoantibodies at screening.</p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="MOESM3"><media xlink:href="12916_2025_4225_MOESM3_ESM.docx"><caption><p>Additional File 3: Tables S1 &#x02013; S4. Table S1: Five-year risk estimates of T1D progression stratified by islet autoantibody status in TEDDY and TrialNet individuals. Table S2: Model Coefficients for TEDDY-Derived Prediction Models and Recalibrated Model in TrialNet. Table S3: Comparison of time-dependent ROC AUC between the Original prediction model and stratified prediction across different landmark and horizons. Table S4: Individualised table of risk estimates with confidence interval in TrialNet Pathway to Prevention participants.</p></caption></media></supplementary-material></p></sec></body><back><app-group><app id="App1"><sec id="Sec29"><title>Appendix A</title><sec id="Sec30"><title>TrialNet study group members</title><p id="Par84">Steering Committee: Kevan Herold (Yale University), Mark Anderson (University of California, San Francisco), Mark A. Atkinson (University of Florida), Todd Brusko (University of Florida), Jane Buckner (Benaroya Research Institute), Mark Clements (The Children&#x02019;s Mercy Hospital), Peter G. Colman (Walter &#x00026; Eliza Hall Institute of Medical Research), Mark Daniels (Children&#x02019;s Hospital of Orange County), Linda DiMeglio (Indiana University), Carmella Evans-Molina (Indiana University), Jason Gaglia (Joslin Diabetes Center), Stephen E. Gitelman (University of California, San Francisco), Robin Goland (Columbia University), Peter Gottlieb (Barbara Davis Center for Childhood Diabetes), Michael Haller (University of Florida), Carla J. Greenbaum (Benaroya Research Institute), Martin Hessner (Medical College of Wisconsin), Jeffrey P. Krischer (University of South Florida), Megan Levings (University of British Columbia), Ingrid Libman (University of Pittsburgh), Peter Linsley (Benaroya Research Institute), Alice Long (Benaroya Research Institute), Sandra Lord (Benaroya Research Institute), Wayne Moore (The Children&#x02019;s Mercy Hospital), Antoinette Moran (University of Minnesota), Andrew Muir (Emory Children&#x02019;s Center), Priya Pralahad (Stanford University), William Russell (Vanderbilt Eskind Diabetes Clinic), Jennifer Sherr (Yale University), Lisa Spain (National Institute of Diabetes and Digestive and Kidney Diseases [NIDDK]), Andrea Steck (Barbara Davis Center for Childhood Diabetes), John Wentworth (Walter &#x00026; Eliza Hall Institute of Medical Research), Diane Wherrett (University of Toronto), Perrin White (University of Texas Southwestern), Darrell M. Wilson (Stanford University), William Winter (University of Florida).</p><p id="Par85">Past Members: Peter Antinozzi (Wake Forest University), David A. Baidal (University of Miami), Manuela Battaglia (San Raffaele University), Dorothy Becker (University of Pittsburgh), Penelope Bingley (University of Bristol), Emanuele Bosi (San Raffaele University), Richard Insel (JDRF), Thomas Kay (St. Vincent&#x02019;s Institute of Medical Research), Mikael Knip (University of Helsinki), &#x000c5;ke Lernmark (Skane University Hospital), Yuk-Fun Liu (University of Bristol), Jennifer B. Marks (University of Miami), Jerry Palmer (University of Washington), Mark Peakman (King&#x02019;s College), Louis Philipson (University of Chicago), Alberto Pugliese (University of Miami), Philip Raskin (University of Texas Southwestern), Maria Redondo (Baylor College of Medicine), Henry Rodriguez (University of South Florida Diabetes and Endocrinology Center), Bart Roep (Leiden University Medical Center), Desmond A. Schatz (University of Florida), Jay S. Skyler (University of Miami), Jay M. Sosenko (University of Miami), Jorma Toppari (Hospital District of Southwest Finland), Anette Ziegler (Technical University Munich).</p><p id="Par86">Executive Committee: Kevan Herold (Yale University), Linda DiMeglio (Indiana University), Carla J. Greenbaum (Benaroya Research Institute), Jeffrey P. Krischer (University of South Florida), Ellen Leschek (National Institute of Diabetes and Digestive and Kidney Diseases [NIDDK]), Lisa Spain (National Institute of Diabetes and Digestive and Kidney Diseases [NIDDK]).</p><p id="Par87">Past Members: Katarzyna Bourcier (National Institute of Allergy and Infectious Diseases [NIAID]), Richard Insel (JDRF), John Ridge (National Institute of Allergy and Infectious Disease [NIAID]), Jay S. Skyler (University of Miami).</p><p id="Par88">Chair&#x02019;s Office: Kevan Herold, (Yale University), Lisa Rafkin (University of Miami).</p><p id="Par89">Past Members: Carla J. Greenbaum (Benaroya Research Institute), Irene Santiago (University of Miami), Jay S. Skyler (University of Miami), Jay M. Sosenko (University of Miami).</p><p id="Par90">TrialNet Coordinating Center (University of South Florida): Jeffrey P. Krischer, Brian Bundy, Michael Abbondondolo, Rajesh Adusumalli, Logan Alford, Matthew Boonstra, Jessica Conaty, David Cuthbertson, Julie Ford, Jennifer Garmeson, Veena Gowda, Cameron Hainline, Brian Hays, Kathleen Heyman, Christina Karges, Amy Kunz, Shu Liu, Kristin Maddox, Colleen Maguire, Margaret Moore, Sarah Muller, Melissa Murray, Johanna Nesbitt, Ryan O&#x02019;Donnell, Melissa Parker, MJ Pereyra, Francisco Perez Laras, Aswani Raheja, Devon Rizzo, Ariana Rojas, Cintia Reichert, Lisa Steward, Michael Taylor, Roy Tamura, Dena Tewey, Elon Walker-Veras, Jianmei Wang, Melissa Wroble, Lili Wurmser, Lu You, Kenneth Young.</p><p id="Par91">Past Members: Timothy Adams, Darlene Amado, Ilma Asif, Jenna Bjellquist, Laura Bocchino, Cristina Burroughs, Mario Cleves, Meagan DeSalvatore, Christopher Eberhard, Steve Fiske, Susan Geyer, Courtney Henderson, Martha Henry, Belinda Hsiao, Amanda Kinderman, Beata-Gabriela Koziol, Lindsay Lane, Ashley Leinbach, Jennifer Lloyd, Jamie Malloy, Julie Martin, Cameron McNeill, Jessica Miller, Thuy Nguyen, Jodie Nunez, Nichole Reed, Amy Roberts, Kelly Sadler, Tina Stavros, Christine Sullivan, Megan V. Warnock, Keith Wood, Rebecca Wood, Ping Xu, Vanessa Yanek.</p><p id="Par92">National Institute of Diabetes and Digestive and Kidney Diseases [NIDDK]: Ellen Leschek, Lisa Spain.</p><p id="Par93">Data Safety and Monitoring Board: Emily Blumberg (University of Pennsylvania), Sean Aas (Georgetown University), Gerald Beck (Cleveland Clinic Foundation), Rose Gubitosi-Klug (Case Western Reserve University), Dennis Wallace (Retired).</p><p id="Par94">Past Members: David Brillon (Cornell University), Lori Laffel (Joslin Diabetes Center), Robert Veatch (Georgetown University), Robert Vigersky (Medtronic).</p><p id="Par95">Infectious Disease Safety Committee: Brett Loechelt (Children&#x02019;s National Medical Center), Lindsey Baden (Brigham and Women&#x02019;s Hospital), Peter Gottlieb (Barbara Davis Center for Childhood Diabetes), Michael Green (University of Pittsburgh), Ellen Leschek (National Institute of Diabetes and Digestive and Kidney Diseases [NIDDK]), Ingrid Libman (University of Pittsburgh), Adriana Weinberg (University of Colorado), John Wentworth (Walter &#x00026; Eliza Hall Institute of Medical Research).</p><p id="Par96">Past Members: Nora Bryant (Joslin Diabetes Center),Yuk-Fun Liu (University of Bristol).</p><p id="Par97">Laboratory Directors: Michael Sheldon (Infinity BiologiX), Adriana Weinberg (University of Colorado), William Winter (University of Florida), Liping Yu (Barbara Davis Center for Childhood Diabetes).</p><p id="Par98">Past Members: Santica Marcovina (University of Washington), Jerry P. Palmer (University of Washington), Jay Tischfield (Rutgers University).</p><p id="Par99">Psychosocial Committee: Suzanne Bennett-Johnson (Florida State University), Ryan McDonough (Children&#x02019;s Mercy), Kelli Delallo (University of Pittsburgh), Krim Driscoll (University of Florida), Kevan Herold (Yale University), Christine March (University of Pittsburgh), Sarah McGaugh (Hospital for Sick Children Toronto), Arielle Pagryzinski (HUB), Lisa Rafkin (University of Miami), Jennifer Sherr (Yale University), Korey Hood (Stanford University), Diane Naranjo (Stanford University), Anna Barash (Benaroya), Holly O&#x02019;Donnell (Barbara Davis Center), Brittany Bruggeman (University of Florida), Paige Trojanowski (Barbara Davis Center), Linda DiMeglio (Indiana University), Kali Johnson (University of Minnesota).</p><p id="Par100">TrialNet Clinical Network Hub (Benaroya Research Institute): Arielle Pagryzinski, Emily Batts, Danielle Tom, Catherine Nguyen, Chris Budy.</p><p id="Par101">Past Members: Annie Schultz, Kristin Fitzpatrick, Randy Guerra, Melita Romasco, Annie Shultz, Mary Ramey, Michele Patience-Staal, Meghan Tobin, Diana Skye, Christopher Webb.</p><p id="Par102">Underrepresented Minority Committee: Ananta Addala (Stanford University), Susanne Cabrera (University of California San Francisco), Kevan Herold (Yale University, Lisa Rafkin (University of Miami), Andy Muir (Emory University), Robin Goland (Columbia University), Darrell Wilson (Stanford University), Ingrid Libman (University of Pittsburgh), Wayne Moore (The Children&#x02019;s Mercy Hospital), Carla Greenbaum (Benaroya), Linda DiMeglio (Indiana University), Brittany Bruggeman (University of Florida).</p></sec><sec id="Sec31"><title>Active personnel at clinical centers participating in the TN01 protocol</title><p id="Par103">Barbara Davis Center for Childhood Diabetes, Aurora, Colorado: Andrea K. Steck, Lexie Chesshir, Peter A. Gottlieb, Lisa Meyers, Aaron W. Michels, Marian Rewers, Kimber Simmons, Morgan Sooy, Fatima Tensun, Taylor Triolo, Leah Galvez Valencia, Paula Wadwa, Ruthie Williamson.</p><p id="Par104">Benaroya Research Institute, Seattle, Washington: Carla J. Greenbaum, Jane H. Buckner, Sadiq El&#x02019;Amin-White, Sandra Lord, Bao Ng, Mary Ramey, Michael Richter, Elaine Sachter, Corinna Tordillos, Dana VanBuecken, Kimberly Varner, Heather White, Nancy Wickstrom, Cassandra Williams, Alyssa Ylescupidez.</p><p id="Par105">The Children&#x02019;s Hospital of Orange County: Amrit Bhangoo, Mark Daniels, Daina Dreimane, Mark Daniels, Marissa Erickson, Timothy Flannery, Nikta Forghani, Sarah Hu, Himala Kashmiri, Anabel Palencia, Christina Reh, Francoise Sutton, Heather Speer, Lien Trihn.</p><p id="Par106">The Children&#x02019;s Mercy Hospital, Kansas City, Missouri: Wayne Moore, Fadi Al Muhaisen, Jennifer Boyd, Julia Broussard, Mark Clements, Aliza Elrod, Katelyn Evans, Max Feldt, Kelsee Halpin, Heather Harding, Jennifer James, Terri Luetjen, Ryan McDonough, Susan Mitchell, Tiffany Musick, Emily Paprocki, Rhiannon Pomerantz, Nikita Raje, Luis Sainz y Diaz, Britaney Spartz.</p><p id="Par107">Columbia University, New York, New York: Robin Goland, Mone Anzai, Magdalena Bogun, Rachelle Gandica, Natasha Leibel, Jacqueline Lonier, James Pring, Nathan Schwab, Kristen Williams.</p><p id="Par108">Emory Children&#x02019;s Center, Atlanta, Georgia: Andrew Muir, Amber Antich, Kristina Cossen, Eric Felner, Lynette Gonzalez, Wanda Sanchez, Catherine Simpson.</p><p id="Par109">The Hospital for Sick Children, Toronto, Ontario: Diane K. Wherrett, Lesley Eisel, Sarah McGaugh, Rebecca Stochinsky, Mary Jo Ricci.</p><p id="Par110">Indiana University, Indianapolis, Indiana: Linda A. DiMeglio, Carmella Evans-Molina, Jamie Felton, Heba M. Ismail, Megan Kirchner, Anna Neyman, Juan Sanchez, Corinne Parks-Schenck, Emily K. Sims, Maria Spall.</p><p id="Par111">Stanford University, Stanford, California: Darrell M. Wilson, Bonnie Baker, Karen Barahona, Bruce Buckingham, Priya Prahalad, Trudy Esrey.</p><p id="Par112">University of British Columbia, Kelowna, British Columbia: Constadina Panagiotopoulos, Daniel Metzger, Lauren Semkow.</p><p id="Par113">University of California, San Francisco, California: Stephen Gitelman, Fatema Abdulhussein, Natalie Aceves, Mark Anderson, Julissa Cabrera, Hannah Chessner, Abby Cobb-Walch, Laura Dapkus, Aristides Diamant, Marysol Gonzales Granados, Tina Hu, Karen Ko, Janet Lee, Roger Long, Isabella Niu, Srinath Sanda, Caroline Schulmeister, Priya Srivastava, Lorraine Stiehl, Christine Torok, Rebecca Wesch, Jenise Wong, Kevin Yen.</p><p id="Par114">University of Florida, Gainesville, Florida: Michael Haller, Mark Atkinson, Brittany Bruggeman, Todd M. Brusko, Miriam Cintron, Kristin Dayton, Timothy Foster, Jennifer Hosford, Laura Jacobsen, Sarah Peeling, Danielle Poulton, Desmond Schatz, Malinda Tran, William Winter.</p><p id="Par115">University of Minnesota, Minneapolis, Minnesota: Antoinette Moran, Shannon Beasley, Melena Bellin, Jane Kennedy, Janice Leschyshyn, Brandon Nathan, Beth Pappenfus, Ihsan Rizky, Muna Sunni.</p><p id="Par116">University of Pittsburgh, Pittsburgh, Pennsylvania: Ingrid Libman, Dorothy Becker, Kelli DeLallo, Christine March, Carly Shelleby, Frederico Toledo.</p><p id="Par117">University of Texas Southwestern, Dallas, Texas: Perrin White, Abha Choudhary, Yasmin Dominguez, Philip Raskin, Serey Sao.</p><p id="Par118">Vanderbilt Eskind Diabetes Clinic, Nashville, Tennessee: William Russell, Faith Brendle, Justin Gregory, Brenna Hammel, Jenny Leshko, Daniel Moore, Kimberly Rainer, Tyler Smith.</p><p id="Par119">Walter and Eliza Hall Institute of Medical Research, Parkville, Victoria: Peter Colman, John M. Wentworth, Candice Breen, Marika Bjorasen, Spiros Fourlanos, Leonard Harrison, Felicity Healy, Leanne Redl.</p><p id="Par120">Yale University, New Haven, Connecticut: Jennifer Sherr, Kevan Herold, Lori Carria, Jeanine May, William Tamborlane, Eileen Tichy, Stuart Weinzimer, Kate Weyman.</p></sec></sec></app></app-group><glossary><title>Abbreviations</title><def-list><def-item><term>CI</term><def><p id="Par5">Confidence interval</p></def></def-item><def-item><term>DASP</term><def><p id="Par6">Diabetes Autoantibody Standardisation Programme</p></def></def-item><def-item><term>FDR</term><def><p id="Par7">First-Degree Relative</p></def></def-item><def-item><term>GAD</term><def><p id="Par8">Glutamic acid decarboxylase</p></def></def-item><def-item><term>GRS</term><def><p id="Par9">Genetic Risk Score</p></def></def-item><def-item><term>HLA</term><def><p id="Par10">Human leukocyte antigens</p></def></def-item><def-item><term>IA-2A</term><def><p id="Par11">Insulinoma Associated Antigen 2</p></def></def-item><def-item><term>IQR</term><def><p id="Par12">Interquartile range</p></def></def-item><def-item><term>Max</term><def><p id="Par13">Maximum</p></def></def-item><def-item><term>mg/dL</term><def><p id="Par14">Milligrams per deciliter</p></def></def-item><def-item><term>mIAA</term><def><p id="Par15">Micro-Insulin Antibody Assay</p></def></def-item><def-item><term>min</term><def><p id="Par16">Minimum</p></def></def-item><def-item><term>mmol/L</term><def><p id="Par17">Millimoles per lite</p></def></def-item><def-item><term>mmol/mol</term><def><p id="Par18">Millimoles per mole</p></def></def-item><def-item><term>MODY</term><def><p id="Par19">Maturity onset diabetes of the young</p></def></def-item><def-item><term>OGTT</term><def><p id="Par20">Oral glucose tolerance test</p></def></def-item><def-item><term>PH</term><def><p id="Par21">Proportional hazards</p></def></def-item><def-item><term>ROC-AUC</term><def><p id="Par22">Area under Receiver Operating Characteristic Curve</p></def></def-item><def-item><term>SD</term><def><p id="Par23">Standard deviation</p></def></def-item><def-item><term>SNP</term><def><p id="Par24">Single-nucleotide polymorphism</p></def></def-item><def-item><term>TEDDY</term><def><p id="Par25">The Environmental Determinants of Diabetes in the Young</p></def></def-item><def-item><term>U.S. FDA</term><def><p id="Par26">Food and Drug Administration</p></def></def-item></def-list></glossary><fn-group><fn><p><bold>Publisher&#x02019;s Note</bold></p><p>A list of authors and their affiliations appears at the end of the paper.</p></fn><fn><p>Maria J. Redondo and Richard A. Oram contributed equally to this work.</p></fn></fn-group><ack><title>Acknowledgements</title><p>The Type 1 Diabetes TrialNet Study Group is a clinical trials network currently funded by the National Institutes of Health (NIH) through the National Institute of Diabetes and Digestive and Kidney Diseases, the National Institute of Allergy and Infectious Diseases, and The Eunice Kennedy Shriver National Institute of Child Health and Human Development, through the cooperative agreements U01 DK061010, U01 DK061034, U01 DK061042, U01 DK061058, U01 DK085461, U01 DK085465, U01 DK085466, U01 DK085476, U01 DK085499, U01 DK085509, U01 DK103180, U01 DK103153, U01 DK103266, U01 DK103282, U01 DK106984, U01 DK106994, U01 DK107013, U01 DK107014, UC4 DK106993, UC4 DK117009, and the JDRF.</p><p>The contents of this article are solely the responsibility of the authors and do not necessarily represent the official views of the NIH or the JDRF.</p><p>RAO has received consulting fees from Janssen R &#x00026; D, Provention Bio, and Sanofi, and previously held a UK MRC Confidence in Concept grant to develop a 10 SNP T1D GRS biochip with Randox R &#x00026; D, and research funding from Randox R &#x00026; D and the University of Exeter have a licencing agreement with Randox for a 10 SNP T1D GRS. LAF has received consulting fees from Janssen R &#x00026; D.</p><sec id="FPar1"><title>Guarantor statement</title><p id="Par79">RAO, LAF, and MJR are the guarantors for this manuscript. During the course of preparing this work, the author(s) used ChatGPT for the purpose of generating a title and consolidating the abstract to be under the word count. Following the use of this tool/service, the author(s) formally reviewed the content for its accuracy and edited it as necessary. The author(s) take full responsibility for all the content of this publication.</p></sec></ack><notes notes-type="author-contribution"><title>Authors&#x02019; contributions</title><p>ET, LAF, WAH, RAO, and MJR contributed to the study design, as well as data analysis and interpretation, and wrote the first draft of the manuscript. AKS, JPK, HMP, and LY contributed to the study design and data interpretation. PAG, SO-G, and SR contributed to data interpretation. All authors revised and edited the manuscript. AKS, HMP, LY, PAG, JK, TMT, and MJR were members of the Type 1 Diabetes TrialNet Study Group at the time of the study (Supplementary Table S1). LAF, RAO, and MJR are the guarantors of this article and take full responsibility for the work, including the study design, access to data, and the decision to submit and publish the manuscript.
</p></notes><notes notes-type="funding-information"><title>Funding</title><p>This work was funded by NIH R01 DK121843 (MJR, AKS, LY, SOG, PAG, SSR, JK, RAO), R01 DK124395 (MJR), JDRF 3-SRA-2019-827-S-B (LAF, RAO), NIH K23 DK136931 (TMT). ELT is a PhD student funded by Randox. </p><p>The TEDDY Study is funded by U01 DK63829, U01 DK63861, U01 DK63821, U01 DK63865, U01 DK63863, U01 DK63836, U01 DK63790, UC4 DK63829, UC4 DK63861, UC4 DK63821, UC4 DK63865, UC4 DK63863, UC4 DK63836, UC4 DK95300, UC4 DK100238, UC4 DK106955, UC4 DK112243, UC4 DK117483, U01 DK124166, U01 DK128847, and Contract No. HHSN267200700014C from the National Institute of Diabetes and Digestive and Kidney Diseases (NIDDK), National Institute of Allergy and Infectious Diseases (NIAID), Eunice Kennedy Shriver National Institute of Child Health and Human Development (NICHD), National Institute of Environmental Health Sciences (NIEHS), Centers for Disease Control and Prevention (CDC), and Breakthrough T1D (formerly JDRF). This work is supported in part by the NIH/NCATS Clinical and Translational Science Awards to the University of Florida (UL1 TR000064) and the University of Colorado (UL1 TR002535). The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health.
</p></notes><notes notes-type="data-availability"><title>Data availability</title><p>The online prediction tool is available online at <ext-link ext-link-type="uri" xlink:href="https://t1dpredictor.diabetesgenes.org">https://t1dpredictor.diabetesgenes.org</ext-link>.</p><p>Clinical metadata and GRS genotyping data from TEDDY(11,12) and TrialNet Pathway to Prevention (TN01) (8,9) are available through the NIDDK Central Repository via controlled-access request. Code used in the current study to analyze the datasets is available online (<ext-link ext-link-type="uri" xlink:href="https://github.com/TemplemanErin/recalibration">https://github.com/TemplemanErin/recalibration</ext-link>).</p></notes><notes><title>Declarations</title><notes id="FPar2"><title>Ethics approval and consent to participate</title><p id="Par80">For TEDDY study, all procedures were approved by the ethics committees/institutional review boards including Colorado Multiple Institutional Review Board (04&#x02013;0361); Medical College of Georgia Human Assurance Committee (2004&#x02013;2010)/Georgia Health Sciences University Human Assurance Committee (2011&#x02013;2012)/Georgia Regents University Institutional Review Board (2013&#x02013;2017)/Augusta University Institutional Review Board (2017&#x02013;present) (HAC 0405380); University of Florida Health Center Institutional Review Board (IRB201600277); Washington State Institutional Review Board (2004&#x02013;2012)/Western Institutional Review Board (2013&#x02013;present) (20130211); Ethics Committee of the Hospital District of Southwest Finland (Dnro168/2004); Bayerischen Landes&#x000e4;rztekammer (Bavarian Medical Association) Ethics Committee (04089); and Regional Ethics Board in Lund, Sect.&#x000a0;2 (2004&#x02013;2012)/Lund University Committee for Continuing Ethical Review (2013-present) (217/2004). In addition, TEDDY is monitored by an external evaluation committee formed by the National Institutes of Health, Bethesda, MD, USA.</p><p id="Par81">TrialNet Pathway to Prevention Clinical Trials.gov ID: NCT00097292&#x000a0;TN01 was/is approved by the Central IRB at the University of Miami. This approval began in 2016 with all sites completing the transition to the UM central IRB by the end of 2018 (per NIH requirements). Prior to then, each site had their own local approval.</p></notes><notes id="FPar3"><title>Consent for publication</title><p id="Par82">Not applicable.</p></notes><notes id="FPar4" notes-type="COI-statement"><title>Competing interests</title><p id="Par83">RAO has received consulting fees from Janssen R &#x00026; D, and previously held a UK MRC Confidence in Concept grant to develop a 10 SNP type 1 diabetes GRS biochip with Randox R &#x00026; D, and received research funding from Randox R &#x00026; D. WAH has received consulting fees from Janssen R &#x00026; D and Sanofi, and has received research funding from Sanofi. The remaining authors declare that they have no competing interests.
</p></notes></notes><ref-list id="Bib1"><title>References
</title><ref id="CR1"><label>1.</label><citation-alternatives><element-citation id="ec-CR1" publication-type="journal"><person-group person-group-type="author"><name><surname>Herold</surname><given-names>KC</given-names></name><name><surname>Bundy</surname><given-names>BN</given-names></name><name><surname>Long</surname><given-names>SA</given-names></name><name><surname>Bluestone</surname><given-names>JA</given-names></name><name><surname>DiMeglio</surname><given-names>LA</given-names></name><name><surname>Dufort</surname><given-names>MJ</given-names></name><etal/></person-group><article-title>An Anti-CD3 Antibody, Teplizumab, in Relatives at Risk for Type 1 Diabetes</article-title><source>N Engl J Med</source><year>2019</year><volume>381</volume><issue>7</issue><fpage>603</fpage><lpage>613</lpage><?supplied-pmid 31180194?><pub-id pub-id-type="pmid">31180194</pub-id>
</element-citation><mixed-citation id="mc-CR1" publication-type="journal">Herold KC, Bundy BN, Long SA, Bluestone JA, DiMeglio LA, Dufort MJ, et al. An Anti-CD3 Antibody, Teplizumab, in Relatives at Risk for Type 1 Diabetes. N Engl J Med. 2019;381(7):603&#x02013;13.<pub-id pub-id-type="pmid">31180194</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR2"><label>2.</label><mixed-citation publication-type="other">Sims EK, Bundy BN, Stier K, Serti E, Lim N, Long SA, et al. Teplizumab improves and stabilizes beta cell function in antibody-positive high-risk individuals. Sci Transl Med. 2021;13(583):eabc8980.</mixed-citation></ref><ref id="CR3"><label>3.</label><citation-alternatives><element-citation id="ec-CR3" publication-type="journal"><person-group person-group-type="author"><name><surname>Elding Larsson</surname><given-names>H</given-names></name><name><surname>Vehik</surname><given-names>K</given-names></name><name><surname>Bell</surname><given-names>R</given-names></name><name><surname>Dabelea</surname><given-names>D</given-names></name><name><surname>Dolan</surname><given-names>L</given-names></name><name><surname>Pihoker</surname><given-names>C</given-names></name><etal/></person-group><article-title>Reduced Prevalence of Diabetic Ketoacidosis at Diagnosis of Type 1 Diabetes in Young Children Participating in Longitudinal Follow-Up</article-title><source>Diabetes Care</source><year>2011</year><volume>34</volume><issue>11</issue><fpage>2347</fpage><lpage>2352</lpage><?supplied-pmid 21972409?><pub-id pub-id-type="pmid">21972409</pub-id>
</element-citation><mixed-citation id="mc-CR3" publication-type="journal">Elding Larsson H, Vehik K, Bell R, Dabelea D, Dolan L, Pihoker C, et al. Reduced Prevalence of Diabetic Ketoacidosis at Diagnosis of Type 1 Diabetes in Young Children Participating in Longitudinal Follow-Up. Diabetes Care. 2011;34(11):2347&#x02013;52.<pub-id pub-id-type="pmid">21972409</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR4"><label>4.</label><citation-alternatives><element-citation id="ec-CR4" publication-type="journal"><person-group person-group-type="author"><name><surname>Ziegler</surname><given-names>AG</given-names></name><name><surname>Rewers</surname><given-names>M</given-names></name><name><surname>Simell</surname><given-names>O</given-names></name><name><surname>Simell</surname><given-names>T</given-names></name><name><surname>Lempainen</surname><given-names>J</given-names></name><name><surname>Steck</surname><given-names>A</given-names></name><etal/></person-group><article-title>Seroconversion to Multiple Islet Autoantibodies and Risk of Progression to Diabetes in Children</article-title><source>JAMA</source><year>2013</year><volume>309</volume><issue>23</issue><fpage>2473</fpage><?supplied-pmid 23780460?><pub-id pub-id-type="pmid">23780460</pub-id>
</element-citation><mixed-citation id="mc-CR4" publication-type="journal">Ziegler AG, Rewers M, Simell O, Simell T, Lempainen J, Steck A, et al. Seroconversion to Multiple Islet Autoantibodies and Risk of Progression to Diabetes in Children. JAMA. 2013;309(23):2473.<pub-id pub-id-type="pmid">23780460</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR5"><label>5.</label><citation-alternatives><element-citation id="ec-CR5" publication-type="journal"><person-group person-group-type="author"><name><surname>Redondo</surname><given-names>MJ</given-names></name><name><surname>Geyer</surname><given-names>S</given-names></name><name><surname>Steck</surname><given-names>AK</given-names></name><name><surname>Sharp</surname><given-names>S</given-names></name><name><surname>Wentworth</surname><given-names>JM</given-names></name><name><surname>Weedon</surname><given-names>MN</given-names></name><etal/></person-group><article-title>A Type 1 Diabetes Genetic Risk Score Predicts Progression of Islet Autoimmunity and Development of Type 1 Diabetes in Individuals at Risk</article-title><source>Diabetes Care</source><year>2018</year><volume>41</volume><issue>9</issue><fpage>1887</fpage><lpage>1894</lpage><?supplied-pmid 30002199?><pub-id pub-id-type="pmid">30002199</pub-id>
</element-citation><mixed-citation id="mc-CR5" publication-type="journal">Redondo MJ, Geyer S, Steck AK, Sharp S, Wentworth JM, Weedon MN, et al. A Type 1 Diabetes Genetic Risk Score Predicts Progression of Islet Autoimmunity and Development of Type 1 Diabetes in Individuals at Risk. Diabetes Care. 2018;41(9):1887&#x02013;94.<pub-id pub-id-type="pmid">30002199</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR6"><label>6.</label><citation-alternatives><element-citation id="ec-CR6" publication-type="journal"><person-group person-group-type="author"><name><surname>Ferrat</surname><given-names>LA</given-names></name><name><surname>Vehik</surname><given-names>K</given-names></name><name><surname>Sharp</surname><given-names>SA</given-names></name><name><surname>Lernmark</surname><given-names>&#x000c5;</given-names></name><name><surname>Rewers</surname><given-names>MJ</given-names></name><name><surname>She</surname><given-names>JX</given-names></name><etal/></person-group><article-title>A combined risk score enhances prediction of type 1 diabetes among susceptible children</article-title><source>Nat Med</source><year>2020</year><volume>26</volume><issue>8</issue><fpage>1247</fpage><lpage>1255</lpage><?supplied-pmid 32770166?><pub-id pub-id-type="pmid">32770166</pub-id>
</element-citation><mixed-citation id="mc-CR6" publication-type="journal">Ferrat LA, Vehik K, Sharp SA, Lernmark &#x000c5;, Rewers MJ, She JX, et al. A combined risk score enhances prediction of type 1 diabetes among susceptible children. Nat Med. 2020;26(8):1247&#x02013;55.<pub-id pub-id-type="pmid">32770166</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR7"><label>7.</label><citation-alternatives><element-citation id="ec-CR7" publication-type="journal"><person-group person-group-type="author"><name><surname>Sims</surname><given-names>EK</given-names></name><name><surname>Besser</surname><given-names>REJ</given-names></name><name><surname>Dayan</surname><given-names>C</given-names></name><name><surname>Geno Rasmussen</surname><given-names>C</given-names></name><name><surname>Greenbaum</surname><given-names>C</given-names></name><name><surname>Griffin</surname><given-names>KJ</given-names></name><etal/></person-group><article-title>Screening for Type 1 Diabetes in the General Population: A Status Report and Perspective</article-title><source>Diabetes</source><year>2022</year><volume>71</volume><issue>4</issue><fpage>610</fpage><lpage>623</lpage><?supplied-pmid 35316839?><pub-id pub-id-type="pmid">35316839</pub-id>
</element-citation><mixed-citation id="mc-CR7" publication-type="journal">Sims EK, Besser REJ, Dayan C, Geno Rasmussen C, Greenbaum C, Griffin KJ, et al. Screening for Type 1 Diabetes in the General Population: A Status Report and Perspective. Diabetes. 2022;71(4):610&#x02013;23.<pub-id pub-id-type="pmid">35316839</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR8"><label>8.</label><citation-alternatives><element-citation id="ec-CR8" publication-type="journal"><person-group person-group-type="author"><name><surname>Mahon</surname><given-names>JL</given-names></name><name><surname>Sosenko</surname><given-names>JM</given-names></name><name><surname>Rafkin-Mervis</surname><given-names>L</given-names></name><name><surname>Krause-Steinrauf</surname><given-names>H</given-names></name><name><surname>Lachin</surname><given-names>JM</given-names></name><name><surname>Thompson</surname><given-names>C</given-names></name><etal/></person-group><article-title>The TrialNet Natural History Study of the Development of Type 1 Diabetes: objectives, design, and initial results</article-title><source>Pediatr Diabetes</source><year>2009</year><volume>10</volume><issue>2</issue><fpage>97</fpage><lpage>104</lpage><?supplied-pmid 18823409?><pub-id pub-id-type="pmid">18823409</pub-id>
</element-citation><mixed-citation id="mc-CR8" publication-type="journal">Mahon JL, Sosenko JM, Rafkin-Mervis L, Krause-Steinrauf H, Lachin JM, Thompson C, et al. The TrialNet Natural History Study of the Development of Type 1 Diabetes: objectives, design, and initial results. Pediatr Diabetes. 2009;10(2):97&#x02013;104.<pub-id pub-id-type="pmid">18823409</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR9"><label>9.</label><mixed-citation publication-type="other">Krischer J. TrialNet Pathway To Prevention (formerly Natural History Study) (TN01) . NIDDK Central Repository; 2022. Available from: 10.58020/geaw-2v38.</mixed-citation></ref><ref id="CR10"><label>10.</label><citation-alternatives><element-citation id="ec-CR10" publication-type="journal"><person-group person-group-type="author"><name><surname>P&#x000f6;ll&#x000e4;nen</surname><given-names>PM</given-names></name><name><surname>Ryh&#x000e4;nen</surname><given-names>SJ</given-names></name><name><surname>Toppari</surname><given-names>J</given-names></name><name><surname>Ilonen</surname><given-names>J</given-names></name><name><surname>V&#x000e4;h&#x000e4;salo</surname><given-names>P</given-names></name><name><surname>Veijola</surname><given-names>R</given-names></name><etal/></person-group><article-title>Dynamics of Islet Autoantibodies During Prospective Follow-Up From Birth to Age 15 Years</article-title><source>J Clin Endocrinol Metab</source><year>2020</year><volume>105</volume><issue>12</issue><fpage>e4638</fpage><lpage>e4651</lpage><?supplied-pmid 32882033?><pub-id pub-id-type="pmid">32882033</pub-id>
</element-citation><mixed-citation id="mc-CR10" publication-type="journal">P&#x000f6;ll&#x000e4;nen PM, Ryh&#x000e4;nen SJ, Toppari J, Ilonen J, V&#x000e4;h&#x000e4;salo P, Veijola R, et al. Dynamics of Islet Autoantibodies During Prospective Follow-Up From Birth to Age 15 Years. J Clin Endocrinol Metab. 2020;105(12):e4638&#x02013;51.<pub-id pub-id-type="pmid">32882033</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR11"><label>11.</label><citation-alternatives><element-citation id="ec-CR11" publication-type="journal"><person-group person-group-type="author"><collab>TEDDY Study Group</collab></person-group><article-title>The Environmental Determinants of Diabetes in the Young (TEDDY) Study</article-title><source>Ann N Y Acad Sci</source><year>2008</year><volume>1150</volume><issue>1</issue><fpage>1</fpage><lpage>13</lpage></element-citation><mixed-citation id="mc-CR11" publication-type="journal">TEDDY Study Group. The Environmental Determinants of Diabetes in the Young (TEDDY) Study. Ann N Y Acad Sci. 2008;1150(1):1&#x02013;13.</mixed-citation></citation-alternatives></ref><ref id="CR12"><label>12.</label><mixed-citation publication-type="other">Krischer J. The Environmental Determinants of Diabetes in the Young (TEDDY) . NIDDK Central Repository; 2025. Available from: 10.58020/y3jk-x087.</mixed-citation></ref><ref id="CR13"><label>13.</label><citation-alternatives><element-citation id="ec-CR13" publication-type="journal"><person-group person-group-type="author"><name><surname>Ziegler</surname><given-names>AG</given-names></name><name><surname>Kick</surname><given-names>K</given-names></name><name><surname>Bonifacio</surname><given-names>E</given-names></name><name><surname>Haupt</surname><given-names>F</given-names></name><name><surname>Hippich</surname><given-names>M</given-names></name><name><surname>Dunstheimer</surname><given-names>D</given-names></name><etal/></person-group><article-title>Yield of a Public Health Screening of Children for Islet Autoantibodies in Bavaria, Germany</article-title><source>JAMA</source><year>2020</year><volume>323</volume><issue>4</issue><fpage>339</fpage><?supplied-pmid 31990315?><pub-id pub-id-type="pmid">31990315</pub-id>
</element-citation><mixed-citation id="mc-CR13" publication-type="journal">Ziegler AG, Kick K, Bonifacio E, Haupt F, Hippich M, Dunstheimer D, et al. Yield of a Public Health Screening of Children for Islet Autoantibodies in Bavaria, Germany. JAMA. 2020;323(4):339.<pub-id pub-id-type="pmid">31990315</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR14"><label>14.</label><citation-alternatives><element-citation id="ec-CR14" publication-type="journal"><person-group person-group-type="author"><name><surname>Hippisley-Cox</surname><given-names>J</given-names></name><name><surname>Coupland</surname><given-names>C</given-names></name><name><surname>Vinogradova</surname><given-names>Y</given-names></name><name><surname>Robson</surname><given-names>J</given-names></name><name><surname>Minhas</surname><given-names>R</given-names></name><name><surname>Sheikh</surname><given-names>A</given-names></name><etal/></person-group><article-title>Predicting cardiovascular risk in England and Wales: prospective derivation and validation of QRISK2</article-title><source>BMJ</source><year>2008</year><volume>336</volume><issue>7659</issue><fpage>1475</fpage><lpage>1482</lpage><?supplied-pmid 18573856?><pub-id pub-id-type="pmid">18573856</pub-id>
</element-citation><mixed-citation id="mc-CR14" publication-type="journal">Hippisley-Cox J, Coupland C, Vinogradova Y, Robson J, Minhas R, Sheikh A, et al. Predicting cardiovascular risk in England and Wales: prospective derivation and validation of QRISK2. BMJ. 2008;336(7659):1475&#x02013;82.<pub-id pub-id-type="pmid">18573856</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR15"><label>15.</label><citation-alternatives><element-citation id="ec-CR15" publication-type="journal"><person-group person-group-type="author"><collab>The TEDDY Study Group</collab></person-group><article-title>The Environmental Determinants of Diabetes in the Young (TEDDY) study: study design</article-title><source>Pediatr Diabetes</source><year>2007</year><volume>8</volume><issue>5</issue><fpage>286</fpage><lpage>298</lpage><pub-id pub-id-type="pmid">17850472</pub-id>
</element-citation><mixed-citation id="mc-CR15" publication-type="journal">The TEDDY Study Group. The Environmental Determinants of Diabetes in the Young (TEDDY) study: study design. Pediatr Diabetes. 2007;8(5):286&#x02013;98.<pub-id pub-id-type="pmid">17850472</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR16"><label>16.</label><citation-alternatives><element-citation id="ec-CR16" publication-type="journal"><person-group person-group-type="author"><name><surname>Hagopian</surname><given-names>WA</given-names></name><name><surname>Erlich</surname><given-names>H</given-names></name><name><surname>Lernmark</surname><given-names>&#x000c5;</given-names></name><name><surname>Rewers</surname><given-names>M</given-names></name><name><surname>Ziegler</surname><given-names>AG</given-names></name><name><surname>Simell</surname><given-names>O</given-names></name><etal/></person-group><article-title>The Environmental Determinants of Diabetes in the Young (TEDDY): genetic criteria and international diabetes risk screening of 421 000 infants</article-title><source>Pediatr Diabetes</source><year>2011</year><volume>12</volume><issue>8</issue><fpage>733</fpage><lpage>743</lpage><?supplied-pmid 21564455?><pub-id pub-id-type="pmid">21564455</pub-id>
</element-citation><mixed-citation id="mc-CR16" publication-type="journal">Hagopian WA, Erlich H, Lernmark &#x000c5;, Rewers M, Ziegler AG, Simell O, et al. The Environmental Determinants of Diabetes in the Young (TEDDY): genetic criteria and international diabetes risk screening of 421 000 infants. Pediatr Diabetes. 2011;12(8):733&#x02013;43.<pub-id pub-id-type="pmid">21564455</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR17"><label>17.</label><citation-alternatives><element-citation id="ec-CR17" publication-type="journal"><person-group person-group-type="author"><name><surname>Bonifacio</surname><given-names>E</given-names></name><name><surname>Yu</surname><given-names>L</given-names></name><name><surname>Williams</surname><given-names>AK</given-names></name><name><surname>Eisenbarth</surname><given-names>GS</given-names></name><name><surname>Bingley</surname><given-names>PJ</given-names></name><name><surname>Marcovina</surname><given-names>SM</given-names></name><etal/></person-group><article-title>Harmonization of Glutamic Acid Decarboxylase and Islet Antigen-2 Autoantibody Assays for National Institute of Diabetes and Digestive and Kidney Diseases Consortia</article-title><source>J Clin Endocrinol Metab</source><year>2010</year><volume>95</volume><issue>7</issue><fpage>3360</fpage><lpage>3367</lpage><?supplied-pmid 20444913?><pub-id pub-id-type="pmid">20444913</pub-id>
</element-citation><mixed-citation id="mc-CR17" publication-type="journal">Bonifacio E, Yu L, Williams AK, Eisenbarth GS, Bingley PJ, Marcovina SM, et al. Harmonization of Glutamic Acid Decarboxylase and Islet Antigen-2 Autoantibody Assays for National Institute of Diabetes and Digestive and Kidney Diseases Consortia. J Clin Endocrinol Metab. 2010;95(7):3360&#x02013;7.<pub-id pub-id-type="pmid">20444913</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR18"><label>18.</label><citation-alternatives><element-citation id="ec-CR18" publication-type="journal"><person-group person-group-type="author"><name><surname>Sharp</surname><given-names>SA</given-names></name><name><surname>Rich</surname><given-names>SS</given-names></name><name><surname>Wood</surname><given-names>AR</given-names></name><name><surname>Jones</surname><given-names>SE</given-names></name><name><surname>Beaumont</surname><given-names>RN</given-names></name><name><surname>Harrison</surname><given-names>JW</given-names></name><etal/></person-group><article-title>Development and Standardization of an Improved Type 1 Diabetes Genetic Risk Score for Use in Newborn Screening and Incident Diagnosis</article-title><source>Diabetes Care</source><year>2019</year><volume>42</volume><issue>2</issue><fpage>200</fpage><lpage>207</lpage><?supplied-pmid 30655379?><pub-id pub-id-type="pmid">30655379</pub-id>
</element-citation><mixed-citation id="mc-CR18" publication-type="journal">Sharp SA, Rich SS, Wood AR, Jones SE, Beaumont RN, Harrison JW, et al. Development and Standardization of an Improved Type 1 Diabetes Genetic Risk Score for Use in Newborn Screening and Incident Diagnosis. Diabetes Care. 2019;42(2):200&#x02013;7.<pub-id pub-id-type="pmid">30655379</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR19"><label>19.</label><mixed-citation publication-type="other">Triolo TM, Parikh HM, Tosur M, Ferrat L, You L, Gottlieb PA, et al. Genetic Associations with C-peptide Levels before Type 1 Diabetes Diagnosis in At-Risk Relatives. J Clin Endocrinol Metab. 2025;110(4);e1046-50.</mixed-citation></ref><ref id="CR20"><label>20.</label><citation-alternatives><element-citation id="ec-CR20" publication-type="journal"><person-group person-group-type="author"><name><surname>Taliun</surname><given-names>D</given-names></name><name><surname>Harris</surname><given-names>DN</given-names></name><name><surname>Kessler</surname><given-names>MD</given-names></name><name><surname>Carlson</surname><given-names>J</given-names></name><name><surname>Szpiech</surname><given-names>ZA</given-names></name><name><surname>Torres</surname><given-names>R</given-names></name><etal/></person-group><article-title>Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program</article-title><source>Nature</source><year>2021</year><volume>590</volume><issue>7845</issue><fpage>290</fpage><lpage>299</lpage><?supplied-pmid 33568819?><pub-id pub-id-type="pmid">33568819</pub-id>
</element-citation><mixed-citation id="mc-CR20" publication-type="journal">Taliun D, Harris DN, Kessler MD, Carlson J, Szpiech ZA, Torres R, et al. Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program. Nature. 2021;590(7845):290&#x02013;9.<pub-id pub-id-type="pmid">33568819</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR21"><label>21.</label><citation-alternatives><element-citation id="ec-CR21" publication-type="journal"><person-group person-group-type="author"><name><surname>Das</surname><given-names>S</given-names></name><name><surname>Forer</surname><given-names>L</given-names></name><name><surname>Sch&#x000f6;nherr</surname><given-names>S</given-names></name><name><surname>Sidore</surname><given-names>C</given-names></name><name><surname>Locke</surname><given-names>AE</given-names></name><name><surname>Kwong</surname><given-names>A</given-names></name><etal/></person-group><article-title>Next-generation genotype imputation service and methods</article-title><source>Nat Genet</source><year>2016</year><volume>48</volume><issue>10</issue><fpage>1284</fpage><lpage>1287</lpage><?supplied-pmid 27571263?><pub-id pub-id-type="pmid">27571263</pub-id>
</element-citation><mixed-citation id="mc-CR21" publication-type="journal">Das S, Forer L, Sch&#x000f6;nherr S, Sidore C, Locke AE, Kwong A, et al. Next-generation genotype imputation service and methods. Nat Genet. 2016;48(10):1284&#x02013;7.<pub-id pub-id-type="pmid">27571263</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR22"><label>22.</label><citation-alternatives><element-citation id="ec-CR22" publication-type="journal"><person-group person-group-type="author"><name><surname>Luo</surname><given-names>Y</given-names></name><name><surname>Kanai</surname><given-names>M</given-names></name><name><surname>Choi</surname><given-names>W</given-names></name><name><surname>Li</surname><given-names>X</given-names></name><name><surname>Sakaue</surname><given-names>S</given-names></name><name><surname>Yamamoto</surname><given-names>K</given-names></name><etal/></person-group><article-title>A high-resolution HLA reference panel capturing global population diversity enables multi-ancestry fine-mapping in HIV host response</article-title><source>Nat Genet</source><year>2021</year><volume>53</volume><issue>10</issue><fpage>1504</fpage><lpage>1516</lpage><?supplied-pmid 34611364?><pub-id pub-id-type="pmid">34611364</pub-id>
</element-citation><mixed-citation id="mc-CR22" publication-type="journal">Luo Y, Kanai M, Choi W, Li X, Sakaue S, Yamamoto K, et al. A high-resolution HLA reference panel capturing global population diversity enables multi-ancestry fine-mapping in HIV host response. Nat Genet. 2021;53(10):1504&#x02013;16.<pub-id pub-id-type="pmid">34611364</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR23"><label>23.</label><citation-alternatives><element-citation id="ec-CR23" publication-type="journal"><person-group person-group-type="author"><name><surname>Arni</surname><given-names>AM</given-names></name><name><surname>Fraser</surname><given-names>DP</given-names></name><name><surname>Sharp</surname><given-names>SA</given-names></name><name><surname>Oram</surname><given-names>RA</given-names></name><name><surname>Johnson</surname><given-names>MB</given-names></name><name><surname>Weedon</surname><given-names>MN</given-names></name><etal/></person-group><article-title>Type 1 diabetes genetic risk score variation across ancestries using whole genome sequencing and array-based approaches</article-title><source>Sci Rep</source><year>2024</year><volume>14</volume><issue>1</issue><fpage>31044</fpage><?supplied-pmid 39730838?><pub-id pub-id-type="pmid">39730838</pub-id>
</element-citation><mixed-citation id="mc-CR23" publication-type="journal">Arni AM, Fraser DP, Sharp SA, Oram RA, Johnson MB, Weedon MN, et al. Type 1 diabetes genetic risk score variation across ancestries using whole genome sequencing and array-based approaches. Sci Rep. 2024;14(1):31044.<pub-id pub-id-type="pmid">39730838</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR24"><label>24.</label><mixed-citation publication-type="other">Luckett AM, Oram RA, Deutsch AJ, Ortega HI, Fraser DP, Ashok K, et al. Standardized Measurement of Type 1 Diabetes Polygenic Risk Across Multi-Ancestry Population Cohorts. medRxiv; 2025:2025.01.16.25320691.</mixed-citation></ref><ref id="CR25"><label>25.</label><citation-alternatives><element-citation id="ec-CR25" publication-type="journal"><person-group person-group-type="author"><name><surname>Hoffmann</surname><given-names>VS</given-names></name><name><surname>Wei&#x000df;</surname><given-names>A</given-names></name><name><surname>Winkler</surname><given-names>C</given-names></name><name><surname>Knopff</surname><given-names>A</given-names></name><name><surname>Jolink</surname><given-names>M</given-names></name><name><surname>Bonifacio</surname><given-names>E</given-names></name><etal/></person-group><article-title>Landmark models to define the age-adjusted risk of developing stage 1 type 1 diabetes across childhood and adolescence</article-title><source>BMC Med</source><year>2019</year><volume>17</volume><issue>1</issue><fpage>125</fpage><?supplied-pmid 31286933?><pub-id pub-id-type="pmid">31286933</pub-id>
</element-citation><mixed-citation id="mc-CR25" publication-type="journal">Hoffmann VS, Wei&#x000df; A, Winkler C, Knopff A, Jolink M, Bonifacio E, et al. Landmark models to define the age-adjusted risk of developing stage 1 type 1 diabetes across childhood and adolescence. BMC Med. 2019;17(1):125.<pub-id pub-id-type="pmid">31286933</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR26"><label>26.</label><citation-alternatives><element-citation id="ec-CR26" publication-type="journal"><person-group person-group-type="author"><name><surname>Xue</surname><given-names>X</given-names></name><name><surname>Xie</surname><given-names>X</given-names></name><name><surname>Gunter</surname><given-names>M</given-names></name><name><surname>Rohan</surname><given-names>TE</given-names></name><name><surname>Wassertheil-Smoller</surname><given-names>S</given-names></name><name><surname>Ho</surname><given-names>GY</given-names></name><etal/></person-group><article-title>Testing the proportional hazards assumption in case-cohort analysis</article-title><source>BMC Med Res Methodol</source><year>2013</year><volume>13</volume><issue>1</issue><fpage>88</fpage><?supplied-pmid 23834739?><pub-id pub-id-type="pmid">23834739</pub-id>
</element-citation><mixed-citation id="mc-CR26" publication-type="journal">Xue X, Xie X, Gunter M, Rohan TE, Wassertheil-Smoller S, Ho GY, et al. Testing the proportional hazards assumption in case-cohort analysis. BMC Med Res Methodol. 2013;13(1):88.<pub-id pub-id-type="pmid">23834739</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR27"><label>27.</label><citation-alternatives><element-citation id="ec-CR27" publication-type="journal"><person-group person-group-type="author"><name><surname>Samoli</surname><given-names>E</given-names></name><name><surname>Rodopoulou</surname><given-names>S</given-names></name><name><surname>Hvidtfeldt</surname><given-names>UA</given-names></name><name><surname>Wolf</surname><given-names>K</given-names></name><name><surname>Stafoggia</surname><given-names>M</given-names></name><name><surname>Brunekreef</surname><given-names>B</given-names></name><etal/></person-group><article-title>Modeling multi-level survival data in multi-center epidemiological cohort studies: Applications from the ELAPSE project</article-title><source>Environ Int</source><year>2021</year><volume>147</volume><fpage>106371</fpage><?supplied-pmid 33422970?><pub-id pub-id-type="pmid">33422970</pub-id>
</element-citation><mixed-citation id="mc-CR27" publication-type="journal">Samoli E, Rodopoulou S, Hvidtfeldt UA, Wolf K, Stafoggia M, Brunekreef B, et al. Modeling multi-level survival data in multi-center epidemiological cohort studies: Applications from the ELAPSE project. Environ Int. 2021;147: 106371.<pub-id pub-id-type="pmid">33422970</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR28"><label>28.</label><mixed-citation publication-type="other">Templeman EL, Ferrat LA, Hagopian WA, Vehik K, Redondo MJ, Oram RA. A33 (P354): Improved prediction of type 1 diabetes using a stratified cox model reflecting interactions between genetic risk and autoantibody status. Diabetes UK Professional Conference; 2023; Liverpool, United Kingdom.</mixed-citation></ref><ref id="CR29"><label>29.</label><citation-alternatives><element-citation id="ec-CR29" publication-type="journal"><person-group person-group-type="author"><name><surname>Blanche</surname><given-names>P</given-names></name><name><surname>Dartigues</surname><given-names>JF</given-names></name><name><surname>Jacqmin-Gadda</surname><given-names>H</given-names></name></person-group><article-title>Estimating and comparing time-dependent areas under receiver operating characteristic curves for censored event times with competing risks</article-title><source>Stat Med</source><year>2013</year><volume>32</volume><issue>30</issue><fpage>5381</fpage><lpage>5397</lpage><?supplied-pmid 24027076?><pub-id pub-id-type="pmid">24027076</pub-id>
</element-citation><mixed-citation id="mc-CR29" publication-type="journal">Blanche P, Dartigues JF, Jacqmin-Gadda H. Estimating and comparing time-dependent areas under receiver operating characteristic curves for censored event times with competing risks. Stat Med. 2013;32(30):5381&#x02013;97.<pub-id pub-id-type="pmid">24027076</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR30"><label>30.</label><citation-alternatives><element-citation id="ec-CR30" publication-type="journal"><person-group person-group-type="author"><name><surname>Austin</surname><given-names>PC</given-names></name><name><surname>Harrell</surname><given-names>FE</given-names></name><name><surname>Klaveren</surname><given-names>D</given-names></name></person-group><article-title>Graphical calibration curves and the integrated calibration index (ICI) for survival models</article-title><source>Stat Med</source><year>2020</year><volume>39</volume><issue>21</issue><fpage>2714</fpage><lpage>2742</lpage><?supplied-pmid 32548928?><pub-id pub-id-type="pmid">32548928</pub-id>
</element-citation><mixed-citation id="mc-CR30" publication-type="journal">Austin PC, Harrell FE, Klaveren D. Graphical calibration curves and the integrated calibration index (ICI) for survival models. Stat Med. 2020;39(21):2714&#x02013;42.<pub-id pub-id-type="pmid">32548928</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR31"><label>31.</label><citation-alternatives><element-citation id="ec-CR31" publication-type="journal"><person-group person-group-type="author"><name><surname>Schoop</surname><given-names>R</given-names></name><name><surname>Graf</surname><given-names>E</given-names></name><name><surname>Schumacher</surname><given-names>M</given-names></name></person-group><article-title>Quantifying the Predictive Performance of Prognostic Models for Censored Survival Data with Time-Dependent Covariates</article-title><source>Biometrics</source><year>2008</year><volume>64</volume><issue>2</issue><fpage>603</fpage><lpage>610</lpage><?supplied-pmid 17764480?><pub-id pub-id-type="pmid">17764480</pub-id>
</element-citation><mixed-citation id="mc-CR31" publication-type="journal">Schoop R, Graf E, Schumacher M. Quantifying the Predictive Performance of Prognostic Models for Censored Survival Data with Time-Dependent Covariates. Biometrics. 2008;64(2):603&#x02013;10.<pub-id pub-id-type="pmid">17764480</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR32"><label>32.</label><citation-alternatives><element-citation id="ec-CR32" publication-type="journal"><person-group person-group-type="author"><name><surname>Steyerberg</surname><given-names>EW</given-names></name><name><surname>Borsboom</surname><given-names>GJJM</given-names></name><name><surname>Van Houwelingen</surname><given-names>HC</given-names></name><name><surname>Eijkemans</surname><given-names>MJC</given-names></name><name><surname>Habbema</surname><given-names>JDF</given-names></name></person-group><article-title>Validation and updating of predictive logistic regression models: a study on sample size and shrinkage</article-title><source>Stat Med</source><year>2004</year><volume>23</volume><issue>16</issue><fpage>2567</fpage><lpage>2586</lpage><?supplied-pmid 15287085?><pub-id pub-id-type="pmid">15287085</pub-id>
</element-citation><mixed-citation id="mc-CR32" publication-type="journal">Steyerberg EW, Borsboom GJJM, Van Houwelingen HC, Eijkemans MJC, Habbema JDF. Validation and updating of predictive logistic regression models: a study on sample size and shrinkage. Stat Med. 2004;23(16):2567&#x02013;86.<pub-id pub-id-type="pmid">15287085</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR33"><label>33.</label><mixed-citation publication-type="other">R Core Team. R: A Language and Environment for Statistical Computing. Vienna, Austria: R Foundation for Statistica Computing; 2023.</mixed-citation></ref><ref id="CR34"><label>34.</label><citation-alternatives><element-citation id="ec-CR34" publication-type="book"><person-group person-group-type="author"><name><surname>Therneau</surname><given-names>TM</given-names></name><name><surname>Grambsch</surname><given-names>PM</given-names></name></person-group><source>Modeling Survival Data: Extending the Cox Model</source><year>2000</year><publisher-loc>New York, NY</publisher-loc><publisher-name>Springer</publisher-name></element-citation><mixed-citation id="mc-CR34" publication-type="book">Therneau TM, Grambsch PM. Modeling Survival Data: Extending the Cox Model. New York, NY: Springer; 2000. (Statistics for Biology and Health).</mixed-citation></citation-alternatives></ref><ref id="CR35"><label>35.</label><citation-alternatives><element-citation id="ec-CR35" publication-type="journal"><person-group person-group-type="author"><name><surname>Ferrat</surname><given-names>LA</given-names></name><name><surname>Carr</surname><given-names>A</given-names></name><name><surname>Vehik</surname><given-names>K</given-names></name><name><surname>Steck</surname><given-names>A</given-names></name><name><surname>Patel</surname><given-names>K</given-names></name><name><surname>Jones</surname><given-names>A</given-names></name><etal/></person-group><article-title>Type 1 Diabetes Risk Calculator for clinical researchers and clinicians</article-title><source>Diabetes Technol Ther</source><year>2022</year><volume>24</volume><issue>S2</issue><fpage>A116</fpage><lpage>A116</lpage></element-citation><mixed-citation id="mc-CR35" publication-type="journal">Ferrat LA, Carr A, Vehik K, Steck A, Patel K, Jones A, et al. Type 1 Diabetes Risk Calculator for clinical researchers and clinicians. Diabetes Technol Ther. 2022;24(S2):A116&#x02013;A116.</mixed-citation></citation-alternatives></ref><ref id="CR36"><label>36.</label><mixed-citation publication-type="other">Type 1 Diabetes Predictor. Available from: <ext-link ext-link-type="uri" xlink:href="https://t1dpredictor.diabetesgenes.org/">https://t1dpredictor.diabetesgenes.org/</ext-link>.</mixed-citation></ref><ref id="CR37"><label>37.</label><citation-alternatives><element-citation id="ec-CR37" publication-type="journal"><person-group person-group-type="author"><name><surname>Jacobsen</surname><given-names>LM</given-names></name><name><surname>Bocchino</surname><given-names>L</given-names></name><name><surname>Evans-Molina</surname><given-names>C</given-names></name><name><surname>DiMeglio</surname><given-names>L</given-names></name><name><surname>Goland</surname><given-names>R</given-names></name><name><surname>Wilson</surname><given-names>DM</given-names></name><etal/></person-group><article-title>The risk of progression to type 1 diabetes is highly variable in individuals with multiple autoantibodies following screening</article-title><source>Diabetologia</source><year>2020</year><volume>63</volume><issue>3</issue><fpage>588</fpage><lpage>596</lpage><?supplied-pmid 31768570?><pub-id pub-id-type="pmid">31768570</pub-id>
</element-citation><mixed-citation id="mc-CR37" publication-type="journal">Jacobsen LM, Bocchino L, Evans-Molina C, DiMeglio L, Goland R, Wilson DM, et al. The risk of progression to type 1 diabetes is highly variable in individuals with multiple autoantibodies following screening. Diabetologia. 2020;63(3):588&#x02013;96.<pub-id pub-id-type="pmid">31768570</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR38"><label>38.</label><citation-alternatives><element-citation id="ec-CR38" publication-type="journal"><person-group person-group-type="author"><name><surname>Primavera</surname><given-names>M</given-names></name><name><surname>Giannini</surname><given-names>C</given-names></name><name><surname>Chiarelli</surname><given-names>F</given-names></name></person-group><article-title>Prediction and Prevention of Type 1 Diabetes</article-title><source>Front Endocrinol</source><year>2020</year><volume>11</volume><fpage>248</fpage></element-citation><mixed-citation id="mc-CR38" publication-type="journal">Primavera M, Giannini C, Chiarelli F. Prediction and Prevention of Type 1 Diabetes. Front Endocrinol. 2020;11:248.</mixed-citation></citation-alternatives></ref><ref id="CR39"><label>39.</label><citation-alternatives><element-citation id="ec-CR39" publication-type="journal"><person-group person-group-type="author"><name><surname>Bonifacio</surname><given-names>E</given-names></name></person-group><article-title>Predicting Type 1 Diabetes Using Biomarkers</article-title><source>Diabetes Care</source><year>2015</year><volume>38</volume><issue>6</issue><fpage>989</fpage><lpage>996</lpage><?supplied-pmid 25998291?><pub-id pub-id-type="pmid">25998291</pub-id>
</element-citation><mixed-citation id="mc-CR39" publication-type="journal">Bonifacio E. Predicting Type 1 Diabetes Using Biomarkers. Diabetes Care. 2015;38(6):989&#x02013;96.<pub-id pub-id-type="pmid">25998291</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR40"><label>40.</label><citation-alternatives><element-citation id="ec-CR40" publication-type="journal"><person-group person-group-type="author"><name><surname>Regnell</surname><given-names>SE</given-names></name><name><surname>Lernmark</surname><given-names>&#x000c5;</given-names></name></person-group><article-title>Early prediction of autoimmune (type 1) diabetes</article-title><source>Diabetologia</source><year>2017</year><volume>60</volume><issue>8</issue><fpage>1370</fpage><lpage>1381</lpage><?supplied-pmid 28550517?><pub-id pub-id-type="pmid">28550517</pub-id>
</element-citation><mixed-citation id="mc-CR40" publication-type="journal">Regnell SE, Lernmark &#x000c5;. Early prediction of autoimmune (type 1) diabetes. Diabetologia. 2017;60(8):1370&#x02013;81.<pub-id pub-id-type="pmid">28550517</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR41"><label>41.</label><mixed-citation publication-type="other">Van Calster B, McLernon DJ, van Smeden M, Wynants L, Steyerberg EW, On behalf of Topic Group &#x02018;Evaluating diagnostic tests and prediction models&#x02019; of the STRATOS initiative. Calibration: the Achilles heel of predictive analytics. BMC Med. 2019;17(1):230.</mixed-citation></ref><ref id="CR42"><label>42.</label><citation-alternatives><element-citation id="ec-CR42" publication-type="journal"><person-group person-group-type="author"><name><surname>Bonifacio</surname><given-names>E</given-names></name><name><surname>Wei&#x000df;</surname><given-names>A</given-names></name><name><surname>Winkler</surname><given-names>C</given-names></name><name><surname>Hippich</surname><given-names>M</given-names></name><name><surname>Rewers</surname><given-names>MJ</given-names></name><name><surname>Toppari</surname><given-names>J</given-names></name><etal/></person-group><article-title>An Age-Related Exponential Decline in the Risk of Multiple Islet Autoantibody Seroconversion During Childhood</article-title><source>Diabetes Care</source><year>2021</year><volume>44</volume><issue>10</issue><fpage>2260</fpage><lpage>2268</lpage><?supplied-pmid 33627366?><pub-id pub-id-type="pmid">33627366</pub-id>
</element-citation><mixed-citation id="mc-CR42" publication-type="journal">Bonifacio E, Wei&#x000df; A, Winkler C, Hippich M, Rewers MJ, Toppari J, et al. An Age-Related Exponential Decline in the Risk of Multiple Islet Autoantibody Seroconversion During Childhood. Diabetes Care. 2021;44(10):2260&#x02013;8.<pub-id pub-id-type="pmid">33627366</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR43"><label>43.</label><mixed-citation publication-type="other">Felton JL, Redondo MJ, Oram RA, Speake C, Long SA, Onengut-Gumuscu S, et al. Islet autoantibodies as precision diagnostic tools to characterize heterogeneity in type 1 diabetes: a systematic review. Commun Med. 2024;4(1):66.</mixed-citation></ref><ref id="CR44"><label>44.</label><citation-alternatives><element-citation id="ec-CR44" publication-type="journal"><person-group person-group-type="author"><name><surname>Marzinotto</surname><given-names>I</given-names></name><name><surname>Pittman</surname><given-names>DL</given-names></name><name><surname>Williams</surname><given-names>AJK</given-names></name><name><surname>Long</surname><given-names>AE</given-names></name><name><surname>Achenbach</surname><given-names>P</given-names></name><name><surname>Schlosser</surname><given-names>M</given-names></name><etal/></person-group><article-title>Islet Autoantibody Standardization Program: interlaboratory comparison of insulin autoantibody assay performance in 2018 and 2020 workshops</article-title><source>Diabetologia</source><year>2023</year><volume>66</volume><issue>5</issue><fpage>897</fpage><lpage>912</lpage><?supplied-pmid 36759347?><pub-id pub-id-type="pmid">36759347</pub-id>
</element-citation><mixed-citation id="mc-CR44" publication-type="journal">Marzinotto I, Pittman DL, Williams AJK, Long AE, Achenbach P, Schlosser M, et al. Islet Autoantibody Standardization Program: interlaboratory comparison of insulin autoantibody assay performance in 2018 and 2020 workshops. Diabetologia. 2023;66(5):897&#x02013;912.<pub-id pub-id-type="pmid">36759347</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR45"><label>45.</label><citation-alternatives><element-citation id="ec-CR45" publication-type="journal"><person-group person-group-type="author"><name><surname>Shields</surname><given-names>BM</given-names></name><name><surname>McDonald</surname><given-names>TJ</given-names></name><name><surname>Ellard</surname><given-names>S</given-names></name><name><surname>Campbell</surname><given-names>MJ</given-names></name><name><surname>Hyde</surname><given-names>C</given-names></name><name><surname>Hattersley</surname><given-names>AT</given-names></name></person-group><article-title>The development and validation of a clinical prediction model to determine the probability of MODY in patients with young-onset diabetes</article-title><source>Diabetologia</source><year>2012</year><volume>55</volume><issue>5</issue><fpage>1265</fpage><lpage>1272</lpage><?supplied-pmid 22218698?><pub-id pub-id-type="pmid">22218698</pub-id>
</element-citation><mixed-citation id="mc-CR45" publication-type="journal">Shields BM, McDonald TJ, Ellard S, Campbell MJ, Hyde C, Hattersley AT. The development and validation of a clinical prediction model to determine the probability of MODY in patients with young-onset diabetes. Diabetologia. 2012;55(5):1265&#x02013;72.<pub-id pub-id-type="pmid">22218698</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR46"><label>46.</label><mixed-citation publication-type="other">Hippisley-Cox J, Coupland C, Brindle P. Development and validation of QRISK3 risk prediction algorithms to estimate future risk of cardiovascular disease: prospective cohort study. BMJ. 2017;357:j2099.</mixed-citation></ref><ref id="CR47"><label>47.</label><citation-alternatives><element-citation id="ec-CR47" publication-type="journal"><person-group person-group-type="author"><name><surname>You</surname><given-names>L</given-names></name><name><surname>Ferrat</surname><given-names>LA</given-names></name><name><surname>Oram</surname><given-names>RA</given-names></name><name><surname>Parikh</surname><given-names>HM</given-names></name><name><surname>Steck</surname><given-names>AK</given-names></name><name><surname>Krischer</surname><given-names>J</given-names></name><etal/></person-group><article-title>Identification of type 1 diabetes risk phenotypes using an outcome-guided clustering analysis</article-title><source>Diabetologia</source><year>2024</year><volume>67</volume><issue>11</issue><fpage>2507</fpage><lpage>2517</lpage><?supplied-pmid 39103721?><pub-id pub-id-type="pmid">39103721</pub-id>
</element-citation><mixed-citation id="mc-CR47" publication-type="journal">You L, Ferrat LA, Oram RA, Parikh HM, Steck AK, Krischer J, et al. Identification of type 1 diabetes risk phenotypes using an outcome-guided clustering analysis. Diabetologia. 2024;67(11):2507&#x02013;17.<pub-id pub-id-type="pmid">39103721</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR48"><label>48.</label><citation-alternatives><element-citation id="ec-CR48" publication-type="journal"><person-group person-group-type="author"><name><surname>Kaddis</surname><given-names>JS</given-names></name><name><surname>Perry</surname><given-names>DJ</given-names></name><name><surname>Vu</surname><given-names>AN</given-names></name><name><surname>Rich</surname><given-names>SS</given-names></name><name><surname>Atkinson</surname><given-names>MA</given-names></name><name><surname>Schatz</surname><given-names>DA</given-names></name><etal/></person-group><article-title>Improving the Prediction of Type 1 Diabetes Across Ancestries</article-title><source>Diabetes Care</source><year>2022</year><volume>45</volume><issue>3</issue><fpage>e48</fpage><lpage>50</lpage><?supplied-pmid 35043156?><pub-id pub-id-type="pmid">35043156</pub-id>
</element-citation><mixed-citation id="mc-CR48" publication-type="journal">Kaddis JS, Perry DJ, Vu AN, Rich SS, Atkinson MA, Schatz DA, et al. Improving the Prediction of Type 1 Diabetes Across Ancestries. Diabetes Care. 2022;45(3):e48-50.<pub-id pub-id-type="pmid">35043156</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR49"><label>49.</label><citation-alternatives><element-citation id="ec-CR49" publication-type="journal"><person-group person-group-type="author"><name><surname>Booth</surname><given-names>S</given-names></name><name><surname>Riley</surname><given-names>RD</given-names></name><name><surname>Ensor</surname><given-names>J</given-names></name><name><surname>Lambert</surname><given-names>PC</given-names></name><name><surname>Rutherford</surname><given-names>MJ</given-names></name></person-group><article-title>Temporal recalibration for improving prognostic model development and risk predictions in settings where survival is improving over time</article-title><source>Int J Epidemiol</source><year>2020</year><volume>49</volume><issue>4</issue><fpage>1316</fpage><lpage>1325</lpage><?supplied-pmid 32243524?><pub-id pub-id-type="pmid">32243524</pub-id>
</element-citation><mixed-citation id="mc-CR49" publication-type="journal">Booth S, Riley RD, Ensor J, Lambert PC, Rutherford MJ. Temporal recalibration for improving prognostic model development and risk predictions in settings where survival is improving over time. Int J Epidemiol. 2020;49(4):1316&#x02013;25.<pub-id pub-id-type="pmid">32243524</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR50"><label>50.</label><citation-alternatives><element-citation id="ec-CR50" publication-type="journal"><person-group person-group-type="author"><name><surname>Tanner</surname><given-names>KT</given-names></name><name><surname>Keogh</surname><given-names>RH</given-names></name><name><surname>Coupland</surname><given-names>CAC</given-names></name><name><surname>Hippisley-Cox</surname><given-names>J</given-names></name><name><surname>Diaz-Ordaz</surname><given-names>K</given-names></name></person-group><article-title>Dynamic updating of clinical survival prediction models in a changing environment</article-title><source>Diagn Progn Res</source><year>2023</year><volume>7</volume><issue>1</issue><fpage>24</fpage><?supplied-pmid 38082429?><pub-id pub-id-type="pmid">38082429</pub-id>
</element-citation><mixed-citation id="mc-CR50" publication-type="journal">Tanner KT, Keogh RH, Coupland CAC, Hippisley-Cox J, Diaz-Ordaz K. Dynamic updating of clinical survival prediction models in a changing environment. Diagn Progn Res. 2023;7(1):24.<pub-id pub-id-type="pmid">38082429</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR51"><label>51.</label><citation-alternatives><element-citation id="ec-CR51" publication-type="journal"><person-group person-group-type="author"><name><surname>Vehik</surname><given-names>K</given-names></name><name><surname>Boulware</surname><given-names>D</given-names></name><name><surname>Killian</surname><given-names>M</given-names></name><name><surname>Rewers</surname><given-names>M</given-names></name><name><surname>McIndoe</surname><given-names>R</given-names></name><name><surname>Toppari</surname><given-names>J</given-names></name><etal/></person-group><article-title>Rising Hemoglobin A1c in the Nondiabetic Range Predicts Progression of Type 1 Diabetes As Well As Oral Glucose Tolerance Tests</article-title><source>Diabetes Care</source><year>2022</year><volume>45</volume><issue>10</issue><fpage>2342</fpage><lpage>2349</lpage><?supplied-pmid 36150054?><pub-id pub-id-type="pmid">36150054</pub-id>
</element-citation><mixed-citation id="mc-CR51" publication-type="journal">Vehik K, Boulware D, Killian M, Rewers M, McIndoe R, Toppari J, et al. Rising Hemoglobin A1c in the Nondiabetic Range Predicts Progression of Type 1 Diabetes As Well As Oral Glucose Tolerance Tests. Diabetes Care. 2022;45(10):2342&#x02013;9.<pub-id pub-id-type="pmid">36150054</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR52"><label>52.</label><citation-alternatives><element-citation id="ec-CR52" publication-type="journal"><person-group person-group-type="author"><name><surname>Helminen</surname><given-names>O</given-names></name><name><surname>Aspholm</surname><given-names>S</given-names></name><name><surname>Pokka</surname><given-names>T</given-names></name><name><surname>Hautakangas</surname><given-names>MR</given-names></name><name><surname>Haatanen</surname><given-names>N</given-names></name><name><surname>Lempainen</surname><given-names>J</given-names></name><etal/></person-group><article-title>HbA1c Predicts Time to Diagnosis of Type 1 Diabetes in Children at Risk</article-title><source>Diabetes</source><year>2015</year><volume>64</volume><issue>5</issue><fpage>1719</fpage><lpage>1727</lpage><?supplied-pmid 25524912?><pub-id pub-id-type="pmid">25524912</pub-id>
</element-citation><mixed-citation id="mc-CR52" publication-type="journal">Helminen O, Aspholm S, Pokka T, Hautakangas MR, Haatanen N, Lempainen J, et al. HbA1c Predicts Time to Diagnosis of Type 1 Diabetes in Children at Risk. Diabetes. 2015;64(5):1719&#x02013;27.<pub-id pub-id-type="pmid">25524912</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR53"><label>53.</label><citation-alternatives><element-citation id="ec-CR53" publication-type="journal"><person-group person-group-type="author"><name><surname>Rizopoulos</surname><given-names>D</given-names></name><name><surname>Verbeke</surname><given-names>G</given-names></name><name><surname>Molenberghs</surname><given-names>G</given-names></name></person-group><article-title>Multiple-Imputation-Based Residuals and Diagnostic Plots for Joint Models of Longitudinal and Survival Outcomes</article-title><source>Biometrics</source><year>2010</year><volume>66</volume><issue>1</issue><fpage>20</fpage><lpage>29</lpage><?supplied-pmid 19459832?><pub-id pub-id-type="pmid">19459832</pub-id>
</element-citation><mixed-citation id="mc-CR53" publication-type="journal">Rizopoulos D, Verbeke G, Molenberghs G. Multiple-Imputation-Based Residuals and Diagnostic Plots for Joint Models of Longitudinal and Survival Outcomes. Biometrics. 2010;66(1):20&#x02013;9.<pub-id pub-id-type="pmid">19459832</pub-id>
</mixed-citation></citation-alternatives></ref></ref-list></back></article>