# Multi-omics analysis uncovers predictive biomarkers for the efficacy and outcomes of chemoimmunotherapy in advanced unresectable biliary tract cancers.

<!DOCTYPE article PUBLIC "-//NLM//DTD JATS (Z39.96) Journal Archiving and Interchange DTD with MathML3 v1.3 20210610//EN" "JATS-archivearticle1-3-mathml3.dtd"> 
<article xmlns:ali="http://www.niso.org/schemas/ali/1.0/" xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns:xlink="http://www.w3.org/1999/xlink" article-type="research-article" dtd-version="1.3"><?properties open_access?><?DTDIdentifier.IdentifierValue -//Springer-Verlag//DTD A++ V2.4//EN?><?DTDIdentifier.IdentifierType public?><?SourceDTD.DTDName A++V2.4.dtd?><?SourceDTD.Version 2.4?><?ConverterInfo.XSLTName springer2nlmx2.xsl?><?ConverterInfo.Version 1?><processing-meta base-tagset="archiving" mathml-version="3.0" table-model="xhtml" tagset-family="jats"><restricted-by>pmc</restricted-by></processing-meta><front><journal-meta><journal-id journal-id-type="nlm-ta">BMC Med</journal-id><journal-id journal-id-type="iso-abbrev">BMC Med</journal-id><journal-title-group><journal-title>BMC Medicine</journal-title></journal-title-group><issn pub-type="epub">1741-7015</issn><publisher><publisher-name>BioMed Central</publisher-name><publisher-loc>London</publisher-loc></publisher></journal-meta>
<article-meta><article-id pub-id-type="pmcid">12281707</article-id><article-id pub-id-type="publisher-id">4263</article-id><article-id pub-id-type="doi">10.1186/s12916-025-04263-z</article-id><article-categories><subj-group subj-group-type="heading"><subject>Research</subject></subj-group></article-categories><title-group><article-title>Multi-omics analysis uncovers predictive biomarkers for the efficacy and outcomes of chemoimmunotherapy in advanced unresectable biliary tract cancers</article-title></title-group><contrib-group><contrib contrib-type="author" equal-contrib="yes"><name><surname>Ni</surname><given-names>Jiaojiao</given-names></name><xref ref-type="aff" rid="Aff1">1</xref></contrib><contrib contrib-type="author" equal-contrib="yes"><name><surname>Li</surname><given-names>Chaoqun</given-names></name><xref ref-type="aff" rid="Aff1">1</xref></contrib><contrib contrib-type="author" equal-contrib="yes"><name><surname>Xu</surname><given-names>Xiaoqing</given-names></name><xref ref-type="aff" rid="Aff2">2</xref></contrib><contrib contrib-type="author" equal-contrib="yes"><name><surname>Xu</surname><given-names>Qi</given-names></name><xref ref-type="aff" rid="Aff1">1</xref></contrib><contrib contrib-type="author"><name><surname>Yin</surname><given-names>Qian</given-names></name><xref ref-type="aff" rid="Aff1">1</xref></contrib><contrib contrib-type="author"><name><surname>Chen</surname><given-names>Hanlin</given-names></name><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author"><name><surname>Zhou</surname><given-names>Shurui</given-names></name><xref ref-type="aff" rid="Aff1">1</xref></contrib><contrib contrib-type="author"><name><surname>Li</surname><given-names>Jingjing</given-names></name><xref ref-type="aff" rid="Aff1">1</xref></contrib><contrib contrib-type="author"><name><surname>Zheng</surname><given-names>Qinhong</given-names></name><xref ref-type="aff" rid="Aff4">4</xref></contrib><contrib contrib-type="author"><name><surname>Xie</surname><given-names>Yanru</given-names></name><xref ref-type="aff" rid="Aff5">5</xref></contrib><contrib contrib-type="author"><name><surname>Hua</surname><given-names>Hongjun</given-names></name><xref ref-type="aff" rid="Aff6">6</xref></contrib><contrib contrib-type="author"><name><surname>Jin</surname><given-names>Jianying</given-names></name><xref ref-type="aff" rid="Aff7">7</xref></contrib><contrib contrib-type="author"><name><surname>Xie</surname><given-names>Tieming</given-names></name><xref ref-type="aff" rid="Aff8">8</xref></contrib><contrib contrib-type="author"><name><surname>Zhao</surname><given-names>Haiping</given-names></name><xref ref-type="aff" rid="Aff9">9</xref></contrib><contrib contrib-type="author"><name><surname>Yan</surname><given-names>Feng</given-names></name><xref ref-type="aff" rid="Aff10">10</xref></contrib><contrib contrib-type="author"><name><surname>Shen</surname><given-names>Junjun</given-names></name><xref ref-type="aff" rid="Aff11">11</xref></contrib><contrib contrib-type="author"><name><surname>Huang</surname><given-names>Yihui</given-names></name><xref ref-type="aff" rid="Aff12">12</xref></contrib><contrib contrib-type="author"><name><surname>Zheng</surname><given-names>Wangye</given-names></name><xref ref-type="aff" rid="Aff12">12</xref></contrib><contrib contrib-type="author"><name><surname>Yan</surname><given-names>Junrong</given-names></name><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author"><name><surname>Wu</surname><given-names>Xiaoying</given-names></name><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author"><name><surname>Wang</surname><given-names>Song</given-names></name><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author"><name><surname>Ou</surname><given-names>Qiuxiang</given-names></name><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author"><name><surname>Shao</surname><given-names>Yang</given-names></name><xref ref-type="aff" rid="Aff3">3</xref></contrib><contrib contrib-type="author" corresp="yes"><name><surname>Wu</surname><given-names>Wei</given-names></name><address><email>wuwei@zjcc.org.cn</email></address><xref ref-type="aff" rid="Aff13">13</xref></contrib><contrib contrib-type="author" corresp="yes"><name><surname>Zhuo</surname><given-names>Wei</given-names></name><address><email>wzhuo@zju.edu.cn</email></address><xref ref-type="aff" rid="Aff14">14</xref></contrib><contrib contrib-type="author" corresp="yes"><name><surname>Ying</surname><given-names>Jieer</given-names></name><address><email>jieerying@aliyun.com</email></address><xref ref-type="aff" rid="Aff1">1</xref><xref ref-type="aff" rid="Aff15">15</xref><xref ref-type="aff" rid="Aff16">16</xref></contrib><aff id="Aff1"><label>1</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/034t30j35</institution-id><institution-id institution-id-type="GRID">grid.9227.e</institution-id><institution-id institution-id-type="ISNI">0000000119573309</institution-id><institution>Department of Hepato-Pancreato-Biliary &#x00026; Gastric Medical Oncology, Zhejiang Cancer Hospital, </institution><institution>Hangzhou Institute of Medicine (HIM), Chinese Academy of Sciences, </institution></institution-wrap>Hangzhou, 310022 China </aff><aff id="Aff2"><label>2</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/04zkkh342</institution-id><institution-id institution-id-type="GRID">grid.460137.7</institution-id><institution>Department of Oncology, Hangzhou Xixi Hospital, </institution><institution>Hangzhou Sixth People&#x02019;s Hospital, Hangzhou Xixi Hospital Affiliated to Zhejiang Chinese Medical University, </institution></institution-wrap>Hangzhou, 310023 China </aff><aff id="Aff3"><label>3</label><institution-wrap><institution-id institution-id-type="GRID">grid.518662.e</institution-id><institution>Geneseeq Research Institute, Nanjing Geneseeq Technology Inc., </institution></institution-wrap>Nanjing, 210032 China </aff><aff id="Aff4"><label>4</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/004qehs09</institution-id><institution-id institution-id-type="GRID">grid.459520.f</institution-id><institution>Department of Medical Oncology, </institution><institution>Quzhou Affiliated Hospital of Wenzhou Medical University, Quzhou People&#x02019;s Hospital, </institution></institution-wrap>Quzhou, 324000 China </aff><aff id="Aff5"><label>5</label>Department of Medical Oncology, Lishui Municipal Central Hospital, Lishui, 323000 China </aff><aff id="Aff6"><label>6</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/04z13ha89</institution-id><institution-id institution-id-type="GRID">grid.452555.6</institution-id><institution-id institution-id-type="ISNI">0000 0004 1758 3222</institution-id><institution>Department of Gastroenterology, </institution><institution>Jinhua Municipal Central Hospital, </institution></institution-wrap>Jinhua, 321099 China </aff><aff id="Aff7"><label>7</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/05m0wv206</institution-id><institution-id institution-id-type="GRID">grid.469636.8</institution-id><institution>Department of Medical Oncology, </institution><institution>Taizhou Hospital of Zhejiang Province, </institution></institution-wrap>Taizhou, 317000 China </aff><aff id="Aff8"><label>8</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/034t30j35</institution-id><institution-id institution-id-type="GRID">grid.9227.e</institution-id><institution-id institution-id-type="ISNI">0000000119573309</institution-id><institution>Department of Radiology, Zhejiang Cancer Hospital, </institution><institution>Hangzhou Institute of Medicine (HIM), Chinese Academy of Sciences, </institution></institution-wrap>Hangzhou, 310022 China </aff><aff id="Aff9"><label>9</label>Department of General Surgery, The First People&#x02019;s Hospital of Fuyang, Hangzhou, 311499 China </aff><aff id="Aff10"><label>10</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/02sysn258</institution-id><institution-id institution-id-type="GRID">grid.440280.a</institution-id><institution>Department of Medical Oncology, </institution><institution>The First People&#x02019;s Hospital of Hangzhou Lin&#x02019;an District, </institution></institution-wrap>Hangzhou, 311300 China </aff><aff id="Aff11"><label>11</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/01czx1v82</institution-id><institution-id institution-id-type="GRID">grid.413679.e</institution-id><institution-id institution-id-type="ISNI">0000 0004 0517 0981</institution-id><institution>Department of Medical Oncology, </institution><institution>Huzhou Central Hospital, </institution></institution-wrap>Huzhou, 313000 China </aff><aff id="Aff12"><label>12</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/04epb4p87</institution-id><institution-id institution-id-type="GRID">grid.268505.c</institution-id><institution-id institution-id-type="ISNI">0000 0000 8744 8924</institution-id><institution>Department of Medical Oncology, </institution><institution>The Second Clinical Medical College of Zhejiang Chinese Medical University, </institution></institution-wrap>Hangzhou, 310000 China </aff><aff id="Aff13"><label>13</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/034t30j35</institution-id><institution-id institution-id-type="GRID">grid.9227.e</institution-id><institution-id institution-id-type="ISNI">0000000119573309</institution-id><institution>Department of Pathology, Zhejiang Cancer Hospital, </institution><institution>Hangzhou Institute of Medicine (HIM), Chinese Academy of Sciences, </institution></institution-wrap>Hangzhou, 310022 China </aff><aff id="Aff14"><label>14</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/059cjpv64</institution-id><institution-id institution-id-type="GRID">grid.412465.0</institution-id><institution>Department of Cell Biology and Department Colorectal Surgery and Oncology, the Second Affiliated Hospital, </institution><institution>Zhejiang University School of Medicine, </institution></institution-wrap>Hangzhou, 310058 China </aff><aff id="Aff15"><label>15</label><institution-wrap><institution-id institution-id-type="ROR">https://ror.org/0144s0951</institution-id><institution-id institution-id-type="GRID">grid.417397.f</institution-id><institution-id institution-id-type="ISNI">0000 0004 1808 0985</institution-id><institution>Postgraduate training base Alliance of Wenzhou Medical University (Zhejiang Cancer Hospital), </institution></institution-wrap>Hangzhou, Zhejiang 310022 China </aff><aff id="Aff16"><label>16</label>Zhejiang Key Laboratory of Prevention, Diagnosis, and Therapy of Digestive Tract Tumors, Hangzhou, 310022 China </aff></contrib-group><pub-date pub-type="epub"><day>22</day><month>7</month><year>2025</year></pub-date><pub-date pub-type="pmc-release"><day>22</day><month>7</month><year>2025</year></pub-date><pub-date pub-type="collection"><year>2025</year></pub-date><volume>23</volume><elocation-id>435</elocation-id><history><date date-type="received"><day>28</day><month>9</month><year>2024</year></date><date date-type="accepted"><day>7</day><month>7</month><year>2025</year></date></history><permissions><copyright-statement>&#x000a9; The Author(s) 2025</copyright-statement><copyright-year>2025</copyright-year><license><ali:license_ref specific-use="textmining" content-type="ccbyncndlicense">https://creativecommons.org/licenses/by-nc-nd/4.0/</ali:license_ref><license-p><bold>Open Access</bold> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article&#x02019;s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article&#x02019;s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <ext-link ext-link-type="uri" xlink:href="https://creativecommons.org/licenses/by-nc-nd/4.0/">http://creativecommons.org/licenses/by-nc-nd/4.0/</ext-link>.</license-p></license></permissions><abstract id="Abs1"><sec><title>Background</title><p id="Par6">The limited response rates of immune checkpoint inhibitors in biliary tract cancers (BTC) highlight the need for effective biomarkers to optimize patient selection.</p></sec><sec><title>Methods</title><p id="Par7">Baseline tumor tissues from 125 patients with advanced BTC treated with first-line chemoimmunotherapy (chemoIO) were analyzed using targeted DNA sequencing and bulk RNA sequencing. In vitro and in vivo experiments were conducted to investigate the role of identified biomarkers in BTC.</p></sec><sec><title>Results</title><p id="Par8">Mutations in <italic>TP53</italic>, <italic>BRCA2</italic>, cytokine genes, and high tumor mutation burden were significantly associated with treatment response. In contrast, <italic>KRAS</italic> G12D and <italic>ARID1A</italic> mutations were linked to poorer survival outcomes. High expression levels of <italic>CXCL9</italic> and <italic>CTLA4</italic> expression were associated with improved treatment response, prolonged progression-free survival, and overall survival. Using these biomarkers, patients were categorized into three molecular subtypes, with Type I patients demonstrating the most favorable outcomes under chemoIO. Subsequent RNA analysis revealed that elevated <italic>CXCL9</italic> expression was associated with increased immune checkpoint expression within the tumor and heightened immune activity in the tumor microenvironment. In the mouse orthotopic cholangiocarcinoma model, CXCL9 overexpression enhanced chemoIO efficacy. Immunohistochemistry and flow cytometry showed that CXCL9 promoted T-cell infiltration and activation. In vitro experiments using multiple BTC cell lines further demonstrated that CXCL9 was essential for maintaining T cell cytotoxicity. The immune-modulatory effects of CXCL9/CTLA4 and their predictive value for treatment efficacy were further validated in a multicenter BTC cohort.</p></sec><sec><title>Conclusions</title><p id="Par9">This study identified several predictive biomarkers associated with the response and efficacy of chemoIO in advanced BTC, offering valuable insights into patient stratification and refining therapeutic strategies.</p></sec><sec><title>Supplementary Information</title><p>The online version contains supplementary material available at 10.1186/s12916-025-04263-z.</p></sec></abstract><kwd-group xml:lang="en"><title>Keywords</title><kwd>Immune checkpoint inhibitors</kwd><kwd>Biliary tract cancer</kwd><kwd>Next-generation sequencing</kwd><kwd>CXCL9</kwd></kwd-group><funding-group><award-group><funding-source><institution-wrap><institution-id institution-id-type="FundRef">http://dx.doi.org/10.13039/501100001809</institution-id><institution>National Natural Science Foundation of China</institution></institution-wrap></funding-source><award-id>82473488</award-id></award-group></funding-group><funding-group><award-group><funding-source><institution-wrap><institution-id institution-id-type="FundRef">http://dx.doi.org/10.13039/100018904</institution-id><institution>Beijing Xisike Clinical Oncology Research Foundation</institution></institution-wrap></funding-source><award-id>Y-HR2020MS-0934</award-id></award-group></funding-group><funding-group><award-group><funding-source><institution>Zhejiang Anti-Cancer Association-Qilu Clinical Scientific Research Foundation</institution></funding-source><award-id>ziskaxhqllckyxm202208</award-id></award-group></funding-group><funding-group><award-group><funding-source><institution-wrap><institution-id institution-id-type="FundRef">http://dx.doi.org/10.13039/501100004731</institution-id><institution>Natural Science Foundation of Zhejiang Province</institution></institution-wrap></funding-source><award-id>LTGY23H160010</award-id></award-group></funding-group><funding-group><award-group><funding-source><institution>"Pioneer" and "Leading Goose" R&#x00026;D Program of Zhejiang</institution></funding-source><award-id>2023SDYXSO001-5</award-id></award-group></funding-group><funding-group><award-group><funding-source><institution>Special Funds for Clinical Research of the National Health Commission</institution></funding-source><award-id>WKZX2023CX030001</award-id></award-group></funding-group><funding-group><award-group><funding-source><institution>Medical and Health Science and Technology Program of Zhejiang Province</institution></funding-source><award-id>2021KY087</award-id></award-group></funding-group><custom-meta-group><custom-meta><meta-name>issue-copyright-statement</meta-name><meta-value>&#x000a9; BioMed Central Ltd., part of Springer Nature 2025</meta-value></custom-meta></custom-meta-group></article-meta></front><body><sec id="Sec1"><title>Background</title><p id="Par40">Biliary tract cancers (BTCs) account for approximately 3% of all gastrointestinal malignancies, comprising various subtypes affecting the gallbladder, extrahepatic, perihilar, and intrahepatic bile ducts [<xref ref-type="bibr" rid="CR1">1</xref>, <xref ref-type="bibr" rid="CR2">2</xref>]. BTCs are characterized by their insidious onset, aggressive behavior, and frequent diagnosis at advanced stages, with early postoperative recurrence being common. The prognosis remains poor, with a median overall survival (OS) of less than 1&#x000a0;year and a 5-year OS rate as low as 10% [<xref ref-type="bibr" rid="CR3">3</xref>]. The current first-line standard of care for advanced BTC is gemcitabine plus cisplatin chemotherapy, a therapeutic approach that has remained unchanged for over a decade, offering a median OS of 11.7&#x000a0;months [<xref ref-type="bibr" rid="CR4">4</xref>]. Despite extensive efforts to explore novel therapeutic agents, progress has been limited [<xref ref-type="bibr" rid="CR5">5</xref>&#x02013;<xref ref-type="bibr" rid="CR8">8</xref>]. Targeted therapies in select patient subgroups harboring <italic>FGFR</italic> rearrangements, <italic>IDH</italic> variants, <italic>BRAF</italic> V600E mutations, <italic>HER2</italic> amplifications, and DNA damage repair (DDR) genes have shown promising outcomes [<xref ref-type="bibr" rid="CR9">9</xref>&#x02013;<xref ref-type="bibr" rid="CR13">13</xref>]. However, these alterations are relatively rare, with the first four occurring in approximately 14%, 14%, 5%, and 4% of intrahepatic cholangiocarcinoma (ICC) patients, respectively.
</p><p id="Par41">Given the success of combining immune checkpoint inhibitors (ICIs) with chemotherapy in various solid tumors [<xref ref-type="bibr" rid="CR14">14</xref>&#x02013;<xref ref-type="bibr" rid="CR18">18</xref>], chemoimmunotherapy (chemoIO) has emerged as a promising treatment strategy for patients with advanced BTC. The phase III TOPAZ-1 trial demonstrated that advanced BTC patients treated with durvalumab (PD-L1 inhibitor) plus chemotherapy had a higher overall response rate (ORR: 26.7% vs. 18.7%) and a significantly prolonged median OS (mOS: 12.8 vs. 11.5&#x000a0;months, <italic>P</italic>&#x02009;=&#x02009;0.021) compared to those receiving chemotherapy with placebo [<xref ref-type="bibr" rid="CR19">19</xref>]. These findings led to the FDA approval of durvalumab combined with gemcitabine and cisplatin as the first-line treatment for advanced BTCs. Similarly, the phase III KEYNOTE-966 trial showed that pembrolizumab plus gemcitabine and cisplatin outperformed placebo plus gemcitabine and cisplatin in previously untreated advanced BTCs (mOS: 12.7 vs. 10.9&#x000a0;months, <italic>P</italic>&#x02009;=&#x02009;0.0034) [<xref ref-type="bibr" rid="CR20">20</xref>]. These landmark trials have established chemoimmunotherapy as a new standard treatment for advanced BTC, marking a significant advancement in the therapeutic landscape of this disease.</p><p id="Par42">While ICIs have shown promise in the treatment of advanced BTCs, a substantial proportion of patients remain unresponsive. Overcoming the challenges of treating recalcitrant cancers like BTCs requires the identification of reliable biomarkers to predict responses to ICI monotherapy or combination strategies. Previous studies have proposed potential predictive biomarkers, such as KRAS signaling [<xref ref-type="bibr" rid="CR21">21</xref>] and T cell signatures [<xref ref-type="bibr" rid="CR22">22</xref>]. Furthermore, assessing PD-L1 protein expression, microsatellite instability (MSI), mismatch repair deficiency (dMMR), and tumor mutation burden (TMB) may also aid in patient stratification by identifying subsets of patients most likely to benefit from ICIs [<xref ref-type="bibr" rid="CR23">23</xref>]. In this study, we conducted a comprehensive prospective analysis of baseline genomic and transcriptomic profiles in a large Chinese cohort of advanced BTC patients treated with first-line anti-PD-1/PD-L1 therapies plus chemotherapy (ClinicialTrials.gov ID: NCT06048289). We aim to investigate how these multi-omics characteristics contribute to disease prognosis and serve as predictive indicators for treatment outcomes.</p></sec><sec id="Sec2"><title>Methods</title><sec id="Sec3"><title>Patient enrollment and data collection</title><p id="Par43">This study is a prospective observational cohort study (ClinicialTrials.gov ID: NCT06048289). Eligible patients were 18&#x000a0;years or older with a histologically confirmed diagnosis of stage IV unresectable or metastatic biliary tract adenocarcinoma, including intrahepatic cholangiocarcinoma, extrahepatic cholangiocarcinoma, or gallbladder carcinoma. All patients received first-line treatment with anti-PD-1/PD-L1 therapy combined with chemotherapy between November 2017 and August 2022. The detailed inclusion and exclusion criteria are shown in Additional file 1: Figure S1. All medical records of recruited patients were reviewed, and the genomic and transcriptomic landscapes of baseline tissues were analyzed. Progression-free survival (PFS) was defined as the time from treatment initiation until tumor progression or death, whichever occurs first. Overall survival (OS) was defined as the time from diagnosis until death from any cause. This study was approved by the Ethics Committee of the Zhejiang Cancer Hospital (Approval No.: IRB-2020&#x02013;404) and in accordance with the Declaration of Helsinki. Informed consent was obtained from all the participating patients.
</p></sec><sec id="Sec4"><title>DNA extraction, library preparation, and targeted DNA sequencing</title><p id="Par44">Genomic DNA extraction was performed on formalin-fixed paraffin-embedded (FFPE) tumor samples utilizing the QIAamp DNA FFPE Tissue Kit provided by Qiagen. Subsequently, DNA quantity and quality were evaluated using a Qubit 3.0 fluorometer (Thermo Fisher Scientific) and Nanodrop 2000 (Thermo Fisher Scientific), respectively. The creation of sequencing libraries was accomplished using the KAPA Hyper Prep Kit from KAPA Biosystems. Hybridization-based target enrichment was executed employing the GeneseeqPrime&#x000ae; pan-cancer gene panel along with xGen Lockdown Hybridization and Wash Reagents Kit (Integrated DNA Technologies). Library capture was carried out using Dynabeads M-270 from Life Technologies, followed by amplification using KAPA HiFi HotStart ReadyMix from KAPA Biosystems. Library quantification was performed using qPCR and the KAPA Library quantitative Kit from KAPA Biosystems. Finally, the libraries were subjected to sequencing on an Illumina Hiseq4000 platform.</p></sec><sec id="Sec5"><title>Bioinformatics analysis of DNA sequencing data</title><p id="Par45">Sequencing data was processed as previously described [<xref ref-type="bibr" rid="CR24">24</xref>]. In brief, Trimmomatic (v0.36) was employed to perform quality control on the FASTQ files [<xref ref-type="bibr" rid="CR25">25</xref>]. Leading and trailing low-quality bases (with a quality score below 20) and N bases were removed using the default parameters. Subsequently, the sequencing data was aligned to the reference Human Genome (hg19) using the Burrows-Wheeler Aligner (BWA-mem, v0.7.12), followed by deduplication of alignment results using Sambamba. Base quality recalibration and indel realignment were conducted with the Genome Analysis Toolkit (GATK 3.4.0). Variant calling, including single-nucleotide variations (SNVs) and insertion/deletions (INDELs), was performed using VarScan2. Genomic fusions were identified using FACTERA with default parameters, and copy-number variations (CNVs) were detected utilizing CNVkit with default parameters, applying a threshold of 0.6 for copy-number loss and 2.0 for copy-number gain. The tumor mutational burden (TMB) was defined as the number of somatic, coding, base substitution, and indel mutations per megabase of genome examined, and was calculated as previously described [<xref ref-type="bibr" rid="CR26">26</xref>]. Microsatellite instability (MSI) analysis was conducted as previously described [<xref ref-type="bibr" rid="CR27">27</xref>]. In brief, the stability of 52 microsatellite sites, including classic MSI sites (BAT-25, BAT-26, NR-21, NR-24, and MONO-27), was assessed to generate an overall MSI score. A site was considered qualified with at least 101X coverage, and a sample was classified as MSI if&#x02009;&#x0003e;&#x02009;40% of qualified sites with&#x02009;&#x02265;&#x02009;100X coverage showed instability.</p></sec><sec id="Sec6"><title>RNA sequencing and bioinformatics analysis</title><p id="Par46">RNA extraction and sequencing were performed as previously described [<xref ref-type="bibr" rid="CR28">28</xref>]. In brief, total RNA extraction from FFPE tumor samples was carried out using the RNeasy FFPE Kit (QIAGEN), and the RNA quantity was assessed using a Bioanalyzer 2100 (Agilent Technologies). To eliminate ribosomal RNA and any remaining genomic DNA, enzymatic digestion with RNase H and DNase was performed. Subsequently, sequencing libraries were constructed employing the KAPA Stranded RNA-Seq Library Preparation Kit with RiboErase (HMR) (KAPA Biosystems). Library quality was verified using the Agilent High Sensitivity DNA kit on a Bioanalyzer 2100 (Agilent Technologies). Paired-end sequencing (PE150) was performed on an Illumina HiSeq4000 NGS platform following the manufacturer&#x02019;s instructions. All samples included in downstream analyses passed quality control (Additional file 2).</p><p id="Par47">Illumina bcl2Fastq software (v2.19.0.316) was employed for base calling, resulting in the generation of sequence reads in FASTQ format. Quality control was performed using Trimmomatic (v0.36) to remove leading and trailing low-quality bases (with a quality score below 20) and N bases [<xref ref-type="bibr" rid="CR25">25</xref>]. Transcriptomic mapping was executed through STAR software (v2.7.3a), aligning the data to the reference human genome (hg19). Gene-level quantification was performed using RSEM (version 1.2.31), and gene expression was quantified as transcripts per million (TPM). Patients were stratified into high and low-expression groups based on whether their gene TPM value was above or below the median TPM value across all samples. Differentially expressed genes (DEGs) were analyzed using the R package &#x0201c;DESeq2&#x0201d;. The thresholds for identifying DEGs were log2 (fold change)&#x02009;&#x0003e; &#x000b1;&#x02009;1, and a false discovery rate (FDR)-adjusted <italic>P</italic>-value (padj)&#x02009;&#x0003c;&#x02009;0.05. Gene set enrichment analysis (GSEA) was conducted utilizing fgsea (v1.20.0), with statistical significance defined as |NES|&#x02265;&#x02009;1.5 and P-adjust&#x02009;&#x0003c;&#x02009;0.1.</p></sec><sec id="Sec7"><title>Immune infiltrates analysis</title><p id="Par48">The estimation of the proportions of tumor-infiltrating immune cells was performed using xCell [<xref ref-type="bibr" rid="CR29">29</xref>]. Tumor microenvironment (TME) scores were estimated using the R package TMEscore (<ext-link ext-link-type="uri" xlink:href="https://github.com/DongqiangZeng0808/TMEscore">https://github.com/DongqiangZeng0808/TMEscore</ext-link>) [<xref ref-type="bibr" rid="CR30">30</xref>]. In brief, the TMEscore was calculated using consensus clustering to subgroup patients based on patterns of tumor microenvironment (TME) cell infiltration. Differentially expressed genes were identified among the resulting clusters, followed by dimensionality reduction to generate the TMEscore. TMEscoreA is immune cell-related, characterized by genes implicated in immune activation and response to viruses and IFN&#x003b3;. TMEscoreB is stromal cell-related and enriched for genes involved in extracellular matrix remodeling, epithelial-mesenchymal transition, cell adhesion, and angiogenesis. The sum of these two relevant individual scores was termed TMEscore. T cell dysfunction and exclusion scores were evaluated using Tumor Immune Dysfunction and Exclusion (TIDE, <ext-link ext-link-type="uri" xlink:href="http://tide.dfci.harvard.edu">http://tide.dfci.harvard.edu</ext-link>).</p></sec><sec id="Sec8"><title>In vitro and in vivo experiments</title><p id="Par49">The mouse cholangiocarcinoma model was successfully established through hydrodynamic tail vein injection of AKT/YAP (S127A) plasmids. Co-injection of oe-<italic>Con</italic> and oe-<italic>CXCL9</italic> plasmids achieved the expression regulation of CXCL9. Further immunohistochemistry (IHC) staining and flow cytometry were performed to evaluate the infiltration and activation of T cells in mouse CCA tissues between different groups. A tumor cell-T cell co-culture model and patient-derived organoid-T cell co-culture system were also established to detect the role of CXCL9 on T cell function in vitro. In addition, the immune effects of CXCL9/CTLA4 and its predictive value for chemoIO efficacy were clinically validated through IHC staining based on a multicenter clinical validation cohort (<italic>N</italic>&#x02009;=&#x02009;40). All relevant experimental methods are available in Additional file 1.</p></sec><sec id="Sec9"><title>Statistical analyses</title><p id="Par50">In the genomic and transcriptomic comparisons, Fisher&#x02019;s exact test was employed to assess disparities in categorical variables across groups, while the Wilcoxon rank sum test was utilized to ascertain distinctions in continuous variables between these groups. In the animal and cell line experiments, Student&#x02019;s unpaired <italic>t</italic>-test and one-way ANOVA (Tukey&#x02019;s and Bonferroni&#x02019;s multiple comparison test) were employed, and the Wilcoxon rank-sum test was utilized to assess the differences between IHC staining of patients&#x02019; resection specimens. Trends in the proportion of specific binary categorical variables, either increasing or decreasing, were assessed using the Cochran-Armitage test, whereas the Jonckheere trend test was applied to evaluate trends concerning ordinal variables. Spearman&#x02019;s Rho correlation coefficient was employed to analyze associations among continuous features. Survival analysis involved the generation of PFS and OS curves and respective 95% confidence interval (CI) using the Kaplan&#x02013;Meier method, with comparisons conducted using the log-rank test. Hazard ratios (HRs) were computed and compared using univariate Cox proportional hazards regression models. To account for multiple comparisons, a false discovery rate (FDR) correction was implemented. Two-tailed <italic>P</italic>-value&#x02009;&#x0003c;&#x02009;0.05 was considered statistically significant unless otherwise specified. All statistical analyses were executed using R software (v4.1.2).</p></sec></sec><sec id="Sec10"><title>Results</title><sec id="Sec11"><title>Study cohort and patient characteristics</title><p id="Par51">A total of 125 patients with advanced BTCs who received first-line chemoimmunotherapy between November 2017 and August 2022 were prospectively enrolled in this study, including 54 patients with gallbladder cancer (GC), 57 with intrahepatic cholangiocarcinoma (ICC), and 14 with extrahepatic cholangiocarcinoma (ECCA) (Fig. <xref rid="Fig1" ref-type="fig">1</xref>A). The median age was 63&#x000a0;years (range 34&#x02013;82) and 52.8% (66/125) were male. The median follow-up duration was 14.8&#x000a0;months for the entire cohort. Median PFS and OS were 6.9&#x000a0;months (95% CI 6.2&#x02013;7.9) and 11.8&#x000a0;months (95% CI 10.3&#x02013;14.8), respectively (Fig.&#x000a0;<xref rid="Fig1" ref-type="fig">1</xref>B,C). The median PFS for patients with GC, ICC, and ECCA was 6.83, 6.93, and 6.77&#x000a0;months, respectively, while the median OS was 11.10, 12.75, and 8.61&#x000a0;months, with no significant differences observed (PFS: <italic>P</italic>&#x02009;=&#x02009;0.720, OS: <italic>P</italic>&#x02009;=&#x02009;0.714; Fig.&#x000a0;<xref rid="Fig1" ref-type="fig">1</xref>D,E).<fig id="Fig1"><label>Fig.&#x000a0;1</label><caption><p>Survival and baseline molecular characteristics of patients with advanced biliary tract cancer. <bold>A</bold> Workflow of the study. <bold>B</bold> Kaplan&#x02013;Meier curves for progression-free survival of all patients. <bold>C</bold> Kaplan&#x02013;Meier curves for overall survival of all patients. <bold>D</bold> Kaplan&#x02013;Meier curves for progression-free survival of patients with gallbladder cancer, intrahepatic cholangiocarcinoma, and extrahepatic cholangiocarcinoma. <bold>E</bold> Kaplan&#x02013;Meier curves for overall survival of patients with gallbladder cancer, intrahepatic cholangiocarcinoma, and extrahepatic cholangiocarcinoma. <bold>F</bold> Oncoprint showing baseline molecular characteristics of all patients. Abbreviations: GC, gallbladder cancer; ICC, intrahepatic cholangiocarcinoma; ECCA, extrahepatic cholangiocarcinoma</p></caption><graphic xlink:href="12916_2025_4263_Fig1_HTML" id="MO1"/></fig></p><p id="Par52">All 125 patients underwent targeted DNA sequencing, and 62 of them underwent RNA sequencing (Additional file 1: Figure S1). The mutational profiles are shown in Fig.&#x000a0;<xref rid="Fig1" ref-type="fig">1</xref>F. The most frequently mutated genes were <italic>TP53</italic> (64/125, 51.2%), <italic>KRAS</italic> (34/125, 27.2%), <italic>ERBB2</italic> (18/125, 14.4%), and <italic>ARID1A</italic> (17/125, 13.6%). Higher frequencies of <italic>KRAS</italic> and <italic>ARID1A</italic> mutations were observed in ECCA and ICC, whereas <italic>ERBB2</italic> and <italic>ARID2</italic> mutations were more common in GC. All patients included in the study were microsatellite stable (MSS).</p></sec><sec id="Sec12"><title>Associations between baseline genomic, transcriptomic, and tumor microenvironment features and chemoimmunotherapy response</title><p id="Par53">Of all 125 patients, 6 achieved complete response (CR), 41 experienced partial response (PR), 67 showed stable disease (SD), and 11 had progressive disease (PD), resulting in an ORR of 37.6%. No significant differences in clinical characteristics were observed between the response and non-response groups (Additional file 1: Table S1), and the baseline serum-level markers showed no association with treatment response (Additional file 1: Figure S2).</p><p id="Par54">The mutational landscapes of the response and non-response groups are shown in Fig.&#x000a0;<xref rid="Fig2" ref-type="fig">2</xref>A. The mutation frequencies of <italic>BRCA2</italic> and <italic>TP53</italic>, as well as the TMB, were higher in the response group (<italic>P</italic>&#x02009;=&#x02009;0.002, 0.042, 0.072, respectively) and demonstrated a decreasing trend across the response status (<italic>P</italic>&#x02009;=&#x02009;0.002, 0.067, 0.008, respectively; Fig.&#x000a0;<xref rid="Fig2" ref-type="fig">2</xref>B&#x02013;D). In terms of immune regulatory pathways, the cytokine pathway displayed a significantly higher mutation frequency in the response group (<italic>P</italic>&#x02009;=&#x02009;0.004, Fig.&#x000a0;<xref rid="Fig2" ref-type="fig">2</xref>E). In the context of the TME, the response group exhibited a higher TME score (<italic>P</italic>&#x02009;=&#x02009;0.048), particularly TME score A, which represents immune-relevant signatures (<italic>P</italic>&#x02009;=&#x02009;0.036, Fig.&#x000a0;<xref rid="Fig2" ref-type="fig">2</xref>F).<fig id="Fig2"><label>Fig.&#x000a0;2</label><caption><p>Comparisons of baseline molecular characteristics between patients with advanced biliary tract cancer responding or not responding to chemoimmunotherapy.<bold> A</bold> Baseline mutational profiles of response and non-response patients. <bold>B&#x02013;D</bold> Mutation frequencies of <italic>BRCA2</italic>, <italic>TP53</italic>, and tumor mutation burdens in response and non-response patients, and patients with each RECIST 1.1 response status for chemoimmunotherapy. <bold>E</bold> Differences in mutation frequencies of immune regulatory pathways between chemoimmunotherapy response and non-response patients. <bold>F</bold> Comparisons of TME scores between chemoimmunotherapy response and non-response patients. <bold>G</bold> Distribution of <italic>TP53</italic> mutation status, <italic>BRCA2</italic> mutation status, cytokine gene mutation status, and combination of the 3 alterations (alteration level) in each RECIST 1.1 response status. Alteration level&#x02009;=&#x02009;0 means negative in all the 3 alterations; alteration level&#x02009;=&#x02009;1 means positive in 1 of the 3 alterations; alteration level&#x02009;&#x02265;&#x02009;2 means positive in at least 2 of the 3 alterations. <bold>H</bold> Comparison of the frequency of each RECIST 1.1 response status in three alteration level groups. Abbreviations: GC, gallbladder cancer; ICC, intrahepatic cholangiocarcinoma; ECCA, extrahepatic cholangiocarcinoma; CR, complete response; PR, partial response; SD, stable disease; PD, progressive disease</p></caption><graphic xlink:href="12916_2025_4263_Fig2_HTML" id="MO2"/></fig></p><p id="Par55">In light of the aforementioned discoveries, our subsequent investigation delved into the connection between the number of identified somatic alterations (<italic>BRCA2</italic>, <italic>TP53</italic>, and the cytokines pathway) in patients and their actual response to chemoIO (Fig.&#x000a0;<xref rid="Fig2" ref-type="fig">2</xref>G). Consequently, we observed noteworthy ascending trends in both the rates of patients achieving CR and PR with the escalation of alteration levels (Fig.&#x000a0;<xref rid="Fig2" ref-type="fig">2</xref>H). The results suggest that these mutations are promising predictive biomarkers for chemoIO response in BTC.</p></sec><sec id="Sec13"><title>Predictive biomarkers for chemoimmunotherapy efficacy and survival</title><p id="Par56">The most extensively recognized and studied biomarkers for chemoimmunotherapy efficacy comprise PD-L1 expression, TMB, and MSI. Nevertheless, within our study, neither PD-L1 RNA expression nor TMB proved to be reliable predictors of PFS or OS (Additional file 1: Figure S3). Notably, patients with abnormal serum carcinoembryonic antigen (CEA) levels exhibited worse PFS (<italic>P</italic>&#x02009;=&#x02009;0.025), and those with abnormal serum CEA, carbohydrate antigen 125 (CA125), and C-reactive protein (CRP) levels experienced poorer outcomes in terms of OS (<italic>P</italic>&#x02009;=&#x02009;0.011, 0.003, 0.035, respectively; Additional file 1: Table S2).</p><p id="Par57">Subsequently, we evaluated the impact of all gene mutations with minor allele frequencies (MAF) exceeding 5% on the PFS and OS under chemoimmunotherapy. Patients harboring mutations in <italic>KRAS</italic> and <italic>ARID1A</italic> exhibited notably poorer PFS (median, <italic>KRAS</italic> variant vs. WT: 5.95 vs. 7.13&#x000a0;months, <italic>P</italic>&#x02009;=&#x02009;0.009; <italic>ARID1A</italic> variant vs. WT: 5.39 vs. 7.1&#x000a0;months, <italic>P</italic>&#x02009;=&#x02009;0.009) and OS (median, <italic>KRAS</italic> variant vs. WT: 8.61 vs. 14.36&#x000a0;months, <italic>P</italic>&#x02009;=&#x02009;0.002; <italic>ARID1A</italic> variant vs. WT: 7.97 vs. 12.19&#x000a0;months, <italic>P</italic>&#x02009;=&#x02009;0.012) (Fig.&#x000a0;<xref rid="Fig3" ref-type="fig">3</xref>A&#x02013;D; Additional file 1: Figure S4), and this trend persisted across all three subtypes of BTC (Additional file 1: Figure S5). It is noteworthy that all <italic>ARID1A</italic> mutations identified in this study were loss-of-function mutations. Remarkably, we noticed that patients carrying <italic>KRAS</italic> G12D mutations experienced significantly shorter PFS (median, <italic>KRAS</italic> G12D vs. others: 4.83 vs. 7.13&#x000a0;months, <italic>P</italic>&#x02009;&#x0003c;&#x02009;0.001) and OS (median, <italic>KRAS</italic> G12D vs. others: 7.59 vs. 13.73&#x000a0;months, <italic>P</italic>&#x02009;=&#x02009;0.034) compared to all other patients (Fig.&#x000a0;<xref rid="Fig3" ref-type="fig">3</xref>E,F) in all BTC subtypes (Additional file 1: Figure S6A-B). Even among <italic>KRAS</italic>-mutated patients, patients carrying <italic>KRAS</italic> G12D mutations experienced shorter PFS (Additional file 1: Figure S6C-D), which implies that <italic>KRAS</italic> G12D serves as an adverse biomarker for chemoIO efficacy.<fig id="Fig3"><label>Fig.&#x000a0;3</label><caption><p>Survival analysis through stratification by <italic>ARID1A</italic>, <italic>KRAS</italic>, or <italic>KRAS </italic>G12D mutational status in advanced biliary tract cancer.<bold> A,B</bold> Kaplan&#x02013;Meier curves for progression-free survival and overall survival of patients with <italic>ARID1A</italic> mutations and patients with wild-type <italic>ARID1A</italic>. <bold>C,D</bold> Kaplan&#x02013;Meier curves for progression-free survival and overall survival of patients with <italic>KRAS</italic> mutations and patients with wild-type <italic>KRAS</italic>. <bold>E,F</bold> Kaplan&#x02013;Meier curves for progression-free survival and overall survival of patients with <italic>KRAS</italic> G12D mutations and all other patients</p></caption><graphic xlink:href="12916_2025_4263_Fig3_HTML" id="MO3"/></fig></p><p id="Par58">At the transcriptional level, we found 40 downregulated and 12 upregulated genes in BTC patients who responded to chemoIO compared to non-responders (Additional file 1: Figure S7A). Subsequent network analysis revealed no notable relationships among these up- and downregulated genes (Additional file 1: Figure S7B-C). GSEA demonstrated that cell cycle-related pathways (E2F, G2M checkpoint) and MTORC1 signaling were enriched in responders, whereas hypoxia, TGF beta, and NOTCH signaling were more prominent in non-responders (Additional file 1: Figure S7D). Most immune cell types showed no statistically significant differences between subgroups, except for eosinophils and plasmacytoid dendritic cells (Additional file 1: Figure S7E-F). These findings prompted us to focus on immune-relevant gene sets, where we observed that responders exhibited higher expression of immune checkpoint genes (e.g., <italic>PD-1</italic>, <italic>CTLA4</italic>, <italic>PD-L2</italic>) and several CXCL family members (e.g., <italic>CXCL9</italic>, <italic>CXCL10</italic>, and <italic>CXCL11</italic>), whereas <italic>CXCL14</italic> expression was elevated in non-responders (Additional file 1: Figure S8). Through univariate Cox regression models, we showed that the expression levels of <italic>CTLA4</italic> and <italic>CXCL9</italic> were associated not only with therapeutic response but also with PFS and OS of patients (Fig.&#x000a0;<xref rid="Fig4" ref-type="fig">4</xref>; Additional file 1: Table S3). Specifically, patients exhibited elevated <italic>CTLA4</italic> and <italic>CXCL9</italic> expressions displayed higher response rate (<italic>CTLA4</italic>: <italic>P</italic>&#x02009;=&#x02009;0.067, Fig.&#x000a0;<xref rid="Fig4" ref-type="fig">4</xref>A; <italic>CXCL9</italic>: <italic>P</italic>&#x02009;=&#x02009;0.014; Fig.&#x000a0;<xref rid="Fig4" ref-type="fig">4</xref>B), extended PFS (median, <italic>CTLA4</italic> high vs. low: 9.07 vs. 6.18&#x000a0;months, <italic>P</italic>&#x02009;=&#x02009;0.008, Fig.&#x000a0;<xref rid="Fig4" ref-type="fig">4</xref>C; <italic>CXCL9</italic> high vs. low: 7.66 vs. 6.67&#x000a0;months, <italic>P</italic>&#x02009;=&#x02009;0.018; Fig.&#x000a0;<xref rid="Fig4" ref-type="fig">4</xref>D), and longer OS (median, <italic>CTLA4</italic> high vs. low: 14.32 vs. 10.58&#x000a0;months, <italic>P</italic>&#x02009;=&#x02009;0.008, Fig.&#x000a0;<xref rid="Fig4" ref-type="fig">4</xref>E; <italic>CXCL9</italic> high vs. low: 16.92 vs. 10.58&#x000a0;months, <italic>P</italic>&#x02009;=&#x02009;0.010; Fig.&#x000a0;<xref rid="Fig4" ref-type="fig">4</xref>F). This consistent pattern was observed across all BTC subtypes (Additional file 1: Figure S9).<fig id="Fig4"><label>Fig.&#x000a0;4</label><caption><p>The effects of <italic>CXCL9</italic> or <italic>CTLA4 </italic>RNA expression levels on the response, progression-free survival, and overall survival to immune checkpoint inhibitors in patients with advanced biliary tract cancer.<bold> A</bold> Comparison of <italic>CTLA4</italic> expression levels in response and non-response patients. <bold>B</bold> Comparison of <italic>CXCL9</italic> expression levels in response and non-response patients. <bold>C,D</bold> Kaplan&#x02013;Meier curves for progression-free survival of patients with high <italic>CTLA4/CXCL9</italic> expression levels and patients with low <italic>CTLA4/CXCL9</italic> expression levels, stratified by the median TPM expression of all patients. <bold>E,F</bold> Kaplan&#x02013;Meier curves for overall survival of patients with high <italic>CTLA4/CXCL9</italic> expression levels and patients with low <italic>CTLA4/CXCL9</italic> expression levels, stratified by the median TPM expression of all patients</p></caption><graphic xlink:href="12916_2025_4263_Fig4_HTML" id="MO4"/></fig></p></sec><sec id="Sec14"><title>CTLA4 and CXCL9 are highly correlated with elevated immune activities</title><p id="Par59">Given the pivotal roles of <italic>CTLA4</italic> and <italic>CXCL9</italic> in immune regulation, we subsequently investigated the tumor immune microenvironment in relation to their expression levels using RNA sequencing data. Elevated expression levels of either <italic>CTLA4</italic> or <italic>CXCL9</italic> were correlated with increased expression levels of immune checkpoint-related genes, such as <italic>CD274</italic> (<italic>PD-L1</italic>, <italic>P</italic>&#x02009;&#x0003c;&#x02009;0.001; Additional file 1: Figure S10A-B) and <italic>PDCD1</italic> (<italic>PD-1</italic>, <italic>P</italic>&#x02009;&#x0003c;&#x02009;0.001; Additional file 1: Figure S10C-D), as well as significantly heightened TME scores (both <italic>P</italic>&#x02009;&#x0003c;&#x02009;0.001; Additional file 1: Figure S10E-F), indicating immune activation. In GSEA analysis, we observed significant enrichment of multiple immune-related pathways in patients with high <italic>CTLA4</italic> and <italic>CXCL9</italic> expression levels, encompassing pathways such as interferon-gamma response, interferon alpha response, inflammatory response, and IL2/STAT5 signaling pathways (Additional file 1: Figure S10G-H). These findings underscore heightened immune activity in this subgroup of patients. We further observed a strong positive correlation between the expressions of <italic>CTLA4</italic> and <italic>CXCL9</italic> (<italic>P</italic>&#x02009;&#x0003c;&#x02009;0.001, R&#x02009;=&#x02009;0.71, Additional file 1: Figure S10I), suggesting potential co-expression of these two genes.</p><p id="Par60">To validate our findings, we incorporated an independent internal validation cohort consisting of 33 patients with advanced BTC with available baseline RNA sequencing data, along with an external validation cohort consisting of 255 patients with advanced intrahepatic cholangiocarcinoma and baseline RNA sequencing data [<xref ref-type="bibr" rid="CR31">31</xref>, <xref ref-type="bibr" rid="CR32">32</xref>]. The expressions of <italic>CXCL9</italic> and <italic>CTLA4</italic> remained highly correlated in both internal (<italic>P</italic>&#x02009;=&#x02009;0.002, R&#x02009;=&#x02009;0.52, Additional file 1: Figure S10J) and external (<italic>P</italic>&#x02009;&#x0003c;&#x02009;0.001, R&#x02009;=&#x02009;0.53, Additional file 1: Figure S10K) validation data sets. Consistently, elevated expression levels of immune checkpoint genes <italic>CD274</italic> (<italic>PD-L1</italic>) and <italic>PDCD1</italic> (<italic>PD-1</italic>), along with enhanced immune regulatory activity, were verified in both the internal and external validation cohorts (Additional file 1: Figure S11).</p></sec><sec id="Sec15"><title>Genomic-transcriptomic subtypes predict outcomes under chemoimmunotherapy</title><p id="Par61">After investigating predictive biomarkers of treatment response and efficacy in BTCs at both genomic and transcriptomic levels, we categorized the patients into three subtypes based on these biomarkers. In Fig.&#x000a0;<xref rid="Fig5" ref-type="fig">5</xref>A, patients with high <italic>CTLA4</italic> or <italic>CXCL9</italic> expressions without <italic>KRAS</italic> G12D and <italic>ARID1A</italic> mutations were categorized as Type I, those with low <italic>CTLA4</italic> and <italic>CXCL9</italic> expressions but with <italic>KRAS</italic> G12D or <italic>ARID1A</italic> mutations as Type II, and all other patients as Type III. In survival analysis, Type I exhibited the highest survival rates, followed by Type III and Type II, for both PFS (median, Type I vs. Type II vs. Type III: 9.07 vs. 4.42 vs. 6.70&#x000a0;months, <italic>P</italic>&#x02009;=&#x02009;0.002; Fig.&#x000a0;<xref rid="Fig5" ref-type="fig">5</xref>B, D) and OS (median, Type I vs. Type II vs. Type III: not reached [NR] vs. 7.16 vs. 11.10&#x000a0;months, <italic>P</italic>&#x02009;=&#x02009;0.002; Fig.&#x000a0;<xref rid="Fig5" ref-type="fig">5</xref>C, E). Multivariate Cox regression analysis was performed to adjust for potential confounding variables, including age, sex, and tumor subtype. Molecular subtype emerged as a significant independent factor associated with both PFS and OS (Additional file 1: Table S4).<fig id="Fig5"><label>Fig.&#x000a0;5</label><caption><p>Combination survival analysis based on <italic>KRAS</italic> G12D and <italic>ARID1A</italic> mutation status, as well as <italic>CXCL9</italic> and <italic>CTLA4</italic> expression levels, in patients with advanced biliary tract cancer treated with first-line immune checkpoint inhibitors in combination with chemotherapy.<bold> A</bold> Oncoprint shows how patients were categorized into three types based on the mutation status of <italic>KRAS</italic> G12D and <italic>ARID1A</italic>, as well as the expression levels of <italic>CTLA4</italic> and <italic>CXCL9</italic>. Patients with high <italic>CTLA4</italic> or <italic>CXCL9</italic> expressions without <italic>KRAS</italic> G12D and <italic>ARID1A</italic> mutations were categorized as Type I, those with low <italic>CTLA4</italic> and <italic>CXCL9</italic> expressions but with <italic>KRAS</italic> G12D or <italic>ARID1A</italic> mutations as Type II, and all other patients as Type III. <bold>B,C</bold> Kaplan&#x02013;Meier curves for progression-free survival and overall survival of Type I, Type II, and Type III patients. <bold>D,E</bold> Cox analysis of progression-free survival and overall survival among Type I, Type II, and Type III patients. Abbreviations: GC, gallbladder cancer; ICC, intrahepatic cholangiocarcinoma; ECCA, extrahepatic cholangiocarcinoma</p></caption><graphic xlink:href="12916_2025_4263_Fig5_HTML" id="MO5"/></fig></p></sec><sec id="Sec16"><title>High CXCL9 expression enhances chemoimmunotherapy efficacy by promoting T-cell infiltration and activation</title><p id="Par62">Considering the promising predictive value of <italic>CXCL9</italic> expression in chemoIO response, PFS, and OS, along with the associated highly regulated immune pathways, we are motivated to explore the role of CXCL9 in the therapeutic effects elicited by chemoimmunotherapy. We established a mouse cholangiocarcinoma model via hydrodynamic transfection of AKT/YAP(S127A) plasmids [<xref ref-type="bibr" rid="CR33">33</xref>], as illustrated in Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>A and Additional file 1: Figure S12A. Hydrodynamic injection of CXCL9-overexpression plasmids led to successful CXCL9 overexpression in the mouse model, as validated by IHC staining and qRT-PCR analysis (Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>B; Additional file 1: Figure S12B). Following anti-PD-1 antibody and gemcitabine treatment (Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>A), we collected the liver tissues for subsequent analyses. Compared with the control group, chemoIO delayed tumor progression (Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>B). Notably, overexpression of CXCL9 significantly enhanced treatment efficacy and reduced tumor burden, as indicated by decreased tumor size, liver weight, and H&#x00026;E staining (Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>B,C). We next analyzed the tumor immune microenvironment using the mouse model. As expected, chemoIO induced a robust anti-tumor immune response, as indicated by elevated infiltration of CD3<sup>+</sup>, CD4<sup>+</sup>, and CD8<sup>+</sup>-T cells via IHC staining (Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>D). Strikingly, overexpression of CXCL9 further increased the infiltration of T cells following chemoIO. Flow cytometry analysis was further applied to measure the activities of T cells and showed that overexpression of CXCL9 with chemoIO dramatically enhanced the production of TNF&#x003b1;, IFN-&#x003b3;, and perforin intratumorally in CD8<sup>+</sup>-T cell subsets compared with the control group, while the overall trend of Granzyme B (GZMB) was not significant (Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>E).<fig id="Fig6"><label>Fig.&#x000a0;6</label><caption><p>High CXCL9 expression enhances chemoimmunotherapy efficacy and T cell infiltration and activation in biliary tract cancer.<bold> A</bold> Schematic illustration for establishing a mouse cholangiocarcinoma model using the hydrodynamic method and subsequent chemoimmunotherapy regimen (By Figdraw. ID: RTAllaaa2a). Mice were all injected with AKT/YAP(S127A) plasmid and were divided into four groups: oe-Con, oe-<italic>CXCL9</italic>, oe-Con&#x02009;+&#x02009;chemoimmunotherapy group, oe-<italic>CXCL9</italic>&#x02009;+&#x02009;chemoimmunotherapy group. <italic>N</italic>&#x02009;=&#x02009;6 per group. <bold>B</bold> Images of tumor-bearing livers, representative H&#x00026;E staining (50X), and CXCL9 IHC staining (66X). <bold>C</bold> Measurement of liver weight for each group. <bold>D</bold> Representative images and statistical graphs of CD3, CD4, and CD8 IHC staining in four groups of tumor tissues. Scale bars: 200&#x000a0;&#x000b5;m and 25&#x000a0;&#x000b5;m. <bold>E</bold> Flow cytometry analysis of TNF&#x003b1;, IFN-&#x003b3;, Perforin, and GZMB expression by CD8<sup>+</sup> T cells in tumor tissues from four groups. <italic>N</italic>&#x02009;=&#x02009;6 for each group. <bold>F</bold> Schematic diagram of human PBMC isolation, T cell activation, and cancer cell co-culture models (Created with BioRender.com). Activated T cells derived from PBMCs were co-cultured with multiple BTC cells with modified CXCL9 expression at a 2:1 ratio for 24&#x000a0;h. <bold>G</bold> Flow cytometry analysis of GZMB, Perforin, and TNF&#x003b1; expression by CD8<sup>+</sup> T cells when co-cultured with RBE cells harboring sh-Con or sh-<italic>CXCL9</italic>. The experiments were repeated three times. <bold>H, I</bold> Representative images and statistical graphs of CD3, CD4, and CD8 IHC staining (200X) in human tumor tissues with high (<italic>N</italic>&#x02009;=&#x02009;21) and low (<italic>N</italic>&#x02009;=&#x02009;19) expression of CXCL9, stratified by the median positivity proportion. <bold>J</bold> Validation of CXCL9 expression levels in response and non-response patients of the clinical validation cohort. <bold>K, L</bold> Kaplan&#x02013;Meier curves for progression-free survival (<bold>K</bold>) and overall survival (<bold>L</bold>) of patients with high versus low CXCL9 expression levels. Note: ns, <italic>P</italic>&#x02009;&#x0003e;&#x02009;0.05; *, <italic>P</italic>&#x02009;&#x0003c;&#x02009;0.05; **, <italic>P</italic>&#x02009;&#x0003c;&#x02009;0.01; ***, <italic>P</italic>&#x02009;&#x0003c;&#x02009;0.001; ****, <italic>P</italic>&#x02009;&#x0003c;&#x02009;0.0001</p></caption><graphic xlink:href="12916_2025_4263_Fig6_HTML" id="MO6"/></fig></p><p id="Par63">To further confirm the effect of CXCL9 on T cell functions, we established tumor cell-T cell co-culture models using three BTC cell lines (Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>F). <italic>CXCL9</italic> was knocked down in RBE and HuCCT1 cells using two independent shRNAs, while lentivirus-mediated <italic>CXCL9</italic> overexpression was performed in HCCC-9810 cells (Additional file 1: Figure S12C-E). Activated T cells isolated from PBMCs were co-cultured with these cells, and their cytotoxicity was detected by flow cytometry. Flow cytometry analysis revealed that <italic>CXCL9</italic> knockdown in both RBE cells and HuCCT1 cells significantly reduced expression levels of GZMB, perforin, TNF&#x003b1; in co-cultured CD8<sup>+</sup> T cells (Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>F, G; Additional file 1: Figure S13A). In contrast, CXCL9 overexpression in HCCC-9810 cells led to a significant increase in the expression levels of these markers (Additional file 1: Figure S13B). For further validation, we established patient-derived BTC organoids (Additional file 1: Figure S14A). In the organoid-T cell co-culture system, exogenous CXCL9 enhanced the cytotoxic function of CD8<sup>+</sup> T cells, as demonstrated by increased expression of GZMB, perforin, and TNF&#x003b1; (Additional file 1: Figure S14B). Consistently, ELISA analysis showed elevated IFN&#x003b3; production in the co-culture supernatant following CXCL9 treatment (Additional file 1: Figure S14C). Collectively, the findings from both the cell line and patient-derived organoids are consistent with those observed in the mouse model, providing additional evidence for the critical role of CXCL9 in maintaining T cell cytotoxicity in BTC.</p><p id="Par64">As previously shown in Additional file 1: Figure S10, high <italic>CXCL9</italic>/<italic>CTLA4</italic> expression was associated with the upregulation of pathways related to interferon-gamma response, interferon-alpha response, inflammatory response, and IL2/STAT5 signaling. To validate these results, we targeted STAT1 and STAT5 using small molecule inhibitors, given that STAT1 is a key transcriptional factor in the INF-&#x003b3; and INF-&#x003b1; signaling pathways, while STAT5 mediates IL2 pathway activation. The addition of fludarabine and STAT5-IN-1 effectively suppressed the phosphorylation of STAT1 and STAT5, respectively (Additional file 1: Figure S15A, B). Flow cytometry analysis revealed that STAT1 inhibition by fludarabine significantly reduced the level of GZMB, perforin, and TNF&#x003b1; in&#x000a0;T cells co-cultured with both HuCCT1 and HCCC-9810 cell lines (Additional file 1: Figure S15C, D). STAT5 inhibition led to significantly reduced expression of TNF&#x003b1; and perforin in&#x000a0;T cells co-cultured with HuCCT1 and HCCC-9810 cells, respectively, with consistent trends observed for other markers (Additional file 1: Figure S15E, F). These findings align with the differential expression analysis results from RNA-seq, providing valuable mechanistic insights into the role of CTLA4/CXCL9 in immune modulation.</p><p id="Par65">To verify the above results in clinical settings, we launched an independent multicenter clinical validation cohort consisting of 40 patients receiving first-line chemoIO (Additional file 1: Table S5). This cohort included three major subtypes of BTC: 11 patients with GC, 25 with ICC, and 4 with ECCA. IHC staining of CXCL9, CTLA4, CD3, CD4, and CD8 was performed on tumor resection specimens from these patients. The IHC results indicated that the distribution of CD3<sup>+</sup>, CD4<sup>+</sup>, and CD8<sup>+</sup> T cells was markedly higher in CXCL9/CTLA4 high-expression patients than in CXCL9/CTLA4 low-expression patients (Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>H,I; Additional file 1: Figure S16A-B). In terms of treatment efficacy, responders have higher expression levels of CXCL9 and CTLA4 compared to non-responders (Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>J; Additional file 1: Figure S16C). Besides, patients exhibited elevated CXCL9 and CTLA4 expressions displayed extended PFS (CXCL9: <italic>P</italic>&#x02009;=&#x02009;0.002, CTLA4: <italic>P</italic>&#x02009;=&#x02009;0.039; Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>K; Additional file 1: Figure S16D) and OS (CXCL9: <italic>P</italic>&#x02009;=&#x02009;0.024, CTLA4: <italic>P</italic>&#x02009;=&#x02009;0.090; Fig.&#x000a0;<xref rid="Fig6" ref-type="fig">6</xref>L; Additional file 1: Figure S16E). Given the rarity of ECCA patients in the clinical validation cohort, subgroup analyses were conducted on the remaining two subtypes based on the primary tumor site. In both ICC and GC patients, high CXCL9/CTLA4 expression was associated with increased T-cell infiltration, improved response to chemoIO, and better prognosis (Additional file 1: Figure S17 and S18). However, the small sample size may have reduced the statistical power of these findings. Taken together, CXCL9/CTLA4 plays critical roles in the therapeutic efficacy and T cell activation of chemoimmunotherapy in BTC patients.</p></sec></sec><sec id="Sec17"><title>Discussion</title><p id="Par66">The advent of ICI treatment has kindled optimism in the management of BTC. However, only a subset of patients derives substantial and lasting benefits from ICI therapies [<xref ref-type="bibr" rid="CR34">34</xref>]. As such, identifying predictive biomarkers for ICI efficacy and patient survival has become a key focus in clinical research. Single biomarkers, such as PD-L1 expression, MSI, and TMB, often fail to capture the intricate interactions within the tumor microenvironment, underscoring the need for more comprehensive approaches. This study addressed this challenge by leveraging a multi-omic approach to explore the genomic and transcriptomic landscapes of advanced BTC, revealing associations between genetic variants and RNA expression patterns with treatment response and prognostic outcomes in the context of first-line chemoIO. Notably, the study led to the development of a genetic signature-based classification approach, categorizing patients into three molecular subtypes to identify those most likely to benefit from treatment, while providing novel insights for therapeutic development.</p><p id="Par67">The most widely recognized and extensively studied biomarkers for evaluating the effectiveness of ICIs include PD-L1 expression, TMB, and MSI [<xref ref-type="bibr" rid="CR35">35</xref>&#x02013;<xref ref-type="bibr" rid="CR38">38</xref>]. Despite extensive investigations across various cancer types, their roles in predicting ICI response in BTC remain unclear [<xref ref-type="bibr" rid="CR39">39</xref>, <xref ref-type="bibr" rid="CR40">40</xref>]. Notably, all patients in this study had MSS tumors, and TMB was associated with a better response to chemoIO. Our findings reveal significant associations between chemoIO response and mutations in <italic>TP53</italic>, <italic>BRCA2</italic>, and the cytokine pathway. Regarding underlying mechanisms, <italic>TP53</italic> and <italic>BRCA2</italic> mutations are linked to impaired DNA damage repair processes, resulting in an accumulation of genetic mutations, increased neoantigen loads, and enhanced immunogenicity of cancer cells [<xref ref-type="bibr" rid="CR41">41</xref>&#x02013;<xref ref-type="bibr" rid="CR43">43</xref>]. Cytokines play a crucial role in immune system communication by rapidly propagating immune signaling and stimulating immune cells, which is essential for anti-tumor responses and informs cytokine-based cancer therapies [<xref ref-type="bibr" rid="CR44">44</xref>]. However, the functional impact of cytokine mutations on treatment response requires further investigation.</p><p id="Par68">This study identified <italic>KRAS</italic> G12D and <italic>ARID1A</italic> loss-of-function mutations as negative prognostic indicators, while high expressions of <italic>CTLA4</italic> and <italic>CXCL9</italic> were associated with a favorable response to cheomIO and improved survival outcomes. A previous investigation showed that ARID1A abnormalities limit access to chromatin in IFN-responsive genes, resulting in decreased IFN expression [<xref ref-type="bibr" rid="CR45">45</xref>]. This, in turn, diminishes cytotoxic T-cell infiltration, weakens tumor immunity, and shortens survival. Importantly, compromised IFN signaling has been associated with less-effective responses to ICI treatments [<xref ref-type="bibr" rid="CR45">45</xref>, <xref ref-type="bibr" rid="CR46">46</xref>]. Meanwhile, <italic>KRAS</italic> G12D mutation has been associated with immune suppression and primary resistance to PD-1/PD-L1 inhibitors in non-small cell lung cancer, through the downregulation of PD-L1 expression and a reduction in CD8<sup>+</sup> T cell infiltration [<xref ref-type="bibr" rid="CR47">47</xref>]. Nevertheless, the impact of these mutations on the effectiveness of chemoIO in BTC warrants further investigation.</p><p id="Par69">The expression levels of <italic>CXCL9</italic> and <italic>CTLA4</italic> consistently demonstrated predictive value for chemoIO response, PFS, and OS. CTLA4 is an immune checkpoint protein that binds with greater affinity to CD80 and CD86 compared to CD28. It acts as a competitive inhibitor of CD28 on antigen-presenting cells and is a frequently targeted molecule for ICIs [<xref ref-type="bibr" rid="CR48">48</xref>]. CXCL9, on the other hand, plays a crucial role in immune regulation and is a strong indicator of chemoIO efficacy across various cancer types [<xref ref-type="bibr" rid="CR49">49</xref>]. In our study, <italic>CXCL9</italic>, along with its family members <italic>CXCL10</italic> and <italic>CXCL11</italic>, all of which were upregulated in the response group, likely promotes the infiltration of tumor-suppressive lymphocytes by regulating the recruitment and activation of T cells [<xref ref-type="bibr" rid="CR50">50</xref>]. Overexpression of CXCL9 led to T-cell accumulation and improved survival, contingent upon the activation of adaptive immune responses [<xref ref-type="bibr" rid="CR51">51</xref>].</p><p id="Par70">Interestingly, we found that patients with elevated <italic>CXCL9/CTLA4</italic> expressions often exhibited heightened levels of immune checkpoints, such as <italic>CTLA4</italic>, <italic>PD-L1</italic>, and <italic>PD-1</italic>, indicating a highly activated immune environment in baseline tumor samples. These findings were corroborated by both internal and external validation cohorts. Elevated immune checkpoint levels are a hallmark of the TME, where they inhibit T cell activity, allowing tumors to persist [<xref ref-type="bibr" rid="CR52">52</xref>]. ICIs block this interaction, enabling T cells to more effectively target and eliminate cancer cells, which may explain their high efficacy in this patient group. The immune-modulatory roles of CXCL9 and CTLA4 were validated both in vitro and in vitro. Elevated CXCL9/CTLA4 expression enhances chemoIO efficacy by promoting T-cell infiltration and activation. Notably, the three molecular subtypes, integrating mutational and transcriptional biomarkers, provide valuable insights for identifying patients most likely to benefit from chemoimmunotherapy and for developing novel therapeutic strategies to improve survival outcomes in advanced BTC patients. Besides, future advances in BTC research are anticipated, driven by the development of innovative combination therapies involving ICIs, anti-angiogenic agents, bispecific antibodies, and emerging molecules like TIGIT inhibitors. The integration of cutting-edge technologies, such as spatial transcriptomics and microbiome analyses, will likely deepen our understanding of BTC biology and resistance mechanisms, advancing personalized treatment strategies through patient stratification.</p><p id="Par71">The study had various limitations that need to be acknowledged. Firstly, the lack of external validation for the identified predictive biomarkers at the mutational level represents a significant constraint. While our analysis provides compelling evidence for their potential utility, further studies in independent cohorts are essential to confirm their predictive power and ensure their generalizability across diverse populations. Secondly, the relatively small sample size of patients with ECCA may affect the generalizability and robustness of the findings within this specific subgroup. Thirdly, being a single-center study, the potential population heterogeneity could limit the applicability of our findings in other research contexts. Lastly, validation in other preclinical models, such as patient-derived xenografts, would further strengthen our findings and enhance the understanding of the underlying mechanisms and clinical relevance.</p></sec><sec id="Sec18"><title>Conclusions</title><p id="Par72">In conclusion, this study identified promising predictive biomarkers associated with treatment response and survival in advanced BTCs in the context of chemoimmunotherapy. Future large-scale studies are warranted to validate these findings.</p></sec><sec id="Sec20" sec-type="supplementary-material"><title>Supplementary Information</title><p>
<supplementary-material content-type="local-data" id="MOESM1"><media xlink:href="12916_2025_4263_MOESM1_ESM.docx"><caption><p>Additional file 1: Figures S1-S18 and Tables S1-S4. Figure S1. Workflow of the study. Figure S2. Associations between baseline clinical characteristics and the overall response rate to chemoimmunotherapy in patients with advanced biliary tract cancer. Figure S3. Impact of TMB and PD-L1 RNA expression on PFS and OS. Figure S4. Associations between somatic mutations (mutation frequency&#x02009;&#x0003e;&#x02009;5%) and prognosis of patients. Figure S5. Effects of <italic>ARID1A</italic> and <italic>KRAS</italic> somatic alterations on PFS and OS across three biliary tract cancer subtypes under chemoimmunotherapy. Figure S6. Prognostic impact of <italic>KRAS</italic> G12D mutations on PFS and OS in three biliary tract cancer subtypes. Figure S7. Transcriptomic differences between responders and non-responders to chemoimmunotherapy. Figure S8. Differential expression analysis of immune-related genes in biliary tract cancer patients. Figure S9. Impact of <italic>CXCL9</italic> and <italic>CTLA4</italic> RNA expression levels on PFS and OS in patients receiving chemoimmunotherapy. Figure S10. Correlation between <italic>CXCL9</italic>/<italic>CTLA4</italic> expressions and immune checkpoint expression and immune activity. Figure S11. Internal and external validation of tumor microenvironment differences based on <italic>CXCL9</italic> and <italic>CTLA4</italic> expression. Figure S12. Validation of <italic>CXCL9</italic> knockdown and overexpression in orthotopic model and biliary tract cancer cell lines. Figure S13. CXCL9 regulates T cell effector functions. Figure S14. CXCL9 enhances T cell function in the patient-derived BTC organoid model. Figure S15. STAT1/STAT5 inhibition reduces T cell cytotoxicity. Figure S16. Clinical validation of CTLA4 expression levels in predicting first-line chemoimmunotherapy efficacy in an independent multicenter validation cohort. Figure S17. Subgroup analysis of patients stratified by CXCL9 expression. Figure S18. Subgroup analysis of patients stratified by CTLA4 expression. Table S1. Comparison of clinical features across different therapeutic response groups. Table S2. Impact of baseline clinical characteristics on PFS and OS. Table S3. Univariate analysis of immune-related gene transcription levels. Table S4. Multivariate Cox regression analysis of clinicopathological features. Table S5. Clinical features of the clinical validation cohort.</p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="MOESM2"><media xlink:href="12916_2025_4263_MOESM2_ESM.xlsx"><caption><p>Additional file 2: RNA QC.</p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="MOESM3"><media xlink:href="12916_2025_4263_MOESM3_ESM.xlsx"><caption><p>Additional file 3: Count matrix.</p></caption></media></supplementary-material><supplementary-material content-type="local-data" id="MOESM4"><media xlink:href="12916_2025_4263_MOESM4_ESM.doc"><caption><p>Additional file 4: Original&#x000a0;Western&#x000a0;blot&#x000a0;images&#x000a0;corresponding&#x000a0;to&#x000a0;Supplementary&#x000a0;Figure&#x000a0;S15A&#x000a0;and&#x000a0;S15B.</p></caption></media></supplementary-material></p></sec></body><back><glossary><title>Abbreviations</title><def-list><def-item><term>BTC</term><def><p id="Par10">Biliary tract cancer</p></def></def-item><def-item><term>CA125</term><def><p id="Par11">Carbohydrate antigen 125</p></def></def-item><def-item><term>CEA</term><def><p id="Par12">Carcinoembryonic antigen</p></def></def-item><def-item><term>ChemoIO</term><def><p id="Par13">Chemoimmunotherapy</p></def></def-item><def-item><term>CI</term><def><p id="Par14">Confidence interval</p></def></def-item><def-item><term>CNV</term><def><p id="Par15">Copy-number variation</p></def></def-item><def-item><term>CR</term><def><p id="Par16">Complete response</p></def></def-item><def-item><term>CRP</term><def><p id="Par17">C-reactive protein</p></def></def-item><def-item><term>ECCA</term><def><p id="Par18">Extrahepatic cholangiocarcinoma</p></def></def-item><def-item><term>FDR</term><def><p id="Par19">False discovery rate</p></def></def-item><def-item><term>FFPE</term><def><p id="Par20">Formalin-fixed paraffin-embedded</p></def></def-item><def-item><term>GC</term><def><p id="Par21">Gallbladder cancer</p></def></def-item><def-item><term>GSEA</term><def><p id="Par22">Gene set enrichment analysis</p></def></def-item><def-item><term>HR</term><def><p id="Par23">Hazard ratio</p></def></def-item><def-item><term>ICC</term><def><p id="Par24">Intrahepatic cholangiocarcinoma</p></def></def-item><def-item><term>ICI</term><def><p id="Par25">Immune checkpoint inhibitor</p></def></def-item><def-item><term>IHC</term><def><p id="Par26">Immunohistochemistry</p></def></def-item><def-item><term>INDEL</term><def><p id="Par27">Insertion/deletion</p></def></def-item><def-item><term>MAF</term><def><p id="Par28">Minor allele frequencies</p></def></def-item><def-item><term>MSI</term><def><p id="Par29">Microsatellite instability</p></def></def-item><def-item><term>ORR</term><def><p id="Par30">Overall response rate</p></def></def-item><def-item><term>OS</term><def><p id="Par31">Overall survival</p></def></def-item><def-item><term>PD</term><def><p id="Par32">Progressive disease</p></def></def-item><def-item><term>PFS</term><def><p id="Par33">Progression-free survival</p></def></def-item><def-item><term>PR</term><def><p id="Par34">Partial response</p></def></def-item><def-item><term>SD</term><def><p id="Par35">Stable disease</p></def></def-item><def-item><term>SNV</term><def><p id="Par36">Single-nucleotide variation</p></def></def-item><def-item><term>TIDE</term><def><p id="Par37">Tumor Immune Dysfunction and Exclusion</p></def></def-item><def-item><term>TMB</term><def><p id="Par38">Tumor mutational burden</p></def></def-item><def-item><term>TME</term><def><p id="Par39">Tumor microenvironment</p></def></def-item></def-list></glossary><fn-group><fn><p><bold>Publisher&#x02019;s Note</bold></p><p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></fn><fn><p>Jiaojiao Ni, Chaoqun Li, Xiaoqing Xu and Qi Xu contributed equally to this work.</p></fn></fn-group><ack><title>Acknowledgements
</title><p>We thank all the patients who participated in this study and their families.</p></ack><notes notes-type="author-contribution"><title>Authors&#x02019; contributions
</title><p>JN, WZ2, and JY2 designed the study. XX, QX, QY, JL, QZ, YX, HH, JJ, TX, HZ, FY, JS, YH, and WZ1 collected and extracted data. CL, JN, and SZ performed the in vitro and in vivo experiments. HC, JY1, XW, SW, QO and YS analyzed and interpreted the data. JN, WW, WZ2, and JY2 critically revised the manuscript. All authors read and approved the final manuscript.</p></notes><notes notes-type="funding-information"><title>Funding</title><p>This work was supported by the National Natural Science Foundation of China (82473488), Beijing Xisike Clinical Oncology Research Foundation (Y-HR2020MS-0934), Zhejiang Anti-Cancer Association-Qilu Clinical Scientific Research Foundation (ziskaxhqllckyxm202208), Natural Science Foundation of Zhejiang Province (LTGY23H160010), Medical and Health Science and Technology Program of Zhejiang Province (2021KY087), &#x0201c;Pioneer&#x0201d; and &#x0201c;Leading Goose&#x0201d; R&#x00026;D Program of Zhejiang (2023SDYXSO001-5), and Special Funds for Clinical Research of the National Health Commission (WKZX2023CX030001).</p></notes><notes notes-type="data-availability"><title>Data availability
</title><p>The sequencing data presented in the study have been deposited in the Genome Sequence Archive for Human (GSA-Human) repository (<ext-link ext-link-type="uri" xlink:href="https://ngdc.cncb.ac.cn/gsa-human/">https://ngdc.cncb.ac.cn/gsa-human/</ext-link>) under the accession number HRA005369 for DNA and RNA data of the study cohort, and HRA009951 for RNA data of the internal validation cohort. Additional information is available in the supplementary files or upon reasonable request to the corresponding author.</p></notes><notes><title>Declarations</title><notes id="FPar2"><title>Ethics approval and consent to participate</title><p id="Par73">This study was approved by the Ethics Committee of the Zhejiang Cancer Hospital (Approval No.: IRB-2020&#x02013;404), and in accordance with the Declaration of Helsinki. Informed consent was obtained from all the participating patients. All animal experiments were approved by the Animal Experimental Ethics Committee of Zhejiang Cancer Hospital (2023&#x02013;06-014) and were performed under the guidelines for the care and use of laboratory animals.</p></notes><notes id="FPar3"><title>Consent for publication</title><p id="Par74">Not applicable.</p></notes><notes id="FPar4" notes-type="COI-statement"><title>Competing interests
</title><p id="Par75">Hanlin Chen, Junrong Yan, Xiaoying Wu, Song Wang, Qiuxiang Ou and Yang Shao are employees of Nanjing Geneseeq Technology, Inc. All the other authors declared no conflicts of interest.</p></notes></notes><ref-list id="Bib1"><title>References</title><ref id="CR1"><label>1.</label><citation-alternatives><element-citation id="ec-CR1" publication-type="journal"><person-group person-group-type="author"><name><surname>Ebata</surname><given-names>T</given-names></name><name><surname>Ercolani</surname><given-names>G</given-names></name><name><surname>Alvaro</surname><given-names>D</given-names></name><name><surname>Ribero</surname><given-names>D</given-names></name><name><surname>Di Tommaso</surname><given-names>L</given-names></name><name><surname>Valle</surname><given-names>JW</given-names></name></person-group><article-title>Current Status on Cholangiocarcinoma and Gallbladder Cancer</article-title><source>Liver Cancer</source><year>2016</year><volume>6</volume><issue>1</issue><fpage>59</fpage><lpage>65</lpage><?supplied-pmid 27995089?><pub-id pub-id-type="pmid">27995089</pub-id>
</element-citation><mixed-citation id="mc-CR1" publication-type="journal">Ebata T, Ercolani G, Alvaro D, Ribero D, Di Tommaso L, Valle JW. Current Status on Cholangiocarcinoma and Gallbladder Cancer. Liver Cancer. 2016;6(1):59&#x02013;65.<pub-id pub-id-type="pmid">27995089</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR2"><label>2.</label><citation-alternatives><element-citation id="ec-CR2" publication-type="journal"><person-group person-group-type="author"><name><surname>Valle</surname><given-names>JW</given-names></name><name><surname>Kelley</surname><given-names>RK</given-names></name><name><surname>Nervi</surname><given-names>B</given-names></name><name><surname>Oh</surname><given-names>DY</given-names></name><name><surname>Zhu</surname><given-names>AX</given-names></name></person-group><article-title>Biliary tract cancer</article-title><source>Lancet</source><year>2021</year><volume>397</volume><issue>10272</issue><fpage>428</fpage><lpage>444</lpage><?supplied-pmid 33516341?><pub-id pub-id-type="pmid">33516341</pub-id>
</element-citation><mixed-citation id="mc-CR2" publication-type="journal">Valle JW, Kelley RK, Nervi B, Oh DY, Zhu AX. Biliary tract cancer. Lancet. 2021;397(10272):428&#x02013;44.<pub-id pub-id-type="pmid">33516341</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR3"><label>3.</label><citation-alternatives><element-citation id="ec-CR3" publication-type="journal"><person-group person-group-type="author"><name><surname>Lamarca</surname><given-names>A</given-names></name><name><surname>Barriuso</surname><given-names>J</given-names></name><name><surname>McNamara</surname><given-names>MG</given-names></name><name><surname>Valle</surname><given-names>JW</given-names></name></person-group><article-title>Molecular targeted therapies: Ready for "prime time" in biliary tract cancer</article-title><source>J Hepatol</source><year>2020</year><volume>73</volume><issue>1</issue><fpage>170</fpage><lpage>185</lpage><?supplied-pmid 32171892?><pub-id pub-id-type="pmid">32171892</pub-id>
</element-citation><mixed-citation id="mc-CR3" publication-type="journal">Lamarca A, Barriuso J, McNamara MG, Valle JW. Molecular targeted therapies: Ready for &#x0201c;prime time&#x0201d; in biliary tract cancer. J Hepatol. 2020;73(1):170&#x02013;85.<pub-id pub-id-type="pmid">32171892</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR4"><label>4.</label><citation-alternatives><element-citation id="ec-CR4" publication-type="journal"><person-group person-group-type="author"><name><surname>Valle</surname><given-names>J</given-names></name><name><surname>Wasan</surname><given-names>H</given-names></name><name><surname>Palmer</surname><given-names>DH</given-names></name><name><surname>Cunningham</surname><given-names>D</given-names></name><name><surname>Anthoney</surname><given-names>A</given-names></name><name><surname>Maraveyas</surname><given-names>A</given-names></name><name><surname>Madhusudan</surname><given-names>S</given-names></name><name><surname>Iveson</surname><given-names>T</given-names></name><name><surname>Hughes</surname><given-names>S</given-names></name><name><surname>Pereira</surname><given-names>SP</given-names></name><etal/></person-group><article-title>Cisplatin plus gemcitabine versus gemcitabine for biliary tract cancer</article-title><source>N Engl J Med</source><year>2010</year><volume>362</volume><issue>14</issue><fpage>1273</fpage><lpage>1281</lpage><?supplied-pmid 20375404?><pub-id pub-id-type="pmid">20375404</pub-id>
</element-citation><mixed-citation id="mc-CR4" publication-type="journal">Valle J, Wasan H, Palmer DH, Cunningham D, Anthoney A, Maraveyas A, Madhusudan S, Iveson T, Hughes S, Pereira SP, et al. Cisplatin plus gemcitabine versus gemcitabine for biliary tract cancer. N Engl J Med. 2010;362(14):1273&#x02013;81.<pub-id pub-id-type="pmid">20375404</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR5"><label>5.</label><citation-alternatives><element-citation id="ec-CR5" publication-type="journal"><person-group person-group-type="author"><name><surname>Valle</surname><given-names>JW</given-names></name><name><surname>Wasan</surname><given-names>H</given-names></name><name><surname>Lopes</surname><given-names>A</given-names></name><name><surname>Backen</surname><given-names>AC</given-names></name><name><surname>Palmer</surname><given-names>DH</given-names></name><name><surname>Morris</surname><given-names>K</given-names></name><name><surname>Duggan</surname><given-names>M</given-names></name><name><surname>Cunningham</surname><given-names>D</given-names></name><name><surname>Anthoney</surname><given-names>DA</given-names></name><name><surname>Corrie</surname><given-names>P</given-names></name><etal/></person-group><article-title>Cediranib or placebo in combination with cisplatin and gemcitabine chemotherapy for patients with advanced biliary tract cancer (ABC-03): a randomised phase 2 trial</article-title><source>Lancet Oncol</source><year>2015</year><volume>16</volume><issue>8</issue><fpage>967</fpage><lpage>978</lpage><?supplied-pmid 26179201?><pub-id pub-id-type="pmid">26179201</pub-id>
</element-citation><mixed-citation id="mc-CR5" publication-type="journal">Valle JW, Wasan H, Lopes A, Backen AC, Palmer DH, Morris K, Duggan M, Cunningham D, Anthoney DA, Corrie P, et al. Cediranib or placebo in combination with cisplatin and gemcitabine chemotherapy for patients with advanced biliary tract cancer (ABC-03): a randomised phase 2 trial. Lancet Oncol. 2015;16(8):967&#x02013;78.<pub-id pub-id-type="pmid">26179201</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR6"><label>6.</label><citation-alternatives><element-citation id="ec-CR6" publication-type="journal"><person-group person-group-type="author"><name><surname>Valle</surname><given-names>JW</given-names></name><name><surname>Vogel</surname><given-names>A</given-names></name><name><surname>Denlinger</surname><given-names>CS</given-names></name><name><surname>He</surname><given-names>AR</given-names></name><name><surname>Bai</surname><given-names>LY</given-names></name><name><surname>Orlova</surname><given-names>R</given-names></name><name><surname>Van Cutsem</surname><given-names>E</given-names></name><name><surname>Adeva</surname><given-names>J</given-names></name><name><surname>Chen</surname><given-names>LT</given-names></name><name><surname>Obermannova</surname><given-names>R</given-names></name><etal/></person-group><article-title>Addition of ramucirumab or merestinib to standard first-line chemotherapy for locally advanced or metastatic biliary tract cancer: a randomised, double-blind, multicentre, phase 2 study</article-title><source>Lancet Oncol</source><year>2021</year><volume>22</volume><issue>10</issue><fpage>1468</fpage><lpage>1482</lpage><?supplied-pmid 34592180?><pub-id pub-id-type="pmid">34592180</pub-id>
</element-citation><mixed-citation id="mc-CR6" publication-type="journal">Valle JW, Vogel A, Denlinger CS, He AR, Bai LY, Orlova R, Van Cutsem E, Adeva J, Chen LT, Obermannova R, et al. Addition of ramucirumab or merestinib to standard first-line chemotherapy for locally advanced or metastatic biliary tract cancer: a randomised, double-blind, multicentre, phase 2 study. Lancet Oncol. 2021;22(10):1468&#x02013;82.<pub-id pub-id-type="pmid">34592180</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR7"><label>7.</label><citation-alternatives><element-citation id="ec-CR7" publication-type="journal"><person-group person-group-type="author"><name><surname>Morizane</surname><given-names>C</given-names></name><name><surname>Okusaka</surname><given-names>T</given-names></name><name><surname>Mizusawa</surname><given-names>J</given-names></name><name><surname>Katayama</surname><given-names>H</given-names></name><name><surname>Ueno</surname><given-names>M</given-names></name><name><surname>Ikeda</surname><given-names>M</given-names></name><name><surname>Ozaka</surname><given-names>M</given-names></name><name><surname>Okano</surname><given-names>N</given-names></name><name><surname>Sugimori</surname><given-names>K</given-names></name><name><surname>Fukutomi</surname><given-names>A</given-names></name><etal/></person-group><article-title>Combination gemcitabine plus S-1 versus gemcitabine plus cisplatin for advanced/recurrent biliary tract cancer: the FUGA-BT (JCOG1113) randomized phase III clinical trial</article-title><source>Ann Oncol</source><year>2019</year><volume>30</volume><issue>12</issue><fpage>1950</fpage><lpage>1958</lpage><?supplied-pmid 31566666?><pub-id pub-id-type="pmid">31566666</pub-id>
</element-citation><mixed-citation id="mc-CR7" publication-type="journal">Morizane C, Okusaka T, Mizusawa J, Katayama H, Ueno M, Ikeda M, Ozaka M, Okano N, Sugimori K, Fukutomi A, et al. Combination gemcitabine plus S-1 versus gemcitabine plus cisplatin for advanced/recurrent biliary tract cancer: the FUGA-BT (JCOG1113) randomized phase III clinical trial. Ann Oncol. 2019;30(12):1950&#x02013;8.<pub-id pub-id-type="pmid">31566666</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR8"><label>8.</label><citation-alternatives><element-citation id="ec-CR8" publication-type="journal"><person-group person-group-type="author"><name><surname>Kim</surname><given-names>ST</given-names></name><name><surname>Kang</surname><given-names>JH</given-names></name><name><surname>Lee</surname><given-names>J</given-names></name><name><surname>Lee</surname><given-names>HW</given-names></name><name><surname>Oh</surname><given-names>SY</given-names></name><name><surname>Jang</surname><given-names>JS</given-names></name><name><surname>Lee</surname><given-names>MA</given-names></name><name><surname>Sohn</surname><given-names>BS</given-names></name><name><surname>Yoon</surname><given-names>SY</given-names></name><name><surname>Choi</surname><given-names>HJ</given-names></name><etal/></person-group><article-title>Capecitabine plus oxaliplatin versus gemcitabine plus oxaliplatin as first-line therapy for advanced biliary tract cancers: a multicenter, open-label, randomized, phase III, noninferiority trial</article-title><source>Ann Oncol</source><year>2019</year><volume>30</volume><issue>5</issue><fpage>788</fpage><lpage>795</lpage><?supplied-pmid 30785198?><pub-id pub-id-type="pmid">30785198</pub-id>
</element-citation><mixed-citation id="mc-CR8" publication-type="journal">Kim ST, Kang JH, Lee J, Lee HW, Oh SY, Jang JS, Lee MA, Sohn BS, Yoon SY, Choi HJ, et al. Capecitabine plus oxaliplatin versus gemcitabine plus oxaliplatin as first-line therapy for advanced biliary tract cancers: a multicenter, open-label, randomized, phase III, noninferiority trial. Ann Oncol. 2019;30(5):788&#x02013;95.<pub-id pub-id-type="pmid">30785198</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR9"><label>9.</label><citation-alternatives><element-citation id="ec-CR9" publication-type="journal"><person-group person-group-type="author"><name><surname>Goyal</surname><given-names>L</given-names></name><name><surname>Meric-Bernstam</surname><given-names>F</given-names></name><name><surname>Hollebecque</surname><given-names>A</given-names></name><name><surname>Valle</surname><given-names>JW</given-names></name><name><surname>Morizane</surname><given-names>C</given-names></name><name><surname>Karasic</surname><given-names>TB</given-names></name><name><surname>Abrams</surname><given-names>TA</given-names></name><name><surname>Furuse</surname><given-names>J</given-names></name><name><surname>Kelley</surname><given-names>RK</given-names></name><name><surname>Cassier</surname><given-names>PA</given-names></name><etal/></person-group><article-title>Futibatinib for FGFR2-Rearranged Intrahepatic Cholangiocarcinoma</article-title><source>N Engl J Med</source><year>2023</year><volume>388</volume><issue>3</issue><fpage>228</fpage><lpage>239</lpage><?supplied-pmid 36652354?><pub-id pub-id-type="pmid">36652354</pub-id>
</element-citation><mixed-citation id="mc-CR9" publication-type="journal">Goyal L, Meric-Bernstam F, Hollebecque A, Valle JW, Morizane C, Karasic TB, Abrams TA, Furuse J, Kelley RK, Cassier PA, et al. Futibatinib for FGFR2-Rearranged Intrahepatic Cholangiocarcinoma. N Engl J Med. 2023;388(3):228&#x02013;39.<pub-id pub-id-type="pmid">36652354</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR10"><label>10.</label><citation-alternatives><element-citation id="ec-CR10" publication-type="journal"><person-group person-group-type="author"><name><surname>Zhu</surname><given-names>AX</given-names></name><name><surname>Macarulla</surname><given-names>T</given-names></name><name><surname>Javle</surname><given-names>MM</given-names></name><name><surname>Kelley</surname><given-names>RK</given-names></name><name><surname>Lubner</surname><given-names>SJ</given-names></name><name><surname>Adeva</surname><given-names>J</given-names></name><name><surname>Cleary</surname><given-names>JM</given-names></name><name><surname>Catenacci</surname><given-names>DVT</given-names></name><name><surname>Borad</surname><given-names>MJ</given-names></name><name><surname>Bridgewater</surname><given-names>JA</given-names></name><etal/></person-group><article-title>Final Overall Survival Efficacy Results of Ivosidenib for Patients With Advanced Cholangiocarcinoma With IDH1 Mutation: The Phase 3 Randomized Clinical ClarIDHy Trial</article-title><source>JAMA Oncol</source><year>2021</year><volume>7</volume><issue>11</issue><fpage>1669</fpage><lpage>1677</lpage><?supplied-pmid 34554208?><pub-id pub-id-type="pmid">34554208</pub-id>
</element-citation><mixed-citation id="mc-CR10" publication-type="journal">Zhu AX, Macarulla T, Javle MM, Kelley RK, Lubner SJ, Adeva J, Cleary JM, Catenacci DVT, Borad MJ, Bridgewater JA, et al. Final Overall Survival Efficacy Results of Ivosidenib for Patients With Advanced Cholangiocarcinoma With IDH1 Mutation: The Phase 3 Randomized Clinical ClarIDHy Trial. JAMA Oncol. 2021;7(11):1669&#x02013;77.<pub-id pub-id-type="pmid">34554208</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR11"><label>11.</label><citation-alternatives><element-citation id="ec-CR11" publication-type="journal"><person-group person-group-type="author"><name><surname>Abou-Alfa</surname><given-names>GK</given-names></name><name><surname>Macarulla</surname><given-names>T</given-names></name><name><surname>Javle</surname><given-names>MM</given-names></name><name><surname>Kelley</surname><given-names>RK</given-names></name><name><surname>Lubner</surname><given-names>SJ</given-names></name><name><surname>Adeva</surname><given-names>J</given-names></name><name><surname>Cleary</surname><given-names>JM</given-names></name><name><surname>Catenacci</surname><given-names>DV</given-names></name><name><surname>Borad</surname><given-names>MJ</given-names></name><name><surname>Bridgewater</surname><given-names>J</given-names></name><etal/></person-group><article-title>Ivosidenib in IDH1-mutant, chemotherapy-refractory cholangiocarcinoma (ClarIDHy): a multicentre, randomised, double-blind, placebo-controlled, phase 3 study</article-title><source>Lancet Oncol</source><year>2020</year><volume>21</volume><issue>6</issue><fpage>796</fpage><lpage>807</lpage><?supplied-pmid 32416072?><pub-id pub-id-type="pmid">32416072</pub-id>
</element-citation><mixed-citation id="mc-CR11" publication-type="journal">Abou-Alfa GK, Macarulla T, Javle MM, Kelley RK, Lubner SJ, Adeva J, Cleary JM, Catenacci DV, Borad MJ, Bridgewater J, et al. Ivosidenib in IDH1-mutant, chemotherapy-refractory cholangiocarcinoma (ClarIDHy): a multicentre, randomised, double-blind, placebo-controlled, phase 3 study. Lancet Oncol. 2020;21(6):796&#x02013;807.<pub-id pub-id-type="pmid">32416072</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR12"><label>12.</label><citation-alternatives><element-citation id="ec-CR12" publication-type="journal"><person-group person-group-type="author"><name><surname>Subbiah</surname><given-names>V</given-names></name><name><surname>Lassen</surname><given-names>U</given-names></name><name><surname>Elez</surname><given-names>E</given-names></name><name><surname>Italiano</surname><given-names>A</given-names></name><name><surname>Curigliano</surname><given-names>G</given-names></name><name><surname>Javle</surname><given-names>M</given-names></name><name><surname>de Braud</surname><given-names>F</given-names></name><name><surname>Prager</surname><given-names>GW</given-names></name><name><surname>Greil</surname><given-names>R</given-names></name><name><surname>Stein</surname><given-names>A</given-names></name><etal/></person-group><article-title>Dabrafenib plus trametinib in patients with BRAF(V600E)-mutated biliary tract cancer (ROAR): a phase 2, open-label, single-arm, multicentre basket trial</article-title><source>Lancet Oncol</source><year>2020</year><volume>21</volume><issue>9</issue><fpage>1234</fpage><lpage>1243</lpage><?supplied-pmid 32818466?><pub-id pub-id-type="pmid">32818466</pub-id>
</element-citation><mixed-citation id="mc-CR12" publication-type="journal">Subbiah V, Lassen U, Elez E, Italiano A, Curigliano G, Javle M, de Braud F, Prager GW, Greil R, Stein A, et al. Dabrafenib plus trametinib in patients with BRAF(V600E)-mutated biliary tract cancer (ROAR): a phase 2, open-label, single-arm, multicentre basket trial. Lancet Oncol. 2020;21(9):1234&#x02013;43.<pub-id pub-id-type="pmid">32818466</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR13"><label>13.</label><citation-alternatives><element-citation id="ec-CR13" publication-type="journal"><person-group person-group-type="author"><name><surname>Ricci</surname><given-names>AD</given-names></name><name><surname>Rizzo</surname><given-names>A</given-names></name><name><surname>Brandi</surname><given-names>G</given-names></name></person-group><article-title>The DNA damage repair (DDR) pathway in biliary tract cancer (BTC): a new Pandora's box?</article-title><source>ESMO Open</source><year>2020</year><volume>5</volume><issue>5</issue><fpage>e001042</fpage><?supplied-pmid 32994319?><pub-id pub-id-type="pmid">32994319</pub-id>
</element-citation><mixed-citation id="mc-CR13" publication-type="journal">Ricci AD, Rizzo A, Brandi G. The DNA damage repair (DDR) pathway in biliary tract cancer (BTC): a new Pandora&#x02019;s box? ESMO Open. 2020;5(5): e001042.<pub-id pub-id-type="pmid">32994319</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR14"><label>14.</label><citation-alternatives><element-citation id="ec-CR14" publication-type="journal"><person-group person-group-type="author"><name><surname>Robert</surname><given-names>C</given-names></name><name><surname>Thomas</surname><given-names>L</given-names></name><name><surname>Bondarenko</surname><given-names>I</given-names></name><name><surname>O'Day</surname><given-names>S</given-names></name><name><surname>Weber</surname><given-names>J</given-names></name><name><surname>Garbe</surname><given-names>C</given-names></name><name><surname>Lebbe</surname><given-names>C</given-names></name><name><surname>Baurain</surname><given-names>JF</given-names></name><name><surname>Testori</surname><given-names>A</given-names></name><name><surname>Grob</surname><given-names>JJ</given-names></name><etal/></person-group><article-title>Ipilimumab plus dacarbazine for previously untreated metastatic melanoma</article-title><source>N Engl J Med</source><year>2011</year><volume>364</volume><issue>26</issue><fpage>2517</fpage><lpage>2526</lpage><?supplied-pmid 21639810?><pub-id pub-id-type="pmid">21639810</pub-id>
</element-citation><mixed-citation id="mc-CR14" publication-type="journal">Robert C, Thomas L, Bondarenko I, O&#x02019;Day S, Weber J, Garbe C, Lebbe C, Baurain JF, Testori A, Grob JJ, et al. Ipilimumab plus dacarbazine for previously untreated metastatic melanoma. N Engl J Med. 2011;364(26):2517&#x02013;26.<pub-id pub-id-type="pmid">21639810</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR15"><label>15.</label><citation-alternatives><element-citation id="ec-CR15" publication-type="journal"><person-group person-group-type="author"><name><surname>Emens</surname><given-names>LA</given-names></name><name><surname>Middleton</surname><given-names>G</given-names></name></person-group><article-title>The interplay of immunotherapy and chemotherapy: harnessing potential synergies</article-title><source>Cancer Immunol Res</source><year>2015</year><volume>3</volume><issue>5</issue><fpage>436</fpage><lpage>443</lpage><?supplied-pmid 25941355?><pub-id pub-id-type="pmid">25941355</pub-id>
</element-citation><mixed-citation id="mc-CR15" publication-type="journal">Emens LA, Middleton G. The interplay of immunotherapy and chemotherapy: harnessing potential synergies. Cancer Immunol Res. 2015;3(5):436&#x02013;43.<pub-id pub-id-type="pmid">25941355</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR16"><label>16.</label><citation-alternatives><element-citation id="ec-CR16" publication-type="journal"><person-group person-group-type="author"><name><surname>Fournel</surname><given-names>L</given-names></name><name><surname>Wu</surname><given-names>Z</given-names></name><name><surname>Stadler</surname><given-names>N</given-names></name><name><surname>Damotte</surname><given-names>D</given-names></name><name><surname>Lococo</surname><given-names>F</given-names></name><name><surname>Boulle</surname><given-names>G</given-names></name><name><surname>Segal-Bendirdjian</surname><given-names>E</given-names></name><name><surname>Bobbio</surname><given-names>A</given-names></name><name><surname>Icard</surname><given-names>P</given-names></name><name><surname>Tredaniel</surname><given-names>J</given-names></name><etal/></person-group><article-title>Cisplatin increases PD-L1 expression and optimizes immune check-point blockade in non-small cell lung cancer</article-title><source>Cancer Lett</source><year>2019</year><volume>464</volume><fpage>5</fpage><lpage>14</lpage><?supplied-pmid 31404614?><pub-id pub-id-type="pmid">31404614</pub-id>
</element-citation><mixed-citation id="mc-CR16" publication-type="journal">Fournel L, Wu Z, Stadler N, Damotte D, Lococo F, Boulle G, Segal-Bendirdjian E, Bobbio A, Icard P, Tredaniel J, et al. Cisplatin increases PD-L1 expression and optimizes immune check-point blockade in non-small cell lung cancer. Cancer Lett. 2019;464:5&#x02013;14.<pub-id pub-id-type="pmid">31404614</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR17"><label>17.</label><citation-alternatives><element-citation id="ec-CR17" publication-type="journal"><person-group person-group-type="author"><name><surname>Luo</surname><given-names>H</given-names></name><name><surname>Lu</surname><given-names>J</given-names></name><name><surname>Bai</surname><given-names>Y</given-names></name><name><surname>Mao</surname><given-names>T</given-names></name><name><surname>Wang</surname><given-names>J</given-names></name><name><surname>Fan</surname><given-names>Q</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Zhao</surname><given-names>K</given-names></name><name><surname>Chen</surname><given-names>Z</given-names></name><name><surname>Gao</surname><given-names>S</given-names></name><etal/></person-group><article-title>Effect of Camrelizumab vs Placebo Added to Chemotherapy on Survival and Progression-Free Survival in Patients With Advanced or Metastatic Esophageal Squamous Cell Carcinoma: The ESCORT-1st Randomized Clinical Trial</article-title><source>JAMA</source><year>2021</year><volume>326</volume><issue>10</issue><fpage>916</fpage><lpage>925</lpage><?supplied-pmid 34519801?><pub-id pub-id-type="pmid">34519801</pub-id>
</element-citation><mixed-citation id="mc-CR17" publication-type="journal">Luo H, Lu J, Bai Y, Mao T, Wang J, Fan Q, Zhang Y, Zhao K, Chen Z, Gao S, et al. Effect of Camrelizumab vs Placebo Added to Chemotherapy on Survival and Progression-Free Survival in Patients With Advanced or Metastatic Esophageal Squamous Cell Carcinoma: The ESCORT-1st Randomized Clinical Trial. JAMA. 2021;326(10):916&#x02013;25.<pub-id pub-id-type="pmid">34519801</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR18"><label>18.</label><citation-alternatives><element-citation id="ec-CR18" publication-type="journal"><person-group person-group-type="author"><name><surname>Forde</surname><given-names>PM</given-names></name><name><surname>Spicer</surname><given-names>J</given-names></name><name><surname>Lu</surname><given-names>S</given-names></name><name><surname>Provencio</surname><given-names>M</given-names></name><name><surname>Mitsudomi</surname><given-names>T</given-names></name><name><surname>Awad</surname><given-names>MM</given-names></name><name><surname>Felip</surname><given-names>E</given-names></name><name><surname>Broderick</surname><given-names>SR</given-names></name><name><surname>Brahmer</surname><given-names>JR</given-names></name><name><surname>Swanson</surname><given-names>SJ</given-names></name><etal/></person-group><article-title>Neoadjuvant Nivolumab plus Chemotherapy in Resectable Lung Cancer</article-title><source>N Engl J Med</source><year>2022</year><volume>386</volume><issue>21</issue><fpage>1973</fpage><lpage>1985</lpage><?supplied-pmid 35403841?><pub-id pub-id-type="pmid">35403841</pub-id>
</element-citation><mixed-citation id="mc-CR18" publication-type="journal">Forde PM, Spicer J, Lu S, Provencio M, Mitsudomi T, Awad MM, Felip E, Broderick SR, Brahmer JR, Swanson SJ, et al. Neoadjuvant Nivolumab plus Chemotherapy in Resectable Lung Cancer. N Engl J Med. 2022;386(21):1973&#x02013;85.<pub-id pub-id-type="pmid">35403841</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR19"><label>19.</label><citation-alternatives><element-citation id="ec-CR19" publication-type="journal"><person-group person-group-type="author"><name><surname>Oh</surname><given-names>D-Y</given-names></name><name><surname>He</surname><given-names>AR</given-names></name><name><surname>Qin</surname><given-names>S</given-names></name><name><surname>Chen</surname><given-names>L-T</given-names></name><name><surname>Okusaka</surname><given-names>T</given-names></name><name><surname>Vogel</surname><given-names>A</given-names></name><name><surname>Kim</surname><given-names>JW</given-names></name><name><surname>Suksombooncharoen</surname><given-names>T</given-names></name><name><surname>Lee</surname><given-names>MA</given-names></name><name><surname>Kitano</surname><given-names>M</given-names></name><etal/></person-group><article-title>Durvalumab plus Gemcitabine and Cisplatin in Advanced Biliary Tract Cancer</article-title><source>NEJM Evid.</source><year>2022</year><volume>1</volume><issue>8</issue><fpage>EVIDoa2200015</fpage><?supplied-pmid 38319896?><pub-id pub-id-type="pmid">38319896</pub-id>
</element-citation><mixed-citation id="mc-CR19" publication-type="journal">Oh D-Y, He AR, Qin S, Chen L-T, Okusaka T, Vogel A, Kim JW, Suksombooncharoen T, Lee MA, Kitano M, et al. Durvalumab plus Gemcitabine and Cisplatin in Advanced Biliary Tract Cancer. NEJM Evid. 2022;1(8):EVIDoa2200015.<pub-id pub-id-type="pmid">38319896</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR20"><label>20.</label><citation-alternatives><element-citation id="ec-CR20" publication-type="journal"><person-group person-group-type="author"><name><surname>Kelley</surname><given-names>RK</given-names></name><name><surname>Ueno</surname><given-names>M</given-names></name><name><surname>Yoo</surname><given-names>C</given-names></name><name><surname>Finn</surname><given-names>RS</given-names></name><name><surname>Furuse</surname><given-names>J</given-names></name><name><surname>Ren</surname><given-names>Z</given-names></name><name><surname>Yau</surname><given-names>T</given-names></name><name><surname>Klumpen</surname><given-names>HJ</given-names></name><name><surname>Chan</surname><given-names>SL</given-names></name><name><surname>Ozaka</surname><given-names>M</given-names></name><etal/></person-group><article-title>Pembrolizumab in combination with gemcitabine and cisplatin compared with gemcitabine and cisplatin alone for patients with advanced biliary tract cancer (KEYNOTE-966): a randomised, double-blind, placebo-controlled, phase 3 trial</article-title><source>Lancet</source><year>2023</year><volume>401</volume><issue>10391</issue><fpage>1853</fpage><lpage>1865</lpage><?supplied-pmid 37075781?><pub-id pub-id-type="pmid">37075781</pub-id>
</element-citation><mixed-citation id="mc-CR20" publication-type="journal">Kelley RK, Ueno M, Yoo C, Finn RS, Furuse J, Ren Z, Yau T, Klumpen HJ, Chan SL, Ozaka M, et al. Pembrolizumab in combination with gemcitabine and cisplatin compared with gemcitabine and cisplatin alone for patients with advanced biliary tract cancer (KEYNOTE-966): a randomised, double-blind, placebo-controlled, phase 3 trial. Lancet. 2023;401(10391):1853&#x02013;65.<pub-id pub-id-type="pmid">37075781</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR21"><label>21.</label><mixed-citation publication-type="other">Chen X, Wang D, Liu J, Qiu J, Zhou J, Ying J, Shi Y, Wang Z, Lou H, Cui J, et al. Genomic alterations in biliary tract cancer predict prognosis and immunotherapy outcomes. J Immunother Cancer. 2021;9(11):e003214.</mixed-citation></ref><ref id="CR22"><label>22.</label><citation-alternatives><element-citation id="ec-CR22" publication-type="journal"><person-group person-group-type="author"><name><surname>Zeng</surname><given-names>TM</given-names></name><name><surname>Yang</surname><given-names>G</given-names></name><name><surname>Lou</surname><given-names>C</given-names></name><name><surname>Wei</surname><given-names>W</given-names></name><name><surname>Tao</surname><given-names>CJ</given-names></name><name><surname>Chen</surname><given-names>XY</given-names></name><name><surname>Han</surname><given-names>Q</given-names></name><name><surname>Cheng</surname><given-names>Z</given-names></name><name><surname>Shang</surname><given-names>PP</given-names></name><name><surname>Dong</surname><given-names>YL</given-names></name><etal/></person-group><article-title>Clinical and biomarker analyses of sintilimab plus gemcitabine and cisplatin as first-line treatment for patients with advanced biliary tract cancer</article-title><source>Nat Commun</source><year>2023</year><volume>14</volume><issue>1</issue><fpage>1340</fpage><?supplied-pmid 36906670?><pub-id pub-id-type="pmid">36906670</pub-id>
</element-citation><mixed-citation id="mc-CR22" publication-type="journal">Zeng TM, Yang G, Lou C, Wei W, Tao CJ, Chen XY, Han Q, Cheng Z, Shang PP, Dong YL, et al. Clinical and biomarker analyses of sintilimab plus gemcitabine and cisplatin as first-line treatment for patients with advanced biliary tract cancer. Nat Commun. 2023;14(1):1340.<pub-id pub-id-type="pmid">36906670</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR23"><label>23.</label><citation-alternatives><element-citation id="ec-CR23" publication-type="journal"><person-group person-group-type="author"><name><surname>Ricci</surname><given-names>AD</given-names></name><name><surname>Rizzo</surname><given-names>A</given-names></name><name><surname>Brandi</surname><given-names>G</given-names></name></person-group><article-title>Immunotherapy in Biliary Tract Cancer: Worthy of a Second Look</article-title><source>Cancer Control</source><year>2020</year><volume>27</volume><issue>3</issue><fpage>1073274820948047</fpage><?supplied-pmid 32806956?><pub-id pub-id-type="pmid">32806956</pub-id>
</element-citation><mixed-citation id="mc-CR23" publication-type="journal">Ricci AD, Rizzo A, Brandi G. Immunotherapy in Biliary Tract Cancer: Worthy of a Second Look. Cancer Control. 2020;27(3):1073274820948047.<pub-id pub-id-type="pmid">32806956</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR24"><label>24.</label><citation-alternatives><element-citation id="ec-CR24" publication-type="journal"><person-group person-group-type="author"><name><surname>Li</surname><given-names>J</given-names></name><name><surname>Xue</surname><given-names>X</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Ding</surname><given-names>F</given-names></name><name><surname>Wu</surname><given-names>W</given-names></name><name><surname>Liu</surname><given-names>C</given-names></name><name><surname>Xu</surname><given-names>Y</given-names></name><name><surname>Chen</surname><given-names>H</given-names></name><name><surname>Ou</surname><given-names>Q</given-names></name><name><surname>Shao</surname><given-names>Y</given-names></name><etal/></person-group><article-title>The differences in immune features and genomic profiling between squamous cell carcinoma and adenocarcinoma - A multi-center study in Chinese patients with uterine cervical cancer</article-title><source>Gynecol Oncol</source><year>2023</year><volume>175</volume><fpage>133</fpage><lpage>141</lpage><?supplied-pmid 37356314?><pub-id pub-id-type="pmid">37356314</pub-id>
</element-citation><mixed-citation id="mc-CR24" publication-type="journal">Li J, Xue X, Zhang Y, Ding F, Wu W, Liu C, Xu Y, Chen H, Ou Q, Shao Y, et al. The differences in immune features and genomic profiling between squamous cell carcinoma and adenocarcinoma - A multi-center study in Chinese patients with uterine cervical cancer. Gynecol Oncol. 2023;175:133&#x02013;41.<pub-id pub-id-type="pmid">37356314</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR25"><label>25.</label><citation-alternatives><element-citation id="ec-CR25" publication-type="journal"><person-group person-group-type="author"><name><surname>Bolger</surname><given-names>AM</given-names></name><name><surname>Lohse</surname><given-names>M</given-names></name><name><surname>Usadel</surname><given-names>B</given-names></name></person-group><article-title>Trimmomatic: a flexible trimmer for Illumina sequence data</article-title><source>Bioinformatics</source><year>2014</year><volume>30</volume><issue>15</issue><fpage>2114</fpage><lpage>2120</lpage><?supplied-pmid 24695404?><pub-id pub-id-type="pmid">24695404</pub-id>
</element-citation><mixed-citation id="mc-CR25" publication-type="journal">Bolger AM, Lohse M, Usadel B. Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics. 2014;30(15):2114&#x02013;20.<pub-id pub-id-type="pmid">24695404</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR26"><label>26.</label><citation-alternatives><element-citation id="ec-CR26" publication-type="journal"><person-group person-group-type="author"><name><surname>Fang</surname><given-names>W</given-names></name><name><surname>Ma</surname><given-names>Y</given-names></name><name><surname>Yin</surname><given-names>JC</given-names></name><name><surname>Hong</surname><given-names>S</given-names></name><name><surname>Zhou</surname><given-names>H</given-names></name><name><surname>Wang</surname><given-names>A</given-names></name><name><surname>Wang</surname><given-names>F</given-names></name><name><surname>Bao</surname><given-names>H</given-names></name><name><surname>Wu</surname><given-names>X</given-names></name><name><surname>Yang</surname><given-names>Y</given-names></name><etal/></person-group><article-title>Comprehensive Genomic Profiling Identifies Novel Genetic Predictors of Response to Anti-PD-(L)1 Therapies in Non-Small Cell Lung Cancer</article-title><source>Clin Cancer Res</source><year>2019</year><volume>25</volume><issue>16</issue><fpage>5015</fpage><lpage>5026</lpage><?supplied-pmid 31085721?><pub-id pub-id-type="pmid">31085721</pub-id>
</element-citation><mixed-citation id="mc-CR26" publication-type="journal">Fang W, Ma Y, Yin JC, Hong S, Zhou H, Wang A, Wang F, Bao H, Wu X, Yang Y, et al. Comprehensive Genomic Profiling Identifies Novel Genetic Predictors of Response to Anti-PD-(L)1 Therapies in Non-Small Cell Lung Cancer. Clin Cancer Res. 2019;25(16):5015&#x02013;26.<pub-id pub-id-type="pmid">31085721</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR27"><label>27.</label><mixed-citation publication-type="other">Ying J, Yang L, Yin JC, Xia G, Xing M, Chen X, Pang J, Wu Y, Bao H, Wu X, et al. Additive effects of variants of unknown significance in replication repair-associated DNA polymerase genes on mutational burden and prognosis across diverse cancers. J Immunother Cancer. 2021;9(9):e002336.</mixed-citation></ref><ref id="CR28"><label>28.</label><citation-alternatives><element-citation id="ec-CR28" publication-type="journal"><person-group person-group-type="author"><name><surname>Liu</surname><given-names>J</given-names></name><name><surname>Ji</surname><given-names>Y</given-names></name><name><surname>Weng</surname><given-names>X</given-names></name><name><surname>Shao</surname><given-names>W</given-names></name><name><surname>Zhao</surname><given-names>J</given-names></name><name><surname>Chen</surname><given-names>H</given-names></name><name><surname>Shen</surname><given-names>L</given-names></name><name><surname>Wang</surname><given-names>F</given-names></name><name><surname>Meng</surname><given-names>Q</given-names></name><name><surname>Wu</surname><given-names>X</given-names></name><etal/></person-group><article-title>Immune microenvironment analysis and novel biomarkers of early-stage lung adenocarcinoma evolution</article-title><source>Front Oncol</source><year>2023</year><volume>13</volume><fpage>1150098</fpage><?supplied-pmid 37427097?><pub-id pub-id-type="pmid">37427097</pub-id>
</element-citation><mixed-citation id="mc-CR28" publication-type="journal">Liu J, Ji Y, Weng X, Shao W, Zhao J, Chen H, Shen L, Wang F, Meng Q, Wu X, et al. Immune microenvironment analysis and novel biomarkers of early-stage lung adenocarcinoma evolution. Front Oncol. 2023;13:1150098.<pub-id pub-id-type="pmid">37427097</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR29"><label>29.</label><citation-alternatives><element-citation id="ec-CR29" publication-type="journal"><person-group person-group-type="author"><name><surname>Aran</surname><given-names>D</given-names></name><name><surname>Hu</surname><given-names>Z</given-names></name><name><surname>Butte</surname><given-names>AJ</given-names></name></person-group><article-title>xCell: digitally portraying the tissue cellular heterogeneity landscape</article-title><source>Genome Biol</source><year>2017</year><volume>18</volume><issue>1</issue><fpage>220</fpage><?supplied-pmid 29141660?><pub-id pub-id-type="pmid">29141660</pub-id>
</element-citation><mixed-citation id="mc-CR29" publication-type="journal">Aran D, Hu Z, Butte AJ. xCell: digitally portraying the tissue cellular heterogeneity landscape. Genome Biol. 2017;18(1):220.<pub-id pub-id-type="pmid">29141660</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR30"><label>30.</label><citation-alternatives><element-citation id="ec-CR30" publication-type="journal"><person-group person-group-type="author"><name><surname>Zeng</surname><given-names>D</given-names></name><name><surname>Li</surname><given-names>M</given-names></name><name><surname>Zhou</surname><given-names>R</given-names></name><name><surname>Zhang</surname><given-names>J</given-names></name><name><surname>Sun</surname><given-names>H</given-names></name><name><surname>Shi</surname><given-names>M</given-names></name><name><surname>Bin</surname><given-names>J</given-names></name><name><surname>Liao</surname><given-names>Y</given-names></name><name><surname>Rao</surname><given-names>J</given-names></name><name><surname>Liao</surname><given-names>W</given-names></name></person-group><article-title>Tumor Microenvironment Characterization in Gastric Cancer Identifies Prognostic and Immunotherapeutically Relevant Gene Signatures</article-title><source>Cancer Immunol Res</source><year>2019</year><volume>7</volume><issue>5</issue><fpage>737</fpage><lpage>750</lpage><?supplied-pmid 30842092?><pub-id pub-id-type="pmid">30842092</pub-id>
</element-citation><mixed-citation id="mc-CR30" publication-type="journal">Zeng D, Li M, Zhou R, Zhang J, Sun H, Shi M, Bin J, Liao Y, Rao J, Liao W. Tumor Microenvironment Characterization in Gastric Cancer Identifies Prognostic and Immunotherapeutically Relevant Gene Signatures. Cancer Immunol Res. 2019;7(5):737&#x02013;50.<pub-id pub-id-type="pmid">30842092</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR31"><label>31.</label><mixed-citation publication-type="other">Dong L. Proteogenomic Characterization Identifies Clinically-relevant Subgroups of Intrahepatic Cholangiocarcinoma. National Omics Data Encyclopedia (NODE) 2021. <ext-link ext-link-type="uri" xlink:href="https://www.biosino.org/node/analysis/detail/OEZ00008243">https://www.biosino.org/node/analysis/detail/OEZ00008243</ext-link>.</mixed-citation></ref><ref id="CR32"><label>32.</label><citation-alternatives><element-citation id="ec-CR32" publication-type="journal"><person-group person-group-type="author"><name><surname>Dong</surname><given-names>L</given-names></name><name><surname>Lu</surname><given-names>D</given-names></name><name><surname>Chen</surname><given-names>R</given-names></name><name><surname>Lin</surname><given-names>Y</given-names></name><name><surname>Zhu</surname><given-names>H</given-names></name><name><surname>Zhang</surname><given-names>Z</given-names></name><name><surname>Cai</surname><given-names>S</given-names></name><name><surname>Cui</surname><given-names>P</given-names></name><name><surname>Song</surname><given-names>G</given-names></name><name><surname>Rao</surname><given-names>D</given-names></name><etal/></person-group><article-title>Proteogenomic characterization identifies clinically relevant subgroups of intrahepatic cholangiocarcinoma</article-title><source>Cancer Cell.</source><year>2022</year><volume>40</volume><issue>1</issue><fpage>70</fpage><lpage>87</lpage><?supplied-pmid 34971568?><pub-id pub-id-type="pmid">34971568</pub-id>
</element-citation><mixed-citation id="mc-CR32" publication-type="journal">Dong L, Lu D, Chen R, Lin Y, Zhu H, Zhang Z, Cai S, Cui P, Song G, Rao D, et al. Proteogenomic characterization identifies clinically relevant subgroups of intrahepatic cholangiocarcinoma. Cancer Cell. 2022;40(1):70&#x02013;87 e15.<pub-id pub-id-type="pmid">34971568</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR33"><label>33.</label><citation-alternatives><element-citation id="ec-CR33" publication-type="journal"><person-group person-group-type="author"><name><surname>Zhang</surname><given-names>S</given-names></name><name><surname>Song</surname><given-names>X</given-names></name><name><surname>Cao</surname><given-names>D</given-names></name><name><surname>Xu</surname><given-names>Z</given-names></name><name><surname>Fan</surname><given-names>B</given-names></name><name><surname>Che</surname><given-names>L</given-names></name><name><surname>Hu</surname><given-names>J</given-names></name><name><surname>Chen</surname><given-names>B</given-names></name><name><surname>Dong</surname><given-names>M</given-names></name><name><surname>Pilo</surname><given-names>MG</given-names></name><etal/></person-group><article-title>Pan-mTOR inhibitor MLN0128 is effective against intrahepatic cholangiocarcinoma in mice</article-title><source>J Hepatol</source><year>2017</year><volume>67</volume><issue>6</issue><fpage>1194</fpage><lpage>1203</lpage><?supplied-pmid 28733220?><pub-id pub-id-type="pmid">28733220</pub-id>
</element-citation><mixed-citation id="mc-CR33" publication-type="journal">Zhang S, Song X, Cao D, Xu Z, Fan B, Che L, Hu J, Chen B, Dong M, Pilo MG, et al. Pan-mTOR inhibitor MLN0128 is effective against intrahepatic cholangiocarcinoma in mice. J Hepatol. 2017;67(6):1194&#x02013;203.<pub-id pub-id-type="pmid">28733220</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR34"><label>34.</label><citation-alternatives><element-citation id="ec-CR34" publication-type="journal"><person-group person-group-type="author"><name><surname>Sharma</surname><given-names>P</given-names></name><name><surname>Hu-Lieskovan</surname><given-names>S</given-names></name><name><surname>Wargo</surname><given-names>JA</given-names></name><name><surname>Ribas</surname><given-names>A</given-names></name></person-group><article-title>Primary, Adaptive, and Acquired Resistance to Cancer Immunotherapy</article-title><source>Cell</source><year>2017</year><volume>168</volume><issue>4</issue><fpage>707</fpage><lpage>723</lpage><?supplied-pmid 28187290?><pub-id pub-id-type="pmid">28187290</pub-id>
</element-citation><mixed-citation id="mc-CR34" publication-type="journal">Sharma P, Hu-Lieskovan S, Wargo JA, Ribas A. Primary, Adaptive, and Acquired Resistance to Cancer Immunotherapy. Cell. 2017;168(4):707&#x02013;23.<pub-id pub-id-type="pmid">28187290</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR35"><label>35.</label><citation-alternatives><element-citation id="ec-CR35" publication-type="journal"><person-group person-group-type="author"><name><surname>Chan</surname><given-names>TA</given-names></name><name><surname>Yarchoan</surname><given-names>M</given-names></name><name><surname>Jaffee</surname><given-names>E</given-names></name><name><surname>Swanton</surname><given-names>C</given-names></name><name><surname>Quezada</surname><given-names>SA</given-names></name><name><surname>Stenzinger</surname><given-names>A</given-names></name><name><surname>Peters</surname><given-names>S</given-names></name></person-group><article-title>Development of tumor mutation burden as an immunotherapy biomarker: utility for the oncology clinic</article-title><source>Ann Oncol</source><year>2019</year><volume>30</volume><issue>1</issue><fpage>44</fpage><lpage>56</lpage><?supplied-pmid 30395155?><pub-id pub-id-type="pmid">30395155</pub-id>
</element-citation><mixed-citation id="mc-CR35" publication-type="journal">Chan TA, Yarchoan M, Jaffee E, Swanton C, Quezada SA, Stenzinger A, Peters S. Development of tumor mutation burden as an immunotherapy biomarker: utility for the oncology clinic. Ann Oncol. 2019;30(1):44&#x02013;56.<pub-id pub-id-type="pmid">30395155</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR36"><label>36.</label><citation-alternatives><element-citation id="ec-CR36" publication-type="journal"><person-group person-group-type="author"><name><surname>Doroshow</surname><given-names>DB</given-names></name><name><surname>Bhalla</surname><given-names>S</given-names></name><name><surname>Beasley</surname><given-names>MB</given-names></name><name><surname>Sholl</surname><given-names>LM</given-names></name><name><surname>Kerr</surname><given-names>KM</given-names></name><name><surname>Gnjatic</surname><given-names>S</given-names></name><name><surname>Wistuba</surname><given-names>II</given-names></name><name><surname>Rimm</surname><given-names>DL</given-names></name><name><surname>Tsao</surname><given-names>MS</given-names></name><name><surname>Hirsch</surname><given-names>FR</given-names></name></person-group><article-title>PD-L1 as a biomarker of response to immune-checkpoint inhibitors</article-title><source>Nat Rev Clin Oncol</source><year>2021</year><volume>18</volume><issue>6</issue><fpage>345</fpage><lpage>362</lpage><?supplied-pmid 33580222?><pub-id pub-id-type="pmid">33580222</pub-id>
</element-citation><mixed-citation id="mc-CR36" publication-type="journal">Doroshow DB, Bhalla S, Beasley MB, Sholl LM, Kerr KM, Gnjatic S, Wistuba II, Rimm DL, Tsao MS, Hirsch FR. PD-L1 as a biomarker of response to immune-checkpoint inhibitors. Nat Rev Clin Oncol. 2021;18(6):345&#x02013;62.<pub-id pub-id-type="pmid">33580222</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR37"><label>37.</label><citation-alternatives><element-citation id="ec-CR37" publication-type="journal"><person-group person-group-type="author"><name><surname>Zhao</surname><given-names>P</given-names></name><name><surname>Li</surname><given-names>L</given-names></name><name><surname>Jiang</surname><given-names>X</given-names></name><name><surname>Li</surname><given-names>Q</given-names></name></person-group><article-title>Mismatch repair deficiency/microsatellite instability-high as a predictor for anti-PD-1/PD-L1 immunotherapy efficacy</article-title><source>J Hematol Oncol</source><year>2019</year><volume>12</volume><issue>1</issue><fpage>54</fpage><?supplied-pmid 31151482?><pub-id pub-id-type="pmid">31151482</pub-id>
</element-citation><mixed-citation id="mc-CR37" publication-type="journal">Zhao P, Li L, Jiang X, Li Q. Mismatch repair deficiency/microsatellite instability-high as a predictor for anti-PD-1/PD-L1 immunotherapy efficacy. J Hematol Oncol. 2019;12(1):54.<pub-id pub-id-type="pmid">31151482</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR38"><label>38.</label><citation-alternatives><element-citation id="ec-CR38" publication-type="journal"><person-group person-group-type="author"><name><surname>Duffy</surname><given-names>MJ</given-names></name><name><surname>Crown</surname><given-names>J</given-names></name></person-group><article-title>Biomarkers for Predicting Response to Immunotherapy with Immune Checkpoint Inhibitors in Cancer Patients</article-title><source>Clin Chem</source><year>2019</year><volume>65</volume><issue>10</issue><fpage>1228</fpage><lpage>1238</lpage><?supplied-pmid 31315901?><pub-id pub-id-type="pmid">31315901</pub-id>
</element-citation><mixed-citation id="mc-CR38" publication-type="journal">Duffy MJ, Crown J. Biomarkers for Predicting Response to Immunotherapy with Immune Checkpoint Inhibitors in Cancer Patients. Clin Chem. 2019;65(10):1228&#x02013;38.<pub-id pub-id-type="pmid">31315901</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR39"><label>39.</label><mixed-citation publication-type="other">Frega G, Cossio FP, Banales JM, Cardinale V, Macias RIR, Braconi C, Lamarca A. Lacking Immunotherapy Biomarkers for Biliary Tract Cancer: A Comprehensive Systematic Literature Review and Meta-Analysis. Cells. 2023;12(16):2098.</mixed-citation></ref><ref id="CR40"><label>40.</label><mixed-citation publication-type="other">Rizzo A, Ricci AD, Brandi G. PD-L1, TMB, MSI, and Other Predictors of Response to Immune Checkpoint Inhibitors in Biliary Tract Cancer. Cancers (Basel). 2021;13(3):558.</mixed-citation></ref><ref id="CR41"><label>41.</label><citation-alternatives><element-citation id="ec-CR41" publication-type="journal"><person-group person-group-type="author"><name><surname>Gu</surname><given-names>M</given-names></name><name><surname>Xu</surname><given-names>T</given-names></name><name><surname>Chang</surname><given-names>P</given-names></name></person-group><article-title>KRAS/LKB1 and KRAS/TP53 co-mutations create divergent immune signatures in lung adenocarcinomas</article-title><source>Ther Adv Med Oncol</source><year>2021</year><volume>13</volume><fpage>17588359211006950</fpage><?supplied-pmid 33995590?><pub-id pub-id-type="pmid">33995590</pub-id>
</element-citation><mixed-citation id="mc-CR41" publication-type="journal">Gu M, Xu T, Chang P. KRAS/LKB1 and KRAS/TP53 co-mutations create divergent immune signatures in lung adenocarcinomas. Ther Adv Med Oncol. 2021;13:17588359211006950.<pub-id pub-id-type="pmid">33995590</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR42"><label>42.</label><citation-alternatives><element-citation id="ec-CR42" publication-type="journal"><person-group person-group-type="author"><name><surname>Zhou</surname><given-names>Z</given-names></name><name><surname>Li</surname><given-names>M</given-names></name></person-group><article-title>Evaluation of BRCA1 and BRCA2 as Indicators of Response to Immune Checkpoint Inhibitors</article-title><source>JAMA Netw Open</source><year>2021</year><volume>4</volume><issue>5</issue><fpage>e217728</fpage><?supplied-pmid 33961040?><pub-id pub-id-type="pmid">33961040</pub-id>
</element-citation><mixed-citation id="mc-CR42" publication-type="journal">Zhou Z, Li M. Evaluation of BRCA1 and BRCA2 as Indicators of Response to Immune Checkpoint Inhibitors. JAMA Netw Open. 2021;4(5): e217728.<pub-id pub-id-type="pmid">33961040</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR43"><label>43.</label><citation-alternatives><element-citation id="ec-CR43" publication-type="journal"><person-group person-group-type="author"><name><surname>Samstein</surname><given-names>RM</given-names></name><name><surname>Krishna</surname><given-names>C</given-names></name><name><surname>Ma</surname><given-names>X</given-names></name><name><surname>Pei</surname><given-names>X</given-names></name><name><surname>Lee</surname><given-names>KW</given-names></name><name><surname>Makarov</surname><given-names>V</given-names></name><name><surname>Kuo</surname><given-names>F</given-names></name><name><surname>Chung</surname><given-names>J</given-names></name><name><surname>Srivastava</surname><given-names>RM</given-names></name><name><surname>Purohit</surname><given-names>TA</given-names></name><etal/></person-group><article-title>Mutations in BRCA1 and BRCA2 differentially affect the tumor microenvironment and response to checkpoint blockade immunotherapy</article-title><source>Nat Cancer</source><year>2021</year><volume>1</volume><issue>12</issue><fpage>1188</fpage><lpage>1203</lpage><?supplied-pmid 33834176?><pub-id pub-id-type="pmid">33834176</pub-id>
</element-citation><mixed-citation id="mc-CR43" publication-type="journal">Samstein RM, Krishna C, Ma X, Pei X, Lee KW, Makarov V, Kuo F, Chung J, Srivastava RM, Purohit TA, et al. Mutations in BRCA1 and BRCA2 differentially affect the tumor microenvironment and response to checkpoint blockade immunotherapy. Nat Cancer. 2021;1(12):1188&#x02013;203.<pub-id pub-id-type="pmid">33834176</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR44"><label>44.</label><citation-alternatives><element-citation id="ec-CR44" publication-type="journal"><person-group person-group-type="author"><name><surname>Lee</surname><given-names>S</given-names></name><name><surname>Margolin</surname><given-names>K</given-names></name></person-group><article-title>Cytokines in cancer immunotherapy</article-title><source>Cancers (Basel)</source><year>2011</year><volume>3</volume><issue>4</issue><fpage>3856</fpage><lpage>3893</lpage><?supplied-pmid 24213115?><pub-id pub-id-type="pmid">24213115</pub-id>
</element-citation><mixed-citation id="mc-CR44" publication-type="journal">Lee S, Margolin K. Cytokines in cancer immunotherapy. Cancers (Basel). 2011;3(4):3856&#x02013;93.<pub-id pub-id-type="pmid">24213115</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR45"><label>45.</label><citation-alternatives><element-citation id="ec-CR45" publication-type="journal"><person-group person-group-type="author"><name><surname>Li</surname><given-names>J</given-names></name><name><surname>Wang</surname><given-names>W</given-names></name><name><surname>Zhang</surname><given-names>Y</given-names></name><name><surname>Cieslik</surname><given-names>M</given-names></name><name><surname>Guo</surname><given-names>J</given-names></name><name><surname>Tan</surname><given-names>M</given-names></name><name><surname>Green</surname><given-names>MD</given-names></name><name><surname>Wang</surname><given-names>W</given-names></name><name><surname>Lin</surname><given-names>H</given-names></name><name><surname>Li</surname><given-names>W</given-names></name><etal/></person-group><article-title>Epigenetic driver mutations in ARID1A shape cancer immune phenotype and immunotherapy</article-title><source>J Clin Invest</source><year>2020</year><volume>130</volume><issue>5</issue><fpage>2712</fpage><lpage>2726</lpage><?supplied-pmid 32027624?><pub-id pub-id-type="pmid">32027624</pub-id>
</element-citation><mixed-citation id="mc-CR45" publication-type="journal">Li J, Wang W, Zhang Y, Cieslik M, Guo J, Tan M, Green MD, Wang W, Lin H, Li W, et al. Epigenetic driver mutations in ARID1A shape cancer immune phenotype and immunotherapy. J Clin Invest. 2020;130(5):2712&#x02013;26.<pub-id pub-id-type="pmid">32027624</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR46"><label>46.</label><mixed-citation publication-type="other">Chen X, Wu X, Wu H, Gu Y, Shao Y, Shao Q, Zhu F, Li X, Qian X, Hu J, et al. Camrelizumab plus gemcitabine and oxaliplatin (GEMOX) in patients with advanced biliary tract cancer: a single-arm, open-label, phase II trial. J Immunother Cancer. 2020;8(2):e001240.</mixed-citation></ref><ref id="CR47"><label>47.</label><citation-alternatives><element-citation id="ec-CR47" publication-type="journal"><person-group person-group-type="author"><name><surname>Liu</surname><given-names>C</given-names></name><name><surname>Zheng</surname><given-names>S</given-names></name><name><surname>Wang</surname><given-names>Z</given-names></name><name><surname>Wang</surname><given-names>S</given-names></name><name><surname>Wang</surname><given-names>X</given-names></name><name><surname>Yang</surname><given-names>L</given-names></name><name><surname>Xu</surname><given-names>H</given-names></name><name><surname>Cao</surname><given-names>Z</given-names></name><name><surname>Feng</surname><given-names>X</given-names></name><name><surname>Xue</surname><given-names>Q</given-names></name><etal/></person-group><article-title>KRAS-G12D mutation drives immune suppression and the primary resistance of anti-PD-1/PD-L1 immunotherapy in non-small cell lung cancer</article-title><source>Cancer Commun (Lond)</source><year>2022</year><volume>42</volume><issue>9</issue><fpage>828</fpage><lpage>847</lpage><?supplied-pmid 35811500?><pub-id pub-id-type="pmid">35811500</pub-id>
</element-citation><mixed-citation id="mc-CR47" publication-type="journal">Liu C, Zheng S, Wang Z, Wang S, Wang X, Yang L, Xu H, Cao Z, Feng X, Xue Q, et al. KRAS-G12D mutation drives immune suppression and the primary resistance of anti-PD-1/PD-L1 immunotherapy in non-small cell lung cancer. Cancer Commun (Lond). 2022;42(9):828&#x02013;47.<pub-id pub-id-type="pmid">35811500</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR48"><label>48.</label><mixed-citation publication-type="other">Sobhani N, Tardiel-Cyril DR, Davtyan A, Generali D, Roudi R, Li Y. CTLA-4 in Regulatory T Cells for Cancer Immunotherapy. Cancers (Basel). 2021;13(6):1440.</mixed-citation></ref><ref id="CR49"><label>49.</label><citation-alternatives><element-citation id="ec-CR49" publication-type="journal"><person-group person-group-type="author"><name><surname>Litchfield</surname><given-names>K</given-names></name><name><surname>Reading</surname><given-names>JL</given-names></name><name><surname>Puttick</surname><given-names>C</given-names></name><name><surname>Thakkar</surname><given-names>K</given-names></name><name><surname>Abbosh</surname><given-names>C</given-names></name><name><surname>Bentham</surname><given-names>R</given-names></name><name><surname>Watkins</surname><given-names>TBK</given-names></name><name><surname>Rosenthal</surname><given-names>R</given-names></name><name><surname>Biswas</surname><given-names>D</given-names></name><name><surname>Rowan</surname><given-names>A</given-names></name><etal/></person-group><article-title>Meta-analysis of tumor- and T cell-intrinsic mechanisms of sensitization to checkpoint inhibition</article-title><source>Cell</source><year>2021</year><volume>184</volume><issue>3</issue><fpage>596</fpage><lpage>614</lpage><?supplied-pmid 33508232?><pub-id pub-id-type="pmid">33508232</pub-id>
</element-citation><mixed-citation id="mc-CR49" publication-type="journal">Litchfield K, Reading JL, Puttick C, Thakkar K, Abbosh C, Bentham R, Watkins TBK, Rosenthal R, Biswas D, Rowan A, et al. Meta-analysis of tumor- and T cell-intrinsic mechanisms of sensitization to checkpoint inhibition. Cell. 2021;184(3):596&#x02013;614 e514.<pub-id pub-id-type="pmid">33508232</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR50"><label>50.</label><citation-alternatives><element-citation id="ec-CR50" publication-type="journal"><person-group person-group-type="author"><name><surname>Tokunaga</surname><given-names>R</given-names></name><name><surname>Zhang</surname><given-names>W</given-names></name><name><surname>Naseem</surname><given-names>M</given-names></name><name><surname>Puccini</surname><given-names>A</given-names></name><name><surname>Berger</surname><given-names>MD</given-names></name><name><surname>Soni</surname><given-names>S</given-names></name><name><surname>McSkane</surname><given-names>M</given-names></name><name><surname>Baba</surname><given-names>H</given-names></name><name><surname>Lenz</surname><given-names>HJ</given-names></name></person-group><article-title>CXCL9, CXCL10, CXCL11/CXCR3 axis for immune activation - A target for novel cancer therapy</article-title><source>Cancer Treat Rev</source><year>2018</year><volume>63</volume><fpage>40</fpage><lpage>47</lpage><?supplied-pmid 29207310?><pub-id pub-id-type="pmid">29207310</pub-id>
</element-citation><mixed-citation id="mc-CR50" publication-type="journal">Tokunaga R, Zhang W, Naseem M, Puccini A, Berger MD, Soni S, McSkane M, Baba H, Lenz HJ. CXCL9, CXCL10, CXCL11/CXCR3 axis for immune activation - A target for novel cancer therapy. Cancer Treat Rev. 2018;63:40&#x02013;7.<pub-id pub-id-type="pmid">29207310</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR51"><label>51.</label><citation-alternatives><element-citation id="ec-CR51" publication-type="journal"><person-group person-group-type="author"><name><surname>Seitz</surname><given-names>S</given-names></name><name><surname>Dreyer</surname><given-names>TF</given-names></name><name><surname>Stange</surname><given-names>C</given-names></name><name><surname>Steiger</surname><given-names>K</given-names></name><name><surname>Brauer</surname><given-names>R</given-names></name><name><surname>Scheutz</surname><given-names>L</given-names></name><name><surname>Multhoff</surname><given-names>G</given-names></name><name><surname>Weichert</surname><given-names>W</given-names></name><name><surname>Kiechle</surname><given-names>M</given-names></name><name><surname>Magdolen</surname><given-names>V</given-names></name><etal/></person-group><article-title>CXCL9 inhibits tumour growth and drives anti-PD-L1 therapy in ovarian cancer</article-title><source>Br J Cancer</source><year>2022</year><volume>126</volume><issue>10</issue><fpage>1470</fpage><lpage>1480</lpage><?supplied-pmid 35314795?><pub-id pub-id-type="pmid">35314795</pub-id>
</element-citation><mixed-citation id="mc-CR51" publication-type="journal">Seitz S, Dreyer TF, Stange C, Steiger K, Brauer R, Scheutz L, Multhoff G, Weichert W, Kiechle M, Magdolen V, et al. CXCL9 inhibits tumour growth and drives anti-PD-L1 therapy in ovarian cancer. Br J Cancer. 2022;126(10):1470&#x02013;80.<pub-id pub-id-type="pmid">35314795</pub-id>
</mixed-citation></citation-alternatives></ref><ref id="CR52"><label>52.</label><citation-alternatives><element-citation id="ec-CR52" publication-type="journal"><person-group person-group-type="author"><name><surname>He</surname><given-names>X</given-names></name><name><surname>Xu</surname><given-names>C</given-names></name></person-group><article-title>Immune checkpoint signaling and cancer immunotherapy</article-title><source>Cell Res</source><year>2020</year><volume>30</volume><issue>8</issue><fpage>660</fpage><lpage>669</lpage><?supplied-pmid 32467592?><pub-id pub-id-type="pmid">32467592</pub-id>
</element-citation><mixed-citation id="mc-CR52" publication-type="journal">He X, Xu C. Immune checkpoint signaling and cancer immunotherapy. Cell Res. 2020;30(8):660&#x02013;9.<pub-id pub-id-type="pmid">32467592</pub-id>
</mixed-citation></citation-alternatives></ref></ref-list></back></article>